{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "##Load metadata from file\n",
    "\n",
    "import ast\n",
    "\n",
    "d = {}\n",
    "nary = {}\n",
    "with open(\"metadata_trial4.txt\", 'r') as f:\n",
    "    for line in f:\n",
    "        d = (ast.literal_eval(line))\n",
    "        nary.update({d['id'] : d})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Filter terms that have children and aspect\n",
    "\n",
    "filnary = {}\n",
    "kilnary = {}\n",
    "nochild = {}\n",
    "\n",
    "for key, value in nary.items():\n",
    "    if value.get('children') != None:\n",
    "        filnary = ({'id' : value['id'], 'name' : value['name'], 'aspect' : value['aspect'], \n",
    "                    'children' : value['children']})\n",
    "        kilnary.update({key : filnary})\n",
    "    else:\n",
    "        filnary = ({'id' : value['id'], 'name' : value['name'], 'aspect' : value['aspect']})\n",
    "        nochild.update({key : filnary})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "#remove anything that's not a biological process or molecular function from kilnary\n",
    "\n",
    "keys = []\n",
    "for key, value in kilnary.items():\n",
    "    if value['aspect'] != 'biological_process' and value['aspect'] != 'molecular_function':\n",
    "        keys.append(key)\n",
    "        \n",
    "for i in keys:\n",
    "    del kilnary[i]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "#get parents\n",
    "\n",
    "for key, value in kilnary.items():\n",
    "    for item in value['children']:\n",
    "        if nary[item['id']].get('parent') == None:\n",
    "            nary[item['id']]['parent'] = []\n",
    "            nary[item['id']]['parent'].append({'id' : key, 'relation' : item['relation']})\n",
    "        else:\n",
    "            nary[item['id']]['parent'].append({'id' : key, 'relation' : item['relation']})\n",
    "            \n",
    "for key, value in nary.items():\n",
    "    if value.get('parent') == None:\n",
    "        value['parent'] = []"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'GO:0000001': {'id': 'GO:0000001',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrion inheritance',\n",
       "  'definition': {'text': 'The distribution of mitochondria, including the mitochondrial genome, into daughter cells after mitosis or meiosis, mediated by interactions between mitochondria and the cytoskeleton.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10873824'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11389764'}]},\n",
       "  'synonyms': [{'name': 'mitochondrial inheritance', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048308', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048311', 'relation': 'is_a'}]},\n",
       " 'GO:0000002': {'id': 'GO:0000002',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial genome maintenance',\n",
       "  'definition': {'text': 'The maintenance of the structure and integrity of the mitochondrial genome; includes replication and segregation of the mitochondrial chromosome.'},\n",
       "  'children': [{'id': 'GO:0033955', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032042', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007005', 'relation': 'is_a'}]},\n",
       " 'GO:0000003': {'id': 'GO:0000003',\n",
       "  'isObsolete': False,\n",
       "  'name': 'reproduction',\n",
       "  'definition': {'text': 'The production of new individuals that contain some portion of genetic material inherited from one or more parent organisms.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'reproductive physiological process',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0032504', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003006', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0019098', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0019954', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061887', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019953', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032505', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022414', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008150', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007582', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000004', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044699', 'relation': 'is_a'}]},\n",
       " 'GO:0000006': {'id': 'GO:0000006',\n",
       "  'isObsolete': False,\n",
       "  'name': 'high-affinity zinc transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of zinc ions (Zn2+) from one side of a membrane to the other, probably powered by proton motive force. In high-affinity transport the transporter is able to bind the solute even if it is only present at very low concentrations.',\n",
       "   'xrefs': [{'dbCode': 'TC', 'dbId': '2.A.5.1.1'}]},\n",
       "  'synonyms': [{'name': 'high-affinity zinc uptake transmembrane transporter activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'high affinity zinc uptake transmembrane transporter activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005385', 'relation': 'is_a'}]},\n",
       " 'GO:0000007': {'id': 'GO:0000007',\n",
       "  'isObsolete': False,\n",
       "  'name': 'low-affinity zinc ion transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Zn2+ = Zn2+, probably powered by proton motive force. In low-affinity transport the transporter is able to bind the solute only if it is present at very high concentrations.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815340729'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005385', 'relation': 'is_a'}]},\n",
       " 'GO:0000009': {'id': 'GO:0000009',\n",
       "  'isObsolete': False,\n",
       "  'name': 'alpha-1,6-mannosyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the transfer of a mannose residue to an oligosaccharide, forming an alpha-(1->6) linkage.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '2644248'}]},\n",
       "  'synonyms': [{'name': '1,6-alpha-mannosyltransferase activity',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0033164', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000136', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0052917', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052824', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000030', 'relation': 'is_a'}]},\n",
       " 'GO:0000010': {'id': 'GO:0000010',\n",
       "  'isObsolete': False,\n",
       "  'name': 'trans-hexaprenyltranstransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: all-trans-hexaprenyl diphosphate + isopentenyl diphosphate = all-trans-heptaprenyl diphosphate + diphosphate.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9708911'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016765', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016766', 'relation': 'is_a'}]},\n",
       " 'GO:0000011': {'id': 'GO:0000011',\n",
       "  'isObsolete': False,\n",
       "  'name': 'vacuole inheritance',\n",
       "  'definition': {'text': 'The distribution of vacuoles into daughter cells after mitosis or meiosis, mediated by interactions between vacuoles and the cytoskeleton.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10873824'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14616069'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007033', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048308', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044086', 'relation': 'is_a'}]},\n",
       " 'GO:0000012': {'id': 'GO:0000012',\n",
       "  'isObsolete': False,\n",
       "  'name': 'single strand break repair',\n",
       "  'definition': {'text': 'The repair of single strand breaks in DNA. Repair of such breaks is mediated by the same enzyme systems as are used in base excision repair.',\n",
       "   'xrefs': [{'dbCode': 'URL',\n",
       "     'dbId': 'http://www.ultranet.com/~jkimball/BiologyPages/D/DNArepair.html'}]},\n",
       "  'children': [{'id': 'GO:1903516', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1903517', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1903518', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1990396', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903823', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006281', 'relation': 'is_a'}]},\n",
       " 'GO:0000014': {'id': 'GO:0000014',\n",
       "  'isObsolete': False,\n",
       "  'name': 'single-stranded DNA endodeoxyribonuclease activity',\n",
       "  'definition': {'text': 'Catalysis of the hydrolysis of ester linkages within a single-stranded deoxyribonucleic acid molecule by creating internal breaks.'},\n",
       "  'synonyms': [{'name': 'single-stranded DNA specific endodeoxyribonuclease activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'ssDNA-specific endodeoxyribonuclease activity',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:1990599', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990601', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004520', 'relation': 'is_a'}]},\n",
       " 'GO:0000015': {'id': 'GO:0000015',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phosphopyruvate hydratase complex',\n",
       "  'definition': {'text': 'A multimeric enzyme complex, usually a dimer or an octamer, that catalyzes the conversion of 2-phospho-D-glycerate to phosphoenolpyruvate and water.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'enolase complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004634', 'relation': 'capable_of'}]},\n",
       " 'GO:0000016': {'id': 'GO:0000016',\n",
       "  'isObsolete': False,\n",
       "  'name': 'lactase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: lactose + H2O = D-glucose + D-galactose.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12023280'}]},\n",
       "  'synonyms': [{'name': 'lactose galactohydrolase activity', 'type': 'exact'},\n",
       "   {'name': 'lactase-phlorizin hydrolase activity', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004553', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016800', 'relation': 'is_a'}]},\n",
       " 'GO:0000017': {'id': 'GO:0000017',\n",
       "  'isObsolete': False,\n",
       "  'name': 'alpha-glucoside transport',\n",
       "  'definition': {'text': 'The directed movement of alpha-glucosides into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore. Alpha-glucosides are glycosides in which the sugar group is a glucose residue, and the anomeric carbon of the bond is in an alpha configuration.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'},\n",
       "    {'dbCode': 'URL', 'dbId': 'http://www.biochem.purdue.edu/'}]},\n",
       "  'children': [{'id': 'GO:0015151', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0042946', 'relation': 'is_a'}]},\n",
       " 'GO:0000018': {'id': 'GO:0000018',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of DNA recombination',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of DNA recombination, a DNA metabolic process in which a new genotype is formed by reassortment of genes resulting in gene combinations different from those that were present in the parents.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'children': [{'id': 'GO:0000337', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060542', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010569', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903110', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061806', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045191', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010520', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000019', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045911', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045910', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072695', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006310', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0051052', 'relation': 'is_a'}]},\n",
       " 'GO:0000019': {'id': 'GO:0000019',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of mitotic recombination',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of DNA recombination during mitosis.'},\n",
       "  'synonyms': [{'name': 'regulation of recombination within rDNA repeats',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:1903221', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045950', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032207', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045951', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000018', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006312', 'relation': 'regulates'}]},\n",
       " 'GO:0000022': {'id': 'GO:0000022',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic spindle elongation',\n",
       "  'definition': {'text': 'The cell cycle process in which the distance is lengthened between poles of the mitotic spindle. Mitotic spindle elongation begins during mitotic prophase and ends during mitotic anaphase B.'},\n",
       "  'synonyms': [{'name': 'spindle elongation during mitosis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1905121', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061804', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051256', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061805', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902845', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1902846', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0032888', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000070', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000278', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007052', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051231', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903047', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016359', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007067', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030472', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000071', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043148', 'relation': 'part_of'}]},\n",
       " 'GO:0000023': {'id': 'GO:0000023',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maltose metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the disaccharide maltose (4-O-alpha-D-glucopyranosyl-D-glucopyranose), an intermediate in the catabolism of glycogen and starch.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'malt sugar metabolic process', 'type': 'exact'},\n",
       "   {'name': 'maltose metabolism', 'type': 'exact'},\n",
       "   {'name': 'malt sugar metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000025', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032450', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000024', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005984', 'relation': 'is_a'}]},\n",
       " 'GO:0000024': {'id': 'GO:0000024',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maltose biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of the disaccharide maltose (4-O-alpha-D-glucopyranosyl-D-glucopyranose).',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'malt sugar biosynthetic process', 'type': 'exact'},\n",
       "   {'name': 'maltose synthesis', 'type': 'exact'},\n",
       "   {'name': 'malt sugar biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'maltose anabolism', 'type': 'exact'},\n",
       "   {'name': 'maltose biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'maltose formation', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000023', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046351', 'relation': 'is_a'}]},\n",
       " 'GO:0000025': {'id': 'GO:0000025',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maltose catabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of the disaccharide maltose (4-O-alpha-D-glucopyranosyl-D-glucopyranose).',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'maltose degradation', 'type': 'exact'},\n",
       "   {'name': 'maltose hydrolysis', 'type': 'narrow'},\n",
       "   {'name': 'maltose breakdown', 'type': 'exact'},\n",
       "   {'name': 'malt sugar catabolism', 'type': 'exact'},\n",
       "   {'name': 'malt sugar catabolic process', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000023', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046352', 'relation': 'is_a'}]},\n",
       " 'GO:0000026': {'id': 'GO:0000026',\n",
       "  'isObsolete': False,\n",
       "  'name': 'alpha-1,2-mannosyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the transfer of a mannose residue to an oligosaccharide, forming an alpha-(1->2) linkage.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10521541'}]},\n",
       "  'children': [{'id': 'GO:0052918', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052926', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004377', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000030', 'relation': 'is_a'}]},\n",
       " 'GO:0000027': {'id': 'GO:0000027',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ribosomal large subunit assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of constituent RNAs and proteins to form the large ribosomal subunit.'},\n",
       "  'synonyms': [{'name': '50S ribosomal subunit assembly', 'type': 'narrow'},\n",
       "   {'name': '60S ribosomal subunit assembly', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:1902775', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034463', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0022618', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042255', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042273', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042257', 'relation': 'part_of'}]},\n",
       " 'GO:0000028': {'id': 'GO:0000028',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ribosomal small subunit assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of constituent RNAs and proteins to form the small ribosomal subunit.'},\n",
       "  'synonyms': [{'name': '30S ribosomal subunit assembly', 'type': 'narrow'},\n",
       "   {'name': '40S ribosomal subunit assembly', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0034462', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0034463', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0022618', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042255', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042274', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042257', 'relation': 'part_of'}]},\n",
       " 'GO:0000030': {'id': 'GO:0000030',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mannosyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the transfer of a mannosyl group to an acceptor molecule, typically another carbohydrate or a lipid.'},\n",
       "  'children': [{'id': 'GO:0000026', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000033', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016438', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004169', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000009', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0047267', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031501', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0019187', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0047252', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0047264', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051751', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004376', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004582', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004584', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016758', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097502', 'relation': 'part_of'}]},\n",
       " 'GO:0000031': {'id': 'GO:0000031',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mannosylphosphate transferase activity',\n",
       "  'definition': {'text': 'Catalysis of the transfer of a mannosylphosphate group from one compound to another.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016740', 'relation': 'is_a'}]},\n",
       " 'GO:0000032': {'id': 'GO:0000032',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell wall mannoprotein biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of cell wall mannoproteins, any cell wall protein that contains covalently bound mannose residues.'},\n",
       "  'synonyms': [{'name': 'cell wall mannoprotein anabolism', 'type': 'exact'},\n",
       "   {'name': 'cell wall mannoprotein biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'cell wall mannoprotein formation', 'type': 'exact'},\n",
       "   {'name': 'cell wall mannoprotein synthesis', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006057', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031506', 'relation': 'is_a'}]},\n",
       " 'GO:0000033': {'id': 'GO:0000033',\n",
       "  'isObsolete': False,\n",
       "  'name': 'alpha-1,3-mannosyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the transfer of a mannose residue to an oligosaccharide, forming an alpha-(1->3) linkage.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10521541'}]},\n",
       "  'children': [{'id': 'GO:0052925', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000030', 'relation': 'is_a'}]},\n",
       " 'GO:0000034': {'id': 'GO:0000034',\n",
       "  'isObsolete': False,\n",
       "  'name': 'adenine deaminase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: adenine + H2O = hypoxanthine + NH3.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.5.4.2'}]},\n",
       "  'synonyms': [{'name': 'adenine aminase activity', 'type': 'exact'},\n",
       "   {'name': 'adenase activity', 'type': 'exact'},\n",
       "   {'name': 'adenine aminohydrolase activity', 'type': 'exact'},\n",
       "   {'name': 'ADase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016814', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019239', 'relation': 'is_a'}]},\n",
       " 'GO:0000035': {'id': 'GO:0000035',\n",
       "  'isObsolete': False,\n",
       "  'name': 'acyl binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an acyl group, any group formally derived by removal of the hydroxyl group from the acid function of a carboxylic acid.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'acyl-CoA or acyl binding', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005488', 'relation': 'is_a'}]},\n",
       " 'GO:0000036': {'id': 'GO:0000036',\n",
       "  'isObsolete': False,\n",
       "  'name': 'acyl carrier activity',\n",
       "  'definition': {'text': 'Binding an acyl group and presenting it for processing or offloading to a cognate enzyme. Covalently binds the acyl group via a phosphopantetheine prosthetic group and mediates protein-protein interactions with the enzyme conferring specificity. The acyl carrier protein (ACP) presents substrates to enzymes involved in fatty acid biosynthesis or in polyketide secondary metabolite biosynthesis.',\n",
       "   'xrefs': [{'dbCode': 'CHEBI', 'dbId': '22221'}]},\n",
       "  'synonyms': [{'name': 'ACP phosphopantetheine attachment site binding involved in fatty acid biosynthetic process',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0036370', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904091', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006633', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0044620', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140104', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000037', 'relation': 'part_of'}]},\n",
       " 'GO:0000038': {'id': 'GO:0000038',\n",
       "  'isObsolete': False,\n",
       "  'name': 'very long-chain fatty acid metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving a fatty acid which has a chain length greater than C22.',\n",
       "   'xrefs': [{'dbCode': 'CHEBI', 'dbId': '27283'}]},\n",
       "  'synonyms': [{'name': 'very long chain fatty acid metabolic process',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'very-long-chain fatty acid metabolic process', 'type': 'exact'},\n",
       "   {'name': 'very-long-chain fatty acid metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0042761', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042760', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006631', 'relation': 'is_a'}]},\n",
       " 'GO:0000041': {'id': 'GO:0000041',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transition metal ion transport',\n",
       "  'definition': {'text': 'The directed movement of transition metal ions into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore. A transition metal is an element whose atom has an incomplete d-subshell of extranuclear electrons, or which gives rise to a cation or cations with an incomplete d-subshell. Transition metals often have more than one valency state. Biologically relevant transition metals include vanadium, manganese, iron, copper, cobalt, nickel, molybdenum and silver.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'transition metal transport', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0015691', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006825', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015694', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036249', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006826', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015673', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006824', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015675', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006829', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006828', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015676', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046915', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030001', 'relation': 'is_a'}]},\n",
       " 'GO:0000045': {'id': 'GO:0000045',\n",
       "  'isObsolete': False,\n",
       "  'name': 'autophagosome assembly',\n",
       "  'definition': {'text': 'The formation of a double membrane-bounded structure, the autophagosome, that occurs when a specialized membrane sac, called the isolation membrane, starts to enclose a portion of the cytoplasm.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9412464'}]},\n",
       "  'synonyms': [{'name': 'PAS formation', 'type': 'narrow'},\n",
       "   {'name': 'autophagic vacuole formation', 'type': 'related'},\n",
       "   {'name': 'autophagosome formation', 'type': 'exact'},\n",
       "   {'name': 'autophagosome biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'autophagic vacuole assembly', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0061736', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061753', 'relation': 'part_of'},\n",
       "   {'id': 'GO:2000786', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0061908', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0061739', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0034274', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0034497', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1990316', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0062079', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:1902902', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0035032', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:1990462', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0000407', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:2000785', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0070925', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905037', 'relation': 'is_a'}]},\n",
       " 'GO:0000048': {'id': 'GO:0000048',\n",
       "  'isObsolete': False,\n",
       "  'name': 'peptidyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: peptidyl-tRNA(1) + aminoacyl-tRNA(2) = tRNA(1) + peptidylaminoacyl-tRNA(2).',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.3.2.12'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11433365'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9242921'}]},\n",
       "  'synonyms': [{'name': 'peptidyl-tRNA:aminoacyl-tRNA N-peptidyltransferase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006412', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0016755', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006453', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006416', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043037', 'relation': 'part_of'}]},\n",
       " 'GO:0000049': {'id': 'GO:0000049',\n",
       "  'isObsolete': False,\n",
       "  'name': 'tRNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with transfer RNA.'},\n",
       "  'synonyms': [{'name': 'base pairing with tRNA', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0030557', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990856', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003723', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000498', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044822', 'relation': 'is_a'}]},\n",
       " 'GO:0000050': {'id': 'GO:0000050',\n",
       "  'isObsolete': False,\n",
       "  'name': 'urea cycle',\n",
       "  'definition': {'text': 'The sequence of reactions by which arginine is synthesized from ornithine, then cleaved to yield urea and regenerate ornithine. The overall reaction equation is NH3 + CO2 + aspartate + 3 ATP + 2 H2O = urea + fumarate + 2 ADP + 2 phosphate + AMP + diphosphate.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'urea biosynthetic process', 'type': 'exact'},\n",
       "   {'name': 'ornithine cycle', 'type': 'exact'},\n",
       "   {'name': 'urea biosynthesis', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019627', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043604', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901566', 'relation': 'is_a'}]},\n",
       " 'GO:0000052': {'id': 'GO:0000052',\n",
       "  'isObsolete': False,\n",
       "  'name': 'citrulline metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving citrulline, N5-carbamoyl-L-ornithine, an alpha amino acid not found in proteins.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'citrulline metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0019240', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019241', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:1901605', 'relation': 'is_a'}]},\n",
       " 'GO:0000053': {'id': 'GO:0000053',\n",
       "  'isObsolete': False,\n",
       "  'name': 'argininosuccinate metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving argininosuccinate, 2-(N(omega)-arginino)succinate, an intermediate in the ornithine-urea cycle, where it is synthesized from citrulline and aspartate.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'argininosuccinate metabolism', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006575', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072350', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901605', 'relation': 'is_a'}]},\n",
       " 'GO:0000054': {'id': 'GO:0000054',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ribosomal subunit export from nucleus',\n",
       "  'definition': {'text': 'The directed movement of a ribosomal subunit from the nucleus into the cytoplasm.'},\n",
       "  'synonyms': [{'name': 'ribosomal subunit export from cell nucleus',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ribosomal subunit transport from nucleus to cytoplasm',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ribosomal subunit-nucleus export', 'type': 'exact'},\n",
       "   {'name': 'ribosome export from nucleus', 'type': 'related'},\n",
       "   {'name': 'ribosomal subunit export out of nucleus', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:2000201', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0000056', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000055', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2000200', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000202', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0033750', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042254', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051656', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071428', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033753', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007046', 'relation': 'part_of'}]},\n",
       " 'GO:0000055': {'id': 'GO:0000055',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ribosomal large subunit export from nucleus',\n",
       "  'definition': {'text': 'The directed movement of a ribosomal large subunit from the nucleus into the cytoplasm.'},\n",
       "  'synonyms': [{'name': 'ribosomal large subunit transport from nucleus to cytoplasm',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ribosomal large subunit-nucleus export', 'type': 'exact'},\n",
       "   {'name': 'ribosomal large subunit export from cell nucleus',\n",
       "    'type': 'exact'},\n",
       "   {'name': '60S ribosomal subunit export from nucleus', 'type': 'narrow'},\n",
       "   {'name': 'ribosomal large subunit export out of nucleus', 'type': 'exact'},\n",
       "   {'name': '50S ribosomal subunit export from nucleus', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:2000205', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:2000203', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000204', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000054', 'relation': 'is_a'}]},\n",
       " 'GO:0000056': {'id': 'GO:0000056',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ribosomal small subunit export from nucleus',\n",
       "  'definition': {'text': 'The directed movement of a ribosomal small subunit from the nucleus into the cytoplasm.'},\n",
       "  'synonyms': [{'name': '40S ribosomal subunit export from nucleus',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'ribosomal small subunit-nucleus export', 'type': 'exact'},\n",
       "   {'name': 'ribosomal small subunit transport from nucleus to cytoplasm',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ribosomal small subunit export from cell nucleus',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ribosomal small subunit export out of nucleus', 'type': 'exact'},\n",
       "   {'name': '30S ribosomal subunit export from nucleus', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:2000206', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000207', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:2000208', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000054', 'relation': 'is_a'}]},\n",
       " 'GO:0000062': {'id': 'GO:0000062',\n",
       "  'isObsolete': False,\n",
       "  'name': 'fatty-acyl-CoA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with acyl-CoA, any derivative of coenzyme A in which the sulfhydryl group is in thiolester linkage with a fatty acyl group.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'acyl-CoA or acyl binding', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0036042', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0032559', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033218', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043168', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050662', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901567', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901681', 'relation': 'is_a'}]},\n",
       " 'GO:0000064': {'id': 'GO:0000064',\n",
       "  'isObsolete': False,\n",
       "  'name': 'L-ornithine transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of L-ornithine from one side of a membrane to the other. L-ornithine is 2,5-diaminopentanoic acid.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815340729'}]},\n",
       "  'synonyms': [{'name': 'histidine/arginine/lysine/ornithine porter activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'L-ornithine transporter activity', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0015496', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097627', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008324', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015179', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903352', 'relation': 'part_of'}]},\n",
       " 'GO:0000070': {'id': 'GO:0000070',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic sister chromatid segregation',\n",
       "  'definition': {'text': 'The cell cycle process in which replicated homologous chromosomes are organized and then physically separated and apportioned to two sets during the mitotic cell cycle. Each replicated chromosome, composed of two sister chromatids, aligns at the cell equator, paired with its homologous partner. One homolog of each morphologic type goes into each of the resulting chromosome sets.'},\n",
       "  'synonyms': [{'name': 'mitotic chromosome segregation', 'type': 'exact'},\n",
       "   {'name': 'mitotic sister-chromatid adhesion release', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0007080', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0033047', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0000022', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0062033', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0007076', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007079', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007064', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007083', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051306', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0033048', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000819', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140014', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903047', 'relation': 'is_a'}]},\n",
       " 'GO:0000073': {'id': 'GO:0000073',\n",
       "  'isObsolete': False,\n",
       "  'name': 'initial mitotic spindle pole body separation',\n",
       "  'definition': {'text': 'The release of duplicated mitotic spindle pole bodies (SPBs) that begins with the nucleation of microtubules from each SPB within the nucleus, leading to V-shaped spindle microtubules. Interpolar microtubules that elongate from each pole are interconnected, forming overlapping microtubules. Capturing and antiparallel sliding apart of microtubules promotes the initial separation of the SPB.'},\n",
       "  'children': [{'id': 'GO:0010695', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0010697', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0010696', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0061804', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0110100', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903047', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905047', 'relation': 'part_of'}]},\n",
       " 'GO:0000075': {'id': 'GO:0000075',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell cycle checkpoint',\n",
       "  'definition': {'text': 'A cell cycle process that controls cell cycle progression by monitoring the integrity of specific cell cycle events. A cell cycle checkpoint begins with detection of deficiencies or defects and ends with signal transduction.'},\n",
       "  'comment': 'This term should not be used in direct manual annotation as it should always be possible to minimally designate mitotic or meiotic checkpoint, and usually to additionally specify the checkpoint (i.e mitotic spindle assembly checkpoint, mitotic DNA damage checkpoint etc). Note also that the effector processes are not part of the checkpoint but are positively regulated by the checkpoint signaling and should not be annotated here.',\n",
       "  'synonyms': [{'name': 'G2/M transition checkpoint', 'type': 'narrow'},\n",
       "   {'name': 'G1/S checkpoint', 'type': 'narrow'},\n",
       "   {'name': 'G1/S transition checkpoint', 'type': 'narrow'},\n",
       "   {'name': 'G2/M checkpoint', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0031570', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901977', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0072394', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072395', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1901978', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0033313', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031577', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007093', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901976', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Electronic',\n",
       "  'parent': [{'id': 'GO:0009987', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045786', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050875', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044763', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008151', 'relation': 'is_a'}]},\n",
       " 'GO:0000076': {'id': 'GO:0000076',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA replication checkpoint',\n",
       "  'definition': {'text': 'A cell cycle checkpoint that prevents the initiation of nuclear division until DNA replication is complete, thereby ensuring that progeny inherit a full complement of the genome.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11728327'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12537518'}]},\n",
       "  'children': [{'id': 'GO:0072436', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072437', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0033315', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033314', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031570', 'relation': 'is_a'}]},\n",
       " 'GO:0000077': {'id': 'GO:0000077',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA damage checkpoint',\n",
       "  'definition': {'text': 'A cell cycle checkpoint that regulates progression through the cell cycle in response to DNA damage. A DNA damage checkpoint may blocks cell cycle progression (in G1, G2 or metaphase) or slow the rate at which S phase proceeds.'},\n",
       "  'synonyms': [{'name': 'DNA damage response, signal transduction resulting in cell cycle arrest',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:2000001', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000002', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0031572', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072421', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0044773', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044783', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072422', 'relation': 'part_of'},\n",
       "   {'id': 'GO:2000003', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006974', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031570', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034984', 'relation': 'is_a'}]},\n",
       " 'GO:0000079': {'id': 'GO:0000079',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of cyclin-dependent protein serine/threonine kinase activity',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of cyclin-dependent protein serine/threonine kinase activity.'},\n",
       "  'synonyms': [{'name': 'regulation of CDK activity', 'type': 'exact'},\n",
       "   {'name': 'regulation of cyclin-dependent protein kinase activity',\n",
       "    'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0019907', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0031660', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045736', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045737', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016538', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004693', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0071900', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904029', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016537', 'relation': 'regulates'}]},\n",
       " 'GO:0000080': {'id': 'GO:0000080',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic G1 phase',\n",
       "  'definition': {'text': \"The cell cycle 'gap' phase which is the interval between the completion of DNA segregation by mitosis and the beginning of DNA synthesis.\"},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'synonyms': [{'name': 'G1 phase of mitotic cell cycle', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0051318', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051329', 'relation': 'part_of'}]},\n",
       " 'GO:0000082': {'id': 'GO:0000082',\n",
       "  'isObsolete': False,\n",
       "  'name': 'G1/S transition of mitotic cell cycle',\n",
       "  'definition': {'text': 'The mitotic cell cycle transition by which a cell in G1 commits to S phase. The process begins with the build up of G1 cyclin-dependent kinase (G1 CDK), resulting in the activation of transcription of G1 cyclins. The process ends with the positive feedback of the G1 cyclins on the G1 CDK which commits the cell to S phase, in which DNA replication is initiated.'},\n",
       "  'children': [{'id': 'GO:0000083', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071930', 'relation': 'part_of'},\n",
       "   {'id': 'GO:2000045', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0071931', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031571', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1900477', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:2000134', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1900087', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0031568', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000278', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0044772', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044843', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007067', 'relation': 'part_of'}]},\n",
       " 'GO:0000083': {'id': 'GO:0000083',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of transcription involved in G1/S transition of mitotic cell cycle',\n",
       "  'definition': {'text': 'Any process that regulates transcription such that the target genes are involved in the transition between G1 and S phase of the mitotic cell cycle.'},\n",
       "  'children': [{'id': 'GO:0071931', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071930', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000082', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006355', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903047', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061019', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032583', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045449', 'relation': 'is_a'}]},\n",
       " 'GO:0000084': {'id': 'GO:0000084',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic S phase',\n",
       "  'definition': {'text': 'The cell cycle phase, following G1, during which DNA synthesis takes place as part of a mitotic cell cycle.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'synonyms': [{'name': 'S phase of mitotic cell cycle', 'type': 'exact'},\n",
       "   {'name': 'S-phase of mitotic cell cycle', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0051320', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051329', 'relation': 'part_of'}]},\n",
       " 'GO:0000085': {'id': 'GO:0000085',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic G2 phase',\n",
       "  'definition': {'text': \"The cell cycle 'gap' phase which is the interval between the completion of DNA synthesis and the beginning of DNA segregation by mitosis.\"},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'synonyms': [{'name': 'G2 phase of mitotic cell cycle', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0051319', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051329', 'relation': 'part_of'}]},\n",
       " 'GO:0000086': {'id': 'GO:0000086',\n",
       "  'isObsolete': False,\n",
       "  'name': 'G2/M transition of mitotic cell cycle',\n",
       "  'definition': {'text': 'The mitotic cell cycle transition by which a cell in G2 commits to M phase. The process begins when the kinase activity of M cyclin/CDK complex reaches a threshold high enough for the cell cycle to proceed. This is accomplished by activating a positive feedback loop that results in the accumulation of unphosphorylated and active M cyclin/CDK complex.'},\n",
       "  'synonyms': [{'name': 'mitotic G2/M transition', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0031569', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0000117', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031661', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0031660', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0010389', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0090419', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010972', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0010971', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0090282', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000278', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0044772', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044839', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007067', 'relation': 'part_of'}]},\n",
       " 'GO:0000087': {'id': 'GO:0000087',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic M phase',\n",
       "  'definition': {'text': 'A cell cycle phase during which nuclear division occurs, and which is comprises the phases: prophase, metaphase, anaphase and telophase and occurs as part of a mitotic cell cycle.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'synonyms': [{'name': 'M phase of mitotic cell cycle', 'type': 'exact'},\n",
       "   {'name': 'M-phase of mitotic cell cycle', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000089', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000088', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000090', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000093', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0000279', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098763', 'relation': 'is_a'}]},\n",
       " 'GO:0000088': {'id': 'GO:0000088',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic prophase',\n",
       "  'definition': {'text': 'The cell cycle phase which is the first stage of M phase of mitosis and during which chromosomes condense and the two daughter centrioles and their asters migrate toward the poles of the cell.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0000087', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051324', 'relation': 'is_a'}]},\n",
       " 'GO:0000089': {'id': 'GO:0000089',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic metaphase',\n",
       "  'definition': {'text': 'The cell cycle phase, following prophase, during which chromosomes become aligned on the equatorial plate of the cell as part of a mitotic cell cycle.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0000087', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051323', 'relation': 'is_a'}]},\n",
       " 'GO:0000090': {'id': 'GO:0000090',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic anaphase',\n",
       "  'definition': {'text': 'The cell cycle phase during which chromosomes separate and migrate towards the poles of the spindle the as part of a mitotic cell cycle.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'children': [{'id': 'GO:0000092', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000091', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0000087', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051322', 'relation': 'is_a'}]},\n",
       " 'GO:0000091': {'id': 'GO:0000091',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic anaphase A',\n",
       "  'definition': {'text': 'The cell cycle phase during which the kinetochore microtubules shorten as chromosomes move toward the spindle poles as part of mitosis.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0000090', 'relation': 'is_a'}]},\n",
       " 'GO:0000092': {'id': 'GO:0000092',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic anaphase B',\n",
       "  'definition': {'text': 'The cell cycle phase during which the polar microtubules elongate and the two poles of the spindle move farther apart as part of mitosis.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0000090', 'relation': 'is_a'}]},\n",
       " 'GO:0000093': {'id': 'GO:0000093',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic telophase',\n",
       "  'definition': {'text': 'The cell cycle phase which follows anaphase during M phase of mitosis and during which the chromosomes arrive at the poles of the cell and the division of the cytoplasm starts.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0000087', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051326', 'relation': 'is_a'}]},\n",
       " 'GO:0000095': {'id': 'GO:0000095',\n",
       "  'isObsolete': False,\n",
       "  'name': 'S-adenosyl-L-methionine transmembrane transporter activity',\n",
       "  'definition': {'text': \"Enables the transfer of S-adenosylmethionine from one side of a membrane to the other. S-adenosylmethionine is S-(5'-adenosyl)-L-methionine, an important intermediate in one-carbon metabolism.\"},\n",
       "  'synonyms': [{'name': 'S-adenosylmethionine transporter activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'S-adenosyl methionine transporter activity', 'type': 'exact'},\n",
       "   {'name': 'S-adenosylmethionine transmembrane transporter activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'SAM transmembrane transporter activity', 'type': 'exact'},\n",
       "   {'name': 'S-adenosylmethionine permease activity', 'type': 'exact'},\n",
       "   {'name': 'S-adenosyl methionine permease activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0051185', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901682', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901962', 'relation': 'part_of'}]},\n",
       " 'GO:0000096': {'id': 'GO:0000096',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sulfur amino acid metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving amino acids containing sulfur, comprising cysteine, homocysteine, methionine and selenocysteine.'},\n",
       "  'synonyms': [{'name': 'sulphur amino acid metabolic process',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sulfur amino acid metabolism', 'type': 'exact'},\n",
       "   {'name': 'sulphur amino acid metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0031337', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0031335', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0050667', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000098', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000097', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006534', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006555', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033321', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052698', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031336', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0030329', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006520', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006790', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006519', 'relation': 'is_a'}]},\n",
       " 'GO:0000097': {'id': 'GO:0000097',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sulfur amino acid biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of amino acids containing sulfur, comprising cysteine, methionine and selenocysteine.'},\n",
       "  'synonyms': [{'name': 'sulfur amino acid synthesis', 'type': 'exact'},\n",
       "   {'name': 'sulphur amino acid biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'sulfur amino acid formation', 'type': 'exact'},\n",
       "   {'name': 'sulfur amino acid biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'sulfur amino acid anabolism', 'type': 'exact'},\n",
       "   {'name': 'sulphur amino acid biosynthetic process', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0046311', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009086', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071268', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019344', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033322', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052699', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000096', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008652', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044272', 'relation': 'is_a'}]},\n",
       " 'GO:0000098': {'id': 'GO:0000098',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sulfur amino acid catabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of amino acids containing sulfur, comprising cysteine, methionine and selenocysteine.'},\n",
       "  'synonyms': [{'name': 'sulfur amino acid breakdown', 'type': 'exact'},\n",
       "   {'name': 'sulfur amino acid catabolism', 'type': 'exact'},\n",
       "   {'name': 'sulfur amino acid degradation', 'type': 'exact'},\n",
       "   {'name': 'sulphur amino acid catabolic process', 'type': 'exact'},\n",
       "   {'name': 'sulphur amino acid catabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0009093', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052700', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009087', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043418', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030328', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000096', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009063', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044273', 'relation': 'is_a'}]},\n",
       " 'GO:0000099': {'id': 'GO:0000099',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sulfur amino acid transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of sulfur amino acids from one side of a membrane to the other. Sulphur amino acids contain sulfur in the form of cystine, methionine or their derivatives.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815340729'}]},\n",
       "  'synonyms': [{'name': 'sulfur amino acid transporter activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'sulphur amino acid transmembrane transporter activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sulphur amino acid transporter activity', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0015184', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033229', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043865', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901680', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015171', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901682', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015359', 'relation': 'is_a'}]},\n",
       " 'GO:0000100': {'id': 'GO:0000100',\n",
       "  'isObsolete': False,\n",
       "  'name': 'S-methylmethionine transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of S-methylmethionine from one side of a membrane to the other.'},\n",
       "  'synonyms': [{'name': 'S-methylmethionine permease activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'S-methylmethionine transporter activity', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015806', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072349', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901682', 'relation': 'is_a'}]},\n",
       " 'GO:0000101': {'id': 'GO:0000101',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sulfur amino acid transport',\n",
       "  'definition': {'text': 'The directed movement of amino acids containing sulfur (cystine, methionine and their derivatives) into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore.'},\n",
       "  'synonyms': [{'name': 'sulphur amino acid transport', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0015811', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042883', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015821', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006865', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072348', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006866', 'relation': 'is_a'}]},\n",
       " 'GO:0000102': {'id': 'GO:0000102',\n",
       "  'isObsolete': False,\n",
       "  'name': 'L-methionine secondary active transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of L-methionine from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Secondary active transporters include symporters and antiporters.'},\n",
       "  'synonyms': [{'name': 'L-methionine porter activity', 'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005294', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015191', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901680', 'relation': 'is_a'}]},\n",
       " 'GO:0000103': {'id': 'GO:0000103',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sulfate assimilation',\n",
       "  'definition': {'text': 'The pathways by which inorganic sulfate is processed and incorporated into sulfated compounds.'},\n",
       "  'synonyms': [{'name': 'sulphate assimilation', 'type': 'exact'},\n",
       "   {'name': 'sulphate assimilation, phosphoadenylyl sulphate reduction by an oxidoreductase, acting on sulphur group of donors, NAD or NADP as acceptor',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'sulfate assimilation, phosphoadenylyl sulfate reduction by an oxidoreductase, acting on sulfur group of donors, NAD or NADP as acceptor',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:1900059', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0010134', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019379', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900058', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006790', 'relation': 'is_a'}]},\n",
       " 'GO:0000104': {'id': 'GO:0000104',\n",
       "  'isObsolete': False,\n",
       "  'name': 'succinate dehydrogenase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: succinate + acceptor = fumarate + reduced acceptor.'},\n",
       "  'synonyms': [{'name': 'fumarate reductase activity', 'type': 'exact'},\n",
       "   {'name': 'succinyl dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'succinodehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'succinic acid dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'succinate:acceptor oxidoreductase activity', 'type': 'exact'},\n",
       "   {'name': 'succinate:(acceptor) oxidoreductase activity', 'type': 'exact'},\n",
       "   {'name': 'succinate oxidoreductase activity', 'type': 'exact'},\n",
       "   {'name': 'succinic dehydrogenase activity', 'type': 'broad'},\n",
       "   {'name': 'fumaric hydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'fumarate dehydrogenase activity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1904229', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1904231', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0008177', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045281', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:1904230', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016627', 'relation': 'is_a'}]},\n",
       " 'GO:0000105': {'id': 'GO:0000105',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histidine biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of histidine, 2-amino-3-(1H-imidazol-4-yl)propanoic acid.'},\n",
       "  'synonyms': [{'name': 'histidine formation', 'type': 'exact'},\n",
       "   {'name': 'histidine biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'histidine anabolism', 'type': 'exact'},\n",
       "   {'name': 'histidine synthesis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0120215', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0120214', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0120213', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006547', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009073', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0018130', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901607', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016089', 'relation': 'is_a'}]},\n",
       " 'GO:0000107': {'id': 'GO:0000107',\n",
       "  'isObsolete': False,\n",
       "  'name': 'imidazoleglycerol-phosphate synthase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: phosphoribulosylformimino-AICAR-P + L-glutamine = D-erythro-imidazole-glycerol-phosphate + aminoimidazole carboxamide ribonucleotide + L-glutamate + 2 H(+).',\n",
       "   'xrefs': [{'dbCode': 'MetaCyc', 'dbId': 'GLUTAMIDOTRANS-RXN'}]},\n",
       "  'synonyms': [{'name': 'glutamine amidotransferase:cyclase activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'imidazole glycerol phosphate synthase activity', 'type': 'exact'},\n",
       "   {'name': 'imidazole-glycerol-phosphate synthase activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'imidazoleglycerol phosphate synthase activity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0009382', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016763', 'relation': 'is_a'}]},\n",
       " 'GO:0000109': {'id': 'GO:0000109',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleotide-excision repair complex',\n",
       "  'definition': {'text': 'Any complex formed of proteins that act in nucleotide-excision repair.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10915862'}]},\n",
       "  'comment': 'Note that process information is included in the term and definition for the purpose of describing and distinguishing the complex.',\n",
       "  'synonyms': [{'name': 'UvrBC complex', 'type': 'narrow'},\n",
       "   {'name': 'UvrB-UvrC complex', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0000112', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000113', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070312', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071942', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000110', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070522', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000111', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070913', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070912', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006289', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0045001', 'relation': 'capable_of_part_of'}]},\n",
       " 'GO:0000110': {'id': 'GO:0000110',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleotide-excision repair factor 1 complex',\n",
       "  'definition': {'text': 'One of several protein complexes involved in nucleotide-excision repair; possesses DNA damage recognition and endodeoxynuclease activities. In S. cerevisiae, it is composed of Rad1p, Rad10p, and Rad14p; in human the subunits are ERCC4/XPF, ERCC1 and XPA, respectively.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10915862'}]},\n",
       "  'comment': 'Note that process and function information are included in the term and definition for the purpose of describing and distinguishing the complex.',\n",
       "  'synonyms': [{'name': 'NEF1 complex', 'type': 'exact'},\n",
       "   {'name': 'XPA-ERCC1-ERCC4 complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000111': {'id': 'GO:0000111',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleotide-excision repair factor 2 complex',\n",
       "  'definition': {'text': 'One of several protein complexes involved in nucleotide-excision repair; possesses damaged DNA binding activity. In S. cerevisiae, it is composed of Rad4p and Rad23p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10915862'}]},\n",
       "  'comment': 'Note that process and function information are included in the term and definition for the purpose of describing and distinguishing the complex.',\n",
       "  'synonyms': [{'name': 'NEF2 complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000112': {'id': 'GO:0000112',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleotide-excision repair factor 3 complex',\n",
       "  'definition': {'text': 'One of several protein complexes involved in nucleotide-excision repair; possesses endodeoxynuclease and DNA helicase activities. In S. cerevisiae, it is composed of Rad2p and the core TFIIH-Ssl2p complex (core TFIIH is composed of Rad3p, Tfb1p, Tfb2p, Ssl1p, Tfb4p and Tfb5p. Note that Ssl2p is also called Rad25p).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10915862'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14500720'},\n",
       "    {'dbCode': 'PMID', 'dbId': '7813015'}]},\n",
       "  'comment': 'Note that process and function information are included in the term and definition for the purpose of describing and distinguishing the complex.',\n",
       "  'synonyms': [{'name': 'NEF3 complex', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000440', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000113': {'id': 'GO:0000113',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleotide-excision repair factor 4 complex',\n",
       "  'definition': {'text': 'One of several protein complexes involved in nucleotide-excision repair; possesses DNA damage recognition and DNA-dependent ATPase activities. In S. cerevisiae, it is composed of Rad7p and Rad16p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10915862'}]},\n",
       "  'comment': 'Note that process and function information are included in the term and definition for the purpose of describing and distinguishing the complex.',\n",
       "  'synonyms': [{'name': 'NEF4 complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000117': {'id': 'GO:0000117',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of transcription involved in G2/M transition of mitotic cell cycle',\n",
       "  'definition': {'text': 'Any process that regulates transcription such that the target genes are transcribed as part of the G2/M transition of the mitotic cell cycle.'},\n",
       "  'synonyms': [{'name': 'G2/M-specific transcription in mitotic cell cycle',\n",
       "    'type': 'related'},\n",
       "   {'name': 'regulation of transcription from RNA polymerase II promoter during G2/M transition of mitotic cell cycle',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0090282', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090419', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000086', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006355', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903047', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061019', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032583', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045449', 'relation': 'is_a'}]},\n",
       " 'GO:0000118': {'id': 'GO:0000118',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histone deacetylase complex',\n",
       "  'definition': {'text': 'A protein complex that possesses histone deacetylase activity.'},\n",
       "  'comment': \"Note that this term represents a location, not a function; the activity possessed by this complex is mentioned in the definition for the purpose of describing and distinguishing the complex. The function of this complex is represented by the molecular function term 'histone deacetylase activity ; GO:0004407'.\",\n",
       "  'synonyms': [{'name': 'HDAC complex', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1990483', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070210', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070211', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070822', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034967', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0110129', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070824', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070823', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000812', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016581', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004407', 'relation': 'capable_of'}]},\n",
       " 'GO:0000120': {'id': 'GO:0000120',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I transcription factor complex',\n",
       "  'definition': {'text': 'A transcription factor complex that acts at a regulatory region of a gene transcribed by RNA polymerase I.'},\n",
       "  'children': [{'id': 'GO:0005668', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070860', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000500', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006356', 'relation': 'capable_of'}]},\n",
       " 'GO:0000121': {'id': 'GO:0000121',\n",
       "  'isObsolete': False,\n",
       "  'name': 'glycerol-1-phosphatase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: glycerol-1-phosphate + H2O = glycerol + phosphate.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.1.3.21'}]},\n",
       "  'synonyms': [{'name': 'glycerol 3-phosphate phosphohydrolase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'glycerol 3-phosphatase activity', 'type': 'exact'},\n",
       "   {'name': 'alpha-glycerophosphatase activity', 'type': 'exact'},\n",
       "   {'name': 'alpha-glycerol phosphatase activity', 'type': 'exact'},\n",
       "   {'name': 'glycerol-1-phosphate phosphohydrolase activity', 'type': 'exact'},\n",
       "   {'name': 'glycerol-3-phosphate phosphatase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016791', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016302', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003869', 'relation': 'is_a'}]},\n",
       " 'GO:0000122': {'id': 'GO:0000122',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative regulation of transcription by RNA polymerase II',\n",
       "  'definition': {'text': 'Any process that stops, prevents, or reduces the frequency, rate or extent of transcription mediated by RNA polymerase II.'},\n",
       "  'synonyms': [{'name': 'negative regulation of transcription from RNA polymerase II promoter, global',\n",
       "    'type': 'related'},\n",
       "   {'name': 'inhibition of transcription from RNA polymerase II promoter',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'inhibition of global transcription from RNA polymerase II promoter',\n",
       "    'type': 'related'},\n",
       "   {'name': 'downregulation of global transcription from RNA polymerase II promoter',\n",
       "    'type': 'related'},\n",
       "   {'name': 'down regulation of global transcription from RNA polymerase II promoter',\n",
       "    'type': 'related'},\n",
       "   {'name': 'negative regulation of global transcription from Pol II promoter',\n",
       "    'type': 'related'},\n",
       "   {'name': 'negative regulation of gene-specific transcription from RNA polymerase II promoter',\n",
       "    'type': 'related'},\n",
       "   {'name': 'down-regulation of global transcription from RNA polymerase II promoter',\n",
       "    'type': 'related'},\n",
       "   {'name': 'down-regulation of transcription from RNA polymerase II promoter',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'down regulation of transcription from RNA polymerase II promoter',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'downregulation of transcription from RNA polymerase II promoter',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'negative regulation of transcription from Pol II promoter',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'negative regulation of transcription from RNA polymerase II promoter',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0046020', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010674', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072365', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034244', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900526', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904489', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000437', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900462', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061427', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902352', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900464', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900463', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902894', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034396', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097201', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902625', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061587', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901581', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900387', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905381', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010768', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010688', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900473', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2000820', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0120191', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035952', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904089', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901227', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001227', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060633', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035949', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900477', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990711', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0072368', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904490', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006357', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006366', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0045892', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006358', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010551', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032568', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0032569', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0016481', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061021', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032582', 'relation': 'is_a'}]},\n",
       " 'GO:0000123': {'id': 'GO:0000123',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histone acetyltransferase complex',\n",
       "  'definition': {'text': 'A protein complex that possesses histone acetyltransferase activity.'},\n",
       "  'comment': \"Note that this term represents a location, not a function; the activity possessed by this complex is mentioned in the definition for the purpose of describing and distinguishing the complex. The function of this complex is represented by the molecular function term 'histone acetyltransferase activity ; GO:0004402'.\",\n",
       "  'synonyms': [{'name': 'histone acetylase complex', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1902562', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070461', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070775', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004402', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0046971', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0043166', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0004406', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0004404', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0004403', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0004405', 'relation': 'capable_of'}]},\n",
       " 'GO:0000124': {'id': 'GO:0000124',\n",
       "  'isObsolete': False,\n",
       "  'name': 'SAGA complex',\n",
       "  'definition': {'text': 'A SAGA-type histone acetyltransferase complex that contains Spt8 (in budding yeast) or a homolog thereof; additional polypeptides include Spt group, consisting of Spt7, Spt3, and Spt20/Ada5, which interact with the TATA-binding protein (TBP); the Ada group, consisting of Ada1, Ada2, Ada3, Ada4/Gcn5, and Ada5/Spt20, which is functionally linked to the nucleosomal HAT activity; Tra1, an ATM/PI-3 kinase-related protein that targets DNA-bound activators for recruitment to promoters; the TBP-associated factor (TAF) proteins, consisting of Taf5, Taf6, Taf9, Taf10, and Taf12, which mediate nucleosomal HAT activity and are thought to help recruit the basal transcription machinery; the ubiquitin specifc protease Ubp-8.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10637607'},\n",
       "    {'dbCode': 'PMID', 'dbId': '17337012'},\n",
       "    {'dbCode': 'PMID', 'dbId': '19056896'},\n",
       "    {'dbCode': 'PMID', 'dbId': '20838651'}]},\n",
       "  'synonyms': [{'name': 'Spt-Ada-Gcn5-acetyltransferase complex',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004843', 'relation': 'capable_of'}]},\n",
       " 'GO:0000125': {'id': 'GO:0000125',\n",
       "  'isObsolete': False,\n",
       "  'name': 'PCAF complex',\n",
       "  'definition': {'text': 'A large multiprotein complex that possesses histone acetyltransferase activity and is involved in regulation of transcription. The composition is similar to that of the SAGA complex, but includes fewer Spt and Ada proteins, and more TAFs.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10637607'}]},\n",
       "  'comment': \"See also the cellular component term 'SAGA complex ; GO:0000124'.\",\n",
       "  'synonyms': [{'name': 'PCAF histone acetylase-associated complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'SPT3-TAF9-PCAF acetylase complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000126': {'id': 'GO:0000126',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription factor TFIIIB complex',\n",
       "  'definition': {'text': \"A transcription factor complex that is involved in regulating transcription from RNA polymerase III (Pol III) promoters. TFIIIB contains the TATA-binding protein (TBP) and two Pol III-specific proteins, B'' and BRF.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11433012'}]},\n",
       "  'children': [{'id': 'GO:0034733', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034732', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000127': {'id': 'GO:0000127',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription factor TFIIIC complex',\n",
       "  'definition': {'text': 'A heterotrimeric transcription factor complex that is involved in regulating transcription from RNA polymerase III (Pol III) promoters. TFIIIC contains three conserved subunits that associate with the proximal Pol III promoter element, and additional subunits that associate with sequence elements downstream of the promoter and are more diverged among species. It also functions as a boundary element to partition genome content into distinct domains outside Pol III promoter regions.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11433012'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16751097'}]},\n",
       "  'children': [{'id': 'GO:0034735', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0034734', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000128': {'id': 'GO:0000128',\n",
       "  'isObsolete': False,\n",
       "  'name': 'flocculation',\n",
       "  'definition': {'text': 'The reversible, non-sexual aggregation of single-celled organisms in suspension to form aggregates of many cells known as flocs.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '21114594'},\n",
       "    {'dbCode': 'PMID', 'dbId': '8740415'}]},\n",
       "  'comment': 'The word floc derives from the Latin word floccus, which means a tuft of wool.',\n",
       "  'children': [{'id': 'GO:0032128', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000501', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060257', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0060256', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0043689', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1900735', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0098630', 'relation': 'is_a'}]},\n",
       " 'GO:0000131': {'id': 'GO:0000131',\n",
       "  'isObsolete': False,\n",
       "  'name': 'incipient cellular bud site',\n",
       "  'definition': {'text': 'The portion of the budding yeast plasma membrane where a daughter cell will emerge. The yeast marks this spot with bud-site selection proteins before bud emergence occurs. Actin is polarized to this spot just prior to and during bud emergence.'},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000132': {'id': 'GO:0000132',\n",
       "  'isObsolete': False,\n",
       "  'name': 'establishment of mitotic spindle orientation',\n",
       "  'definition': {'text': 'A cell cycle process that sets the alignment of mitotic spindle relative to other cellular structures.'},\n",
       "  'synonyms': [{'name': 'mitotic spindle orientation', 'type': 'exact'},\n",
       "   {'name': 'orienting of mitotic spindle', 'type': 'exact'},\n",
       "   {'name': 'establishment of spindle orientation involved in mitotic cell cycle',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'establishment of spindle orientation during mitosis',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0003425', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000278', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0040001', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051294', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007067', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030606', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0018986', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030610', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030605', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030608', 'relation': 'is_a'}]},\n",
       " 'GO:0000133': {'id': 'GO:0000133',\n",
       "  'isObsolete': False,\n",
       "  'name': 'polarisome',\n",
       "  'definition': {'text': 'Protein complex that plays a role in determining cell polarity by directing the localized assembly of actin filaments at polarization sites; in Saccharomyces the polarisome includes Bni1p, Spa2p, Pea2p, and Bud6p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14734532'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14998522'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9632790'}]},\n",
       "  'children': [{'id': 'GO:0031562', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031563', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031500', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031560', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031561', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000136': {'id': 'GO:0000136',\n",
       "  'isObsolete': False,\n",
       "  'name': 'alpha-1,6-mannosyltransferase complex',\n",
       "  'definition': {'text': 'A large, multiprotein complex with alpha-(1->6)-mannosyltransferase activity, located in the cis Golgi membrane; adds mannan to N-linked glycans on proteins.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10037752'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11095735'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18083825'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000009', 'relation': 'capable_of'}]},\n",
       " 'GO:0000137': {'id': 'GO:0000137',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Golgi cis cisterna',\n",
       "  'definition': {'text': 'The Golgi cisterna closest to the endoplasmic reticulum; the first processing compartment through which proteins pass after export from the ER.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815316194'}]},\n",
       "  'children': [{'id': 'GO:1990674', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000136', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000138': {'id': 'GO:0000138',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Golgi trans cisterna',\n",
       "  'definition': {'text': 'The Golgi cisterna farthest from the endoplasmic reticulum; the final processing compartment through which proteins pass before exiting the Golgi apparatus; the compartment in which N-linked protein glycosylation is completed.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815316194'}]},\n",
       "  'synonyms': [{'name': 'late Golgi', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:1990676', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000139': {'id': 'GO:0000139',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Golgi membrane',\n",
       "  'definition': {'text': 'The lipid bilayer surrounding any of the compartments of the Golgi apparatus.'},\n",
       "  'children': [{'id': 'GO:0031228', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090498', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098548', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048194', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0098547', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032588', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030173', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090114', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0035964', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0032580', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000140': {'id': 'GO:0000140',\n",
       "  'isObsolete': False,\n",
       "  'name': 'acylglycerone-phosphate reductase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 1-palmitoylglycerol-3-phosphate + NADP+ = palmitoylglycerone phosphate + NADPH + H(+).',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.1.1.101'}]},\n",
       "  'synonyms': [{'name': '1-palmitoylglycerol-3-phosphate:NADP+ oxidoreductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'palmitoyl-dihydroxyacetone-phosphate reductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'palmitoyl dihydroxyacetone phosphate reductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'palmitoyldihydroxyacetone-phosphate reductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': '1-acyldihydroxyacetone-phosphate reductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'acyldihydroxyacetone phosphate reductase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016616', 'relation': 'is_a'}]},\n",
       " 'GO:0000142': {'id': 'GO:0000142',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cellular bud neck contractile ring',\n",
       "  'definition': {'text': 'A contractile ring, i.e. a cytoskeletal structure composed of actin filaments and myosin, that forms beneath the plasma membrane at the mother-bud neck in mitotic cells that divide by budding in preparation for completing cytokinesis. An example of this structure is found in Saccharomyces cerevisiae.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16009555'}]},\n",
       "  'synonyms': [{'name': 'neck ring', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000144': {'id': 'GO:0000144',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cellular bud neck septin ring',\n",
       "  'definition': {'text': 'A ring-shaped structure that forms at the site of cytokinesis in the bud neck of a budding cell; composed of members of the conserved family of filament forming proteins called septins as well as septin-associated proteins. In S. cerevisiae, this structure forms at the time of bud emergence and the septins show a high rate of exchange.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16009555'}]},\n",
       "  'children': [{'id': 'GO:1990317', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000145': {'id': 'GO:0000145',\n",
       "  'isObsolete': False,\n",
       "  'name': 'exocyst',\n",
       "  'definition': {'text': 'A protein complex peripherally associated with the plasma membrane that determines where vesicles dock and fuse. At least eight complex components are conserved between yeast and mammals.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15292201'},\n",
       "    {'dbCode': 'PMID', 'dbId': '27243008'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9700152'}]},\n",
       "  'synonyms': [{'name': 'Sec6/8 complex', 'type': 'exact'},\n",
       "   {'name': 'exocyst complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0090522', 'relation': 'capable_of_part_of'}]},\n",
       " 'GO:0000146': {'id': 'GO:0000146',\n",
       "  'isObsolete': False,\n",
       "  'name': 'microfilament motor activity',\n",
       "  'definition': {'text': 'Catalysis of movement along a microfilament, coupled to the hydrolysis of a nucleoside triphosphate (usually ATP).',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815316194'}]},\n",
       "  'synonyms': [{'name': 'myosin ATPase activity', 'type': 'related'},\n",
       "   {'name': 'muscle motor activity', 'type': 'narrow'},\n",
       "   {'name': 'actin-filament motor activity', 'type': 'exact'},\n",
       "   {'name': 'actin filament motor activity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0120081', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0120079', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0120080', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0060002', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060001', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003774', 'relation': 'is_a'}]},\n",
       " 'GO:0000147': {'id': 'GO:0000147',\n",
       "  'isObsolete': False,\n",
       "  'name': 'actin cortical patch assembly',\n",
       "  'definition': {'text': 'Assembly of an actin cortical patch, a discrete actin-containing structure found at the plasma membrane of fungal cells.'},\n",
       "  'children': [{'id': 'GO:0120133', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0022607', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030866', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044396', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071844', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033109', 'relation': 'is_a'}]},\n",
       " 'GO:0000148': {'id': 'GO:0000148',\n",
       "  'isObsolete': False,\n",
       "  'name': '1,3-beta-D-glucan synthase complex',\n",
       "  'definition': {'text': 'A protein complex that catalyzes the transfer of a glucose group from UDP-glucose to a (1->3)-beta-D-glucan chain.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.4.1.34'}]},\n",
       "  'synonyms': [{'name': '1,3-beta-glucan synthase complex', 'type': 'exact'},\n",
       "   {'name': '(1->3)-beta-glucan synthase complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003843', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0009981', 'relation': 'capable_of'}]},\n",
       " 'GO:0000149': {'id': 'GO:0000149',\n",
       "  'isObsolete': False,\n",
       "  'name': 'SNARE binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a SNARE (soluble N-ethylmaleimide-sensitive factor attached protein receptor) protein.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12642621'}]},\n",
       "  'synonyms': [{'name': 'SNAP receptor binding', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0019905', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005515', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045308', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001948', 'relation': 'is_a'}]},\n",
       " 'GO:0000150': {'id': 'GO:0000150',\n",
       "  'isObsolete': False,\n",
       "  'name': 'recombinase activity',\n",
       "  'definition': {'text': 'Catalysis of the identification and base-pairing of homologous sequences between single-stranded DNA and double-stranded DNA.'},\n",
       "  'comment': \"Note that this term represents activities that do not break or form phosphodiester bonds, and is therefore not a parent of 'site-specific recombinase activity ; GO:0009009'.\",\n",
       "  'synonyms': [{'name': 'strand transferase', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0009009', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006310', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140097', 'relation': 'is_a'}]},\n",
       " 'GO:0000151': {'id': 'GO:0000151',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ubiquitin ligase complex',\n",
       "  'definition': {'text': 'A protein complex that includes a ubiquitin-protein ligase and enables ubiquitin protein ligase activity. The complex also contains other proteins that may confer substrate specificity on the complex.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9529603'}]},\n",
       "  'children': [{'id': 'GO:0033503', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031461', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000153', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990353', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990352', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990452', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071797', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097505', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000152', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044695', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0061630', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0090622', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:1904264', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:1904822', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0090302', 'relation': 'capable_of'}]},\n",
       " 'GO:0000152': {'id': 'GO:0000152',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear ubiquitin ligase complex',\n",
       "  'definition': {'text': 'A ubiquitin ligase complex found in the nucleus.'},\n",
       "  'children': [{'id': 'GO:0033768', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043494', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031436', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140261', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043224', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005680', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035102', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097658', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000153': {'id': 'GO:0000153',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytoplasmic ubiquitin ligase complex',\n",
       "  'definition': {'text': 'A ubiquitin ligase complex found in the cytoplasm.'},\n",
       "  'children': [{'id': 'GO:0030891', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043223', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000835', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000154': {'id': 'GO:0000154',\n",
       "  'isObsolete': False,\n",
       "  'name': 'rRNA modification',\n",
       "  'definition': {'text': 'The covalent alteration of one or more nucleotides within an rRNA molecule to produce an rRNA molecule with a sequence that differs from that coded genetically.'},\n",
       "  'comment': \"The term 'RNA editing' (GO:0016547) was merged into 'RNA modification' (GO:0009451) on the basis of statements in the preface of Modification and Editing of RNA (ISBN:1555811337) that there is no clear distinction between modification and editing. Parallel changes were made for substrate (e.g. tRNA, rRNA, etc.) specific child terms of 'RNA editing'.\",\n",
       "  'synonyms': [{'name': 'rRNA editing', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0031118', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002110', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990882', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031167', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006364', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009451', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006365', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016547', 'relation': 'is_a'}]},\n",
       " 'GO:0000155': {'id': 'GO:0000155',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phosphorelay sensor kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the phosphorylation of a histidine residue in response to detection of an extracellular signal such as a chemical ligand or change in environment, to initiate a change in cell state or activity. The two-component sensor is a histidine kinase that autophosphorylates a histidine residue in its active site. The phosphate is then transferred to an aspartate residue in a downstream response regulator, to trigger a response.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10966457'},\n",
       "    {'dbCode': 'PMID', 'dbId': '20223701'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9191038'}]},\n",
       "  'synonyms': [{'name': 'two-component system sensor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'two-component sensor molecule', 'type': 'narrow'},\n",
       "   {'name': 'two-component sensor activity', 'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000160', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0004673', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0023014', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140299', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008896', 'relation': 'is_a'}]},\n",
       " 'GO:0000156': {'id': 'GO:0000156',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phosphorelay response regulator activity',\n",
       "  'definition': {'text': 'Responds to a phosphorelay sensor to initiate a change in cell state or activity. The activity of the response regulator is regulated by transfer of a phosphate from a histidine residue in the sensor, to an aspartate residue in the response regulator. Many but not all response regulators act as transcriptional regulators to elicit a response.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10966457'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11842140'}]},\n",
       "  'comment': 'Added is_a molecular transducer activity see https://github.com/geneontology/go-ontology/issues/15611',\n",
       "  'synonyms': [{'name': 'two-component response regulator activity',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000160', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060089', 'relation': 'is_a'}]},\n",
       " 'GO:0000159': {'id': 'GO:0000159',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein phosphatase type 2A complex',\n",
       "  'definition': {'text': 'A protein complex that has protein serine/threonine phosphatase activity that is polycation-stimulated (PCS), being directly stimulated by protamine, polylysine, or histone H1; it constitutes a subclass of several enzymes activated by different histones and polylysine, and consists of catalytic, scaffolding, and regulatory subunits. The catalytic and scaffolding subunits form the core enzyme, and the holoenzyme also includes the regulatory subunit.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198547684'},\n",
       "    {'dbCode': 'PMID', 'dbId': '17245430'}]},\n",
       "  'synonyms': [{'name': 'protein phosphatase 2 complex', 'type': 'related'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000160': {'id': 'GO:0000160',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phosphorelay signal transduction system',\n",
       "  'definition': {'text': 'A conserved series of molecular signals found in prokaryotes and eukaryotes; involves autophosphorylation of a histidine kinase and the transfer of the phosphate group to an aspartate that then acts as a phospho-donor to response regulator proteins.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9191038'}]},\n",
       "  'synonyms': [{'name': 'phosphorelay signal transduction system',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'histidyl-aspartyl phosphorelay', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0070298', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0000155', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000156', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0070299', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0007234', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009873', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070297', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0035556', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0023013', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0023034', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007242', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007243', 'relation': 'is_a'}]},\n",
       " 'GO:0000161': {'id': 'GO:0000161',\n",
       "  'isObsolete': False,\n",
       "  'name': 'osmosensory signaling MAPK cascade',\n",
       "  'definition': {'text': 'A series of molecular signals in which a stress-activated protein kinase (SAPK) cascade relays one or more of the signals, containing at least a Hog1/Sty1 family MAPK, a Pbs2/Wis1 family MAPKK and a Ssk2/Win1 family MAP3K.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17604854'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'MAPK cascade involved in osmosensory signaling pathway',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'MAPKKK cascade involved in osmosensory signalling pathway',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'High Osmolarity Glycerol (HOG) MAPK pathway', 'type': 'narrow'},\n",
       "   {'name': 'Hog1 MAPK pathway', 'type': 'narrow'},\n",
       "   {'name': 'MAPKKK cascade involved in osmosensory signaling pathway',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'MAPKKK cascade during osmolarity sensing', 'type': 'exact'},\n",
       "   {'name': 'osmolarity sensing, MAPKKK cascade', 'type': 'exact'},\n",
       "   {'name': 'Hog1/Sty1 stress-activated MAPK cascade', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0051403', 'relation': 'is_a'}]},\n",
       " 'GO:0000162': {'id': 'GO:0000162',\n",
       "  'isObsolete': False,\n",
       "  'name': 'tryptophan biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of tryptophan, the chiral amino acid 2-amino-3-(1H-indol-3-yl)propanoic acid; tryptophan is synthesized from chorismate via anthranilate.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0471331309'},\n",
       "    {'dbCode': 'MetaCyc', 'dbId': 'TRPSYN-PWY'}]},\n",
       "  'synonyms': [{'name': 'tryptophan anabolism', 'type': 'exact'},\n",
       "   {'name': 'tryptophan biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'tryptophan formation', 'type': 'exact'},\n",
       "   {'name': 'tryptophan synthesis', 'type': 'exact'},\n",
       "   {'name': 'aromatic amino acid family biosynthetic process, anthranilate pathway',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0004834', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006568', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009073', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046219', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901607', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016089', 'relation': 'is_a'}]},\n",
       " 'GO:0000164': {'id': 'GO:0000164',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein phosphatase type 1 complex',\n",
       "  'definition': {'text': \"A protein complex that possesses magnesium-dependent protein serine/threonine phosphatase (AMD phosphatase) activity, and consists of a catalytic subunit and one or more regulatory subunits that dictates the phosphatase's substrate specificity, function, and activity.\"},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000165': {'id': 'GO:0000165',\n",
       "  'isObsolete': False,\n",
       "  'name': 'MAPK cascade',\n",
       "  'definition': {'text': 'An intracellular protein kinase cascade containing at least a MAPK, a MAPKK and a MAP3K. The cascade can also contain an additional tiers: the upstream MAP4K. The kinases in each tier phosphorylate and activate the kinase in the downstream tier to transmit a signal within a cell.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20811974'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'comment': 'MAPK cascades lie downstream of many cell surface receptors and cooperate in transmitting various extracellular signals to the nucleus. One way by which the specificity of each cascade is regulated is through the existence of several distinct components in each tier of the different cascades. The cascades are typically named according to the component in the MAPK tier.',\n",
       "  'synonyms': [{'name': 'MAPKKK cascade during sporulation', 'type': 'narrow'},\n",
       "   {'name': 'MAPK signalling', 'type': 'related'},\n",
       "   {'name': 'MAPK signal transduction', 'type': 'exact'},\n",
       "   {'name': 'mitogen-activated protein kinase cascade', 'type': 'exact'},\n",
       "   {'name': 'MAPKKK cascade', 'type': 'exact'},\n",
       "   {'name': 'ERK/MAPK cascade', 'type': 'narrow'},\n",
       "   {'name': 'MAPK signaling', 'type': 'related'},\n",
       "   {'name': 'MAP kinase kinase kinase cascade', 'type': 'exact'},\n",
       "   {'name': 'MAP kinase cascade', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0043409', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0043408', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0008349', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043410', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0035420', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090080', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1903616', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004709', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051403', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004708', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071507', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070375', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070371', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004707', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903695', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0023014', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035556', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0023013', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0023034', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007242', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007243', 'relation': 'is_a'}]},\n",
       " 'GO:0000166': {'id': 'GO:0000166',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleotide binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a nucleotide, any compound consisting of a nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose or deoxyribose.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198547684'}]},\n",
       "  'children': [{'id': 'GO:0051287', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008327', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032553', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050660', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0017076', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050661', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032552', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019103', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010429', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010428', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030551', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0036094', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901265', 'relation': 'is_a'}]},\n",
       " 'GO:0000167': {'id': 'GO:0000167',\n",
       "  'isObsolete': False,\n",
       "  'name': 'activation of MAPKKK activity involved in osmosensory signaling pathway',\n",
       "  'definition': {'text': 'Any process that initiates the activity of the inactive enzyme MAP kinase kinase kinase during osmolarity sensing.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'activation of MAP kinase kinase kinase activity during osmolarity sensing',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'osmolarity sensing, activation of MAPKKK activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'osmolarity sensing, activation of MAP kinase kinase kinase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPKKK activity involved in osmosensory signalling pathway',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPKKK activity during osmolarity sensing',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000185', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007231', 'relation': 'part_of'}]},\n",
       " 'GO:0000168': {'id': 'GO:0000168',\n",
       "  'isObsolete': False,\n",
       "  'name': 'activation of MAPKK activity involved in osmosensory signaling pathway',\n",
       "  'definition': {'text': 'Any process that initiates the activity of the inactive enzyme MAP kinase kinase during osmolarity sensing.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'activation of MAP kinase kinase activity during osmolarity sensing',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPKK activity involved in osmosensory signalling pathway',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPKK activity during osmolarity sensing',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'osmolarity sensing, activation of MAPKK activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'osmolarity sensing, activation of MAP kinase kinase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of Pbs2 kinase', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000186', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007231', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007255', 'relation': 'is_a'}]},\n",
       " 'GO:0000169': {'id': 'GO:0000169',\n",
       "  'isObsolete': False,\n",
       "  'name': 'activation of MAPK activity involved in osmosensory signaling pathway',\n",
       "  'definition': {'text': 'Any process that initiates the activity of the inactive enzyme MAP kinase activity during osmolarity sensing.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'activation of MAPK activity during osmolarity sensing',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPK activity involved in osmosensory signalling pathway',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'osmolarity sensing, activation of MAPK activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000187', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007231', 'relation': 'part_of'}]},\n",
       " 'GO:0000170': {'id': 'GO:0000170',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sphingosine hydroxylase activity',\n",
       "  'definition': {'text': 'Catalysis of the hydroxylation of sphingolipid long chain bases.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9556590'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016491', 'relation': 'is_a'}]},\n",
       " 'GO:0000171': {'id': 'GO:0000171',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ribonuclease MRP activity',\n",
       "  'definition': {'text': 'Catalysis of the site-specific cleavage of RNA by a catalytic RNA-mediated mechanism; substrates include the A3 site in the ITS1 of pre-rRNA.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17881380'}]},\n",
       "  'synonyms': [{'name': 'RNase MRP', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000172', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004540', 'relation': 'is_a'}]},\n",
       " 'GO:0000172': {'id': 'GO:0000172',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ribonuclease MRP complex',\n",
       "  'definition': {'text': 'A ribonucleoprotein complex that contains an RNA molecule of the snoRNA family, and cleaves the rRNA precursor as part of rRNA transcript processing. It also has other roles: In S. cerevisiae it is involved in cell cycle-regulated degradation of daughter cell-specific mRNAs, while in mammalian cells it also enters the mitochondria and processes RNAs to create RNA primers for DNA replication.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14729943'},\n",
       "    {'dbCode': 'PMID', 'dbId': '7510714'}]},\n",
       "  'synonyms': [{'name': 'RNase MRP complex', 'type': 'exact'},\n",
       "   {'name': 'ribonuclease mitochondrial RNA processing complex',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000171', 'relation': 'capable_of'}]},\n",
       " 'GO:0000173': {'id': 'GO:0000173',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inactivation of MAPK activity involved in osmosensory signaling pathway',\n",
       "  'definition': {'text': 'Any process that terminates the activity of the active enzyme MAP kinase during osmolarity sensing.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'inactivation of MAPK activity involved in osmosensory signalling pathway',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'osmolarity sensing, inactivation of MAPK', 'type': 'exact'},\n",
       "   {'name': 'osmolarity sensing, termination of MAPK activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'termination of MAPK activity during osmolarity sensing',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'inactivation of MAPK activity during osmolarity sensing',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000188', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007231', 'relation': 'part_of'}]},\n",
       " 'GO:0000175': {'id': 'GO:0000175',\n",
       "  'isObsolete': False,\n",
       "  'name': \"3'-5'-exoribonuclease activity\",\n",
       "  'definition': {'text': \"Catalysis of the sequential cleavage of mononucleotides from a free 3' terminus of an RNA molecule.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198547684'}]},\n",
       "  'synonyms': [{'name': \"3'-5' exoribonuclease activity\", 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1990838', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000178', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0004535', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044748', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008408', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016896', 'relation': 'is_a'}]},\n",
       " 'GO:0000176': {'id': 'GO:0000176',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear exosome (RNase complex)',\n",
       "  'definition': {'text': 'A ribonuclease complex that has 3-prime to 5-prime processive and distributive hydrolytic exoribonuclease activity and endoribonuclease activity, producing 5-prime-phosphomonoesters. Participates in a multitude of cellular RNA processing and degradation events preventing nuclear export and/or translation of aberrant RNAs. Restricted to processing linear and circular single-stranded RNAs (ssRNA) only. RNAs with complex secondary structures may have to be unwound or pre-processed by co-factors prior to entering the complex, esp if the 3-prime end is structured.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17174896'},\n",
       "    {'dbCode': 'PMID', 'dbId': '20531386'},\n",
       "    {'dbCode': 'PMID', 'dbId': '26726035'}]},\n",
       "  'synonyms': [{'name': 'nuclear exosome multienzyme ribonuclease complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'eukaryotic exosome multienzyme ribonuclease complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'nuclear exosome (ribonuclease complex)', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0101019', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000177': {'id': 'GO:0000177',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytoplasmic exosome (RNase complex)',\n",
       "  'definition': {'text': 'A ribonuclease complex that has 3-prime to 5-prime processive hydrolytic exoribonuclease activity producing 5-prime-phosphomonoesters. Participates in a multitude of cellular RNA processing and degradation events preventing nuclear export and/or translation of aberrant RNAs. Restricted to processing linear and circular single-stranded RNAs (ssRNA) only. RNAs with complex secondary structures may have to be unwound or pre-processed by co-factors prior to entering the complex, esp if the 3-prime end is structured.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17174896'},\n",
       "    {'dbCode': 'PMID', 'dbId': '20531386'},\n",
       "    {'dbCode': 'PMID', 'dbId': '26726035'}]},\n",
       "  'synonyms': [{'name': 'cytoplasmic exosome multienzyme ribonuclease complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'prokaryotic exosome multienzyme ribonuclease complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cytoplasmic exosome (ribonuclease complex)', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1990061', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045025', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000178': {'id': 'GO:0000178',\n",
       "  'isObsolete': False,\n",
       "  'name': 'exosome (RNase complex)',\n",
       "  'definition': {'text': 'A ribonuclease complex that has 3-prime to 5-prime exoribonuclease activity and possibly endoribonuclease activity, producing 5-prime-phosphomonoesters. Participates in a multitude of cellular RNA processing and degradation events preventing nuclear export and/or translation of aberrant RNAs. Restricted to processing linear and circular single-stranded RNAs (ssRNA) only. RNAs with complex secondary structures may have to be unwound or pre-processed by co-factors prior to entering the complex, esp if the 3-prime end is structured.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17174896'},\n",
       "    {'dbCode': 'PMID', 'dbId': '20531386'},\n",
       "    {'dbCode': 'PMID', 'dbId': '26726035'}]},\n",
       "  'comment': \"Note that this term should not be confused with 'exosome' used in the context of vesicles released from multivesicular bodies.\",\n",
       "  'synonyms': [{'name': 'exosome (ribonucleasease complex)', 'type': 'exact'},\n",
       "   {'name': 'exosome multienzyme ribonuclease complex', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000177', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000176', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000175', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0003723', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0006396', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0006401', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0000498', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0044822', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0006394', 'relation': 'capable_of'}]},\n",
       " 'GO:0000179': {'id': 'GO:0000179',\n",
       "  'isObsolete': False,\n",
       "  'name': 'rRNA (adenine-N6,N6-)-dimethyltransferase activity',\n",
       "  'definition': {'text': \"Catalysis of the dimethylation two adjacent A residues in the loop closing the 3'-terminal stem of the 18S rRNA, using S-adenosyl-L-methionine as a methyl donor.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '1555811337'},\n",
       "    {'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'synonyms': [{'name': '18S rRNA dimethylase activity', 'type': 'exact'},\n",
       "   {'name': \"S-adenosylmethionine-6-N', N'-adenosyl(rRNA) dimethyltransferase activity\",\n",
       "    'type': 'exact'},\n",
       "   {'name': 'dimethyladenosine transferase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008170', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016433', 'relation': 'is_a'}]},\n",
       " 'GO:0000182': {'id': 'GO:0000182',\n",
       "  'isObsolete': False,\n",
       "  'name': 'rDNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with DNA sequences encoding ribosomal RNA.'},\n",
       "  'synonyms': [{'name': 'ribosomal DNA binding', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0110016', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0061773', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0043110', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080084', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:1990837', 'relation': 'is_a'}]},\n",
       " 'GO:0000183': {'id': 'GO:0000183',\n",
       "  'isObsolete': False,\n",
       "  'name': 'chromatin silencing at rDNA',\n",
       "  'definition': {'text': 'Repression of transcription of ribosomal DNA by altering the structure of chromatin.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10219245'}]},\n",
       "  'synonyms': [{'name': 'chromatin silencing at ribosomal DNA',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'heterochromatic silencing at rDNA', 'type': 'exact'},\n",
       "   {'name': 'rDNA chromatin silencing', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0061188', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0061187', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000749', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0061773', 'relation': 'capable_of_part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006342', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016440', 'relation': 'is_a'}]},\n",
       " 'GO:0000184': {'id': 'GO:0000184',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear-transcribed mRNA catabolic process, nonsense-mediated decay',\n",
       "  'definition': {'text': 'The nonsense-mediated decay pathway for nuclear-transcribed mRNAs degrades mRNAs in which an amino-acid codon has changed to a nonsense codon; this prevents the translation of such mRNAs into truncated, and potentially harmful, proteins.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10025395'}]},\n",
       "  'synonyms': [{'name': 'mRNA breakdown, nonsense-mediated decay',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'nonsense-mediated mRNA decay', 'type': 'exact'},\n",
       "   {'name': 'mRNA degradation, nonsense-mediated decay', 'type': 'exact'},\n",
       "   {'name': 'mRNA catabolism, nonsense-mediated', 'type': 'exact'},\n",
       "   {'name': 'nuclear mRNA catabolic process, nonsense-mediated decay',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'mRNA catabolic process, nonsense-mediated', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:2000623', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0070479', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070478', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2000622', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000624', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000956', 'relation': 'is_a'}]},\n",
       " 'GO:0000185': {'id': 'GO:0000185',\n",
       "  'isObsolete': False,\n",
       "  'name': 'activation of MAPKKK activity',\n",
       "  'definition': {'text': 'Any process that initiates the activity of the inactive enzyme MAP kinase kinase kinase (MAPKKK).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'activation of MAP3K activity', 'type': 'exact'},\n",
       "   {'name': 'activation of MAPKKK activity during sporulation',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'positive regulation of MAPKKK activity', 'type': 'broad'},\n",
       "   {'name': 'positive regulation of MAP kinase kinase kinase activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'activation of MAP kinase kinase kinase', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0071510', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008349', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042655', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000167', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000197', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004709', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0032147', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043406', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004710', 'relation': 'positively_regulates'}]},\n",
       " 'GO:0000186': {'id': 'GO:0000186',\n",
       "  'isObsolete': False,\n",
       "  'name': 'activation of MAPKK activity',\n",
       "  'definition': {'text': 'The initiation of the activity of the inactive enzyme MAP kinase kinase (MAPKK).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'positive regulation of MAPKK activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'activation of MAP/ERK kinase kinase', 'type': 'exact'},\n",
       "   {'name': 'activation of MAP2K activity', 'type': 'exact'},\n",
       "   {'name': 'activation of MAP kinase kinase activity', 'type': 'exact'},\n",
       "   {'name': 'activation of MAPKK activity during sporulation',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0000198', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000168', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004709', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007256', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071509', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004708', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0032147', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043406', 'relation': 'is_a'}]},\n",
       " 'GO:0000187': {'id': 'GO:0000187',\n",
       "  'isObsolete': False,\n",
       "  'name': 'activation of MAPK activity',\n",
       "  'definition': {'text': 'The initiation of the activity of the inactive enzyme MAP kinase (MAPK).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'activation of MAPK activity during sporulation',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'activation of MAP kinase', 'type': 'exact'},\n",
       "   {'name': 'MAPK activation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000169', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000199', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004708', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071508', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071883', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007257', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0032147', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043406', 'relation': 'is_a'}]},\n",
       " 'GO:0000188': {'id': 'GO:0000188',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inactivation of MAPK activity',\n",
       "  'definition': {'text': 'Any process that terminates the activity of the active enzyme MAP kinase.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'termination of MAPK activity', 'type': 'exact'},\n",
       "   {'name': 'inactivation of MAPK during sporulation', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0071511', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033549', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0035423', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000173', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000200', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0043407', 'relation': 'is_a'}]},\n",
       " 'GO:0000196': {'id': 'GO:0000196',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell wall integrity MAPK cascade',\n",
       "  'definition': {'text': 'A MAPK cascade that contributes to cell wall organization or biogenesis.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17604854'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'MAPK cascade involved in cell wall organization or biogenesis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cell integrity MAPK pathway', 'type': 'exact'},\n",
       "   {'name': 'cell wall biogenesis, MAPKKK cascade', 'type': 'exact'},\n",
       "   {'name': 'Pmk1 MAPK cell integrity signaling', 'type': 'narrow'},\n",
       "   {'name': 'Pmk1 mitogen-activated protein kinase (MAPK) cell integrity pathway',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'PMK1-MAPK signal transduction pathway', 'type': 'narrow'},\n",
       "   {'name': 'MAPK cascade involved in cell wall biogenesis', 'type': 'narrow'},\n",
       "   {'name': 'Mpk1 cascade', 'type': 'narrow'},\n",
       "   {'name': 'Slt2 cascade', 'type': 'narrow'},\n",
       "   {'name': 'MAPKKK cascade involved in cell wall biogenesis',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1903138', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1903139', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1903137', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0051403', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071554', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0070882', 'relation': 'part_of'}]},\n",
       " 'GO:0000197': {'id': 'GO:0000197',\n",
       "  'isObsolete': False,\n",
       "  'name': 'activation of MAPKKK activity involved in cell wall organization or biogenesis',\n",
       "  'definition': {'text': 'Any process that initiates the activity of the inactive enzyme MAP kinase kinase kinase in the context of cell wall organization or biogenesis.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'cell wall biogenesis, activation of MAPKKK activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cell wall biogenesis, activation of MAP kinase kinase kinase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPKKK activity involved in cell wall integrity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAP kinase kinase kinase activity during cell wall biogenesis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPKKK activity involved in cell wall biogenesis',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000185', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071554', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0070882', 'relation': 'part_of'}]},\n",
       " 'GO:0000198': {'id': 'GO:0000198',\n",
       "  'isObsolete': False,\n",
       "  'name': 'activation of MAPKK activity involved in cell wall organization or biogenesis',\n",
       "  'definition': {'text': 'Any process that initiates the activity of the inactive enzyme MAP kinase kinase in the context of cell wall organization or biogenesis.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'activation of MAPKK activity involved in cell wall integrity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cell wall biogenesis, activation of MAPKK activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cell wall biogenesis, activation of MAP kinase kinase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPKK activity involved in cell wall biogenesis',\n",
       "    'type': 'related'},\n",
       "   {'name': 'activation of MAP kinase kinase activity during cell wall biogenesis',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000186', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071554', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007255', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070882', 'relation': 'part_of'}]},\n",
       " 'GO:0000199': {'id': 'GO:0000199',\n",
       "  'isObsolete': False,\n",
       "  'name': 'activation of MAPK activity involved in cell wall organization or biogenesis',\n",
       "  'definition': {'text': 'Any process that initiates the activity of the inactive enzyme MAP kinase in the context of cell wall biogenesis, the assembly and arrangement of the cell wall, the rigid or semi-rigid envelope lying outside the cell membrane of plant, fungal and most prokaryotic cells.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'cell wall biogenesis, activation of MAPK activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPK activity involved in cell wall integrity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of MAPK activity involved in cell wall biogenesis',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000187', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071554', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0070882', 'relation': 'part_of'}]},\n",
       " 'GO:0000200': {'id': 'GO:0000200',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inactivation of MAPK activity involved in cell wall organization or biogenesis',\n",
       "  'definition': {'text': 'Any process that terminates the activity of the active enzyme MAP kinase in the context of cell wall organization or biogenesis.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9561267'}]},\n",
       "  'synonyms': [{'name': 'cell wall biogenesis, inactivation of MAPK activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cell wall biogenesis, termination of MAPK activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'termination of MAPK activity during cell wall biogenesis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'inactivation of MAPK activity involved in cell wall organization or biogenesis',\n",
       "    'type': 'related'},\n",
       "   {'name': 'inactivation of MAPK activity involved in cell wall integrity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000188', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071554', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0070882', 'relation': 'part_of'}]},\n",
       " 'GO:0000209': {'id': 'GO:0000209',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein polyubiquitination',\n",
       "  'definition': {'text': 'Addition of multiple ubiquitin groups to a protein, forming a ubiquitin chain.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815316194'}]},\n",
       "  'synonyms': [{'name': 'protein polyubiquitylation', 'type': 'exact'},\n",
       "   {'name': 'protein polyubiquitinylation', 'type': 'exact'},\n",
       "   {'name': 'polyubiquitin', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0044314', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901044', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097039', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070979', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902914', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0070936', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070534', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990390', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902916', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0035519', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0085020', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902915', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016567', 'relation': 'is_a'}]},\n",
       " 'GO:0000210': {'id': 'GO:0000210',\n",
       "  'isObsolete': False,\n",
       "  'name': 'NAD+ diphosphatase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: NAD+ + H2O = AMP + NMN.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.6.1.22'}]},\n",
       "  'synonyms': [{'name': 'NAD pyrophosphatase activity', 'type': 'exact'},\n",
       "   {'name': 'NAD+ phosphohydrolase activity', 'type': 'exact'},\n",
       "   {'name': 'nicotinamide adenine dinucleotide pyrophosphatase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'NADP pyrophosphatase activity', 'type': 'broad'},\n",
       "   {'name': 'NAD(+) pyrophosphatase activity', 'type': 'exact'},\n",
       "   {'name': 'NAD+ pyrophosphatase activity', 'type': 'exact'},\n",
       "   {'name': 'NAD diphosphatase activity', 'type': 'exact'},\n",
       "   {'name': 'NADH pyrophosphatase activity', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004551', 'relation': 'is_a'}]},\n",
       " 'GO:0000212': {'id': 'GO:0000212',\n",
       "  'isObsolete': False,\n",
       "  'name': 'meiotic spindle organization',\n",
       "  'definition': {'text': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of the microtubule spindle during a meiotic cell cycle.'},\n",
       "  'synonyms': [{'name': 'meiotic spindle organization and biogenesis',\n",
       "    'type': 'related'},\n",
       "   {'name': 'spindle organization during meiosis', 'type': 'exact'},\n",
       "   {'name': 'meiotic spindle organisation', 'type': 'exact'},\n",
       "   {'name': 'meiotic spindle stabilization', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0090306', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051232', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1990395', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051229', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007051', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051321', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903046', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043146', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007126', 'relation': 'part_of'}]},\n",
       " 'GO:0000213': {'id': 'GO:0000213',\n",
       "  'isObsolete': False,\n",
       "  'name': 'tRNA-intron endonuclease activity',\n",
       "  'definition': {'text': \"Catalysis of the endonucleolytic cleavage of pre-tRNA, producing 5'-hydroxyl and 2',3'-cyclic phosphate termini, and specifically removing the intron.\",\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.1.27.9'}]},\n",
       "  'synonyms': [{'name': 'tRNA-intron endoribonuclease activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'tRNATRPintron endonuclease activity', 'type': 'exact'},\n",
       "   {'name': 'transfer splicing endonuclease activity', 'type': 'exact'},\n",
       "   {'name': 'transfer ribonucleate intron endoribonuclease activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'tRNA-splicing endonuclease activity', 'type': 'exact'},\n",
       "   {'name': 'splicing endonuclease activity', 'type': 'exact'},\n",
       "   {'name': 'tRNA splicing endonuclease activity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000214', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004549', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016892', 'relation': 'is_a'}]},\n",
       " 'GO:0000214': {'id': 'GO:0000214',\n",
       "  'isObsolete': False,\n",
       "  'name': 'tRNA-intron endonuclease complex',\n",
       "  'definition': {'text': \"A protein complex that catalyzes the endonucleolytic cleavage of pre-tRNA, producing 5'-hydroxyl and 2',3'-cyclic phosphate termini, and specifically removing the intron.\",\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.1.27.9'}]},\n",
       "  'synonyms': [{'name': 'SEN complex', 'type': 'exact'},\n",
       "   {'name': 'tRNA splicing endonuclease complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000213', 'relation': 'capable_of'}]},\n",
       " 'GO:0000215': {'id': 'GO:0000215',\n",
       "  'isObsolete': False,\n",
       "  'name': \"tRNA 2'-phosphotransferase activity\",\n",
       "  'definition': {'text': \"Catalysis of the reaction: 2'-phospho-[ligated tRNA] + NAD+ = mature tRNA + ADP ribose 1'',2''-phosphate + nicotinamide + H2O. This reaction is the transfer of the splice junction 2-phosphate from ligated tRNA to NAD+ to produce ADP-ribose 1'-2' cyclic phosphate.\",\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.7.1.160'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9148937'}]},\n",
       "  'synonyms': [{'name': \"2'-phosphotransferase activity\", 'type': 'broad'},\n",
       "   {'name': \"yeast 2'-phosphotransferase activity\", 'type': 'narrow'},\n",
       "   {'name': 'Tpt1p', 'type': 'related'},\n",
       "   {'name': 'Tpt1', 'type': 'related'},\n",
       "   {'name': \"2'-phospho-[ligated tRNA]:NAD+ phosphotransferase activity\",\n",
       "    'type': 'exact'},\n",
       "   {'name': \"2'-phospho-tRNA:NAD+ phosphotransferase activity\",\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016773', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140101', 'relation': 'is_a'}]},\n",
       " 'GO:0000217': {'id': 'GO:0000217',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA secondary structure binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with DNA containing secondary structure elements such as four-way junctions, bubbles, loops, Y-form DNA, or double-strand/single-strand junctions.'},\n",
       "  'children': [{'id': 'GO:0000403', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000404', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000405', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032448', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000400', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0062037', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000406', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003677', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043566', 'relation': 'is_a'}]},\n",
       " 'GO:0000220': {'id': 'GO:0000220',\n",
       "  'isObsolete': False,\n",
       "  'name': 'vacuolar proton-transporting V-type ATPase, V0 domain',\n",
       "  'definition': {'text': 'The V0 domain of a proton-transporting V-type ATPase found in the vacuolar membrane.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16449553'}]},\n",
       "  'comment': 'Note that this domain often consists of five subunits, although in some mammalian tissues it may have an additional subunit.',\n",
       "  'synonyms': [{'name': 'vacuolar hydrogen ion-transporting ATPase V0 domain',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0046610', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000221': {'id': 'GO:0000221',\n",
       "  'isObsolete': False,\n",
       "  'name': 'vacuolar proton-transporting V-type ATPase, V1 domain',\n",
       "  'definition': {'text': 'The V1 domain of a proton-transporting V-type ATPase found in the vacuolar membrane.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16449553'}]},\n",
       "  'comment': 'Note that this domain generally consists of eight subunits.',\n",
       "  'synonyms': [{'name': 'vacuolar hydrogen ion-transporting ATPase V1 domain',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0046612', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000222': {'id': 'GO:0000222',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plasma membrane proton-transporting V-type ATPase, V0 domain',\n",
       "  'definition': {'text': 'The V0 domain of a proton-transporting V-type ATPase found in the plasma membrane.'},\n",
       "  'synonyms': [{'name': 'plasma membrane hydrogen ion-transporting ATPase V0 domain',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000223': {'id': 'GO:0000223',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plasma membrane proton-transporting V-type ATPase, V1 domain',\n",
       "  'definition': {'text': 'The V1 domain of a proton-transporting V-type ATPase found in the plasma membrane.'},\n",
       "  'synonyms': [{'name': 'plasma membrane hydrogen ion-transporting ATPase V1 domain',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000224': {'id': 'GO:0000224',\n",
       "  'isObsolete': False,\n",
       "  'name': 'peptide-N4-(N-acetyl-beta-glucosaminyl)asparagine amidase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 4-N-(N-acetyl-D-glucosaminyl)-protein + H2O = N-acetyl-beta-D-glucosaminylamine + peptide L-aspartate. This reaction is the hydrolysis of an N4-(acetyl-beta-D-glucosaminyl)asparagine residue in which the N-acetyl-D-glucosamine residue may be further glycosylated, to yield a (substituted) N-acetyl-beta-D-glucosaminylamine and the peptide containing an aspartic residue.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.5.1.52'}]},\n",
       "  'synonyms': [{'name': 'PNGase', 'type': 'exact'},\n",
       "   {'name': 'peptide:N-glycanase', 'type': 'related'},\n",
       "   {'name': 'jack-bean glycopeptidase', 'type': 'narrow'},\n",
       "   {'name': 'PNGase F', 'type': 'related'},\n",
       "   {'name': 'N-linked-glycopeptide-(N-acetyl-beta-D-glucosaminyl)-L-asparagine amidohydrolase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'PNGase A', 'type': 'related'},\n",
       "   {'name': 'glycopeptide N-glycosidase activity', 'type': 'broad'},\n",
       "   {'name': 'glycopeptidase activity', 'type': 'broad'},\n",
       "   {'name': 'N-oligosaccharide glycopeptidase activity', 'type': 'related'},\n",
       "   {'name': 'N-glycanase activity', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0120125', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016811', 'relation': 'is_a'}]},\n",
       " 'GO:0000225': {'id': 'GO:0000225',\n",
       "  'isObsolete': False,\n",
       "  'name': 'N-acetylglucosaminylphosphatidylinositol deacetylase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: N-acetyl-D-glucosaminylphosphatidylinositol + H2O = D-glucosaminylphosphatidylinositol + acetate. This reaction is the second step of the biosynthesis of glycosylphosphatidylinositol (GPI), used to anchor various eukaryotic proteins to the cell-surface membrane.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.5.1.89'}]},\n",
       "  'comment': 'Note that this function was formerly EC:3.1.1.69.',\n",
       "  'synonyms': [{'name': 'N-acetylglucosaminylphosphatidylinositol de-N-acetylase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'GlcNAc-PI de-N-acetylase activity', 'type': 'exact'},\n",
       "   {'name': 'N-acetyl-D-glucosaminylphosphatidylinositol acetylhydrolase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'GlcNAc-PI deacetylase activity', 'type': 'exact'},\n",
       "   {'name': '6-(N-acetyl-alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol acetylhydrolase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'acetylglucosaminylphosphatidylinositol deacetylase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016811', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019213', 'relation': 'is_a'}]},\n",
       " 'GO:0000226': {'id': 'GO:0000226',\n",
       "  'isObsolete': False,\n",
       "  'name': 'microtubule cytoskeleton organization',\n",
       "  'definition': {'text': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising microtubules and their associated proteins.'},\n",
       "  'synonyms': [{'name': 'microtubule dynamics', 'type': 'exact'},\n",
       "   {'name': 'microtubule cytoskeleton organisation', 'type': 'exact'},\n",
       "   {'name': 'microtubule cytoskeleton organization and biogenesis',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0051293', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031121', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031122', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031322', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031023', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051013', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070507', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0061863', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090176', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032118', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090172', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0021815', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010245', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007051', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035044', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007020', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030951', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080175', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034453', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031109', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902850', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001578', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007010', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007017', 'relation': 'is_a'}]},\n",
       " 'GO:0000227': {'id': 'GO:0000227',\n",
       "  'isObsolete': False,\n",
       "  'name': 'oxaloacetate secondary active transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of oxaloacetate from one side of a membrane to the other, up its concentration gradient. The transporter binds the solute and undergoes a series of conformational changes. Transport works equally well in either direction and is driven by a chemiosmotic source of energy. Secondary active transporters include symporters and antiporters.'},\n",
       "  'synonyms': [{'name': 'oxaloacetate carrier activity', 'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015131', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015291', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022853', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015404', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015570', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015353', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015290', 'relation': 'is_a'}]},\n",
       " 'GO:0000228': {'id': 'GO:0000228',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear chromosome',\n",
       "  'definition': {'text': 'A chromosome that encodes the nuclear genome and is found in the nucleus of a eukaryotic cell during the cell cycle phases when the nucleus is intact.'},\n",
       "  'synonyms': [{'name': 'nuclear interphase chromosome', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0000811', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005724', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000790', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005720', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001740', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005719', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0008622', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071162', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000794', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005664', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0033557', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000784', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043596', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000229': {'id': 'GO:0000229',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytoplasmic chromosome',\n",
       "  'definition': {'text': 'A chromosome found in the cytoplasm.'},\n",
       "  'synonyms': [{'name': 'cytoplasmic interphase chromosome',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0000262', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000789', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000809', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043597', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0009508', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000234': {'id': 'GO:0000234',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phosphoethanolamine N-methyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: S-adenosyl-L-methionine + ethanolamine phosphate = S-adenosyl-L-homocysteine + N-methylethanolamine phosphate.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.1.1.103'}]},\n",
       "  'synonyms': [{'name': 'phosphoethanolamine methyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'S-adenosyl-L-methionine:ethanolamine-phosphate N-methyltransferase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008170', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008757', 'relation': 'is_a'}]},\n",
       " 'GO:0000235': {'id': 'GO:0000235',\n",
       "  'isObsolete': False,\n",
       "  'name': 'astral microtubule',\n",
       "  'definition': {'text': 'Any of the spindle microtubules that radiate in all directions from the spindle poles and are thought to contribute to the forces that separate the poles and position them in relation to the rest of the cell.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815316194'}]},\n",
       "  'children': [{'id': 'GO:0061673', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990574', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000236': {'id': 'GO:0000236',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic prometaphase',\n",
       "  'definition': {'text': 'The cell cycle phase in higher eukaryotes which follows mitotic prophase and during which the nuclear envelope is disrupted and breaks into membrane vesicles, and the spindle microtubules enter the nuclear region. Kinetochores mature on each centromere and attach to some of the spindle microtubules. Kinetochore microtubules begin the process of aligning chromosomes in one plane halfway between the poles.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0098763', 'relation': 'is_a'}]},\n",
       " 'GO:0000237': {'id': 'GO:0000237',\n",
       "  'isObsolete': False,\n",
       "  'name': 'leptotene',\n",
       "  'definition': {'text': 'The cell cycle phase which is the first stage of prophase I in meiosis, and during which the chromosomes first become visible.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0007128', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098764', 'relation': 'is_a'}]},\n",
       " 'GO:0000238': {'id': 'GO:0000238',\n",
       "  'isObsolete': False,\n",
       "  'name': 'zygotene',\n",
       "  'definition': {'text': 'The cell cycle phase which follows leptotene during prophase I of meiosis, and during which each chromosome pairs with its homolog; the two become aligned and crossing over may occur.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0007128', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098764', 'relation': 'is_a'}]},\n",
       " 'GO:0000239': {'id': 'GO:0000239',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pachytene',\n",
       "  'definition': {'text': 'The cell cycle phase which follows zygotene during prophase I of meiosis, and during which crossing over occurs between a chromatid in one partner and another chromatid in the homologous chromosome.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0007128', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098764', 'relation': 'is_a'}]},\n",
       " 'GO:0000240': {'id': 'GO:0000240',\n",
       "  'isObsolete': False,\n",
       "  'name': 'diplotene',\n",
       "  'definition': {'text': 'The cell cycle phase which follows pachytene during prophase I of meiosis, during which the homologous chromosomes begin to separate and the synaptonemal complex dissolves.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0007128', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098764', 'relation': 'is_a'}]},\n",
       " 'GO:0000241': {'id': 'GO:0000241',\n",
       "  'isObsolete': False,\n",
       "  'name': 'diakinesis',\n",
       "  'definition': {'text': 'The cell cycle phase which follows diplotene during prophase I of meiosis, the separation of homologous chromosomes is complete and crossing over has occurred.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0007128', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098764', 'relation': 'is_a'}]},\n",
       " 'GO:0000242': {'id': 'GO:0000242',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pericentriolar material',\n",
       "  'definition': {'text': 'A network of small fibers that surrounds the centrioles in cells; contains the microtubule nucleating activity of the centrosome.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815316194'}]},\n",
       "  'children': [{'id': 'GO:0000924', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000243': {'id': 'GO:0000243',\n",
       "  'isObsolete': False,\n",
       "  'name': 'commitment complex',\n",
       "  'definition': {'text': \"A spliceosomal complex that is formed by association of the U1 snRNP with the 5' splice site of an unspliced intron in an RNA transcript.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9150140'}]},\n",
       "  'synonyms': [{'name': 'mammalian spliceosomal E complex', 'type': 'narrow'},\n",
       "   {'name': 'mammalian spliceosomal complex E', 'type': 'narrow'},\n",
       "   {'name': 'yeast spliceosomal complex CC', 'type': 'narrow'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000244': {'id': 'GO:0000244',\n",
       "  'isObsolete': False,\n",
       "  'name': 'spliceosomal tri-snRNP complex assembly',\n",
       "  'definition': {'text': 'The formation of a tri-snRNP complex containing U4 and U6 (or U4atac and U6atac) snRNAs and U5 snRNAs and associated proteins. This includes reannealing of U4 and U6 (or U4atac and U6atac) snRNAs released from previous rounds of splicing to reform the U4/U6 snRNP (or U4atac/U6atac snRNP) as well as the subsequent association of the U5 snRNP with the U4/U6 snRNP (or U4atac/U6atac snRNP) to form a tri-snRNP that is ready to reassemble into another spliceosome complex.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9452384'}]},\n",
       "  'synonyms': [{'name': 'spliceosomal tri-snRNP U4/U6.U5 assembly',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'assembly of spliceosomal tri-snRNP U4atac/U6atac.U5',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'snRNP recycling', 'type': 'broad'},\n",
       "   {'name': 'assembly of spliceosomal tri-snRNP', 'type': 'exact'},\n",
       "   {'name': 'spliceosomal tri-snRNP U4atac/U6atac.U5 assembly',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'assembly of spliceosomal tri-snRNP U4/U6.U5', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal tri-snRNP assembly', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000387', 'relation': 'is_a'}]},\n",
       " 'GO:0000245': {'id': 'GO:0000245',\n",
       "  'isObsolete': False,\n",
       "  'name': 'spliceosomal complex assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of a spliceosomal complex, a ribonucleoprotein apparatus that catalyzes nuclear mRNA splicing via transesterification reactions.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9476892'}]},\n",
       "  'synonyms': [{'name': 'spliceosome assembly', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0006376', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903241', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000352', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000348', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000354', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000398', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006375', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006374', 'relation': 'part_of'}]},\n",
       " 'GO:0000246': {'id': 'GO:0000246',\n",
       "  'isObsolete': False,\n",
       "  'name': 'delta24(24-1) sterol reductase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ergosterol + NADP(+) = ergosta-5,7,22,24(24(1))-tetraen-3beta-ol + H(+) + NADPH.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.3.1.71'},\n",
       "    {'dbCode': 'RHEA', 'dbId': '18501'}]},\n",
       "  'synonyms': [{'name': 'sterol Delta(24(28))-methylene reductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'C-24(28) sterol reductase activity', 'type': 'exact'},\n",
       "   {'name': 'sterol Delta(24(28))-reductase activity', 'type': 'exact'},\n",
       "   {'name': 'sterol delta24(28)-reductase activity', 'type': 'exact'},\n",
       "   {'name': 'sterol delta24(28)-methylene reductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ergosterol:NADP+ delta24(241)-oxidoreductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'delta24(241)-sterol reductase activity', 'type': 'exact'},\n",
       "   {'name': 'sterol delta-24(28) methylene reductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sterol delta-24(28) reductase activity', 'type': 'exact'},\n",
       "   {'name': 'D24(24-1)-sterol reductase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016628', 'relation': 'is_a'}]},\n",
       " 'GO:0000247': {'id': 'GO:0000247',\n",
       "  'isObsolete': False,\n",
       "  'name': 'C-8 sterol isomerase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction which results in unsaturation at C-7 in the B ring of sterols.',\n",
       "   'xrefs': [{'dbCode': 'MetaCyc', 'dbId': 'RXN3O-203'},\n",
       "    {'dbCode': 'PMID', 'dbId': '8988026'}]},\n",
       "  'synonyms': [{'name': 'delta-8-delta-7 sterol isomerase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016863', 'relation': 'is_a'}]},\n",
       " 'GO:0000248': {'id': 'GO:0000248',\n",
       "  'isObsolete': False,\n",
       "  'name': 'C-5 sterol desaturase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 5,7,24(28)-ergostatrienol + O2 + NADPH = 5,7,22,24(28)-ergostatetraenol + 2 H2O + NADP+.',\n",
       "   'xrefs': [{'dbCode': 'MetaCyc', 'dbId': 'RXN3O-227'}]},\n",
       "  'synonyms': [{'name': 'sterol-C5-desaturase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016717', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070704', 'relation': 'is_a'}]},\n",
       " 'GO:0000249': {'id': 'GO:0000249',\n",
       "  'isObsolete': False,\n",
       "  'name': 'C-22 sterol desaturase activity',\n",
       "  'definition': {'text': 'Catalysis of the formation of the C-22(23) double bond in the sterol side chain. An example reaction: 5,7,24(28)-ergostatrienol + O2 + NADPH = 5,7,22,24(28)-ergostatetraenol + 2 H2O + NADP+.',\n",
       "   'xrefs': [{'dbCode': 'MetaCyc', 'dbId': 'RXN3O-227'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016717', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070704', 'relation': 'is_a'}]},\n",
       " 'GO:0000250': {'id': 'GO:0000250',\n",
       "  'isObsolete': False,\n",
       "  'name': 'lanosterol synthase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: (S)-2,3-epoxysqualene = lanosterol. This is a cyclization reaction that forms the sterol nucleus.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '5.4.99.7'},\n",
       "    {'dbCode': 'RHEA', 'dbId': '14621'}]},\n",
       "  'synonyms': [{'name': '2,3-oxidosqualene sterol cyclase activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'squalene epoxidase-cyclase activity', 'type': 'broad'},\n",
       "   {'name': 'OSC', 'type': 'exact'},\n",
       "   {'name': 'squalene-2,3-oxide-lanosterol cyclase activity', 'type': 'exact'},\n",
       "   {'name': 'oxidosqualene--lanosterol cyclase activity', 'type': 'exact'},\n",
       "   {'name': '2,3-epoxysqualene--lanosterol cyclase activity', 'type': 'exact'},\n",
       "   {'name': 'squalene 2,3-epoxide:lanosterol cyclase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'oxidosqualene-lanosterol cyclase activity', 'type': 'exact'},\n",
       "   {'name': 'lanosterol 2,3-oxidosqualene cyclase activity', 'type': 'exact'},\n",
       "   {'name': '2,3-oxidosqualene-lanosterol cyclase activity', 'type': 'exact'},\n",
       "   {'name': '2,3-epoxysqualene-lanosterol cyclase activity', 'type': 'exact'},\n",
       "   {'name': '2,3-epoxysqualene lanosterol cyclase activity', 'type': 'exact'},\n",
       "   {'name': '(S)-2,3-epoxysqualene mutase (cyclizing, lanosterol-forming)',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'oxidosqualene:lanosterol cyclase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031559', 'relation': 'is_a'}]},\n",
       " 'GO:0000252': {'id': 'GO:0000252',\n",
       "  'isObsolete': False,\n",
       "  'name': 'C-3 sterol dehydrogenase (C-4 sterol decarboxylase) activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 3-beta-hydroxy-4-beta-methyl-5-alpha-cholesta-8,24-dien-4-alpha-carboxylate + NAD(P)+ = 4-alpha-methyl-5-alpha-cholesta-8,24-dien-3-one + CO2 + NAD(P)H.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.1.1.170'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9811880'}]},\n",
       "  'comment': 'Note that zymosterol is cholesta-8,24-dien-3-ol.',\n",
       "  'synonyms': [{'name': '3beta-hydroxy-4alpha-methylcholestenecarboxylate 3-dehydrogenase (decarboxylating)',\n",
       "    'type': 'related'},\n",
       "   {'name': 'sterol 4alpha-carboxylic decarboxylase activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'C-3 sterol dehydrogenase activity', 'type': 'related'},\n",
       "   {'name': 'sterol-4-carboxylate 3-dehydrogenase (decarboxylating) activity',\n",
       "    'type': 'related'},\n",
       "   {'name': '3beta-hydroxy-4beta-methyl-5alpha-cholest-7-ene-4alpha-carboxylate:NAD(P)+ 3-oxidoreductase (decarboxylating)',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sterol-4alpha-carboxylate 3-dehydrogenase (decarboxylating)',\n",
       "    'type': 'related'},\n",
       "   {'name': '3beta-hydroxy-4beta-methylcholestenecarboxylate 3-dehydrogenase (decarboxylating)',\n",
       "    'type': 'related'},\n",
       "   {'name': '3beta-hydroxy-4beta-methylcholestenoate dehydrogenase activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'C-3 sterol dehydrogenase (C-4 decarboxylase) activity',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016616', 'relation': 'is_a'}]},\n",
       " 'GO:0000253': {'id': 'GO:0000253',\n",
       "  'isObsolete': False,\n",
       "  'name': '3-keto sterol reductase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: a 3-beta-hydroxyl sterol + NADP+ = a 3-keto sterol + NADPH + H(+).',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.1.1.270'},\n",
       "    {'dbCode': 'MetaCyc', 'dbId': '1.1.1.270-RXN'},\n",
       "    {'dbCode': 'MetaCyc', 'dbId': 'RXN3O-4110'},\n",
       "    {'dbCode': 'MetaCyc', 'dbId': 'RXN66-19'},\n",
       "    {'dbCode': 'MetaCyc', 'dbId': 'RXN66-24'},\n",
       "    {'dbCode': 'MetaCyc', 'dbId': 'RXN66-314'},\n",
       "    {'dbCode': 'MetaCyc', 'dbId': 'RXN66-319'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9811880'}]},\n",
       "  'comment': 'Note that zymosterol is cholesta-8,24-dien-3-ol.',\n",
       "  'synonyms': [{'name': '3-keto-steroid reductase activity', 'type': 'exact'},\n",
       "   {'name': '3-KSR activity', 'type': 'exact'},\n",
       "   {'name': '3beta-hydroxy-steroid:NADP+ 3-oxidoreductase',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016616', 'relation': 'is_a'}]},\n",
       " 'GO:0000254': {'id': 'GO:0000254',\n",
       "  'isObsolete': False,\n",
       "  'name': 'C-4 methylsterol oxidase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 4,4-dimethyl-5-alpha-cholesta-8,24-dien-3-beta-ol + NAD(P)H + H(+) + O2 = 4-beta-hydroxymethyl-4-alpha-methyl-5-alpha-cholesta-8,24-dien-3-beta-ol + NAD(P)+ + H2O.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.14.18.9'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9811880'}]},\n",
       "  'comment': 'Note that zymosterol is cholesta-8,24-dien-3-ol.',\n",
       "  'synonyms': [{'name': '4,4-dimethyl-5alpha-cholest-7-en-3beta-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)',\n",
       "    'type': 'exact'},\n",
       "   {'name': '4,4-dimethyl-5alpha-cholest-7-en-3beta-ol,hydrogen-donor:oxygen oxidoreductase (hydroxylating)',\n",
       "    'type': 'exact'},\n",
       "   {'name': '4-methylsterol oxidase activity', 'type': 'exact'},\n",
       "   {'name': 'methylsterol hydroxylase activity', 'type': 'related'},\n",
       "   {'name': 'methylsterol monooxygenase activity', 'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016709', 'relation': 'is_a'}]},\n",
       " 'GO:0000255': {'id': 'GO:0000255',\n",
       "  'isObsolete': False,\n",
       "  'name': 'allantoin metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving allantoin, (2,5-dioxo-4-imidazolidinyl)urea, an intermediate or end product of purine catabolism.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198547684'}]},\n",
       "  'synonyms': [{'name': 'allantoin metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0019428', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000256', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0017144', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043603', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046483', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901360', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901564', 'relation': 'is_a'}]},\n",
       " 'GO:0000256': {'id': 'GO:0000256',\n",
       "  'isObsolete': False,\n",
       "  'name': 'allantoin catabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of allantoin, (2,5-dioxo-4-imidazolidinyl)urea.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198547684'}]},\n",
       "  'synonyms': [{'name': 'allantoin degradation', 'type': 'exact'},\n",
       "   {'name': 'allantoin catabolism', 'type': 'exact'},\n",
       "   {'name': 'allantoin breakdown', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0009442', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000255', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042737', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043605', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046700', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901361', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901565', 'relation': 'is_a'}]},\n",
       " 'GO:0000257': {'id': 'GO:0000257',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nitrilase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: a nitrile + H2O = a carboxylate + NH3. Acts on a wide range of aromatic nitriles including (indole-3-yl)-acetonitrile and some aliphatic nitriles, and on the corresponding acid amides.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.5.5.1'}]},\n",
       "  'synonyms': [{'name': 'nitrile aminohydrolase activity', 'type': 'exact'},\n",
       "   {'name': 'acetonitrilase activity', 'type': 'exact'},\n",
       "   {'name': 'benzonitrilase activity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0018762', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0018761', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080061', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0047426', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0047427', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0047428', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016815', 'relation': 'is_a'}]},\n",
       " 'GO:0000262': {'id': 'GO:0000262',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial chromosome',\n",
       "  'definition': {'text': 'A chromosome found in the mitochondrion of a eukaryotic cell.'},\n",
       "  'synonyms': [{'name': 'mitochondrial DNA', 'type': 'narrow'},\n",
       "   {'name': 'mtDNA', 'type': 'narrow'},\n",
       "   {'name': 'mitochondrial genome', 'type': 'related'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000266': {'id': 'GO:0000266',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial fission',\n",
       "  'definition': {'text': 'The division of a mitochondrion within a cell to form two or more separate mitochondrial compartments.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11038192'}]},\n",
       "  'synonyms': [{'name': 'mitochondrial division', 'type': 'exact'},\n",
       "   {'name': 'mitochondrial proliferation', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0090145', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090140', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0090146', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090149', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0033955', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090258', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0090141', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0090152', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007005', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048285', 'relation': 'is_a'}]},\n",
       " 'GO:0000268': {'id': 'GO:0000268',\n",
       "  'isObsolete': False,\n",
       "  'name': 'peroxisome targeting sequence binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a peroxisomal targeting sequence, any of several sequences of amino acids within a protein that can act as a signal for the localization of the protein into the peroxisome.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879693568'}]},\n",
       "  'synonyms': [{'name': 'peroxisome targeting signal receptor',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'PTS binding', 'type': 'exact'},\n",
       "   {'name': 'PTS receptor', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0005053', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005052', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033328', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005048', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008249', 'relation': 'is_a'}]},\n",
       " 'GO:0000269': {'id': 'GO:0000269',\n",
       "  'isObsolete': False,\n",
       "  'name': 'toxin export channel activity',\n",
       "  'definition': {'text': 'Enables the energy independent passage of toxins, sized less than 1000 Da, across a membrane towards the outside of the cell. The transmembrane portions of porins consist exclusively of beta-strands which form a beta-barrel. They are found in the outer membranes of Gram-negative bacteria, mitochondria, plastids and possibly acid-fast Gram-positive bacteria.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015288', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019534', 'relation': 'is_a'}]},\n",
       " 'GO:0000270': {'id': 'GO:0000270',\n",
       "  'isObsolete': False,\n",
       "  'name': 'peptidoglycan metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving peptidoglycans, any of a class of glycoconjugates found only in bacterial cell walls and consisting of strands of glycosaminoglycan cross-linked by oligopeptides to form a huge and rigid network.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'},\n",
       "    {'dbCode': 'URL', 'dbId': 'http://www.dsmz.de/species/murein.htm'}]},\n",
       "  'synonyms': [{'name': 'peptidoglycan metabolism', 'type': 'exact'},\n",
       "   {'name': 'murein metabolism', 'type': 'exact'},\n",
       "   {'name': 'murein metabolic process', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0009254', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009253', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009252', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030203', 'relation': 'is_a'}]},\n",
       " 'GO:0000271': {'id': 'GO:0000271',\n",
       "  'isObsolete': False,\n",
       "  'name': 'polysaccharide biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of a polysaccharide, a polymer of many (typically more than 10) monosaccharide residues linked glycosidically.',\n",
       "   'xrefs': [{'dbCode': 'CHEBI', 'dbId': '18154'}]},\n",
       "  'synonyms': [{'name': 'polysaccharide biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'glycan biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'glycan biosynthetic process', 'type': 'exact'},\n",
       "   {'name': 'polysaccharide anabolism', 'type': 'exact'},\n",
       "   {'name': 'polysaccharide synthesis', 'type': 'exact'},\n",
       "   {'name': 'polysaccharide formation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0032885', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0051070', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035884', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045489', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033692', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010246', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005976', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009059', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016051', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044263', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043284', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006093', 'relation': 'is_a'}]},\n",
       " 'GO:0000272': {'id': 'GO:0000272',\n",
       "  'isObsolete': False,\n",
       "  'name': 'polysaccharide catabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of a polysaccharide, a polymer of many (typically more than 10) monosaccharide residues linked glycosidically.'},\n",
       "  'synonyms': [{'name': 'polysaccharide breakdown', 'type': 'exact'},\n",
       "   {'name': 'polysaccharide catabolism', 'type': 'exact'},\n",
       "   {'name': 'polysaccharide degradation', 'type': 'exact'},\n",
       "   {'name': 'multicellular organismal polysaccharide catabolic process',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0033393', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009251', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044247', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052786', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045490', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052787', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0093001', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044347', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052359', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051682', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2000884', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2000885', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031222', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005976', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009057', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016052', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044263', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044266', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043285', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006095', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044724', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044276', 'relation': 'is_a'}]},\n",
       " 'GO:0000274': {'id': 'GO:0000274',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial proton-transporting ATP synthase, stator stalk',\n",
       "  'definition': {'text': 'One of two stalks that connect the catalytic core of the hydrogen-transporting ATP synthase to the mitochondrial membrane-associated F0 proteins; is thought to prevent futile rotation of the catalytic core.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10838056'}]},\n",
       "  'synonyms': [{'name': 'mitochondrial proton-transporting ATP synthase, peripheral stalk',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000275': {'id': 'GO:0000275',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial proton-transporting ATP synthase complex, catalytic core F(1)',\n",
       "  'definition': {'text': 'The catalytic sector of the mitochondrial hydrogen-transporting ATP synthase; it comprises the catalytic core and central stalk, and is peripherally associated with the mitochondrial inner membrane when the entire ATP synthase is assembled.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10838056'}]},\n",
       "  'comment': \"See also the cellular component term 'mitochondrial inner membrane ; GO:0005743'.\",\n",
       "  'synonyms': [{'name': 'proton-transporting ATP synthase complex, catalytic core F(1)',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'hydrogen-transporting ATP synthase, F1 sector', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0005756', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005754', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000276': {'id': 'GO:0000276',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial proton-transporting ATP synthase complex, coupling factor F(o)',\n",
       "  'definition': {'text': 'All non-F1 subunits of the mitochondrial hydrogen-transporting ATP synthase, including integral and peripheral mitochondrial inner membrane proteins.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10838056'}]},\n",
       "  'children': [{'id': 'GO:0000274', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000277': {'id': 'GO:0000277',\n",
       "  'isObsolete': False,\n",
       "  'name': '[cytochrome c]-lysine N-methyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: S-adenosyl-L-methionine + cytochrome c L-lysine = S-adenosyl-L-homocysteine + cytochrome c N6-methyl-L-lysine. This is the addition of a methyl group to the N6 atom of a lysine residue in cytochrome c.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.1.1.59'}]},\n",
       "  'synonyms': [{'name': 'cytochrome c (lysine) methyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cytochrome c-lysine N-methyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'S-adenosyl-L-methionine:cytochrome c-L-lysine N6-methyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cytochrome c-specific protein-lysine methyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cytochrome c methyltransferase activity', 'type': 'exact'},\n",
       "   {'name': 'cytochrome c-specific protein methylase III activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'S-adenosyl-L-methionine:cytochrome c-L-lysine 6-N-methyltransferase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016279', 'relation': 'is_a'}]},\n",
       " 'GO:0000278': {'id': 'GO:0000278',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic cell cycle',\n",
       "  'definition': {'text': 'Progression through the phases of the mitotic cell cycle, the most common eukaryotic cell cycle, which canonically comprises four successive phases called G1, S, G2, and M and includes replication of the genome and the subsequent segregation of chromosomes into daughter cells. In some variant cell cycles nuclear replication or nuclear division may not be followed by cell division, or G1 and G2 phases may be absent.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815316194'},\n",
       "    {'dbCode': 'Reactome', 'dbId': '69278'}]},\n",
       "  'synonyms': [{'name': 'mitosis', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0061805', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007084', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007113', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007080', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061781', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902850', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000082', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051228', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903673', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902406', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1990758', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902404', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1990505', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0033277', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044774', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902408', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0044772', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061796', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007079', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007052', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007077', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007094', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000086', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0033314', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0033301', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034087', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007346', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0071174', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902990', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902410', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045931', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1903047', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0040001', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061804', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007064', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007083', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045448', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903486', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072766', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903087', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061780', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902407', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0016330', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902405', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000132', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0044773', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902981', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902969', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007076', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000022', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007095', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903474', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007093', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007091', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140014', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045930', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0000281', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007049', 'relation': 'is_a'}]},\n",
       " 'GO:0000279': {'id': 'GO:0000279',\n",
       "  'isObsolete': False,\n",
       "  'name': 'M phase',\n",
       "  'definition': {'text': 'A cell cycle phase during which nuclear division occurs, and which is comprises the phases: prophase, metaphase, anaphase and telophase.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).\",\n",
       "  'synonyms': [{'name': 'M-phase', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000087', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051327', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051326', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051324', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051323', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'None',\n",
       "  'parent': [{'id': 'GO:0022403', 'relation': 'is_a'}]},\n",
       " 'GO:0000280': {'id': 'GO:0000280',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear division',\n",
       "  'definition': {'text': 'The division of a cell nucleus into two nuclei, with DNA and other nuclear contents distributed between the daughter nuclei.'},\n",
       "  'synonyms': [{'name': 'karyokinesis', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0140014', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051784', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0140013', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051337', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051785', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0075034', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051783', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048285', 'relation': 'is_a'}]},\n",
       " 'GO:0000281': {'id': 'GO:0000281',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic cytokinesis',\n",
       "  'definition': {'text': 'A cell cycle process that results in the division of the cytoplasm of a cell after mitosis, resulting in the separation of the original cell into two daughter cells.'},\n",
       "  'synonyms': [{'name': 'cytokinesis after mitosis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1990274', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000282', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903490', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1904498', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902413', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1903475', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902412', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0140278', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032186', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010069', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009833', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902410', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000278', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061640', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903047', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007067', 'relation': 'part_of'}]},\n",
       " 'GO:0000282': {'id': 'GO:0000282',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cellular bud site selection',\n",
       "  'definition': {'text': 'The specification of the site where a daughter cell will form, in organisms that reproduce by budding. An example of this process is found in Saccharomyces cerevisiae.'},\n",
       "  'comment': \"Note that this term was split from 'bud site selection/establishment of cell polarity (sensu Saccharomyces) ; GO:0007115' (a sibling term, 'establishment of cell polarity (sensu Saccharomyces) ; GO:0000283', was created but has since been merged with 'establishment of cell polarity' ; GO:0030010).\",\n",
       "  'synonyms': [{'name': 'bud site selection/establishment of cell polarity',\n",
       "    'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0007121', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007120', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000281', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030010', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903047', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030468', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000283', 'relation': 'is_a'}]},\n",
       " 'GO:0000285': {'id': 'GO:0000285',\n",
       "  'isObsolete': False,\n",
       "  'name': '1-phosphatidylinositol-3-phosphate 5-kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: a 1-phosphatidyl-1D-myo-inositol 3-phosphate + ATP = a 1-phosphatidyl-1D-myo-inositol 3,5-bisphosphate + ADP + 2 H(+).',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.7.1.150'},\n",
       "    {'dbCode': 'RHEA', 'dbId': '13609'}]},\n",
       "  'synonyms': [{'name': 'type III PIP kinase activity', 'type': 'related'},\n",
       "   {'name': 'ATP:1-phosphatidyl-1D-myo-inositol-3-phosphate 5-phosphotransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'phosphatidylinositol-3-phosphate 5-kinase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'phosphatidylinositol 3-phosphate 5-kinase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016307', 'relation': 'is_a'}]},\n",
       " 'GO:0000286': {'id': 'GO:0000286',\n",
       "  'isObsolete': False,\n",
       "  'name': 'alanine dehydrogenase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: L-alanine + H2O + NAD+ = pyruvate + NH3 + NADH + H(+).',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.4.1.1'}]},\n",
       "  'synonyms': [{'name': 'NAD-linked alanine dehydrogenase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'NAD-dependent alanine dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'L-alanine:NAD+ oxidoreductase (deaminating)', 'type': 'exact'},\n",
       "   {'name': 'L-alanine dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'AlaDH', 'type': 'related'},\n",
       "   {'name': 'alanine oxidoreductase activity', 'type': 'exact'},\n",
       "   {'name': 'alpha-alanine dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'NADH-dependent alanine dehydrogenase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016638', 'relation': 'is_a'}]},\n",
       " 'GO:0000287': {'id': 'GO:0000287',\n",
       "  'isObsolete': False,\n",
       "  'name': 'magnesium ion binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with magnesium (Mg) ions.'},\n",
       "  'synonyms': [{'name': 'magnesium binding', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1990710', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0046872', 'relation': 'is_a'}]},\n",
       " 'GO:0000288': {'id': 'GO:0000288',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear-transcribed mRNA catabolic process, deadenylation-dependent decay',\n",
       "  'definition': {'text': 'A major pathway of degradation of nuclear-transcribed mRNAs that proceeds through a series of ordered steps that includes poly(A) tail shortening and that can regulate mRNA stability.'},\n",
       "  'synonyms': [{'name': 'mRNA breakdown, deadenylation-dependent decay',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'mRNA catabolism, deadenylation-dependent', 'type': 'exact'},\n",
       "   {'name': 'mRNA catabolism, deadenylylation-dependent', 'type': 'exact'},\n",
       "   {'name': 'nuclear mRNA catabolic process, deadenylation-dependent decay',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'mRNA catabolic process, deadenylation-dependent',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'mRNA catabolic process, deadenylylation-dependent',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'deadenylation-dependent mRNA decay', 'type': 'exact'},\n",
       "   {'name': 'mRNA degradation, deadenylation-dependent decay',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000289', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1900153', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0000290', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1900151', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0043928', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1900152', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000956', 'relation': 'is_a'}]},\n",
       " 'GO:0000289': {'id': 'GO:0000289',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear-transcribed mRNA poly(A) tail shortening',\n",
       "  'definition': {'text': 'Shortening of the poly(A) tail of a nuclear-transcribed mRNA from full length to an oligo(A) length.'},\n",
       "  'synonyms': [{'name': 'nuclear mRNA poly(A) tail shortening',\n",
       "    'type': 'related'},\n",
       "   {'name': 'mRNA deadenylation', 'type': 'related'},\n",
       "   {'name': \"3' to 5' mRNA deadenylation\", 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0060212', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0060213', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0098806', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060211', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000288', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000956', 'relation': 'is_a'}]},\n",
       " 'GO:0000290': {'id': 'GO:0000290',\n",
       "  'isObsolete': False,\n",
       "  'name': 'deadenylation-dependent decapping of nuclear-transcribed mRNA',\n",
       "  'definition': {'text': \"Cleavage of the 5'-cap of a nuclear mRNA triggered by shortening of the poly(A) tail to below a minimum functional length.\"},\n",
       "  'synonyms': [{'name': 'deadenylylation-dependent decapping',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'deadenylation-dependent decapping of nuclear mRNA',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000288', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000956', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0110156', 'relation': 'is_a'}]},\n",
       " 'GO:0000291': {'id': 'GO:0000291',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear-transcribed mRNA catabolic process, exonucleolytic',\n",
       "  'definition': {'text': \"The chemical reactions and pathways resulting in the breakdown of the transcript body of a nuclear-transcribed mRNA that occurs when the ends are not protected by the 5'-cap or the 3'-poly(A) tail.\"},\n",
       "  'synonyms': [{'name': 'nuclear mRNA catabolic process, exonucleolytic',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'exonucleolytic degradation of mRNA', 'type': 'exact'},\n",
       "   {'name': 'mRNA degradation, exonucleolytic', 'type': 'exact'},\n",
       "   {'name': 'mRNA breakdown, exonucleolytic', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0043927', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070480', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034428', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034427', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043928', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000956', 'relation': 'is_a'}]},\n",
       " 'GO:0000292': {'id': 'GO:0000292',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA fragment catabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of a fragment of RNA, such as excised introns or sequences removed from ribosomal RNA during processing.'},\n",
       "  'synonyms': [{'name': 'group I intron catabolic process', 'type': 'narrow'},\n",
       "   {'name': 'RNA fragment degradation', 'type': 'exact'},\n",
       "   {'name': 'RNA fragment catabolism', 'type': 'exact'},\n",
       "   {'name': 'RNA fragment breakdown', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006401', 'relation': 'is_a'}]},\n",
       " 'GO:0000293': {'id': 'GO:0000293',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ferric-chelate reductase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 2 Fe2+ + NAD+ = 2 Fe3+ + NADH + H(+).',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.16.1.7'}]},\n",
       "  'comment': 'Note that this function was formerly EC:1.6.99.13.',\n",
       "  'synonyms': [{'name': 'NADH:Fe(3+)-EDTA reductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ferric chelate reductase activity', 'type': 'exact'},\n",
       "   {'name': 'iron chelate reductase activity', 'type': 'related'},\n",
       "   {'name': 'NADH2:Fe3+ oxidoreductase activity', 'type': 'exact'},\n",
       "   {'name': 'NADH:Fe3+ oxidoreductase activity', 'type': 'exact'},\n",
       "   {'name': 'NADH:Fe3+-EDTA reductase activity', 'type': 'exact'},\n",
       "   {'name': 'NADH:Fe(3+) oxidoreductase activity', 'type': 'exact'},\n",
       "   {'name': 'Fe(II):NAD+ oxidoreductase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016723', 'relation': 'is_a'}]},\n",
       " 'GO:0000294': {'id': 'GO:0000294',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear-transcribed mRNA catabolic process, endonucleolytic cleavage-dependent decay',\n",
       "  'definition': {'text': 'A minor degradation pathway nuclear-transcribed mRNAs that begins with an endonucleolytic cleavage to generate unprotected ends.'},\n",
       "  'synonyms': [{'name': 'mRNA catabolic process, endonucleolytic',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'mRNA breakdown, endonucleolytic cleavage-dependent decay',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'endonucleolytic mRNA decay', 'type': 'exact'},\n",
       "   {'name': 'nuclear mRNA catabolic process, endonucleolytic cleavage-dependent decay',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'mRNA catabolism, endonucleolytic', 'type': 'exact'},\n",
       "   {'name': 'mRNA degradation, endonucleolytic cleavage-dependent decay',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0043927', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000956', 'relation': 'is_a'}]},\n",
       " 'GO:0000295': {'id': 'GO:0000295',\n",
       "  'isObsolete': False,\n",
       "  'name': 'adenine nucleotide transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of adenine nucleotides (AMP, ADP, and ATP) from one side of a membrane to the other.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11566870'}]},\n",
       "  'children': [{'id': 'GO:1902557', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005347', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015217', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140360', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080122', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071077', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046964', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015216', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051503', 'relation': 'part_of'}]},\n",
       " 'GO:0000296': {'id': 'GO:0000296',\n",
       "  'isObsolete': False,\n",
       "  'name': 'spermine transport',\n",
       "  'definition': {'text': 'The directed movement of spermine, N,N-bis(3-aminopropyl)-1,4-diaminobutane, a polyamine formed by the transfer of a propylamine group from decarboxylated S-adenosylmethionine to spermidine, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'children': [{'id': 'GO:1903710', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015696', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015846', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015893', 'relation': 'is_a'}]},\n",
       " 'GO:0000297': {'id': 'GO:0000297',\n",
       "  'isObsolete': False,\n",
       "  'name': 'spermine transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of spermine from one side of a membrane to the other. Spermine is a polybasic amine found in human sperm, in ribosomes and in some viruses, which is involved in nucleic acid packaging. Synthesis is regulated by ornithine decarboxylase which plays a key role in control of DNA replication.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008519', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015203', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903710', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051739', 'relation': 'is_a'}]},\n",
       " 'GO:0000298': {'id': 'GO:0000298',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endopolyphosphatase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: polyphosphate + n H2O = (n+1) oligophosphate. The product contains 4 or 5 phosphate residues.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.6.1.10'}]},\n",
       "  'synonyms': [{'name': 'polymetaphosphatase activity', 'type': 'exact'},\n",
       "   {'name': 'polyphosphate depolymerase activity', 'type': 'broad'},\n",
       "   {'name': 'polyphosphatase activity', 'type': 'broad'},\n",
       "   {'name': 'metaphosphatase activity', 'type': 'broad'},\n",
       "   {'name': 'polyphosphate polyphosphohydrolase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016462', 'relation': 'is_a'}]},\n",
       " 'GO:0000301': {'id': 'GO:0000301',\n",
       "  'isObsolete': False,\n",
       "  'name': 'retrograde transport, vesicle recycling within Golgi',\n",
       "  'definition': {'text': 'The retrograde movement of substances within the Golgi, mediated by COP I vesicles. Cis-Golgi vesicles are constantly moving forward through the Golgi stack by cisternal progression, eventually becoming trans-Golgi vesicles. They then selectively transport membrane and luminal proteins from the trans- to the medial-Golgi while leaving others behind in the trans-Golgi cisternae; similarly, they selectively move proteins from the medial- to the cis-Golgi.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0716731363'}]},\n",
       "  'synonyms': [{'name': 'retrograde (vesicle recycling within Golgi) transport',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1990690', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1990692', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006891', 'relation': 'is_a'}]},\n",
       " 'GO:0000302': {'id': 'GO:0000302',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to reactive oxygen species',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a reactive oxygen species stimulus. Reactive oxygen species include singlet oxygen, superoxide, and oxygen free radicals.'},\n",
       "  'synonyms': [{'name': 'response to active oxygen species', 'type': 'exact'},\n",
       "   {'name': 'response to reactive oxidative species', 'type': 'exact'},\n",
       "   {'name': 'response to reactive oxygen intermediate', 'type': 'exact'},\n",
       "   {'name': 'response to ROS', 'type': 'exact'},\n",
       "   {'name': 'response to AOS', 'type': 'exact'},\n",
       "   {'name': 'response to ROI', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0042542', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901031', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1901530', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901033', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1901032', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0010193', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001315', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071731', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000305', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034614', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000304', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006979', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901700', 'relation': 'is_a'}]},\n",
       " 'GO:0000303': {'id': 'GO:0000303',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to superoxide',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a superoxide stimulus. Superoxide is the anion, oxygen-, formed by addition of one electron to dioxygen (O2) or any compound containing the superoxide anion.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'children': [{'id': 'GO:0071451', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000305', 'relation': 'is_a'}]},\n",
       " 'GO:0000304': {'id': 'GO:0000304',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to singlet oxygen',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a singlet oxygen stimulus. Singlet oxygen is a dioxygen (O2) molecule in which two 2p electrons have similar spin. Singlet oxygen is more highly reactive than the form in which these electrons are of opposite spin, and it is produced in mutant chloroplasts lacking carotenoids and by leukocytes during metabolic burst.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0124325653'},\n",
       "    {'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'children': [{'id': 'GO:0071452', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000302', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007584', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010035', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042493', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0017035', 'relation': 'is_a'}]},\n",
       " 'GO:0000305': {'id': 'GO:0000305',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to oxygen radical',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an oxygen radical stimulus. An oxygen radical is any oxygen species that carries a free electron; examples include hydroxyl radicals and the superoxide anion.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0124325653'}]},\n",
       "  'children': [{'id': 'GO:0000303', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071450', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000302', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010035', 'relation': 'is_a'}]},\n",
       " 'GO:0000306': {'id': 'GO:0000306',\n",
       "  'isObsolete': False,\n",
       "  'name': 'extrinsic component of vacuolar membrane',\n",
       "  'definition': {'text': 'The component of a vacuolar membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region.'},\n",
       "  'synonyms': [{'name': 'extrinsic to vacuolar membrane', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0097042', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032419', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097635', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000307': {'id': 'GO:0000307',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cyclin-dependent protein kinase holoenzyme complex',\n",
       "  'definition': {'text': 'Cyclin-dependent protein kinases (CDKs) are enzyme complexes that contain a kinase catalytic subunit associated with a regulatory cyclin partner.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11602261'}]},\n",
       "  'synonyms': [{'name': 'CDK holoenzyme', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0097128', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097127', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031387', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097129', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097124', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097123', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097126', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097134', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097125', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097135', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097132', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990860', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097133', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097122', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097130', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097121', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097131', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000308', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002945', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002944', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019908', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0097472', 'relation': 'capable_of'}]},\n",
       " 'GO:0000308': {'id': 'GO:0000308',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytoplasmic cyclin-dependent protein kinase holoenzyme complex',\n",
       "  'definition': {'text': 'Cyclin-dependent protein kinase (CDK) complex found in the cytoplasm.'},\n",
       "  'synonyms': [{'name': 'CDK holoenzyme', 'type': 'broad'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000309': {'id': 'GO:0000309',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nicotinamide-nucleotide adenylyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + nicotinamide nucleotide = diphosphate + NAD+.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.7.7.1'}]},\n",
       "  'synonyms': [{'name': 'NAD+ pyrophosphorylase activity', 'type': 'exact'},\n",
       "   {'name': 'NAD+ diphosphorylase activity', 'type': 'exact'},\n",
       "   {'name': 'ATP:NMN adenylyltransferase activity', 'type': 'exact'},\n",
       "   {'name': 'nicotinamide mononucleotide adenylyltransferase activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'nicotinamide adenine dinucleotide pyrophosphorylase activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'diphosphopyridine nucleotide pyrophosphorylase activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'adenosine triphosphate-nicotinamide mononucleotide transadenylase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'NMNAT activity', 'type': 'narrow'},\n",
       "   {'name': 'NMN adenylyltransferase activity', 'type': 'narrow'},\n",
       "   {'name': 'ATP:nicotinamide-nucleotide adenylyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'NAD(+) diphosphorylase activity', 'type': 'narrow'},\n",
       "   {'name': 'NAD(+) pyrophosphorylase activity', 'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0070566', 'relation': 'is_a'}]},\n",
       " 'GO:0000310': {'id': 'GO:0000310',\n",
       "  'isObsolete': False,\n",
       "  'name': 'xanthine phosphoribosyltransferase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: 5-phospho-alpha-D-ribose 1-diphosphate + xanthine = (9-D-ribosylxanthine)-5'-phosphate + diphosphate.\",\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.4.2.22'}]},\n",
       "  'synonyms': [{'name': 'xanthine-guanine phosphoribosyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'Xan phosphoribosyltransferase activity', 'type': 'exact'},\n",
       "   {'name': 'XMP pyrophosphorylase activity', 'type': 'exact'},\n",
       "   {'name': '5-phospho-alpha-D-ribose-1-diphosphate:xanthine phospho-D-ribosyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'XMP:diphosphate 5-phospho-alpha-D-ribosyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': \"xanthosine 5'-phosphate pyrophosphorylase activity\",\n",
       "    'type': 'exact'},\n",
       "   {'name': 'xanthylate pyrophosphorylase activity', 'type': 'exact'},\n",
       "   {'name': 'xanthylic pyrophosphorylase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0106130', 'relation': 'is_a'}]},\n",
       " 'GO:0000311': {'id': 'GO:0000311',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid large ribosomal subunit',\n",
       "  'definition': {'text': 'The larger of the two subunits of a plastid ribosome. Two sites on the ribosomal large subunit are involved in translation: the aminoacyl site (A site) and peptidyl site (P site).'},\n",
       "  'children': [{'id': 'GO:0022628', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000312': {'id': 'GO:0000312',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid small ribosomal subunit',\n",
       "  'definition': {'text': 'The smaller of the two subunits of a plastid ribosome.'},\n",
       "  'children': [{'id': 'GO:0022629', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000313': {'id': 'GO:0000313',\n",
       "  'isObsolete': False,\n",
       "  'name': 'organellar ribosome',\n",
       "  'definition': {'text': 'A ribosome contained within a subcellular membrane-bounded organelle.'},\n",
       "  'children': [{'id': 'GO:0009547', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005761', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000315', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000314', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000314': {'id': 'GO:0000314',\n",
       "  'isObsolete': False,\n",
       "  'name': 'organellar small ribosomal subunit',\n",
       "  'definition': {'text': 'The smaller of the two subunits of an organellar ribosome.'},\n",
       "  'children': [{'id': 'GO:0000312', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005763', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000315': {'id': 'GO:0000315',\n",
       "  'isObsolete': False,\n",
       "  'name': 'organellar large ribosomal subunit',\n",
       "  'definition': {'text': 'The larger of the two subunits of an organellar ribosome. Two sites on the ribosomal large subunit are involved in translation: the aminoacyl site (A site) and peptidyl site (P site).'},\n",
       "  'children': [{'id': 'GO:0000311', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005762', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000316': {'id': 'GO:0000316',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sulfite transport',\n",
       "  'definition': {'text': 'The directed movement of sulfite into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore.'},\n",
       "  'synonyms': [{'name': 'sulphite transport', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1900072', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1900071', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0061426', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0000319', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015698', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072348', 'relation': 'is_a'}]},\n",
       " 'GO:0000319': {'id': 'GO:0000319',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sulfite transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of sulfite ions from one side of a membrane to the other.'},\n",
       "  'synonyms': [{'name': 'sulphite transporter activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000316', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0015103', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901682', 'relation': 'is_a'}]},\n",
       " 'GO:0000320': {'id': 'GO:0000320',\n",
       "  'isObsolete': False,\n",
       "  'name': 're-entry into mitotic cell cycle',\n",
       "  'definition': {'text': 'The resumption of the mitotic cell division cycle by cells that were in a quiescent or other non-dividing state.'},\n",
       "  'children': [{'id': 'GO:0000321', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0022402', 'relation': 'is_a'}]},\n",
       " 'GO:0000321': {'id': 'GO:0000321',\n",
       "  'isObsolete': False,\n",
       "  'name': 're-entry into mitotic cell cycle after pheromone arrest',\n",
       "  'definition': {'text': 'The resumption of the mitotic cell division cycle by pheromone-arrested cells that have not mated. An example of this process is found in Saccharomyces cerevisiae.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9927449'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000320', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000754', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007331', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030453', 'relation': 'is_a'}]},\n",
       " 'GO:0000322': {'id': 'GO:0000322',\n",
       "  'isObsolete': False,\n",
       "  'name': 'storage vacuole',\n",
       "  'definition': {'text': 'A vacuole that functions primarily in the storage of materials, including nutrients, pigments, waste products, and small molecules.'},\n",
       "  'children': [{'id': 'GO:0106123', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000326', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000324', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000323': {'id': 'GO:0000323',\n",
       "  'isObsolete': False,\n",
       "  'name': 'lytic vacuole',\n",
       "  'definition': {'text': 'A vacuole that is maintained at an acidic pH and which contains degradative enzymes, including a wide variety of acid hydrolases.'},\n",
       "  'children': [{'id': 'GO:0000324', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098852', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0106123', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005764', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000327', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010282', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000324': {'id': 'GO:0000324',\n",
       "  'isObsolete': False,\n",
       "  'name': 'fungal-type vacuole',\n",
       "  'definition': {'text': 'A vacuole that has both lytic and storage functions. The fungal vacuole is a large, membrane-bounded organelle that functions as a reservoir for the storage of small molecules (including polyphosphate, amino acids, several divalent cations (e.g. calcium), other ions, and other small molecules) as well as being the primary compartment for degradation. It is an acidic compartment, containing an ensemble of acid hydrolases. At least in S. cerevisiae, there are indications that the morphology of the vacuole is variable and correlated with the cell cycle, with logarithmically growing cells having a multilobed, reticulated vacuole, while stationary phase cells contain a single large structure.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879693649'}]},\n",
       "  'synonyms': [{'name': 'vacuole, cell cycle-correlated morphology',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000329', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000328', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000325': {'id': 'GO:0000325',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plant-type vacuole',\n",
       "  'definition': {'text': 'A closed structure that is completely surrounded by a unit membrane, contains liquid, and retains the same shape regardless of cell cycle phase. An example of this structure is found in Arabidopsis thaliana.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815316208'}]},\n",
       "  'synonyms': [{'name': 'vacuole, cell cycle-independent morphology',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000326', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009705', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042807', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000330', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010282', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000326': {'id': 'GO:0000326',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein storage vacuole',\n",
       "  'definition': {'text': 'A storage vacuole that contains a lytic vacuole; identified in plants.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11739409'}]},\n",
       "  'children': [{'id': 'GO:0000327', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032586', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0034495', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000327': {'id': 'GO:0000327',\n",
       "  'isObsolete': False,\n",
       "  'name': 'lytic vacuole within protein storage vacuole',\n",
       "  'definition': {'text': 'A membrane-bounded compartment containing crystals of phytic acid and proteins characteristic of a lytic vacuole, found within a storage vacuole.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11739490'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000328': {'id': 'GO:0000328',\n",
       "  'isObsolete': False,\n",
       "  'name': 'fungal-type vacuole lumen',\n",
       "  'definition': {'text': 'The volume enclosed within the vacuolar membrane of a vacuole, the shape of which correlates with cell cycle phase. An example of this structure is found in Saccharomyces cerevisiae.'},\n",
       "  'synonyms': [{'name': 'lumen of vacuole with cell cycle-correlated morphology',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000329': {'id': 'GO:0000329',\n",
       "  'isObsolete': False,\n",
       "  'name': 'fungal-type vacuole membrane',\n",
       "  'definition': {'text': 'The lipid bilayer surrounding a vacuole, the shape of which correlates with cell cycle phase. The membrane separates its contents from the cytoplasm of the cell. An example of this structure is found in Saccharomyces cerevisiae.'},\n",
       "  'synonyms': [{'name': 'fungal-type vacuolar membrane', 'type': 'related'},\n",
       "   {'name': 'membrane of vacuole with cell cycle-correlated morphology',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0097042', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071627', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071628', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000330': {'id': 'GO:0000330',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plant-type vacuole lumen',\n",
       "  'definition': {'text': 'The volume enclosed within the vacuolar membrane of a vacuole that retains the same shape regardless of cell cycle phase. An example of this is found in Arabidopsis thaliana.'},\n",
       "  'synonyms': [{'name': 'lumen of vacuole with cell cycle-independent morphology',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0034495', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000331': {'id': 'GO:0000331',\n",
       "  'isObsolete': False,\n",
       "  'name': 'contractile vacuole',\n",
       "  'definition': {'text': 'A specialized vacuole of eukaryotic cells, especially Protozoa, that fills with water from the cytoplasm and then discharges this externally by the opening of contractile vacuole pores. One of its functions is osmoregulatory.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10503189'},\n",
       "    {'dbCode': 'PMID', 'dbId': '23890380'}]},\n",
       "  'synonyms': [{'name': 'central bladder', 'type': 'related'},\n",
       "   {'name': 'central vacuole', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0031164', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000332': {'id': 'GO:0000332',\n",
       "  'isObsolete': False,\n",
       "  'name': 'template for synthesis of G-rich strand of telomere DNA activity',\n",
       "  'definition': {'text': 'Provision of the template used by reverse transcriptase to synthesize the G-rich strand of telomeric DNA.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11812242'}]},\n",
       "  'comment': 'Note that this term describes the activity of an RNA gene product that interacts with other nucleic acid molecules via base pairing; it should not be used to annotate proteins.',\n",
       "  'synonyms': [{'name': 'telomerase, template', 'type': 'exact'},\n",
       "   {'name': 'telomerase RNA', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000497', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003720', 'relation': 'part_of'}]},\n",
       " 'GO:0000333': {'id': 'GO:0000333',\n",
       "  'isObsolete': False,\n",
       "  'name': 'telomerase catalytic core complex',\n",
       "  'definition': {'text': 'The minimal catalytic core of telomerase is a ribonucleoprotein complex composed of a catalytic reverse transcriptase subunit and an RNA subunit that provides the template for telomeric DNA addition.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11884619'},\n",
       "    {'dbCode': 'PMID', 'dbId': '1808260'}]},\n",
       "  'synonyms': [{'name': 'TERT-TERC complex', 'type': 'related'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003720', 'relation': 'capable_of'}]},\n",
       " 'GO:0000334': {'id': 'GO:0000334',\n",
       "  'isObsolete': False,\n",
       "  'name': '3-hydroxyanthranilate 3,4-dioxygenase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 3-hydroxyanthranilate + O(2) = cis,cis-2-amino-3-(3-oxoprop-1-enyl)but-2-enedioate + H(+).',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.13.11.6'},\n",
       "    {'dbCode': 'RHEA', 'dbId': '17953'}]},\n",
       "  'synonyms': [{'name': '3-hydroxyanthranilic acid dioxygenase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': '3-hydroxyanthranilic acid oxygenase activity', 'type': 'exact'},\n",
       "   {'name': '3-hydroxyanthranilic oxygenase activity', 'type': 'exact'},\n",
       "   {'name': '3HAO', 'type': 'related'},\n",
       "   {'name': '3-hydroxyanthranilate oxygenase activity', 'type': 'exact'},\n",
       "   {'name': '3-hydroxyanthranilate:oxygen 3,4-oxidoreductase (decyclizing)',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016702', 'relation': 'is_a'}]},\n",
       " 'GO:0000335': {'id': 'GO:0000335',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative regulation of transposition, DNA-mediated',\n",
       "  'definition': {'text': 'Any process that stops, prevents, or reduces the frequency, rate or extent of DNA transposition.'},\n",
       "  'synonyms': [{'name': 'negative regulation of DNA transposition',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'down regulation of DNA transposition', 'type': 'exact'},\n",
       "   {'name': 'inhibition of DNA transposition', 'type': 'narrow'},\n",
       "   {'name': 'downregulation of DNA transposition', 'type': 'exact'},\n",
       "   {'name': 'down-regulation of DNA transposition', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000337', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006313', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0010529', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045910', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006318', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0006317', 'relation': 'negatively_regulates'}]},\n",
       " 'GO:0000336': {'id': 'GO:0000336',\n",
       "  'isObsolete': False,\n",
       "  'name': 'positive regulation of transposition, DNA-mediated',\n",
       "  'definition': {'text': 'Any process that activates or increases the frequency, rate or extent of DNA transposition.'},\n",
       "  'synonyms': [{'name': 'upregulation of DNA transposition', 'type': 'exact'},\n",
       "   {'name': 'up regulation of DNA transposition', 'type': 'exact'},\n",
       "   {'name': 'positive regulation of DNA transposition', 'type': 'exact'},\n",
       "   {'name': 'up-regulation of DNA transposition', 'type': 'exact'},\n",
       "   {'name': 'stimulation of DNA transposition', 'type': 'narrow'},\n",
       "   {'name': 'activation of DNA transposition', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000337', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006313', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0010530', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045911', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006318', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0006317', 'relation': 'positively_regulates'}]},\n",
       " 'GO:0000337': {'id': 'GO:0000337',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of transposition, DNA-mediated',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of DNA transposition, the process of transposing (moving to a different location) a segment of a chromosome or a piece of a DNA molecule.'},\n",
       "  'synonyms': [{'name': 'regulation of DNA transposition', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000335', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000336', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000018', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006313', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0010528', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006318', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0006317', 'relation': 'regulates'}]},\n",
       " 'GO:0000338': {'id': 'GO:0000338',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein deneddylation',\n",
       "  'definition': {'text': 'The removal of a ubiquitin-like protein of the NEDD8 type from a protein.'},\n",
       "  'synonyms': [{'name': 'cullin deneddylation', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0060625', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0008180', 'relation': 'capable_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0070646', 'relation': 'is_a'}]},\n",
       " 'GO:0000339': {'id': 'GO:0000339',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA cap binding',\n",
       "  'definition': {'text': \"Interacting selectively and non-covalently with a 7-methylguanosine (m7G) group or derivative located at the 5' end of an RNA molecule.\"},\n",
       "  'synonyms': [{'name': 'snRNA cap binding', 'type': 'narrow'},\n",
       "   {'name': 'mRNA cap binding', 'type': 'narrow'},\n",
       "   {'name': 'binding to mRNA cap', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0098808', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034518', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0000340', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000341', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000342', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003723', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000498', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044822', 'relation': 'is_a'}]},\n",
       " 'GO:0000340': {'id': 'GO:0000340',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA 7-methylguanosine cap binding',\n",
       "  'definition': {'text': \"Interacting selectively and non-covalently with the 7-methylguanosine group added cotranscriptionally to the 5' end of RNA molecules transcribed by polymerase II.\"},\n",
       "  'synonyms': [{'name': 'RNA m7G cap binding', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0106095', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000339', 'relation': 'is_a'}]},\n",
       " 'GO:0000341': {'id': 'GO:0000341',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA trimethylguanosine cap binding',\n",
       "  'definition': {'text': \"Interacting selectively and non-covalently with the trimethylguanosine (m(3)(2,2,7)-GTP) group located at the 5' end of some RNA molecules. Such trimethylated cap structures, generally produced by posttranscriptional modification of a 7-methylguanosine cap, are often found on snRNAs and snoRNAs transcribed by RNA polymerase II, but have also be found on snRNAs transcribed by RNA polymerase III. They have also been found on a subset of the mRNA population in some species, e.g. C. elegans.\"},\n",
       "  'synonyms': [{'name': 'RNA m2,2,7G cap binding', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000339', 'relation': 'is_a'}]},\n",
       " 'GO:0000342': {'id': 'GO:0000342',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA cap 4 binding',\n",
       "  'definition': {'text': \"Interacting selectively and non-covalently with a hypermethylated cap structure consisting of 7-methylguanosine (m(7)G) followed by four methylated nucleotides (cap 4): 7-methylguanosine-ppp-N6, N6, 2'-O-trimethyladenosine-p-2'-O-methyladenosine-p-2'-O-methylcytosine-p-N3, 2'-O-dimethyluridine Such caps are known to be found at the 5' ends of SL RNAs of trypanosomatid protozoa.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10880518'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12121975'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000339', 'relation': 'is_a'}]},\n",
       " 'GO:0000343': {'id': 'GO:0000343',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid-encoded plastid RNA polymerase complex A',\n",
       "  'definition': {'text': 'A plastid-encoded DNA-directed RNA polymerase complex that resembles eubacterial multisubunit RNA polymerases, with a core composed of alpha, beta, and beta-prime subunits. An additional subunit, a sigma factor, is required for promoter recognition. PEP-A is generated from the PEP-B form during chloroplast maturation to generate a complex composed of at least thirteen polypeptides that is not sensitive to the antibiotic rifampicin, like its precursor form the PEP-B complex.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10946105'}]},\n",
       "  'synonyms': [{'name': 'PEP-A', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000344': {'id': 'GO:0000344',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid-encoded plastid RNA polymerase complex B',\n",
       "  'definition': {'text': 'A plastid-encoded DNA-directed RNA polymerase complex that resembles eubacterial multisubunit RNA polymerases with a core composed of alpha, beta, and beta-prime subunits. An additional subunit, a sigma factor, is required for promoter recognition. PEP-B is distinguished from PEP-A by its sensitivity to the antibiotic rifampicin. PEP-B is found in both etioplasts and chloroplasts, but is the predominate form in etioplasts. It forms the core of the PEP-A form; the conversion from PEP-B to PEP-A occurs during chloroplast maturation.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10946105'}]},\n",
       "  'synonyms': [{'name': 'PEP-B', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000345': {'id': 'GO:0000345',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytosolic DNA-directed RNA polymerase complex',\n",
       "  'definition': {'text': 'The eubacterial DNA-directed RNA polymerase is a multisubunit complex with a core composed of the essential subunits beta-prime, beta, and two copies of alpha and a fifth nonessential subunit called omega. An additional subunit, a sigma factor, is required for promoter recognition and specificity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11158566'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000346': {'id': 'GO:0000346',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription export complex',\n",
       "  'definition': {'text': 'The transcription export (TREX) complex couples transcription elongation by RNA polymerase II to mRNA export. The complex associates with the polymerase and travels with it along the length of the transcribed gene. TREX is composed of the THO transcription elongation complex as well as other proteins that couple THO to mRNA export proteins. The TREX complex is known to be found in a wide range of eukaryotes, including S. cerevisiae and metazoans.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11979277'}]},\n",
       "  'synonyms': [{'name': 'TREX complex', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000445', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000347': {'id': 'GO:0000347',\n",
       "  'isObsolete': False,\n",
       "  'name': 'THO complex',\n",
       "  'definition': {'text': 'The THO complex is a nuclear complex that is required for transcription elongation through genes containing tandemly repeated DNA sequences. The THO complex is also part of the TREX (TRanscription EXport) complex that is involved in coupling transcription to export of mRNAs to the cytoplasm. In S. cerevisiae, it is composed of four subunits: Hpr1p, Tho2p, Thp1p, and Mft1p, while the human complex is composed of 7 subunits.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11060033'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11979277'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16983072'}]},\n",
       "  'children': [{'id': 'GO:0000446', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000445', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000348': {'id': 'GO:0000348',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mRNA branch site recognition',\n",
       "  'definition': {'text': 'Recognition of the pre-mRNA branch site sequence by components of the assembling spliceosome.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': 'Note that this step represents the formation of the B complex (yeast) or the A complex (mammalian).',\n",
       "  'synonyms': [{'name': 'U2-type nuclear mRNA branch site recognition',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'nuclear mRNA branch site recognition', 'type': 'exact'},\n",
       "   {'name': 'U12-type nuclear mRNA branch site recognition', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal B complex formation', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal B complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal A complex formation', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal A complex biosynthesis', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000245', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'}]},\n",
       " 'GO:0000349': {'id': 'GO:0000349',\n",
       "  'isObsolete': False,\n",
       "  'name': 'generation of catalytic spliceosome for first transesterification step',\n",
       "  'definition': {'text': 'Formation of a catalytic spliceosome complex ready to perform the first splicing reaction. This occurs by an ATP-dependent conformational change of the pre-catalytic spliceosome.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': 'Note that this step represents the formation of the A2-2 complex (yeast) or the C1 complex (mammalian).',\n",
       "  'synonyms': [{'name': 'U2-type catalytic spliceosome formation for first transesterification step',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'U12-type catalytic spliceosome formation for first transesterification step',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'formation of catalytic spliceosome for first transesterification step',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'spliceosomal A2-2 complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal C1 complex formation', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal C1 complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal A2-2 complex formation', 'type': 'narrow'},\n",
       "   {'name': 'catalytic spliceosome assembly for first transesterification step',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000393', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'}]},\n",
       " 'GO:0000350': {'id': 'GO:0000350',\n",
       "  'isObsolete': False,\n",
       "  'name': 'generation of catalytic spliceosome for second transesterification step',\n",
       "  'definition': {'text': 'Conformational rearrangement of the spliceosomal complex containing the RNA products from the 1st step of splicing to form the catalytic site for the second step of splicing.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': 'Note that this step represents formation of the A2-3 complex (yeast) or the C2 complex (mammalian).',\n",
       "  'synonyms': [{'name': 'formation of catalytic U12-type spliceosome for second transesterification step',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'formation of catalytic U2-type spliceosome for second transesterification step',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'spliceosomal A2-3 complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal A2-3 complex formation', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal C2 complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal C2 complex formation', 'type': 'narrow'},\n",
       "   {'name': 'catalytic spliceosome assembly for second transesterification step',\n",
       "    'type': 'related'},\n",
       "   {'name': 'formation of catalytic spliceosome for second transesterification step',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'formation of spliceosomal A2-2 complex', 'type': 'narrow'},\n",
       "   {'name': 'formation of spliceosomal C1 complex', 'type': 'narrow'},\n",
       "   {'name': \"lariat formation, 5'-splice site cleavage\", 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0034403', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000393', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'}]},\n",
       " 'GO:0000352': {'id': 'GO:0000352',\n",
       "  'isObsolete': False,\n",
       "  'name': 'trans assembly of SL-containing precatalytic spliceosome',\n",
       "  'definition': {'text': 'Assembly of a spliceosomal complex containing the SL RNA and the pre-mRNA to be joined, as well as all the spliceosomal snRNPs involved in trans leader splicing. Formation of the trans leader spliceosome brings together the quadruple SL/U4/U5/U6 snRNP and the complex of the U2 snRNP with the splice site of the pre-mRNA.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'synonyms': [{'name': 'trans assembly of spliced leader-containing precatalytic spliceosome',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000245', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045291', 'relation': 'part_of'}]},\n",
       " 'GO:0000353': {'id': 'GO:0000353',\n",
       "  'isObsolete': False,\n",
       "  'name': 'formation of quadruple SL/U4/U5/U6 snRNP',\n",
       "  'definition': {'text': \"Formation of a quadruple snRNP complex composed of the spliced leader (SL) RNA along with the U4/U6-U5 tri-snRNP complex. Interactions that may facilitate this include a duplex between the SL and U6 RNAs and interactions between the U5 RNA and the exon sequence at the 5' splice site within the SL RNA.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': \"Note that this step is analogous to 5' splice site selection in cis-splicing.\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0022618', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045291', 'relation': 'part_of'}]},\n",
       " 'GO:0000354': {'id': 'GO:0000354',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cis assembly of pre-catalytic spliceosome',\n",
       "  'definition': {'text': 'Assembly of a spliceosomal complex containing the intact pre-mRNA and all of the spliceosomal snRNPs. This occurs when the tri-snRNP associates with the pre-mRNA and associated snRNPs in an ATP-dependent manner.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': 'Note that this step represents formation of the A2-1 complex (yeast) or the B1 complex (mammals).',\n",
       "  'synonyms': [{'name': 'spliceosomal A2-1 complex biosynthesis',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'spliceosomal A2-1 complex formation', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal B1 complex formation', 'type': 'narrow'},\n",
       "   {'name': 'formation of spliceosomal A2-1 complex', 'type': 'narrow'},\n",
       "   {'name': 'formation of spliceosomal B1 complex', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal B1 complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'cis assembly of U2-type pre-catalytic spliceosome',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'cis assembly of U12-type pre-catalytic spliceosome',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000245', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045292', 'relation': 'part_of'}]},\n",
       " 'GO:0000365': {'id': 'GO:0000365',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mRNA trans splicing, via spliceosome',\n",
       "  'definition': {'text': 'The joining together of exons from two different primary transcripts of messenger RNA (mRNA) via a spliceosomal mechanism, so that mRNA consisting only of the joined exons is produced.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'synonyms': [{'name': 'nuclear mRNA trans splicing, via spliceosome',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'nuclear mRNA trans splicing, via U2-type spliceosome',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0045291', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000366', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000398', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006375', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006374', 'relation': 'is_a'}]},\n",
       " 'GO:0000366': {'id': 'GO:0000366',\n",
       "  'isObsolete': False,\n",
       "  'name': 'intergenic mRNA trans splicing',\n",
       "  'definition': {'text': 'The joining together of two independently transcribed RNAs from two different genes, each of which also produces mRNA(s) via cis-splicing.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11726664'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12110900'}]},\n",
       "  'synonyms': [{'name': 'intergenic nuclear mRNA trans splicing',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000365', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000380', 'relation': 'is_a'}]},\n",
       " 'GO:0000372': {'id': 'GO:0000372',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Group I intron splicing',\n",
       "  'definition': {'text': 'The splicing of Group I introns. This occurs by a ribozymic mechanism where the intron sequence forms a distinct 3D structure, characteristic of Group I introns and involved in determining the locations of the splice sites (there do not appear to be consensus splice site sequences) as well as having a role in catalyzing the splicing reactions, though protein factors are also required in vivo. Splicing occurs by a series of two transesterification reactions, generally with exogenous guanosine as the initiating nucleophile. The intron is excised as a linear piece (though it may subsequently circularize).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11377794'}]},\n",
       "  'comment': 'Note that Group I introns are known to be found in a number of places: rRNA, mRNA, and tRNA in organelles of fungi, plants, and protists; tRNA and mRNA of bacteria and bacteriophage; rRNA of protists and fungi; and occasionally in mRNA of animal mitochondria (e.g. sea anemone).',\n",
       "  'synonyms': [{'name': 'mRNA splicing', 'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000376', 'relation': 'is_a'}]},\n",
       " 'GO:0000373': {'id': 'GO:0000373',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Group II intron splicing',\n",
       "  'definition': {'text': 'The splicing of Group II introns. This occurs by a ribozymic mechanism where the intron sequence forms a distinct 3D structure, characteristic of Group II introns and containing splice site consensus sequences, that is involved in catalyzing the splicing reactions, though protein factors are also required in vivo. Splicing occurs by a series of two transesterification reactions (mechanistically similar to those for splicing of nuclear mRNAs) initiated by a bulged adenosine residue within the intron sequence as the initiating nucleophile. The intron is excised as a lariat.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11377794'}]},\n",
       "  'comment': 'Note that Group II introns are known to be found in a number of places: rRNA, mRNA, and tRNA in organelles of fungi, plants, and protists; and mRNA of bacteria.',\n",
       "  'synonyms': [{'name': 'mRNA splicing', 'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000377', 'relation': 'is_a'}]},\n",
       " 'GO:0000374': {'id': 'GO:0000374',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Group III intron splicing',\n",
       "  'definition': {'text': 'The splicing of Group III introns. This occurs by a ribozymic mechanism where the intron sequence forms a distinct 3D structure, characteristic of Group III introns, that is involved in catalyzing the splicing reactions, though protein factors are also required in vivo. Splicing occurs by a series of two transesterification reactions begun by a bulged adenosine residue within the intron sequence as the initiating nucleophile. The intron is excised as a lariat. Though very similar in structure and mechanism to Group II introns, Group III introns are smaller and more streamlined and the splice site consensus sequences are not as well conserved.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11377794'}]},\n",
       "  'comment': 'Note that Group III introns are known to be found in mRNA of plastids of euglenoid protists.',\n",
       "  'synonyms': [{'name': 'mRNA splicing', 'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000377', 'relation': 'is_a'}]},\n",
       " 'GO:0000375': {'id': 'GO:0000375',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA splicing, via transesterification reactions',\n",
       "  'definition': {'text': 'Splicing of RNA via a series of two transesterification reactions.'},\n",
       "  'comment': 'Note that nuclear mRNA, Group I, Group II, and Group III introns are all spliced by a series of two transesterification reactions that occur within the RNA itself, or between two RNAs in trans splicing. Some of these require one or more proteins to stabilize the catalytic conformation, while others are autocatalytic. Note that tRNA introns are spliced by a different catalytic mechanism.',\n",
       "  'synonyms': [{'name': 'pre-mRNA splicing factor activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA splicing factor activity, transesterification mechanism',\n",
       "    'type': 'related'},\n",
       "   {'name': 'spliceosomal catalysis', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000376', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000377', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000384', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000386', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008380', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006395', 'relation': 'is_a'}]},\n",
       " 'GO:0000376': {'id': 'GO:0000376',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA splicing, via transesterification reactions with guanosine as nucleophile',\n",
       "  'definition': {'text': 'Splicing of RNA via a series of two transesterification reactions with exogenous guanosine as the initiating nucleophile.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11377794'}]},\n",
       "  'children': [{'id': 'GO:0000372', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000375', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031202', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000385', 'relation': 'is_a'}]},\n",
       " 'GO:0000377': {'id': 'GO:0000377',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA splicing, via transesterification reactions with bulged adenosine as nucleophile',\n",
       "  'definition': {'text': 'Splicing of RNA via a series of two transesterification reactions with a bulged adenosine residue from the intron branch point as the initiating nucleophile. When the initial RNA for the splicing reaction is a single molecule (cis splicing), the excised intron is released in a lariat structure.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11377794'}]},\n",
       "  'synonyms': [{'name': 'lariat RNA formation', 'type': 'related'},\n",
       "   {'name': 'lariat RNA biosynthesis', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000373', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000374', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000398', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000375', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031202', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000385', 'relation': 'is_a'}]},\n",
       " 'GO:0000378': {'id': 'GO:0000378',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA exon ligation',\n",
       "  'definition': {'text': 'The RNA metabolic process that joins two exons, each of which has free ends that were generated by endonucleolytic cleavages, by a ligation reaction.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': 'Note that this is not a part of spliceosomal RNA splicing.',\n",
       "  'children': [{'id': 'GO:0000968', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000394', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0016070', 'relation': 'is_a'}]},\n",
       " 'GO:0000379': {'id': 'GO:0000379',\n",
       "  'isObsolete': False,\n",
       "  'name': 'tRNA-type intron splice site recognition and cleavage',\n",
       "  'definition': {'text': \"RNA processing that begins when the tertiary structure of a tRNA type intron is recognized, and ends when the endonucleolytic cleavage of the RNA at both the 5' and 3' splice sites occurs.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006388', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006396', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006394', 'relation': 'is_a'}]},\n",
       " 'GO:0000380': {'id': 'GO:0000380',\n",
       "  'isObsolete': False,\n",
       "  'name': 'alternative mRNA splicing, via spliceosome',\n",
       "  'definition': {'text': 'The process of generating multiple mRNA molecules from a given set of exons by differential use of exons from the primary transcript(s) to form multiple mature mRNAs that vary in their exon composition.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12110900'}]},\n",
       "  'comment': 'Note that this process most commonly occurs in cis, selecting or skipping exons from the same primary transcript, but it has also been observed to occur in trans at low frequency, at least in some mammals.',\n",
       "  'synonyms': [{'name': 'splice site selection', 'type': 'broad'},\n",
       "   {'name': 'alternative nuclear mRNA splicing, via spliceosome',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000366', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000381', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000398', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006375', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006374', 'relation': 'is_a'}]},\n",
       " 'GO:0000381': {'id': 'GO:0000381',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of alternative mRNA splicing, via spliceosome',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of alternative splicing of nuclear mRNAs.'},\n",
       "  'synonyms': [{'name': 'splice site selection', 'type': 'broad'},\n",
       "   {'name': 'regulation of alternative nuclear mRNA splicing, via spliceosome',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000380', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0048024', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035055', 'relation': 'is_a'}]},\n",
       " 'GO:0000384': {'id': 'GO:0000384',\n",
       "  'isObsolete': False,\n",
       "  'name': 'first spliceosomal transesterification activity',\n",
       "  'definition': {'text': \"Catalysis of the first transesterification reaction of spliceosomal mRNA splicing. The intron branch site adenosine is the nucleophile attacking the 5' splice site, resulting in cleavage at this position. In cis splicing, this is the step that forms a lariat structure of the intron RNA, while it is still joined to the 3' exon.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'synonyms': [{'name': \"lariat formation, 5'-splice site cleavage\",\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0071012', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000375', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140098', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031202', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000385', 'relation': 'part_of'}]},\n",
       " 'GO:0000386': {'id': 'GO:0000386',\n",
       "  'isObsolete': False,\n",
       "  'name': 'second spliceosomal transesterification activity',\n",
       "  'definition': {'text': \"Catalysis of the second transesterification reaction of spliceosomal mRNA splicing. Ligation of the two exons occurs via a transesterification reaction where the free 3'-hydroxyl group of the 5' exon is the nucleophile attacking the 3' splice site. Non-expressed sequences are now detached from the exons. In cis splicing, the intron is in a lariat structure.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'synonyms': [{'name': \"3'-splice site cleavage, exon ligation\",\n",
       "    'type': 'broad'},\n",
       "   {'name': \"lariat formation, 5'-splice site cleavage\", 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0071013', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000375', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140098', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031202', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000385', 'relation': 'part_of'}]},\n",
       " 'GO:0000387': {'id': 'GO:0000387',\n",
       "  'isObsolete': False,\n",
       "  'name': 'spliceosomal snRNP assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of one or more snRNA and multiple protein components to form a ribonucleoprotein complex that is involved in formation of the spliceosome.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'synonyms': [{'name': 'spliceosomal snRNP biogenesis', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000244', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000398', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006375', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006374', 'relation': 'part_of'}]},\n",
       " 'GO:0000388': {'id': 'GO:0000388',\n",
       "  'isObsolete': False,\n",
       "  'name': 'spliceosome conformational change to release U4 (or U4atac) and U1 (or U11)',\n",
       "  'definition': {'text': 'Rearrangement of the pre-catalytic spliceosome containing U4 (or U4atac) and U1 (or U11) snRNPs to unpair U4 (or U4atac) from U6 (or U6atac) and release it from the spliceosomal complex along with U1 (or U11).',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': 'Note that this step represents formation of the A1 complex (yeast) or the B2 complex (mammalian).',\n",
       "  'synonyms': [{'name': \"3'-splice site cleavage, exon ligation\",\n",
       "    'type': 'broad'},\n",
       "   {'name': 'spliceosomal B2 complex formation', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal B2 complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal A1 complex formation', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal A1 complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'U2-type spliceosome conformational change to release U4 and U1',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'U12-type spliceosome conformational change to release U4atac and U11',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000393', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'}]},\n",
       " 'GO:0000389': {'id': 'GO:0000389',\n",
       "  'isObsolete': False,\n",
       "  'name': \"mRNA 3'-splice site recognition\",\n",
       "  'definition': {'text': \"Recognition of the intron 3'-splice site by components of the assembling U2- or U12-type spliceosome.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'synonyms': [{'name': \"U12-type nuclear mRNA 3'-splice site recognition\",\n",
       "    'type': 'narrow'},\n",
       "   {'name': \"U2-type nuclear mRNA 3'-splice site recognition\",\n",
       "    'type': 'narrow'},\n",
       "   {'name': \"nuclear mRNA 3'-splice site recognition\", 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0034403', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006376', 'relation': 'is_a'}]},\n",
       " 'GO:0000390': {'id': 'GO:0000390',\n",
       "  'isObsolete': False,\n",
       "  'name': 'spliceosomal complex disassembly',\n",
       "  'definition': {'text': 'Disassembly of a spliceosomal complex with the ATP-dependent release of the product RNAs, one of which is composed of the joined exons. In cis splicing, the other product is the excised sequence, often a single intron, in a lariat structure.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'synonyms': [{'name': 'U2-type spliceosome disassembly', 'type': 'narrow'},\n",
       "   {'name': 'U12-type spliceosome disassembly', 'type': 'narrow'},\n",
       "   {'name': 'spliceosome disassembly', 'type': 'broad'},\n",
       "   {'name': 'spliceosome complex disassembly', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000398', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032988', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006375', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006374', 'relation': 'part_of'}]},\n",
       " 'GO:0000393': {'id': 'GO:0000393',\n",
       "  'isObsolete': False,\n",
       "  'name': 'spliceosomal conformational changes to generate catalytic conformation',\n",
       "  'definition': {'text': 'Structural rearrangements of the spliceosome complex, containing RNA to be spliced, to generate a catalytic conformation.'},\n",
       "  'synonyms': [{'name': \"3'-splice site cleavage, exon ligation\",\n",
       "    'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0000349', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000388', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000350', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000398', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006375', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006374', 'relation': 'part_of'}]},\n",
       " 'GO:0000394': {'id': 'GO:0000394',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA splicing, via endonucleolytic cleavage and ligation',\n",
       "  'definition': {'text': \"Splicing of RNA via recognition of the folded RNA structure that brings the 5' and 3' splice sites into proximity and cleavage of the RNA at both the 3' and 5' splice sites by an endonucleolytic mechanism, followed by ligation of the exons.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': 'Note that while typically associated with tRNA splicing, this mechanism of splicing is known to be used for some non-tRNA substrates, e.g. HAC1 (YFL031W) in S. cerevisiae and an intron in the 23S rRNA of the Archaeal species Desulfurococcus mobilis.',\n",
       "  'synonyms': [{'name': 'mRNA splicing', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0006388', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070054', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000378', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008380', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006395', 'relation': 'is_a'}]},\n",
       " 'GO:0000395': {'id': 'GO:0000395',\n",
       "  'isObsolete': False,\n",
       "  'name': \"mRNA 5'-splice site recognition\",\n",
       "  'definition': {'text': \"Recognition of the intron 5'-splice site by components of the assembling spliceosome.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': 'Note that this step represents formation of the Commitment Complex (CC, in yeast) or the E complex (mammalian).',\n",
       "  'synonyms': [{'name': \"U12-type nuclear mRNA 5'-splice site recognition\",\n",
       "    'type': 'narrow'},\n",
       "   {'name': \"U12-type nuclear mRNA 5' splice site recognition\",\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'spliceosomal CC complex formation', 'type': 'narrow'},\n",
       "   {'name': \"nuclear mRNA 5'-splice site recognition\", 'type': 'exact'},\n",
       "   {'name': 'spliceosomal E complex formation', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal commitment complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': \"U2-type nuclear mRNA 5' splice site recognition\",\n",
       "    'type': 'narrow'},\n",
       "   {'name': \"nuclear mRNA 5' splice site recognition\", 'type': 'exact'},\n",
       "   {'name': \"U2-type nuclear mRNA 5'-splice site recognition\",\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'spliceosomal CC complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal E complex biosynthesis', 'type': 'narrow'},\n",
       "   {'name': 'spliceosomal commitment complex formation', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0034403', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006376', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045292', 'relation': 'part_of'}]},\n",
       " 'GO:0000398': {'id': 'GO:0000398',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mRNA splicing, via spliceosome',\n",
       "  'definition': {'text': 'The joining together of exons from one or more primary transcripts of messenger RNA (mRNA) and the excision of intron sequences, via a spliceosomal mechanism, so that mRNA consisting only of the joined exons is produced.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'},\n",
       "    {'dbCode': 'ISBN', 'dbId': '0879695897'}]},\n",
       "  'comment': \"Note that although the many U12-type introns have the sequence AT-AC at the intron termini, some introns with these terminal sequences are spliced by the U2-type spliceosome. The distinguishing characteristics are sequences near the 5' splice site and the branch point sequences of the intron. Note that although the majority of U2-type introns have the sequence GU-AG at the intron termini, some introns with these terminal sequences are spliced by the U12-type spliceosome. The distinguishing characteristics are sequences near the 5' splice site and the branch point sequences of the intron.\",\n",
       "  'synonyms': [{'name': 'splicing AT-AC intron', 'type': 'related'},\n",
       "   {'name': 'splicing GT-AG intron', 'type': 'related'},\n",
       "   {'name': 'nuclear mRNA splicing, via spliceosome', 'type': 'exact'},\n",
       "   {'name': 'pre-mRNA splicing', 'type': 'broad'},\n",
       "   {'name': 'mRNA splicing', 'type': 'broad'},\n",
       "   {'name': 'nuclear mRNA splicing via U2-type spliceosome', 'type': 'narrow'},\n",
       "   {'name': 'nuclear mRNA splicing via U12-type spliceosome',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0000365', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048024', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0000390', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000393', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000380', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048025', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0000387', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045292', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000245', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048026', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000377', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006397', 'relation': 'is_a'}]},\n",
       " 'GO:0000399': {'id': 'GO:0000399',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cellular bud neck septin structure',\n",
       "  'definition': {'text': 'Any of a series of septin structures that are localized in the bud neck of a budding fungal cell during the cell cycle.'},\n",
       "  'children': [{'id': 'GO:0000144', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032174', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032177', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000400': {'id': 'GO:0000400',\n",
       "  'isObsolete': False,\n",
       "  'name': 'four-way junction DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with DNA containing four-way junctions, also known as Holliday junctions, a structure where two DNA double strands are held together by reciprocal exchange of two of the four strands, one strand each from the two original helices.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815332181'},\n",
       "    {'dbCode': 'PMID', 'dbId': '15563464'}]},\n",
       "  'synonyms': [{'name': 'Holliday junction binding', 'type': 'exact'},\n",
       "   {'name': 'forked DNA binding', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0000401', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000402', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000217', 'relation': 'is_a'}]},\n",
       " 'GO:0000401': {'id': 'GO:0000401',\n",
       "  'isObsolete': False,\n",
       "  'name': 'open form four-way junction DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with DNA containing the open form of a four-way junction, also known as a Holliday junction, a structure where two DNA double strands are held together by reciprocal exchange of two of the four strands, one strand each from the two original helices. The open form of a four-way junction can be diagrammed without any of the strands crossing over.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815332181'},\n",
       "    {'dbCode': 'PMID', 'dbId': '15563464'}]},\n",
       "  'synonyms': [{'name': 'open form Holliday junction binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000400', 'relation': 'is_a'}]},\n",
       " 'GO:0000402': {'id': 'GO:0000402',\n",
       "  'isObsolete': False,\n",
       "  'name': 'crossed form four-way junction DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with DNA containing the crossed form of a four-way junction, also known as a Holliday junction, a structure where two DNA double strands are held together by reciprocal exchange of two of the four strands, one strand each from the two original helices. The crossed form of a four-way junction cannot be diagrammed without any of the strands crossing over, and instead contains a single crossover between two of the strands.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0815332181'},\n",
       "    {'dbCode': 'PMID', 'dbId': '15563464'}]},\n",
       "  'synonyms': [{'name': 'crossed form Holliday junction binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000400', 'relation': 'is_a'}]},\n",
       " 'GO:0000403': {'id': 'GO:0000403',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Y-form DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with segment of DNA shaped like a Y. This shape occurs when DNA contains a region of paired double-stranded DNA on one end and a region of unpaired DNA strands on the opposite end.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16781730'}]},\n",
       "  'synonyms': [{'name': 'splayed Y-form DNA binding', 'type': 'exact'},\n",
       "   {'name': 'forked DNA binding', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000217', 'relation': 'is_a'}]},\n",
       " 'GO:0000404': {'id': 'GO:0000404',\n",
       "  'isObsolete': False,\n",
       "  'name': 'heteroduplex DNA loop binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with DNA containing a loop. A loop occurs when DNA contains a large insertion or deletion that causes a region of unpaired single-stranded DNA to loop out, while the rest of the DNA is in a paired double-stranded configuration.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16781730'}]},\n",
       "  'synonyms': [{'name': 'loop DNA binding', 'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000217', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032135', 'relation': 'is_a'}]},\n",
       " 'GO:0000405': {'id': 'GO:0000405',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bubble DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with DNA that contains a bubble. A bubble occurs when DNA contains a region of unpaired, single-stranded DNA flanked on both sides by regions of paired, double-stranded DNA.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16781730'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000217', 'relation': 'is_a'}]},\n",
       " 'GO:0000406': {'id': 'GO:0000406',\n",
       "  'isObsolete': False,\n",
       "  'name': 'double-strand/single-strand DNA junction binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a region of DNA that contains double-stranded DNA flanked by a region of single-stranded DNA.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16781730'}]},\n",
       "  'children': [{'id': 'GO:0090655', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000217', 'relation': 'is_a'}]},\n",
       " 'GO:0000407': {'id': 'GO:0000407',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phagophore assembly site',\n",
       "  'definition': {'text': 'Punctate structures proximal to the endoplasmic reticulum which are the sites where the Atg machinery assembles upon autophagy induction.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11689437'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12048214'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12554655'}]},\n",
       "  'synonyms': [{'name': 'pre-autophagosomal structure', 'type': 'narrow'},\n",
       "   {'name': 'PAS', 'type': 'exact'},\n",
       "   {'name': 'perivacuolar space', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0034045', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000045', 'relation': 'capable_of_part_of'}]},\n",
       " 'GO:0000408': {'id': 'GO:0000408',\n",
       "  'isObsolete': False,\n",
       "  'name': 'EKC/KEOPS complex',\n",
       "  'definition': {'text': 'A protein complex involved in t6A tRNA modification; originally proposed to be involved in transcription as well as promoting telomere uncapping and telomere elongation. For example, in Saccharomyces cerevisiae the complex contains Bud32p, Kae1p, Gon7p, Cgi121p, and Pcc1p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16564010'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16874308'},\n",
       "    {'dbCode': 'PMID', 'dbId': '21183954'},\n",
       "    {'dbCode': 'PMID', 'dbId': '23945934'}]},\n",
       "  'synonyms': [{'name': 'KEOPS/EKC complex', 'type': 'exact'},\n",
       "   {'name': 'endopeptidase-like kinase chromatin-associated protein complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'kinase, putative endopeptidase and other proteins of small size protein complex',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002949', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0070526', 'relation': 'capable_of_part_of'}]},\n",
       " 'GO:0000409': {'id': 'GO:0000409',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of transcription by galactose',\n",
       "  'definition': {'text': 'Any process involving galactose that modulates the frequency, rate or extent or transcription.'},\n",
       "  'children': [{'id': 'GO:0000411', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000410', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000431', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006355', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061019', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032583', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045449', 'relation': 'is_a'}]},\n",
       " 'GO:0000410': {'id': 'GO:0000410',\n",
       "  'isObsolete': False,\n",
       "  'name': 'carbon catabolite repression of transcription by galactose',\n",
       "  'definition': {'text': 'A transcription regulation process in which the presence of galactose that leads to a decrease in the frequency, rate, or extent of transcription of specific genes involved in the metabolism of other carbon sources. Carbon catabolite repression is a mechanism of genetic regulation which the accumulation of catabolites of one substance in the cell represses the formation of enzymes that contribute to the catabolism of other substances.'},\n",
       "  'synonyms': [{'name': 'down-regulation of transcription by galactose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'down regulation of transcription by galactose', 'type': 'exact'},\n",
       "   {'name': 'downregulation of transcription by galactose', 'type': 'exact'},\n",
       "   {'name': 'inhibition of transcription by galactose', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0000434', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000409', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045013', 'relation': 'is_a'}]},\n",
       " 'GO:0000411': {'id': 'GO:0000411',\n",
       "  'isObsolete': False,\n",
       "  'name': 'positive regulation of transcription by galactose',\n",
       "  'definition': {'text': 'Any process involving galactose that activates or increases the rate of transcription.'},\n",
       "  'synonyms': [{'name': 'activation of transcription by galactose',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'stimulation of transcription by galactose', 'type': 'narrow'},\n",
       "   {'name': 'upregulation of transcription by galactose', 'type': 'exact'},\n",
       "   {'name': 'up regulation of transcription by galactose', 'type': 'exact'},\n",
       "   {'name': 'up-regulation of transcription by galactose', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000435', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000409', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045991', 'relation': 'is_a'}]},\n",
       " 'GO:0000412': {'id': 'GO:0000412',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histone peptidyl-prolyl isomerization',\n",
       "  'definition': {'text': 'The modification of a histone by cis-trans isomerization of a proline residue.'},\n",
       "  'synonyms': [{'name': 'histone proline isomerization', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000413', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016570', 'relation': 'is_a'}]},\n",
       " 'GO:0000413': {'id': 'GO:0000413',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein peptidyl-prolyl isomerization',\n",
       "  'definition': {'text': 'The modification of a protein by cis-trans isomerization of a proline residue.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16959570'}]},\n",
       "  'synonyms': [{'name': 'protein proline isomerization', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000412', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003755', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0018208', 'relation': 'is_a'}]},\n",
       " 'GO:0000414': {'id': 'GO:0000414',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of histone H3-K36 methylation',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of the covalent addition of a methyl group to the lysine at position 36 of histone H3.'},\n",
       "  'children': [{'id': 'GO:0000415', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2001253', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000416', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0010452', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0031060', 'relation': 'is_a'}]},\n",
       " 'GO:0000415': {'id': 'GO:0000415',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative regulation of histone H3-K36 methylation',\n",
       "  'definition': {'text': 'Any process that stops, prevents, or reduces the frequency, rate or extent of the covalent addition of a methyl group to the lysine at position 36 of histone H3.'},\n",
       "  'synonyms': [{'name': 'down regulation of histone H3-K36 methylation',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'down-regulation of histone H3-K36 methylation', 'type': 'exact'},\n",
       "   {'name': 'inhibition of histone H3-K36 methylation', 'type': 'narrow'},\n",
       "   {'name': 'downregulation of histone H3-K36 methylation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:2001254', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000414', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010452', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0031061', 'relation': 'is_a'}]},\n",
       " 'GO:0000416': {'id': 'GO:0000416',\n",
       "  'isObsolete': False,\n",
       "  'name': 'positive regulation of histone H3-K36 methylation',\n",
       "  'definition': {'text': 'Any process that activates or increases the frequency, rate or extent of the covalent addition of a methyl group to the lysine at position 36 of histone H3.'},\n",
       "  'synonyms': [{'name': 'activation of histone H3-K36 methylation',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'stimulation of histone H3-K36 methylation', 'type': 'narrow'},\n",
       "   {'name': 'up regulation of histone H3-K36 methylation', 'type': 'exact'},\n",
       "   {'name': 'up-regulation of histone H3-K36 methylation', 'type': 'exact'},\n",
       "   {'name': 'upregulation of histone H3-K36 methylation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:2001255', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000414', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010452', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0031062', 'relation': 'is_a'}]},\n",
       " 'GO:0000417': {'id': 'GO:0000417',\n",
       "  'isObsolete': False,\n",
       "  'name': 'HIR complex',\n",
       "  'definition': {'text': 'A protein complex proposed to be involved in replication-independent nucleosome assembly, by promoting histone deposition onto DNA. For example, in Saccharomyces, the complex contains Hir1p, Hir2p, Hir3p, and Hpc2p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16303565'},\n",
       "    {'dbCode': 'PMID', 'dbId': '17180700'}]},\n",
       "  'synonyms': [{'name': 'HIRA complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000418': {'id': 'GO:0000418',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase IV complex',\n",
       "  'definition': {'text': 'RNA polymerase IV is a multisubunit RNA polymerase complex found in the nucleus of plants and involved in accumulation of siRNAs and in DNA methylation-dependent silencing of endogenous repeated sequences. Pol IV is composed of subunits that are paralogous or identical to the 12 subunits of Pol II. The largest and second-largest subunits of Pol IV are the catalytic subunits and share similarity with the corresponding subunits of other eukaryotic and bacterial multisubunit RNA polymerases. The second largest subunit is also found in RNA polymerase V, while the largest subunit is found only in RNAP IV complex.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15692015'},\n",
       "    {'dbCode': 'PMID', 'dbId': '15766525'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16140984'},\n",
       "    {'dbCode': 'PMID', 'dbId': '19110459'}]},\n",
       "  'synonyms': [{'name': 'DNA-directed RNA polymerase IV complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'DNA-directed RNA polymerase IVa complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001057', 'relation': 'capable_of'}]},\n",
       " 'GO:0000419': {'id': 'GO:0000419',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase V complex',\n",
       "  'definition': {'text': 'RNA polymerase V is a multisubunit RNA polymerase complex found in the nucleus of plants and involved in accumulation of siRNAs and in DNA methylation-dependent silencing of endogenous repeated sequences. Pol V is composed of subunits that are paralogous or identical to the 12 subunits of Pol II. Two large subunits comprise the most conserved portion including the catalytic site and share similarity with other eukaryotic and bacterial multisubunit RNA polymerases. The second largest subunit is also found in RNA polymerase IVa, while the largest subunit is found only in the IVa complex and contains an extended C-terminal domain (CTD) that includes multiple repeats of a 16 amino-acid consensus sequence as well as other sequences. The remainder of the complex is composed of smaller subunits.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16140984'},\n",
       "    {'dbCode': 'PMID', 'dbId': '19110459'}]},\n",
       "  'synonyms': [{'name': 'DNA-directed RNA polymerase V complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'DNA-directed RNA polymerase IVb complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001058', 'relation': 'capable_of'}]},\n",
       " 'GO:0000421': {'id': 'GO:0000421',\n",
       "  'isObsolete': False,\n",
       "  'name': 'autophagosome membrane',\n",
       "  'definition': {'text': 'The lipid bilayer surrounding an autophagosome, a double-membrane-bounded vesicle in which endogenous cellular material is sequestered.'},\n",
       "  'synonyms': [{'name': 'autophagic vacuole membrane', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1904930', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097636', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0097635', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0097637', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000422': {'id': 'GO:0000422',\n",
       "  'isObsolete': False,\n",
       "  'name': 'autophagy of mitochondrion',\n",
       "  'definition': {'text': 'The autophagic process in which mitochondria are delivered to a type of vacuole and degraded in response to changing cellular conditions.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15798367'},\n",
       "    {'dbCode': 'PMID', 'dbId': '19289147'},\n",
       "    {'dbCode': 'PMID', 'dbId': '23065344'}]},\n",
       "  'synonyms': [{'name': 'mitochondrion autophagy', 'type': 'exact'},\n",
       "   {'name': 'mitophagy', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:1904925', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0000423', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035695', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904923', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1903146', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1903147', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1903599', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0000424', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006914', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061726', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016238', 'relation': 'is_a'}]},\n",
       " 'GO:0000423': {'id': 'GO:0000423',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitophagy',\n",
       "  'definition': {'text': 'The selective autophagy process in which a mitochondrion is degraded by macroautophagy.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15798367'}]},\n",
       "  'comment': 'Note that this terms refers to the macroautophagy process and is named by common usage. Be aware that there are a separate micromitophagy and mitophagy by induced vacuole formation processes.',\n",
       "  'synonyms': [{'name': 'macromitophagy', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1901525', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1901524', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0098779', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1904924', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1901526', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000422', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061912', 'relation': 'is_a'}]},\n",
       " 'GO:0000424': {'id': 'GO:0000424',\n",
       "  'isObsolete': False,\n",
       "  'name': 'micromitophagy',\n",
       "  'definition': {'text': 'Degradation of a mitochondrion by lysosomal microautophagy.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15798367'},\n",
       "    {'dbCode': 'PMID', 'dbId': '27003723'}]},\n",
       "  'comment': 'Note that this term is not a child of mitophagy because the community genrally uses that latter to refer to the macroautophagy of mitochondria.',\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000422', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016237', 'relation': 'is_a'}]},\n",
       " 'GO:0000425': {'id': 'GO:0000425',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pexophagy',\n",
       "  'definition': {'text': 'The selective autophagy process in which a peroxisome is degraded by macroautophagy.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12914914'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16973210'}]},\n",
       "  'synonyms': [{'name': 'macropexophagy', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030242', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061912', 'relation': 'is_a'}]},\n",
       " 'GO:0000426': {'id': 'GO:0000426',\n",
       "  'isObsolete': False,\n",
       "  'name': 'micropexophagy',\n",
       "  'definition': {'text': 'Degradation of a peroxisome by lysosomal microautophagy.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12914914'},\n",
       "    {'dbCode': 'PMID', 'dbId': '15350980'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16973210'}]},\n",
       "  'comment': 'Note that this term is not a child of pexophagy because the community usually uses the latter to refer to the macroautohagy process.',\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016237', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030242', 'relation': 'is_a'}]},\n",
       " 'GO:0000427': {'id': 'GO:0000427',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid-encoded plastid RNA polymerase complex',\n",
       "  'definition': {'text': 'An RNA polymerase complex containing polypeptides encoded by the plastid genome. Plastid-encoded DNA-directed RNA polymerases resemble eubacterial multisubunit RNA polymerases, with a core composed of alpha, beta, and beta-prime subunits. Some forms contain multiple additional subunits. An additional sigma factor subunit is required for promoter recognition.'},\n",
       "  'children': [{'id': 'GO:0000343', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000344', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000428': {'id': 'GO:0000428',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA-directed RNA polymerase complex',\n",
       "  'definition': {'text': 'A protein complex that possesses DNA-directed RNA polymerase activity.'},\n",
       "  'children': [{'id': 'GO:0000427', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055029', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000345', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034245', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003899', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0000129', 'relation': 'capable_of'}]},\n",
       " 'GO:0000429': {'id': 'GO:0000429',\n",
       "  'isObsolete': False,\n",
       "  'name': 'carbon catabolite regulation of transcription from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'A transcription regulation process in which the presence of one carbon source leads to the modulation of the frequency, rate, or extent of transcription, from an RNA polymerase II promoter, of specific genes involved in the metabolism of other carbon sources.'},\n",
       "  'synonyms': [{'name': 'regulation of transcription from RNA polymerase II promoter by carbon catabolites',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000436', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000437', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061413', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000430', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000431', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006357', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045990', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006358', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010551', 'relation': 'is_a'}]},\n",
       " 'GO:0000430': {'id': 'GO:0000430',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of transcription from RNA polymerase II promoter by glucose',\n",
       "  'definition': {'text': 'Any process involving glucose that modulates the frequency, rate or extent of transcription from an RNA polymerase II promoter.'},\n",
       "  'children': [{'id': 'GO:0061987', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000432', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000429', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046015', 'relation': 'is_a'}]},\n",
       " 'GO:0000431': {'id': 'GO:0000431',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of transcription from RNA polymerase II promoter by galactose',\n",
       "  'definition': {'text': 'Any process involving galactose that modulates the frequency, rate or extent of transcription from an RNA polymerase II promoter.'},\n",
       "  'children': [{'id': 'GO:0000435', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000434', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000409', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000429', 'relation': 'is_a'}]},\n",
       " 'GO:0000432': {'id': 'GO:0000432',\n",
       "  'isObsolete': False,\n",
       "  'name': 'positive regulation of transcription from RNA polymerase II promoter by glucose',\n",
       "  'definition': {'text': 'Any process involving glucose that activates or increases the rate of transcription from an RNA polymerase II promoter.'},\n",
       "  'synonyms': [{'name': 'stimulation of transcription from RNA polymerase II promoter by glucose',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'upregulation of transcription from RNA polymerase II promoter by glucose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'up-regulation of transcription from RNA polymerase II promoter by glucose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'up regulation of transcription from RNA polymerase II promoter by glucose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of transcription from RNA polymerase II promoter by glucose',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000430', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000436', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046016', 'relation': 'is_a'}]},\n",
       " 'GO:0000433': {'id': 'GO:0000433',\n",
       "  'isObsolete': False,\n",
       "  'name': 'carbon catabolite repression of transcription from RNA polymerase II promoter by glucose',\n",
       "  'definition': {'text': 'A transcription regulation process in which the presence of glucose leads to a decrease in the frequency, rate, or extent of transcription of specific RNA polymerase II-transcribed genes involved in the metabolism of other carbon sources. Carbon catabolite repression is a mechanism of genetic regulation which the accumulation of catabolites of one substance in the cell represses the formation of enzymes that contribute to the catabolism of other substances.'},\n",
       "  'synonyms': [{'name': 'down regulation of transcription from RNA polymerase II promoter by glucose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'downregulation of transcription from RNA polymerase II promoter by glucose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'down-regulation of transcription from RNA polymerase II promoter by glucose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'inhibition of transcription from RNA polymerase II promoter by glucose',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0045014', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061987', 'relation': 'is_a'}]},\n",
       " 'GO:0000434': {'id': 'GO:0000434',\n",
       "  'isObsolete': False,\n",
       "  'name': 'carbon catabolite repression of transcription from RNA polymerase II promoter by galactose',\n",
       "  'definition': {'text': 'Any process involving galactose that stops, prevents or reduces the rate of transcription from an RNA polymerase II promoter.'},\n",
       "  'synonyms': [{'name': 'down regulation of transcription from RNA polymerase II promoter by galactose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'down-regulation of transcription from RNA polymerase II promoter by galactose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'downregulation of transcription from RNA polymerase II promoter by galactose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'inhibition of transcription from RNA polymerase II promoter by galactose',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000410', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000431', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000437', 'relation': 'is_a'}]},\n",
       " 'GO:0000435': {'id': 'GO:0000435',\n",
       "  'isObsolete': False,\n",
       "  'name': 'positive regulation of transcription from RNA polymerase II promoter by galactose',\n",
       "  'definition': {'text': 'Any process involving galactose that activates or increases the rate of transcription from an RNA polymerase II promoter.'},\n",
       "  'synonyms': [{'name': 'up-regulation of transcription from RNA polymerase II promoter by galactose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'upregulation of transcription from RNA polymerase II promoter by galactose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'up regulation of transcription from RNA polymerase II promoter by galactose',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'stimulation of transcription from RNA polymerase II promoter by galactose',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'activation of transcription from RNA polymerase II promoter by galactose',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000411', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000431', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000436', 'relation': 'is_a'}]},\n",
       " 'GO:0000436': {'id': 'GO:0000436',\n",
       "  'isObsolete': False,\n",
       "  'name': 'carbon catabolite activation of transcription from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Any process involving carbon catabolites that activates or increases the frequency, rate or extent of transcription from an RNA polymerase II promoter.'},\n",
       "  'synonyms': [{'name': 'positive regulation of transcription from RNA polymerase II promoter by carbon catabolites',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000435', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000432', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061414', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061429', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000429', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045944', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045991', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010552', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045817', 'relation': 'is_a'}]},\n",
       " 'GO:0000437': {'id': 'GO:0000437',\n",
       "  'isObsolete': False,\n",
       "  'name': 'carbon catabolite repression of transcription from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'A transcription regulation process in which the presence of one carbon source leads to a decrease in the frequency, rate, or extent of transcription, from an RNA polymerase II promoter, of specific genes involved in the metabolism of other carbon sources.'},\n",
       "  'synonyms': [{'name': 'negative regulation of transcription from RNA polymerase II promoter by carbon catabolites',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000434', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061415', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000122', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000429', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045013', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010553', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045816', 'relation': 'is_a'}]},\n",
       " 'GO:0000438': {'id': 'GO:0000438',\n",
       "  'isObsolete': False,\n",
       "  'name': 'core TFIIH complex portion of holo TFIIH complex',\n",
       "  'definition': {'text': 'The core TFIIH complex when it is part of the general transcription factor TFIIH.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14500720'},\n",
       "    {'dbCode': 'PMID', 'dbId': '22308316'},\n",
       "    {'dbCode': 'PMID', 'dbId': '22572993'},\n",
       "    {'dbCode': 'PMID', 'dbId': '7813015'}]},\n",
       "  'synonyms': [{'name': 'SSL2-core TFIIH complex portion of holo TFIIH complex',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000439': {'id': 'GO:0000439',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription factor TFIIH core complex',\n",
       "  'definition': {'text': 'The 7 subunit core of TFIIH that is a part of either the general transcription factor holo-TFIIH or the nucleotide-excision repair factor 3 complex. In S. cerevisiae/humans the complex is composed of: Ssl2/XPB, Tfb1/p62, Tfb2/p52, Ssl1/p44, Tfb4/p34, Tfb5/p8 and Rad3/XPD.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14500720'},\n",
       "    {'dbCode': 'PMID', 'dbId': '17215295'},\n",
       "    {'dbCode': 'PMID', 'dbId': '22308316'},\n",
       "    {'dbCode': 'PMID', 'dbId': '22572993'},\n",
       "    {'dbCode': 'PMID', 'dbId': '23028141'},\n",
       "    {'dbCode': 'PMID', 'dbId': '7813015'}]},\n",
       "  'synonyms': [{'name': 'SSL2-core TFIIH complex', 'type': 'exact'},\n",
       "   {'name': 'core TFIIH complex', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000438', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000440', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000440': {'id': 'GO:0000440',\n",
       "  'isObsolete': False,\n",
       "  'name': 'core TFIIH complex portion of NEF3 complex',\n",
       "  'definition': {'text': 'The core TFIIH complex when it is part of the nucleotide-excision repair factor 3 (NEF3).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14500720'},\n",
       "    {'dbCode': 'PMID', 'dbId': '22308316'},\n",
       "    {'dbCode': 'PMID', 'dbId': '22572993'},\n",
       "    {'dbCode': 'PMID', 'dbId': '7813015'}]},\n",
       "  'synonyms': [{'name': 'SSL2-core TFIIH complex portion of NEF3 complex',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000444': {'id': 'GO:0000444',\n",
       "  'isObsolete': False,\n",
       "  'name': 'MIS12/MIND type complex',\n",
       "  'definition': {'text': 'A multiprotein kinetochore subcomplex that binds to centromeric chromatin and forms part of the inner kinetochore. It helps to recruit outer kinetochore subunits that will bind to microtubules. In humans, it consists of MIS12, DSN1, NSL1 and PMF1.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14633972'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16585270'}]},\n",
       "  'synonyms': [{'name': 'Mis12 complex', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0000818', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000445': {'id': 'GO:0000445',\n",
       "  'isObsolete': False,\n",
       "  'name': 'THO complex part of transcription export complex',\n",
       "  'definition': {'text': 'The THO complex when it is part of the TREX (TRanscription EXport) complex that is involved in coupling transcription to export of mRNAs to the cytoplasm. In S. cerevisiae, it is composed of four subunits: Hpr1, Tho2, Thp1, and Mft1, while the human complex is composed of 7 subunits.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11060033'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11979277'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16983072'}]},\n",
       "  'synonyms': [{'name': 'THO complex part of TREX complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000446': {'id': 'GO:0000446',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleoplasmic THO complex',\n",
       "  'definition': {'text': 'The THO complex when it is acting as a nuclear complex that is required for transcription elongation through genes containing tandemly repeated DNA sequences. In S. cerevisiae, it is composed of four subunits: Hpr1, Tho2, Thp2, and Mft1, while the human complex is composed of 7 subunits.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11060033'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11979277'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16983072'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000447': {'id': 'GO:0000447',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endonucleolytic cleavage in ITS1 to separate SSU-rRNA from 5.8S rRNA and LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage between the SSU-rRNA and the 5.8S rRNA of an rRNA molecule originally produced as a tricistronic rRNA transcript that contained the Small SubUnit (SSU) rRNA, the 5.8S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'synonyms': [{'name': 'endonucleolytic cleavage at A2', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000462', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000466', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000479', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990041', 'relation': 'part_of'}]},\n",
       " 'GO:0000448': {'id': 'GO:0000448',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cleavage in ITS2 between 5.8S rRNA and LSU-rRNA of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage within ITS2 between the 5.8S rRNA and the LSU-rRNA of an rRNA molecule originally produced as a tricistronic rRNA transcript that contained the Small SubUnit (SSU) rRNA, the 5.8S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'synonyms': [{'name': 'cleavage at C2', 'type': 'related'},\n",
       "   {'name': 'cleavage in ITS2 of tricistronic rRNA transcript to separate 5.8S and LSU rRNAs (SSU-rRNA, 5.8S rRNA, LSU-rRNA)',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000463', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000466', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000479', 'relation': 'is_a'}]},\n",
       " 'GO:0000449': {'id': 'GO:0000449',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, LSU-rRNA, 5S)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage of a pre-rRNA molecule originally produced as a tricistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, the Large Subunit (LSU) rRNA, and the 5S rRNA, in that order, from 5' to 3' along the primary transcript. For example, primary ribosomal RNA transcripts containing three genes, in this order, are produced in E. coli and other prokaryotic species. Note that the use of the word tricistronic refers only to the number of mature rRNA molecules which will be produced from the primary transcript and ignores tRNAs that may also be present within the primary transcript.\"},\n",
       "  'children': [{'id': 'GO:0000458', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000457', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000478', 'relation': 'is_a'}]},\n",
       " 'GO:0000450': {'id': 'GO:0000450',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cleavage of bicistronic rRNA transcript (SSU-rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage of pre-rRNAs originally produced as a bicistronic rRNA transcript that contains the SSU-rRNA and the LSU-rRNA in that order from 5' to 3' along the primary transcript. Primary ribosomal RNA transcripts with two genes in this order are produced in Archaeal species.\"},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000478', 'relation': 'is_a'}]},\n",
       " 'GO:0000451': {'id': 'GO:0000451',\n",
       "  'isObsolete': False,\n",
       "  'name': \"rRNA 2'-O-methylation\",\n",
       "  'definition': {'text': \"The addition of a methyl group to the 2'-oxygen atom of a nucleotide residue in an rRNA molecule during ribosome biogenesis.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '1555811337'}]},\n",
       "  'children': [{'id': 'GO:0000452', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070039', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000453', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031167', 'relation': 'is_a'}]},\n",
       " 'GO:0000452': {'id': 'GO:0000452',\n",
       "  'isObsolete': False,\n",
       "  'name': \"snoRNA guided rRNA 2'-O-methylation\",\n",
       "  'definition': {'text': \"The posttranscriptional addition of methyl groups to the 2'-oxygen atom of nucleotide residues in an rRNA molecule during ribosome biogenesis using a snoRNA guide that targets the position of methylation.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '1555811337'}]},\n",
       "  'synonyms': [{'name': \"snoRNA guided rRNA 2'-O-ribose methylation\",\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000451', 'relation': 'is_a'}]},\n",
       " 'GO:0000453': {'id': 'GO:0000453',\n",
       "  'isObsolete': False,\n",
       "  'name': \"enzyme-directed rRNA 2'-O-methylation\",\n",
       "  'definition': {'text': \"The addition of methyl groups to the 2'-oxygen atom of nucleotide residues in an rRNA molecule during ribosome biogenesis where the methylase specifies the site that becomes methylated without using a guide RNA.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '1555811337'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000451', 'relation': 'is_a'}]},\n",
       " 'GO:0000454': {'id': 'GO:0000454',\n",
       "  'isObsolete': False,\n",
       "  'name': 'snoRNA guided rRNA pseudouridine synthesis',\n",
       "  'definition': {'text': 'The intramolecular conversion of uridine to pseudouridine in an rRNA molecule during ribosome biogenesis using a snoRNA guide that targets the position of pseudouridylation.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '1555811337'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031118', 'relation': 'is_a'}]},\n",
       " 'GO:0000455': {'id': 'GO:0000455',\n",
       "  'isObsolete': False,\n",
       "  'name': 'enzyme-directed rRNA pseudouridine synthesis',\n",
       "  'definition': {'text': 'The intramolecular conversion of uridine to pseudouridine during ribosome biogenesis where the enzyme specifies the site that becomes pseudouridylated without using a guide RNA.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '1555811337'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031118', 'relation': 'is_a'}]},\n",
       " 'GO:0000457': {'id': 'GO:0000457',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endonucleolytic cleavage between SSU-rRNA and LSU-rRNA of tricistronic rRNA transcript (SSU-rRNA, LSU-rRNA, 5S)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage to separate a pre-SSU-rRNA from a pre-LSU-rRNA originally produced as a tricistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, the Large Subunit (LSU) rRNA, and the 5S rRNA, in that order, from 5' to 3' along the primary transcript. Note that the use of the word tricistronic refers only to the number of mature rRNA molecules which will be produced from the primary transcript and ignores tRNAs that may also be present within the primary transcript.\"},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000449', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002108', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0002109', 'relation': 'part_of'}]},\n",
       " 'GO:0000458': {'id': 'GO:0000458',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endonucleolytic cleavage between LSU-rRNA and 5S rRNA of tricistronic rRNA transcript (SSU-rRNA, LSU-rRNA, 5S)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage to separate a pre-LSU-rRNA from a pre-5S rRNA originally produced as a tricistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, the Large Subunit (LSU) rRNA, and the 5S rRNA, in that order, from 5' to 3' along the primary transcript. Note that the use of the word tricistronic refers only to the number of mature rRNA molecules which will be produced from the primary transcript and ignores tRNAs that may also be present within the primary transcript.\"},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000449', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002108', 'relation': 'part_of'}]},\n",
       " 'GO:0000459': {'id': 'GO:0000459',\n",
       "  'isObsolete': False,\n",
       "  'name': 'exonucleolytic trimming involved in rRNA processing',\n",
       "  'definition': {'text': 'Exonucleolytic digestion of a pre-rRNA molecule in the process to generate a mature rRNA molecule.'},\n",
       "  'synonyms': [{'name': 'exonucleolytic trimming during rRNA processing',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000465', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000467', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000469', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006364', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090503', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006365', 'relation': 'part_of'}]},\n",
       " 'GO:0000460': {'id': 'GO:0000460',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of 5.8S rRNA',\n",
       "  'definition': {'text': 'Any process involved in the maturation of a precursor 5.8S ribosomal RNA (rRNA) molecule into a mature 5.8S rRNA molecule.'},\n",
       "  'children': [{'id': 'GO:0000487', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000466', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006364', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006365', 'relation': 'is_a'}]},\n",
       " 'GO:0000461': {'id': 'GO:0000461',\n",
       "  'isObsolete': False,\n",
       "  'name': \"endonucleolytic cleavage to generate mature 3'-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA)\",\n",
       "  'definition': {'text': \"Endonucleolytic cleavage at the 3'-end of the SSU-rRNA from an originally tricistronic rRNA transcript that contained the Small Subunit (SSU) rRNA, the 5.8S rRNA, and the Large Subunit (LSU) rRNA in that order from 5' to 3' along the primary transcript, to produce the mature end of the SSU-rRNA.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'synonyms': [{'name': 'endonucleolytic cleavage at site D',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000462', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000479', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031125', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990041', 'relation': 'part_of'}]},\n",
       " 'GO:0000462': {'id': 'GO:0000462',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Any process involved in the maturation of a precursor Small SubUnit (SSU) ribosomal RNA (rRNA) molecule into a mature SSU-rRNA molecule from the pre-rRNA molecule originally produced as a tricistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, 5.8S rRNA, and the Large Subunit (LSU) in that order from 5' to 3' along the primary transcript.\"},\n",
       "  'synonyms': [{'name': 'maturation of 18S rRNA', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0000461', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000472', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000480', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000447', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030490', 'relation': 'is_a'}]},\n",
       " 'GO:0000463': {'id': 'GO:0000463',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Any process involved in the maturation of a precursor Large SubUnit (LSU) ribosomal RNA (rRNA) molecule into a mature LSU-rRNA molecule from the pre-rRNA molecule originally produced as a tricistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, 5.8S rRNA, and Large Subunit (LSU) in that order from 5' to 3' along the primary transcript.\"},\n",
       "  'children': [{'id': 'GO:0000448', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000477', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000468', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000464', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000470', 'relation': 'is_a'}]},\n",
       " 'GO:0000464': {'id': 'GO:0000464',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endonucleolytic cleavage in ITS1 upstream of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage within Internal Transcribed Spacer 1 (ITS1) upstream of the 5.8S rRNA derived from an originally tricistronic rRNA transcript that contained the Small Subunit (SSU) rRNA, the 5.8S rRNA, and the Large Subunit (LSU) rRNA in that order from 5' to 3' along the primary transcript. In S. cerevisiae, this endonucleolytic cleavage within ITS1 initiates the maturation of the LSU and the 5.8S rRNAs.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'synonyms': [{'name': 'endonucleolytic cleavage at A3', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000463', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000466', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000479', 'relation': 'is_a'}]},\n",
       " 'GO:0000465': {'id': 'GO:0000465',\n",
       "  'isObsolete': False,\n",
       "  'name': \"exonucleolytic trimming to generate mature 5'-end of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)\",\n",
       "  'definition': {'text': \"Exonucleolytic digestion of a pre-rRNA molecule to generate the mature 5'-end of a 5.8S rRNA molecule derived from an originally tricistronic pre-rRNA transcript that contained the Small Subunit (SSU) rRNA, the 5.8S rRNA, and the Large Subunit (LSU) rRNA in that order from 5' to 3' along the primary transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000459', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000466', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000967', 'relation': 'is_a'}]},\n",
       " 'GO:0000466': {'id': 'GO:0000466',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Any process involved in the maturation of an rRNA molecule originally produced as part of a tricistronic rRNA transcript that contained the Small SubUnit (SSU) rRNA, the 5.8S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'children': [{'id': 'GO:0000464', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000465', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000448', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000467', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000447', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000460', 'relation': 'is_a'}]},\n",
       " 'GO:0000467': {'id': 'GO:0000467',\n",
       "  'isObsolete': False,\n",
       "  'name': \"exonucleolytic trimming to generate mature 3'-end of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)\",\n",
       "  'definition': {'text': \"Exonucleolytic digestion of a pre-rRNA molecule to generate the mature 3'-end of a 5.8S rRNA molecule derived from an originally tricistronic pre-rRNA transcript that contained the Small Subunit (SSU) rRNA, the 5.8S rRNA, and the Large Subunit (LSU) rRNA in that order from 5' to 3' along the primary transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000459', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000466', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031125', 'relation': 'is_a'}]},\n",
       " 'GO:0000468': {'id': 'GO:0000468',\n",
       "  'isObsolete': False,\n",
       "  'name': \"generation of mature 3'-end of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)\",\n",
       "  'definition': {'text': \"Any process involved in generating the mature 3'-end of an LSU-rRNA derived from a tricistronic rRNA transcript that contained the Small SubUnit (SSU) rRNA, the 5.8S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'synonyms': [{'name': 'processing at B2', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000463', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031125', 'relation': 'is_a'}]},\n",
       " 'GO:0000469': {'id': 'GO:0000469',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cleavage involved in rRNA processing',\n",
       "  'definition': {'text': 'Any phosphodiester bond hydrolysis involved in the conversion of a primary ribosomal RNA (rRNA) transcript into a mature rRNA molecule.'},\n",
       "  'synonyms': [{'name': 'cleavage during rRNA processing', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000459', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000477', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000478', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006364', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090501', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006365', 'relation': 'part_of'}]},\n",
       " 'GO:0000470': {'id': 'GO:0000470',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of LSU-rRNA',\n",
       "  'definition': {'text': 'Any process involved in the maturation of a precursor Large SubUnit (LSU) ribosomal RNA (rRNA) molecule into a mature LSU-rRNA molecule.'},\n",
       "  'children': [{'id': 'GO:0000473', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000463', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000488', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002108', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006364', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042273', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006365', 'relation': 'is_a'}]},\n",
       " 'GO:0000471': {'id': 'GO:0000471',\n",
       "  'isObsolete': False,\n",
       "  'name': \"endonucleolytic cleavage in 3'-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)\",\n",
       "  'definition': {'text': \"Endonucleolytic cleavage within the 3'-External Transcribed Spacer (ETS) of a tricistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, the 5.8S rRNA, and the Large Subunit (LSU) rRNA in that order from 5' to 3' along the primary transcript. In S. cerevisiae, endonucleolytic cleavage within the 3'-ETS of the pre-RNA, which may occur cotranscriptionally, is the first step in rRNA processing, and initiates a cascade of subsequent processing and modification events.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000479', 'relation': 'is_a'}]},\n",
       " 'GO:0000472': {'id': 'GO:0000472',\n",
       "  'isObsolete': False,\n",
       "  'name': \"endonucleolytic cleavage to generate mature 5'-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA)\",\n",
       "  'definition': {'text': \"Endonucleolytic cleavage between the 5'-External Transcribed Spacer (5'-ETS) and the 5' end of the SSU-rRNA of a tricistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, the 5.8S rRNA, and the Large Subunit (LSU) rRNA in that order from 5' to 3' along the primary transcript, to produce the mature end of the SSU-rRNA.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'synonyms': [{'name': 'endonucleolytic cleavage at A1', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000462', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000479', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000967', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990041', 'relation': 'part_of'}]},\n",
       " 'GO:0000473': {'id': 'GO:0000473',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of LSU-rRNA from tetracistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, 2S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Any process involved in the maturation of a precursor Large SubUnit (LSU) ribosomal RNA (rRNA) molecule into a mature LSU-rRNA molecule from the pre-rRNA molecule originally produced as a tetracistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, 5.8 S rRNA, 2S rRNA, and Large Subunit (LSU) in that order from 5' to 3' along the primary transcript.\"},\n",
       "  'children': [{'id': 'GO:0000485', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000470', 'relation': 'is_a'}]},\n",
       " 'GO:0000474': {'id': 'GO:0000474',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of SSU-rRNA from tetracistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, 2S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Any process involved in the maturation of a precursor Small SubUnit (SSU) ribosomal RNA (rRNA) molecule into a mature SSU-rRNA molecule from the pre-rRNA molecule originally produced as a tetracistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, 5.8 S rRNA, 2S rRNA, and Large Subunit (LSU) in that order from 5' to 3' along the primary transcript.\"},\n",
       "  'children': [{'id': 'GO:0000484', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030490', 'relation': 'is_a'}]},\n",
       " 'GO:0000475': {'id': 'GO:0000475',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of 2S rRNA',\n",
       "  'definition': {'text': 'Any process involved in the maturation of a precursor 2S ribosomal RNA (rRNA) molecule into a mature 2S rRNA molecule.'},\n",
       "  'children': [{'id': 'GO:0000486', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000485', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006364', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006365', 'relation': 'is_a'}]},\n",
       " 'GO:0000476': {'id': 'GO:0000476',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of 4.5S rRNA',\n",
       "  'definition': {'text': 'Any process involved in the maturation of a precursor 4.5S ribosomal RNA (rRNA) molecule into a mature 4.5S rRNA molecule.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006364', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006365', 'relation': 'is_a'}]},\n",
       " 'GO:0000477': {'id': 'GO:0000477',\n",
       "  'isObsolete': False,\n",
       "  'name': \"generation of mature 5'-end of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)\",\n",
       "  'definition': {'text': \"Cleavage within ITS2 to generate the mature 5'-end of an LSU-rRNA derived from a tricistronic rRNA transcript that contained the Small SubUnit (SSU) rRNA, the 5.8S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'synonyms': [{'name': 'processing at C1', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000463', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000469', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000967', 'relation': 'is_a'}]},\n",
       " 'GO:0000478': {'id': 'GO:0000478',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endonucleolytic cleavage involved in rRNA processing',\n",
       "  'definition': {'text': 'Any endonucleolytic cleavage involved in the conversion of a primary ribosomal RNA (rRNA) transcript into a mature rRNA molecule. Some endonucleolytic cleavages produce the mature end, while others are a step in the process of generating the mature end from the pre-rRNA.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'synonyms': [{'name': 'endonucleolytic cleavage during rRNA processing',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0090730', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0002103', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000450', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000483', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000449', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000479', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000469', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006364', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090502', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006365', 'relation': 'part_of'}]},\n",
       " 'GO:0000479': {'id': 'GO:0000479',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage of a pre-rRNA molecule originally produced as a tricistronic rRNA transcript that contains the Small SubUnit (SSU) rRNA, the 5.8S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript. Primary ribosomal RNA transcripts with three genes, in this order, are produced in the nuclei of many eukaryotic species, including S. cerevisiae.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'}]},\n",
       "  'children': [{'id': 'GO:0000447', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000480', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000471', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000461', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000472', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000464', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000448', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000478', 'relation': 'is_a'}]},\n",
       " 'GO:0000480': {'id': 'GO:0000480',\n",
       "  'isObsolete': False,\n",
       "  'name': \"endonucleolytic cleavage in 5'-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)\",\n",
       "  'definition': {'text': \"Endonucleolytic cleavage within the 5'-External Transcribed Spacer (ETS) of a tricistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, the 5.8S rRNA, and the Large Subunit (LSU) rRNA in that order from 5' to 3' along the primary transcript. Endonucleolytic cleavage within the 5'-ETS of the pre-RNA is conserved as one of the early steps of rRNA processing in all eukaryotes, but the specific position of cleavage is variable.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690410'},\n",
       "    {'dbCode': 'PMID', 'dbId': '15282326'}]},\n",
       "  'synonyms': [{'name': 'endonucleolytic cleavage at A0', 'type': 'narrow'},\n",
       "   {'name': 'endonucleolytic cleavage at A-prime', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000462', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000479', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990041', 'relation': 'part_of'}]},\n",
       " 'GO:0000481': {'id': 'GO:0000481',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of 5S rRNA',\n",
       "  'definition': {'text': 'Any process involved in the maturation of a precursor 5S ribosomal RNA (rRNA) molecule into a mature 5S rRNA molecule.'},\n",
       "  'children': [{'id': 'GO:0002107', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000482', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006364', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006365', 'relation': 'is_a'}]},\n",
       " 'GO:0000482': {'id': 'GO:0000482',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of 5S rRNA from tetracistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Any process involved in the maturation of a precursor 5S ribosomal RNA (rRNA) molecule into a mature 5S rRNA molecule from the pre-rRNA molecule originally produced as a tetracistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, Large Subunit (LSU) the 4.5S rRNA, and the 5S rRNA in that order from 5' to 3' along the primary transcript.\"},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000481', 'relation': 'is_a'}]},\n",
       " 'GO:0000483': {'id': 'GO:0000483',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endonucleolytic cleavage of tetracistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, 2S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage of a pre-rRNA molecule originally produced as a tetracistronic rRNA transcript that contains the Small SubUnit (SSU) rRNA, the 5.8S rRNA, 2S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript. Primary ribosomal RNA transcripts with four genes, in this order, are produced in the nuclei of D. melanogaster as well as in those of other dipteran species.\"},\n",
       "  'children': [{'id': 'GO:0000486', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000485', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000484', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000478', 'relation': 'is_a'}]},\n",
       " 'GO:0000484': {'id': 'GO:0000484',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cleavage between SSU-rRNA and 5.8S rRNA of tetracistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, 2S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage between the SSU-rRNA and the 5.8S rRNA of an rRNA molecule originally produced as a tetracistronic rRNA transcript that contained the Small SubUnit (SSU) rRNA, the 5.8S rRNA, 2S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript.\"},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000474', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000483', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000487', 'relation': 'part_of'}]},\n",
       " 'GO:0000485': {'id': 'GO:0000485',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cleavage between 2S rRNA and LSU-rRNA of tetracistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, 2S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage between the LSU-rRNA and the 2S rRNA of an rRNA molecule originally produced as a tetracistronic rRNA transcript that contained the Small SubUnit (SSU) rRNA, the 5.8S rRNA, 2S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '768488'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000473', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000475', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000483', 'relation': 'is_a'}]},\n",
       " 'GO:0000486': {'id': 'GO:0000486',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cleavage between 5.8S rRNA and 2S rRNA of tetracistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, 2S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Endonucleolytic cleavage between the 5.8S rRNA and the 2S rRNA of an rRNA molecule originally produced as a tetracistronic rRNA transcript that contained the Small SubUnit (SSU) rRNA, the 5.8S rRNA, 2S rRNA, and the Large SubUnit (LSU) rRNA, in that order, from 5' to 3' along the primary transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '768488'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000475', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000483', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000487', 'relation': 'part_of'}]},\n",
       " 'GO:0000487': {'id': 'GO:0000487',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of 5.8S rRNA from tetracistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, 2S rRNA, LSU-rRNA)',\n",
       "  'definition': {'text': \"Any process involved in the maturation of a precursor 5.8S ribosomal RNA (rRNA) molecule into a mature 5.8S rRNA molecule from the pre-rRNA molecule originally produced as a tetracistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, the 8.8S rRNA, the 2S rRNA, and the Large Subunit (LSU) in that order from 5' to 3' along the primary transcript.\"},\n",
       "  'children': [{'id': 'GO:0000484', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000486', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000460', 'relation': 'is_a'}]},\n",
       " 'GO:0000488': {'id': 'GO:0000488',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of LSU-rRNA from tetracistronic rRNA transcript (SSU-rRNA, LSU-rRNA, 4.5S-rRNA, 5S-rRNA)',\n",
       "  'definition': {'text': \"Any process involved in the maturation of a precursor Large SubUnit (LSU) ribosomal RNA (rRNA) molecule into a mature LSU-rRNA molecule from the pre-rRNA molecule originally produced as a tetracistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, Large Subunit (LSU) the 4.5S rRNA, and the 5S rRNA in that order from 5' to 3' along the primary transcript.\"},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000470', 'relation': 'is_a'}]},\n",
       " 'GO:0000489': {'id': 'GO:0000489',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maturation of SSU-rRNA from tetracistronic rRNA transcript (SSU-rRNA, LSU-rRNA, 4.5S-rRNA, 5S-rRNA)',\n",
       "  'definition': {'text': \"Any process involved in the maturation of a precursor Small SubUnit (SSU) ribosomal RNA (rRNA) molecule into a mature SSU-rRNA molecule from the pre-rRNA molecule originally produced as a tetracistronic rRNA transcript that contains the Small Subunit (SSU) rRNA, Large Subunit (LSU) the 4.5S rRNA, and the 5S rRNA in that order from 5' to 3' along the primary transcript.\"},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030490', 'relation': 'is_a'}]},\n",
       " 'GO:0000491': {'id': 'GO:0000491',\n",
       "  'isObsolete': False,\n",
       "  'name': 'small nucleolar ribonucleoprotein complex assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of proteins and a snoRNA to form a small nucleolar ribonucleoprotein (snoRNP) complex.'},\n",
       "  'synonyms': [{'name': 'snoRNP assembly', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000493', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000492', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0022618', 'relation': 'is_a'}]},\n",
       " 'GO:0000492': {'id': 'GO:0000492',\n",
       "  'isObsolete': False,\n",
       "  'name': 'box C/D snoRNP assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of proteins and a box C/D snoRNA to form a box C/D small nucleolar ribonucleoprotein (snoRNP) complex.'},\n",
       "  'synonyms': [{'name': 'box C/D small nucleolar ribonucleoprotein complex assembly',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000491', 'relation': 'is_a'}]},\n",
       " 'GO:0000493': {'id': 'GO:0000493',\n",
       "  'isObsolete': False,\n",
       "  'name': 'box H/ACA snoRNP assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of proteins and a box H/ACA snoRNA to form a box H/ACA small nucleolar ribonucleoprotein (snoRNP) complex.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12515383'}]},\n",
       "  'synonyms': [{'name': 'box H/ACA small nucleolar ribonucleoprotein complex assembly',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000491', 'relation': 'is_a'}]},\n",
       " 'GO:0000494': {'id': 'GO:0000494',\n",
       "  'isObsolete': False,\n",
       "  'name': \"box C/D snoRNA 3'-end processing\",\n",
       "  'definition': {'text': \"Any process involved in forming the mature 3' end of a box C/D snoRNA molecule.\"},\n",
       "  'synonyms': [{'name': \"box C/D snoRNA 3' end processing\", 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031126', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034963', 'relation': 'is_a'}]},\n",
       " 'GO:0000495': {'id': 'GO:0000495',\n",
       "  'isObsolete': False,\n",
       "  'name': \"box H/ACA snoRNA 3'-end processing\",\n",
       "  'definition': {'text': \"Any process involved in forming the mature 3' end of a box H/ACA snoRNA molecule.\"},\n",
       "  'synonyms': [{'name': \"box H/ACA snoRNA 3' end processing\",\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031126', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034964', 'relation': 'is_a'}]},\n",
       " 'GO:0000497': {'id': 'GO:0000497',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA template activity',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with nucleic acid via hydrogen bonds between the bases of a gene product molecule and the bases of a target DNA molecule.'},\n",
       "  'comment': 'Note that with respect to annotation, \"base pairing\" and its child terms are intended to be used to annotate the activity of gene products composed of nucleic acid, presumably RNA, to interact with DNA molecules via base pairing. Internal base pairing with itself is considered part of the secondary structure of the molecule and is not within the scope of GO function.',\n",
       "  'synonyms': [{'name': 'base pairing with DNA', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000332', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003677', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043566', 'relation': 'is_a'}]},\n",
       " 'GO:0000500': {'id': 'GO:0000500',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I upstream activating factor complex',\n",
       "  'definition': {'text': 'A complex required for the transcription of rDNA by RNA polymerase I. In yeast the complex consists of Rrrn5p, Rrn9p, Rrn10p, histones H3 and H4, and Uaf30p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11500378'}]},\n",
       "  'synonyms': [{'name': 'UAF', 'type': 'exact'},\n",
       "   {'name': 'RNA polymerase I upstream activation factor complex',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000501': {'id': 'GO:0000501',\n",
       "  'isObsolete': False,\n",
       "  'name': 'flocculation via cell wall protein-carbohydrate interaction',\n",
       "  'definition': {'text': 'The reversible, non-sexual aggregation of cells mediated by the binding of proteins in the cell wall of one cell to carbohydrates in the cell wall of another cell.  An example of this process is found in Saccharomyces cerevisiae, in which it is pH- and calcium-dependent.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '21114594'}]},\n",
       "  'children': [{'id': 'GO:1900455', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0036350', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900479', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0043690', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1900524', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0036349', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0100054', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000128', 'relation': 'is_a'}]},\n",
       " 'GO:0000502': {'id': 'GO:0000502',\n",
       "  'isObsolete': False,\n",
       "  'name': 'proteasome complex',\n",
       "  'definition': {'text': 'A large multisubunit complex which catalyzes protein degradation, found in eukaryotes, archaea and some bacteria. In eukaryotes, this complex consists of the barrel shaped proteasome core complex and one or two associated proteins or complexes that act in regulating entry into or exit from the core.',\n",
       "   'xrefs': [{'dbCode': 'URL',\n",
       "     'dbId': 'http://en.wikipedia.org/wiki/Proteasome'}]},\n",
       "  'synonyms': [{'name': 'proteasome', 'type': 'exact'},\n",
       "   {'name': '26S proteasome', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:1990111', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005839', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031597', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031595', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022624', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030163', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0044254', 'relation': 'capable_of_part_of'}]},\n",
       " 'GO:0000506': {'id': 'GO:0000506',\n",
       "  'isObsolete': False,\n",
       "  'name': 'glycosylphosphatidylinositol-N-acetylglucosaminyltransferase (GPI-GnT) complex',\n",
       "  'definition': {'text': 'An enzyme complex that catalyzes the transfer of GlcNAc from UDP-GlcNAc to an acceptor phosphatidylinositol, the first step in the production of GPI anchors for cell surface proteins. The complex contains PIG-A, PIG-C, PIG-H, PIG-Q, PIG-P, and DPM2 in human, and Eri1p, Gpi1p, Gpi2p, Gpi15p, Gpi19p, and Spt14p in budding yeast.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10944123'},\n",
       "    {'dbCode': 'PMID', 'dbId': '15163411'}]},\n",
       "  'comment': \"Note that this term should not be confused with 'GPI-anchor transamidase complex ; GO:0042765', which represents a distinct complex with a different catalytic activity.\",\n",
       "  'synonyms': [{'name': 'GPI-GnT complex', 'type': 'exact'},\n",
       "   {'name': 'GPI-GlcNAc transferase complex', 'type': 'exact'},\n",
       "   {'name': 'GPI-N-acetylglucosaminyltransferase complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008194', 'relation': 'capable_of'}]},\n",
       " 'GO:0000578': {'id': 'GO:0000578',\n",
       "  'isObsolete': False,\n",
       "  'name': 'embryonic axis specification',\n",
       "  'definition': {'text': 'The establishment, maintenance and elaboration of a pattern along a line or a point in an embryo.'},\n",
       "  'synonyms': [{'name': 'embryonic axis determination', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0007352', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008595', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009942', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032525', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060834', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0021997', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009798', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009880', 'relation': 'is_a'}]},\n",
       " 'GO:0000700': {'id': 'GO:0000700',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mismatch base pair DNA N-glycosylase activity',\n",
       "  'definition': {'text': \"Catalysis of the removal of single bases present in mismatches by the cleavage the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free base and leaves an apurinic/apyrimidinic (AP) site.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9224623'}]},\n",
       "  'children': [{'id': 'GO:0008263', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000701', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019104', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008578', 'relation': 'is_a'}]},\n",
       " 'GO:0000701': {'id': 'GO:0000701',\n",
       "  'isObsolete': False,\n",
       "  'name': 'purine-specific mismatch base pair DNA N-glycosylase activity',\n",
       "  'definition': {'text': \"Catalysis of the removal of purines present in mismatches, especially opposite oxidized purines, by cleaving the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free base and leaves an apurinic (AP) site.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9224623'}]},\n",
       "  'synonyms': [{'name': 'A/G-specific adenine DNA glycosylase activity',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000700', 'relation': 'is_a'}]},\n",
       " 'GO:0000702': {'id': 'GO:0000702',\n",
       "  'isObsolete': False,\n",
       "  'name': 'oxidized base lesion DNA N-glycosylase activity',\n",
       "  'definition': {'text': \"Catalysis of the removal of oxidized bases by cleaving the N-C1' glycosidic bond between the target damaged DNA base and the deoxyribose sugar. The reaction releases a free base and leaves an apurinic/apyrimidinic (AP) site.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11554296'}]},\n",
       "  'children': [{'id': 'GO:0000703', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008534', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019104', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008578', 'relation': 'is_a'}]},\n",
       " 'GO:0000703': {'id': 'GO:0000703',\n",
       "  'isObsolete': False,\n",
       "  'name': 'oxidized pyrimidine nucleobase lesion DNA N-glycosylase activity',\n",
       "  'definition': {'text': \"Catalysis of the removal oxidized pyrimidine bases by cleaving the N-C1' glycosidic bond between the oxidized pyrimidine and the deoxyribose sugar. The reaction involves formation of a covalent enzyme-pyrimidine base intermediate. Release of the enzyme and free base by a beta-elimination or a beta, gamma-elimination mechanism results in the cleavage of the DNA backbone 3' of the apyrimidinic (AP) site.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11554296'}]},\n",
       "  'comment': \"Consider also annotating to the molecular function term 'DNA-(apurinic or apyrimidinic site) lyase activity ; GO:0003906'.\",\n",
       "  'synonyms': [{'name': 'DNA glycosylase/beta-lyase', 'type': 'broad'},\n",
       "   {'name': 'bifunctional DNA glycosylase', 'type': 'broad'},\n",
       "   {'name': 'endodeoxyribonuclease III', 'type': 'related'},\n",
       "   {'name': 'endonuclease III', 'type': 'related'},\n",
       "   {'name': 'DNA glycosylase/AP-lyase', 'type': 'broad'},\n",
       "   {'name': 'endonuclease VIII activity', 'type': 'related'},\n",
       "   {'name': 'oxidized pyrimidine base lesion DNA N-glycosylase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'pyrimidine-specific oxidized base lesion DNA N-glycosylase activity',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0034043', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034042', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000702', 'relation': 'is_a'}]},\n",
       " 'GO:0000704': {'id': 'GO:0000704',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pyrimidine dimer DNA N-glycosylase activity',\n",
       "  'definition': {'text': \"Catalysis of the removal of pyrimidine dimers by removing the 5' pyrimidine of the dimer by cleaving the N-C1' glycosidic bond between the 5' pyrimidine of the dimer and the deoxyribose sugar. The reaction releases the 5' pyrimidine of the dimer and leaves an apurinic (AP) site. The reaction involves the formation of a covalent enzyme substrate intermediate. Release of the enzyme and free base by a beta-elimination or a beta, gamma-elimination mechanism results in the cleavage of the DNA backbone 3' of the apyrimidinic (AP) site.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9224623'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019104', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008578', 'relation': 'is_a'}]},\n",
       " 'GO:0000705': {'id': 'GO:0000705',\n",
       "  'isObsolete': False,\n",
       "  'name': 'achiasmate meiosis I',\n",
       "  'definition': {'text': 'The first division of meiosis in which homologous chromosomes are paired and segregated from each other, occurring in the constitutive absence of chiasmata.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10690419'}]},\n",
       "  'synonyms': [{'name': 'achiasmate meiosis I nuclear division',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007127', 'relation': 'is_a'}]},\n",
       " 'GO:0000706': {'id': 'GO:0000706',\n",
       "  'isObsolete': False,\n",
       "  'name': 'meiotic DNA double-strand break processing',\n",
       "  'definition': {'text': \"The cell cycle process in which the 5' to 3' exonucleolytic resection of the DNA at the site of the break to form a 3' single-strand DNA overhang occurs. This takes place during meiosis.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9334324'}]},\n",
       "  'children': [{'id': 'GO:0010705', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010704', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000729', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051321', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903046', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007126', 'relation': 'part_of'}]},\n",
       " 'GO:0000707': {'id': 'GO:0000707',\n",
       "  'isObsolete': False,\n",
       "  'name': 'meiotic DNA recombinase assembly',\n",
       "  'definition': {'text': 'During meiosis, the aggregation, arrangement and bonding together of strand exchange proteins (recombinases) to form higher order oligomers on single-stranded DNA.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11459983'}]},\n",
       "  'children': [{'id': 'GO:0010772', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007146', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010773', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000730', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061982', 'relation': 'is_a'}]},\n",
       " 'GO:0000708': {'id': 'GO:0000708',\n",
       "  'isObsolete': False,\n",
       "  'name': 'meiotic strand invasion',\n",
       "  'definition': {'text': 'The cell cycle process in which the nucleoprotein complex (composed of the broken single-strand DNA and the recombinase) searches and identifies a region of homology in intact duplex DNA. The broken single-strand DNA displaces the like strand and forms Watson-Crick base pairs with its complement, forming a duplex in which each strand is from one of the two recombining DNA molecules. This occurs during meiosis.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10915877'}]},\n",
       "  'synonyms': [{'name': 'meiotic D-loop biosynthesis', 'type': 'related'},\n",
       "   {'name': 'meiotic displacement loop formation', 'type': 'related'},\n",
       "   {'name': 'meiotic D-loop formation', 'type': 'related'},\n",
       "   {'name': 'meiotic displacement loop biosynthesis', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0010774', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010775', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0042148', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051321', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061982', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007126', 'relation': 'part_of'}]},\n",
       " 'GO:0000709': {'id': 'GO:0000709',\n",
       "  'isObsolete': False,\n",
       "  'name': 'meiotic joint molecule formation',\n",
       "  'definition': {'text': 'The conversion of the paired broken DNA and homologous duplex DNA into a four-stranded branched intermediate, known as a joint molecule, formed during meiotic recombination. These joint molecules contain Holliday junctions on either side of heteroduplex DNA.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '8521495'}]},\n",
       "  'children': [{'id': 'GO:0010947', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0010946', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007131', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903046', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000021', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007145', 'relation': 'part_of'}]},\n",
       " 'GO:0000710': {'id': 'GO:0000710',\n",
       "  'isObsolete': False,\n",
       "  'name': 'meiotic mismatch repair',\n",
       "  'definition': {'text': 'A system for the identification and correction of base-base mismatches, small insertion-deletion loops, and regions of heterology that are present in duplex DNA formed with strands from two recombining molecules. Correction of the mismatch can result in non-Mendelian segregation of alleles following meiosis.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'children': [{'id': 'GO:0010777', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010776', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006298', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061982', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006300', 'relation': 'is_a'}]},\n",
       " 'GO:0000711': {'id': 'GO:0000711',\n",
       "  'isObsolete': False,\n",
       "  'name': 'meiotic DNA repair synthesis',\n",
       "  'definition': {'text': \"During meiosis, the synthesis of DNA proceeding from the broken 3' single-strand DNA end that uses the homologous intact duplex as the template.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9334324'}]},\n",
       "  'children': [{'id': 'GO:0010778', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010779', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000731', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061982', 'relation': 'is_a'}]},\n",
       " 'GO:0000712': {'id': 'GO:0000712',\n",
       "  'isObsolete': False,\n",
       "  'name': 'resolution of meiotic recombination intermediates',\n",
       "  'definition': {'text': 'The cleavage and rejoining of intermediates, such as Holliday junctions, formed during meiotic recombination to produce two intact molecules in which genetic material has been exchanged.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11733053'}]},\n",
       "  'synonyms': [{'name': 'resolution of meiotic joint molecules as recombinants',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007131', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051307', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061982', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000021', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007145', 'relation': 'part_of'}]},\n",
       " 'GO:0000713': {'id': 'GO:0000713',\n",
       "  'isObsolete': False,\n",
       "  'name': 'meiotic heteroduplex formation',\n",
       "  'definition': {'text': 'During meiosis, the formation of a stable duplex DNA that contains one strand from each of the two recombining DNA molecules.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9334324'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006310', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061982', 'relation': 'is_a'}]},\n",
       " 'GO:0000714': {'id': 'GO:0000714',\n",
       "  'isObsolete': False,\n",
       "  'name': 'meiotic strand displacement',\n",
       "  'definition': {'text': \"The cell cycle process in which the broken 3' single-strand DNA molecule that formed heteroduplex DNA with its complement in an intact duplex DNA is rejected. The Watson-Crick base pairing in the original duplex is restored. The rejected 3' single-strand DNA molecule reanneals with its original complement to reform two intact duplex molecules. This occurs during meiosis.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'synonyms': [{'name': 'meiotic D-loop processing', 'type': 'related'},\n",
       "   {'name': 'meiotic displacement loop dissociation', 'type': 'related'},\n",
       "   {'name': 'meiotic displacement loop processing', 'type': 'related'},\n",
       "   {'name': 'meiotic D-loop dissociation', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:1902346', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000732', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061982', 'relation': 'is_a'}]},\n",
       " 'GO:0000715': {'id': 'GO:0000715',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleotide-excision repair, DNA damage recognition',\n",
       "  'definition': {'text': 'The identification of lesions in DNA, such as pyrimidine-dimers, intrastrand cross-links, and bulky adducts. The wide range of substrate specificity suggests the repair complex recognizes distortions in the DNA helix.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10197977'}]},\n",
       "  'synonyms': [{'name': 'pyrimidine-dimer repair, DNA damage recognition',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:1990249', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0000716', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006289', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051276', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045001', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007001', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051277', 'relation': 'is_a'}]},\n",
       " 'GO:0000716': {'id': 'GO:0000716',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription-coupled nucleotide-excision repair, DNA damage recognition',\n",
       "  'definition': {'text': 'The identification of lesions on the actively transcribed strand of the DNA duplex as well as a small subset of lesions not recognized by the general nucleotide-excision repair pathway.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10197977'}]},\n",
       "  'synonyms': [{'name': 'pyrimidine-dimer repair, DNA damage recognition',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000715', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006283', 'relation': 'part_of'}]},\n",
       " 'GO:0000717': {'id': 'GO:0000717',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleotide-excision repair, DNA duplex unwinding',\n",
       "  'definition': {'text': 'The unwinding, or local denaturation, of the DNA duplex to create a bubble around the site of the DNA damage.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10197977'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006289', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032508', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045001', 'relation': 'part_of'}]},\n",
       " 'GO:0000718': {'id': 'GO:0000718',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleotide-excision repair, DNA damage removal',\n",
       "  'definition': {'text': 'The removal of the oligonucleotide that contains the DNA damage. The oligonucleotide is formed by dual incisions that flank the site of DNA damage.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10197977'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006289', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0044349', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045001', 'relation': 'part_of'}]},\n",
       " 'GO:0000719': {'id': 'GO:0000719',\n",
       "  'isObsolete': False,\n",
       "  'name': 'photoreactive repair',\n",
       "  'definition': {'text': 'The repair of UV-induced T-T, C-T and C-C dimers by directly reversing the damage to restore the original pyrimidines.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10915863'}]},\n",
       "  'synonyms': [{'name': 'pyrimidine-dimer repair by photolyase',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006290', 'relation': 'is_a'}]},\n",
       " 'GO:0000720': {'id': 'GO:0000720',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pyrimidine dimer repair by nucleotide-excision repair',\n",
       "  'definition': {'text': 'The repair of UV-induced T-T, C-T, and C-C dimers by the recognition and removal of the damaged DNA strand from the DNA helix as an oligonucleotide. The small gap left in the DNA helix is filled in by the sequential action of DNA polymerase and DNA ligase.'},\n",
       "  'comment': \"Note that the repair of pyrimidine dimers by nucleotide excision repair involves the same gene products that are involved in general nucleotide excision repair. Consider also annotating to other children of the biological process term 'nucleotide-excision repair ; GO:0006289'.\",\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006289', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006290', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045001', 'relation': 'is_a'}]},\n",
       " 'GO:0000721': {'id': 'GO:0000721',\n",
       "  'isObsolete': False,\n",
       "  'name': '(R,R)-butanediol dehydrogenase activity',\n",
       "  'definition': {'text': 'Catalysis of the reversible reaction: (R,R)-butane-2,3-diol + NAD+ = (R)-acetoin + NADH + H(+).',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.1.1.4'}]},\n",
       "  'synonyms': [{'name': 'D-1-amino-2-propanol:NAD(2) oxidoreductase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'D-aminopropanol dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'D-1-amino-2-propanol dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'D-(-)-butanediol dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': '2,3-butanediol dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': '1-amino-2-propanol oxidoreductase activity', 'type': 'exact'},\n",
       "   {'name': '1-amino-2-propanol dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': '(R)-diacetyl reductase activity', 'type': 'exact'},\n",
       "   {'name': '(R)-2,3-butanediol dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'D-butanediol dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'aminopropanol oxidoreductase activity', 'type': 'exact'},\n",
       "   {'name': 'butylene glycol dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'butyleneglycol dehydrogenase activity', 'type': 'exact'},\n",
       "   {'name': 'diacetyl (acetoin) reductase activity', 'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016616', 'relation': 'is_a'}]},\n",
       " 'GO:0000722': {'id': 'GO:0000722',\n",
       "  'isObsolete': False,\n",
       "  'name': 'telomere maintenance via recombination',\n",
       "  'definition': {'text': 'Any recombinational process that contributes to the maintenance of proper telomeric length.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11850777'}]},\n",
       "  'synonyms': [{'name': 'telomerase-independent telomere maintenance',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0032209', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0032207', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0061819', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032208', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000723', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006312', 'relation': 'is_a'}]},\n",
       " 'GO:0000723': {'id': 'GO:0000723',\n",
       "  'isObsolete': False,\n",
       "  'name': 'telomere maintenance',\n",
       "  'definition': {'text': 'Any process that contributes to the maintenance of proper telomeric length and structure by affecting and monitoring the activity of telomeric proteins, the length of telomeric DNA and the replication and repair of the DNA. These processes includes those that shorten, lengthen, replicate and repair the telomeric DNA sequences.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11092831'}]},\n",
       "  'synonyms': [{'name': 'regulation of telomere length', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0043247', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031627', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010833', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090657', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016233', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043336', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032201', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000722', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090737', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032204', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0032205', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0032206', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006259', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032200', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060249', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055132', 'relation': 'is_a'}]},\n",
       " 'GO:0000724': {'id': 'GO:0000724',\n",
       "  'isObsolete': False,\n",
       "  'name': 'double-strand break repair via homologous recombination',\n",
       "  'definition': {'text': 'The error-free repair of a double-strand break in DNA in which the broken DNA molecule is repaired using homologous sequences. A strand in the broken DNA searches for a homologous region in an intact chromosome to serve as the template for DNA synthesis. The restoration of two intact DNA molecules results in the exchange, reciprocal or nonreciprocal, of genetic material between the intact DNA molecule and the broken DNA molecule.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'synonyms': [{'name': 'homologous recombinational repair', 'type': 'exact'},\n",
       "   {'name': 'HRR', 'type': 'exact'},\n",
       "   {'name': 'Rhp51-dependent recombinational repair', 'type': 'exact'},\n",
       "   {'name': 'Rad51-dependent recombinational repair', 'type': 'exact'},\n",
       "   {'name': 'HDR', 'type': 'exact'},\n",
       "   {'name': 'homology-directed repair', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0010569', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000042', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1905168', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0043150', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000727', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990414', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045003', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000725', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006302', 'relation': 'is_a'}]},\n",
       " 'GO:0000725': {'id': 'GO:0000725',\n",
       "  'isObsolete': False,\n",
       "  'name': 'recombinational repair',\n",
       "  'definition': {'text': 'A DNA repair process that involves the exchange, reciprocal or nonreciprocal, of genetic material between the broken DNA molecule and a homologous region of DNA.'},\n",
       "  'children': [{'id': 'GO:0097552', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036298', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000724', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990396', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006281', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006310', 'relation': 'is_a'}]},\n",
       " 'GO:0000726': {'id': 'GO:0000726',\n",
       "  'isObsolete': False,\n",
       "  'name': 'non-recombinational repair',\n",
       "  'definition': {'text': 'A DNA repair process in which that does not require the exchange of genetic material between the broken DNA molecule and a homologous region of DNA.'},\n",
       "  'children': [{'id': 'GO:0045002', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036299', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006303', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006281', 'relation': 'is_a'}]},\n",
       " 'GO:0000727': {'id': 'GO:0000727',\n",
       "  'isObsolete': False,\n",
       "  'name': 'double-strand break repair via break-induced replication',\n",
       "  'definition': {'text': \"The error-free repair of a double-strand break in DNA in which the centromere-proximal end of a broken chromosome searches for a homologous region in an intact chromosome. DNA synthesis initiates from the 3' end of the invading DNA strand, using the intact chromosome as the template, and progresses to the end of the chromosome.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'children': [{'id': 'GO:1901592', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1901591', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000724', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016924', 'relation': 'is_a'}]},\n",
       " 'GO:0000728': {'id': 'GO:0000728',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gene conversion at mating-type locus, DNA double-strand break formation',\n",
       "  'definition': {'text': 'The site-specific endonucleolytic cleavage of DNA at the mating-type locus which initiates the conversion of one mating-type allele to another.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '7646483'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007534', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022414', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090305', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044702', 'relation': 'is_a'}]},\n",
       " 'GO:0000729': {'id': 'GO:0000729',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA double-strand break processing',\n",
       "  'definition': {'text': \"The 5' to 3' exonucleolytic resection of the DNA at the site of the break to form a 3' single-strand DNA overhang.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'children': [{'id': 'GO:0010792', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031292', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010791', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000706', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903775', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006259', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006302', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0055132', 'relation': 'is_a'}]},\n",
       " 'GO:0000730': {'id': 'GO:0000730',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA recombinase assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of strand exchange proteins (recombinases) into higher order oligomers on single-stranded DNA.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'synonyms': [{'name': 'Rad51 nucleoprotein filament formation',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000707', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034624', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0045003', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0065004', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090735', 'relation': 'is_a'}]},\n",
       " 'GO:0000731': {'id': 'GO:0000731',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA synthesis involved in DNA repair',\n",
       "  'definition': {'text': \"Synthesis of DNA that proceeds from the broken 3' single-strand DNA end and uses the homologous intact duplex as the template.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'synonyms': [{'name': 'mitotic DNA repair synthesis', 'type': 'narrow'},\n",
       "   {'name': 'DNA synthesis during DNA repair', 'type': 'related'},\n",
       "   {'name': 'DNA repair synthesis', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:1904161', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043150', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043151', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000711', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000734', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019985', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006281', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071897', 'relation': 'is_a'}]},\n",
       " 'GO:0000732': {'id': 'GO:0000732',\n",
       "  'isObsolete': False,\n",
       "  'name': 'strand displacement',\n",
       "  'definition': {'text': \"The rejection of the broken 3' single-strand DNA molecule that formed heteroduplex DNA with its complement in an intact duplex DNA. The Watson-Crick base pairing in the original duplex is restored. The rejected 3' single-strand DNA molecule reanneals with its original complement to reform two intact duplex molecules.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'synonyms': [{'name': 'displacement loop processing', 'type': 'related'},\n",
       "   {'name': 'displacement loop dissociation', 'type': 'related'},\n",
       "   {'name': 'D-loop dissociation', 'type': 'related'},\n",
       "   {'name': 'D-loop processing', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000714', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006259', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006310', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0055132', 'relation': 'is_a'}]},\n",
       " 'GO:0000733': {'id': 'GO:0000733',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA strand renaturation',\n",
       "  'definition': {'text': 'The identification and annealing of complementary base pairs in single-strand DNA.'},\n",
       "  'children': [{'id': 'GO:0036310', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061306', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006259', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032392', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055132', 'relation': 'is_a'}]},\n",
       " 'GO:0000734': {'id': 'GO:0000734',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gene conversion at mating-type locus, DNA repair synthesis',\n",
       "  'definition': {'text': \"Synthesis of DNA that proceeds from the broken 3' single-strand DNA end uses the homologous intact duplex as the template during gene conversion at the mating-type locus.\"},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000731', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007534', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022414', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044702', 'relation': 'is_a'}]},\n",
       " 'GO:0000735': {'id': 'GO:0000735',\n",
       "  'isObsolete': False,\n",
       "  'name': 'removal of nonhomologous ends',\n",
       "  'definition': {'text': \"The removal of nonhomologous sequences at the broken 3' single-strand DNA end before DNA repair synthesis can occur.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'children': [{'id': 'GO:0000736', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006259', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055132', 'relation': 'is_a'}]},\n",
       " 'GO:0000736': {'id': 'GO:0000736',\n",
       "  'isObsolete': False,\n",
       "  'name': 'double-strand break repair via single-strand annealing, removal of nonhomologous ends',\n",
       "  'definition': {'text': \"During DSBR via single-strand annealing, the removal of nonhomologous sequences at the broken 3' single-strand DNA end before DNA repair synthesis can occur.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10357855'}]},\n",
       "  'children': [{'id': 'GO:1903776', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000735', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045002', 'relation': 'part_of'}]},\n",
       " 'GO:0000737': {'id': 'GO:0000737',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA catabolic process, endonucleolytic',\n",
       "  'definition': {'text': \"The chemical reactions and pathways resulting in the breakdown of DNA, involving the hydrolysis of internal 3',5'-phosphodiester bonds in one or two strands of deoxyribonucleotides.\"},\n",
       "  'synonyms': [{'name': 'endonucleolytic DNA catabolism', 'type': 'exact'},\n",
       "   {'name': 'DNA breakdown, endonucleolytic', 'type': 'exact'},\n",
       "   {'name': 'DNA degradation, endonucleolytic', 'type': 'exact'},\n",
       "   {'name': 'endonucleolytic degradation of DNA', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0006309', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006308', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090305', 'relation': 'is_a'}]},\n",
       " 'GO:0000738': {'id': 'GO:0000738',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA catabolic process, exonucleolytic',\n",
       "  'definition': {'text': \"The chemical reactions and pathways resulting in the breakdown of DNA, involving the hydrolysis of terminal 3',5'-phosphodiester bonds in one or two strands of deoxyribonucleotides.\"},\n",
       "  'synonyms': [{'name': 'DNA degradation, exonucleolytic', 'type': 'exact'},\n",
       "   {'name': 'DNA breakdown, exonucleolytic', 'type': 'exact'},\n",
       "   {'name': 'exonucleolytic degradation of DNA', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006308', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090305', 'relation': 'is_a'}]},\n",
       " 'GO:0000740': {'id': 'GO:0000740',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear membrane fusion',\n",
       "  'definition': {'text': 'The joining of 2 or more lipid bilayer membranes that surround the nucleus.'},\n",
       "  'children': [{'id': 'GO:0007086', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048288', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0090174', 'relation': 'is_a'}]},\n",
       " 'GO:0000741': {'id': 'GO:0000741',\n",
       "  'isObsolete': False,\n",
       "  'name': 'karyogamy',\n",
       "  'definition': {'text': 'The creation of a single nucleus from multiple nuclei as a result of fusing the lipid bilayers that surround each nuclei.'},\n",
       "  'synonyms': [{'name': 'nuclear fusion', 'type': 'exact'},\n",
       "   {'name': 'nuclear fusion during karyogamy', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0010197', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048288', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032871', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0000744', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000742', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007344', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006997', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048284', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048287', 'relation': 'part_of'}]},\n",
       " 'GO:0000742': {'id': 'GO:0000742',\n",
       "  'isObsolete': False,\n",
       "  'name': 'karyogamy involved in conjugation with cellular fusion',\n",
       "  'definition': {'text': 'During sexual reproduction, the creation of a single nucleus from multiple nuclei as a result of fusing the lipid bilayers that surround each nuclei. This occurs after cytogamy.'},\n",
       "  'synonyms': [{'name': 'karyogamy during conjugation with cellular fusion',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000741', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000747', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022414', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007335', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030477', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007322', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030461', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007333', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0044702', 'relation': 'is_a'}]},\n",
       " 'GO:0000743': {'id': 'GO:0000743',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear migration involved in conjugation with cellular fusion',\n",
       "  'definition': {'text': 'The microtubule-based movement of nuclei towards one another as a prelude to karyogamy in organisms undergoing conjugation with cellular fusion.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16380440'}]},\n",
       "  'synonyms': [{'name': 'nuclear congression', 'type': 'exact'},\n",
       "   {'name': 'nuclear migration during conjugation with cellular fusion',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000747', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022414', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030473', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030477', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007322', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030461', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007333', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0044702', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000065', 'relation': 'is_a'}]},\n",
       " 'GO:0000744': {'id': 'GO:0000744',\n",
       "  'isObsolete': False,\n",
       "  'name': 'karyogamy involved in conjugation with mutual genetic exchange',\n",
       "  'definition': {'text': 'During sexual reproduction, the creation of a single nucleus from two nuclei as a result of fusing the nuclear envelopes that surround each nuclei. This takes place following the mutual exchange of one of the two nuclei produced by the mitosis that follows the second meiotic nuclear division. This occurs in ciliated protozoans such as Tetrahymena.'},\n",
       "  'comment': 'Changed definition https://github.com/geneontology/go-ontology/issues/16248.',\n",
       "  'synonyms': [{'name': 'karyogamy involved in conjugation without cellular fusion',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000745', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000741', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000748', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007335', 'relation': 'is_a'}]},\n",
       " 'GO:0000745': {'id': 'GO:0000745',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear migration involved in conjugation with mutual genetic exchange',\n",
       "  'definition': {'text': 'The net movement of nuclei towards one another, leading to the bilateral transfer of genetic material in organisms undergoing conjugation without cellular fusion.'},\n",
       "  'synonyms': [{'name': 'nuclear migration involved in conjugation without cellular fusion',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'nuclear exchange during conjugation without cellular fusion',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000744', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000748', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007097', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0040023', 'relation': 'is_a'}]},\n",
       " 'GO:0000746': {'id': 'GO:0000746',\n",
       "  'isObsolete': False,\n",
       "  'name': 'conjugation',\n",
       "  'definition': {'text': 'The union or introduction of genetic information from compatible mating types that results in a genetically different individual. Conjugation requires direct cellular contact between the organisms.'},\n",
       "  'children': [{'id': 'GO:0000771', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000747', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009291', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031135', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0031136', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1990277', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000767', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0046999', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0000748', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071832', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0071835', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044764', 'relation': 'is_a'}]},\n",
       " 'GO:0000747': {'id': 'GO:0000747',\n",
       "  'isObsolete': False,\n",
       "  'name': 'conjugation with cellular fusion',\n",
       "  'definition': {'text': 'A conjugation process that results in the union of cellular and genetic information from compatible mating types. An example of this process is found in Saccharomyces cerevisiae.'},\n",
       "  'synonyms': [{'name': 'cell fusion', 'type': 'related'},\n",
       "   {'name': 'mating', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0071631', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1900406', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0031139', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1904541', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032005', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0070871', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000752', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000742', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000753', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000743', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000755', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071833', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0031138', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0071511', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071510', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000749', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0071508', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071509', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031137', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0071507', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000746', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019953', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140253', 'relation': 'is_a'}]},\n",
       " 'GO:0000748': {'id': 'GO:0000748',\n",
       "  'isObsolete': False,\n",
       "  'name': 'conjugation with mutual genetic exchange',\n",
       "  'definition': {'text': 'A conjugation process that results in the mutual exchange and union of only genetic information between compatible mating types. Conjugation without cellular fusion requires direct cellular contact between the organisms without plasma membrane fusion. The organisms involved in conjugation without cellular fusion separate after nuclear exchange.'},\n",
       "  'synonyms': [{'name': 'conjugation without cellular fusion',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000759', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000758', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000745', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000744', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000756', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0000761', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000746', 'relation': 'is_a'}]},\n",
       " 'GO:0000749': {'id': 'GO:0000749',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to pheromone triggering conjugation with cellular fusion',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pheromone stimulus that positively regulates the process of conjugation with cellular fusion. An example of this process is found in Saccharomyces cerevisiae.'},\n",
       "  'synonyms': [{'name': 'response to pheromone during conjugation with cellular fusion',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000750', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000751', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000752', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000754', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000747', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0031139', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071444', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030477', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0007322', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0030461', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0007333', 'relation': 'positively_regulates'}]},\n",
       " 'GO:0000750': {'id': 'GO:0000750',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pheromone-dependent signal transduction involved in conjugation with cellular fusion',\n",
       "  'definition': {'text': 'A signal transduction process resulting in the relay, amplification or dampening of a signal generated in response to pheromone exposure in organisms that undergo conjugation with cellular fusion. An example of this process is found in Saccharomyces cerevisiae.'},\n",
       "  'synonyms': [{'name': 'transduction of mating signal', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0090029', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0090028', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0071507', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010969', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000749', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032005', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007328', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030434', 'relation': 'part_of'}]},\n",
       " 'GO:0000751': {'id': 'GO:0000751',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitotic cell cycle G1 arrest in response to pheromone',\n",
       "  'definition': {'text': 'The cell cycle regulatory process in which the mitotic cell cycle is halted during G1 as a result of a pheromone stimulus. An example of this process is found in Saccharomyces cerevisiae.'},\n",
       "  'synonyms': [{'name': 'cell cycle arrest in response to pheromone',\n",
       "    'type': 'broad'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000749', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071850', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007328', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030434', 'relation': 'part_of'}]},\n",
       " 'GO:0000752': {'id': 'GO:0000752',\n",
       "  'isObsolete': False,\n",
       "  'name': 'agglutination involved in conjugation with cellular fusion',\n",
       "  'definition': {'text': 'During conjugation with cellular fusion, the aggregation or adhesion of compatible mating types via complementary cell-cell interactions. An example of this process is agglutination in Saccharomyces cerevisiae.'},\n",
       "  'synonyms': [{'name': 'cell-cell adhesion during conjugation with cellular fusion',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'agglutination', 'type': 'broad'},\n",
       "   {'name': 'cell-cell adhesion during mating', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000747', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000749', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000771', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044703', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098630', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030477', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007322', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030461', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007333', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007328', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030434', 'relation': 'part_of'}]},\n",
       " 'GO:0000753': {'id': 'GO:0000753',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell morphogenesis involved in conjugation with cellular fusion',\n",
       "  'definition': {'text': 'The change in form (cell shape and size) that occurs during sexual reproduction in order to facilitate direct contact between the compatible mating types in organisms that undergo conjugation cellular fusion.'},\n",
       "  'synonyms': [{'name': 'shmooing', 'type': 'narrow'},\n",
       "   {'name': 'shmoo orientation', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:1905708', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0031382', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000747', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000767', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003006', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030477', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007322', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030461', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007333', 'relation': 'part_of'}]},\n",
       " 'GO:0000754': {'id': 'GO:0000754',\n",
       "  'isObsolete': False,\n",
       "  'name': 'adaptation of signaling pathway by response to pheromone involved in conjugation with cellular fusion',\n",
       "  'definition': {'text': 'In organisms that undergo conjugation with cellular fusion, the process resulting in desensitization following exposure to pheromone stimulus that act to down-regulate further stimulation or block initial conjugation responses. An example of this is the adaptation to pheromone during conjugation with cellular fusion in Saccharomyces cerevisiae.'},\n",
       "  'synonyms': [{'name': 'adaptation of signalling pathway by response to pheromone involved in conjugation with cellular fusion',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'desensitization to pheromone during conjugation with cellular fusion',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'adaptation to pheromone during conjugation with cellular fusion',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000321', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000749', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0023058', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007328', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030434', 'relation': 'part_of'}]},\n",
       " 'GO:0000755': {'id': 'GO:0000755',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytogamy',\n",
       "  'definition': {'text': 'During conjugation with cellular fusion, the process resulting in creating a single cell from complementary mating types. The localized remodeling and dissolution of external protective structures allow the fusion of the plasma membranes and cytoplasmic mixing. An example of this process is found in Saccharomyces cerevisiae.'},\n",
       "  'synonyms': [{'name': 'zygote formation', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0032220', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032219', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000747', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022413', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030477', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007322', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030461', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007333', 'relation': 'part_of'}]},\n",
       " 'GO:0000756': {'id': 'GO:0000756',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to pheromone regulating conjugation with mutual genetic exchange',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pheromone stimulus regulating the process of conjugation without cellular fusion.'},\n",
       "  'synonyms': [{'name': 'response to pheromone triggering conjugation without cellular fusion',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000758', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000760', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000748', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0007154', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046999', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071444', 'relation': 'is_a'}]},\n",
       " 'GO:0000758': {'id': 'GO:0000758',\n",
       "  'isObsolete': False,\n",
       "  'name': 'agglutination involved in conjugation with mutual genetic exchange',\n",
       "  'definition': {'text': 'During conjugation without cellular fusion, the aggregation or adhesion of compatible mating types via complementary cell-cell interactions.'},\n",
       "  'synonyms': [{'name': 'sexual flocculation', 'type': 'exact'},\n",
       "   {'name': 'agglutination involved in conjugation without cellular fusion',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000748', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000756', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000771', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098630', 'relation': 'is_a'}]},\n",
       " 'GO:0000759': {'id': 'GO:0000759',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell morphogenesis involved in conjugation with mutual genetic exchange',\n",
       "  'definition': {'text': 'The change in form (cell shape and size) that occurs during sexual reproduction in order to facilitate direct contact between the compatible mating types in organisms that undergo conjugation without cellular fusion.'},\n",
       "  'synonyms': [{'name': 'cellular morphogenesis involved in conjugation without cellular fusion',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000748', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000767', 'relation': 'is_a'}]},\n",
       " 'GO:0000760': {'id': 'GO:0000760',\n",
       "  'isObsolete': False,\n",
       "  'name': 'adaptation to pheromone regulating conjugation with mutual genetic exchange',\n",
       "  'definition': {'text': 'In organisms that undergo conjugation without cellular fusion, the process resulting in desensitization following exposure to pheromone stimulus that act to down-regulate further stimulation or block initial conjugation responses.'},\n",
       "  'synonyms': [{'name': 'desensitization to pheromone during conjugation without cellular fusion',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'adaptation to pheromone involved conjugation without cellular fusion',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000756', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022401', 'relation': 'is_a'}]},\n",
       " 'GO:0000761': {'id': 'GO:0000761',\n",
       "  'isObsolete': False,\n",
       "  'name': 'conjugant formation',\n",
       "  'definition': {'text': 'During conjugation without cellular fusion, the process that results in pairing complementary mating types. Localized morphological, cytological, and cytoskeletal changes connect the mating types without cytoplasmic mixing.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000748', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000771', 'relation': 'is_a'}]},\n",
       " 'GO:0000762': {'id': 'GO:0000762',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pheromone-induced unidirectional conjugation',\n",
       "  'definition': {'text': \"The process of unidirectional (polarized) transfer of genetic information in response to a pheromone. It involves direct cellular contact between a donor and recipient cell; the contact is followed by the formation of a cellular bridge that physically connects the cells; some or all of the chromosome(s) of one cell ('male') is then transferred into the other cell ('female'); unidirectional conjugation occurs between cells of different mating types.\"},\n",
       "  'children': [{'id': 'GO:0000765', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0070575', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009291', 'relation': 'is_a'}]},\n",
       " 'GO:0000765': {'id': 'GO:0000765',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to pheromone regulating pheromone-induced unidirectional conjugation',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pheromone stimulus that regulates the process of pheromone-induced unidirectional conjugation.'},\n",
       "  'synonyms': [{'name': 'response to pheromone during pheromone-induced unidirectional',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000766', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000762', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0019236', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046999', 'relation': 'is_a'}]},\n",
       " 'GO:0000766': {'id': 'GO:0000766',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative adaptation of signaling pathway by response to pheromone involved in pheromone-induced unidirectional conjugation',\n",
       "  'definition': {'text': 'In organisms that undergo pheromone-induced unidirectional conjugation, the process involved in desensitization following exposure to pheromone stimulus that acts to down-regulate further stimulation or block initial conjugation responses.'},\n",
       "  'synonyms': [{'name': 'negative adaptation of signalling pathway by response to pheromone involved in pheromone-induced unidirectional conjugation',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'desensitization to pheromone during pheromone-induced unidirectional conjugation',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'adaptation to pheromone during pheromone-induced unidirectional conjugation',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000765', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022401', 'relation': 'is_a'}]},\n",
       " 'GO:0000767': {'id': 'GO:0000767',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell morphogenesis involved in conjugation',\n",
       "  'definition': {'text': 'The change in form (cell shape and size) that occurs during sexual reproduction in order to facilitate direct contact between the compatible mating types.'},\n",
       "  'children': [{'id': 'GO:0000753', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000759', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000746', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000902', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007148', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045790', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045791', 'relation': 'is_a'}]},\n",
       " 'GO:0000768': {'id': 'GO:0000768',\n",
       "  'isObsolete': False,\n",
       "  'name': 'syncytium formation by plasma membrane fusion',\n",
       "  'definition': {'text': 'The formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane, by the fusion of the plasma membranes of two or more individual cells.'},\n",
       "  'synonyms': [{'name': 'cell fusion', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0034238', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034242', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0060143', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0060142', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0072675', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007520', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006949', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140253', 'relation': 'is_a'}]},\n",
       " 'GO:0000769': {'id': 'GO:0000769',\n",
       "  'isObsolete': False,\n",
       "  'name': 'syncytium formation by mitosis without cytokinesis',\n",
       "  'definition': {'text': 'The formation of a syncytium, a mass of cytoplasm containing several nuclei enclosed within a single plasma membrane, by one or more rounds of nuclear division without cytokinesis.'},\n",
       "  'synonyms': [{'name': 'syncytium formation by mitosis without cell division',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006949', 'relation': 'is_a'}]},\n",
       " 'GO:0000770': {'id': 'GO:0000770',\n",
       "  'isObsolete': False,\n",
       "  'name': 'peptide pheromone export',\n",
       "  'definition': {'text': 'The directed movement of a peptide pheromone out of a cell by a secretion or export pathway used solely for the export of peptide pheromones.'},\n",
       "  'synonyms': [{'name': 'a-factor export', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0090539', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071832', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090538', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009914', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015833', 'relation': 'is_a'}]},\n",
       " 'GO:0000771': {'id': 'GO:0000771',\n",
       "  'isObsolete': False,\n",
       "  'name': 'agglutination involved in conjugation',\n",
       "  'definition': {'text': 'The aggregation or adhesion of compatible mating types via complementary cell-cell interactions prior to the formation of irreversible cellular contacts during conjugation.'},\n",
       "  'children': [{'id': 'GO:0000761', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000752', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000758', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000746', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098610', 'relation': 'is_a'}]},\n",
       " 'GO:0000772': {'id': 'GO:0000772',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mating pheromone activity',\n",
       "  'definition': {'text': 'The activity of binding to and activating specific cell surface receptors, thereby inducing a behavioral or physiological response(s) from a responding organism or cell that leads to the transfer or union of genetic material between organisms or cells. The mating pheromone can either be retained on the cell surface or secreted.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005186', 'relation': 'is_a'}]},\n",
       " 'GO:0000773': {'id': 'GO:0000773',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phosphatidyl-N-methylethanolamine N-methyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: S-adenosyl-L-methionine + phosphatidyl-N-methylethanolamine = S-adenosyl-L-homocysteine + phosphatidyl-N-dimethylethanolamine.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.1.1.71'}]},\n",
       "  'synonyms': [{'name': 'phospholipid methyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'phosphatidylmonomethylethanolamine methyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'phosphatidylethanolamine methyltransferase I', 'type': 'related'},\n",
       "   {'name': 'phosphatidyl-N-monomethylethanolamine methyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'phosphatidyl-N-methylethanolamine methyltransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'methyltransferase II', 'type': 'related'},\n",
       "   {'name': 'PLMT', 'type': 'exact'},\n",
       "   {'name': 'S-adenosyl-L-methionine:phosphatidyl-N-methylethanolamine N-methyltransferase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008170', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008757', 'relation': 'is_a'}]},\n",
       " 'GO:0000774': {'id': 'GO:0000774',\n",
       "  'isObsolete': False,\n",
       "  'name': 'adenyl-nucleotide exchange factor activity',\n",
       "  'definition': {'text': 'Stimulates the hydrolysis and exchange of adenyl nucleotides by other proteins.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030554', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060590', 'relation': 'is_a'}]},\n",
       " 'GO:0000775': {'id': 'GO:0000775',\n",
       "  'isObsolete': False,\n",
       "  'name': 'chromosome, centromeric region',\n",
       "  'definition': {'text': 'The region of a chromosome that includes the centromeric DNA and associated proteins. In monocentric chromosomes, this region corresponds to a single area of the chromosome, whereas in holocentric chromosomes, it is evenly distributed along the chromosome.'},\n",
       "  'comment': \"Note that this term can be used in place of the obsolete cellular component term 'centromere ; GO:0005698'.\",\n",
       "  'synonyms': [{'name': 'chromosome, pericentric region', 'type': 'related'},\n",
       "   {'name': 'chromosome, centric region', 'type': 'exact'},\n",
       "   {'name': 'centromere', 'type': 'related'},\n",
       "   {'name': 'centromere complex', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0034506', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0034507', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030702', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0031518', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005721', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000776', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0062199', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061838', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000779', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000776': {'id': 'GO:0000776',\n",
       "  'isObsolete': False,\n",
       "  'name': 'kinetochore',\n",
       "  'definition': {'text': 'A multisubunit complex that is located at the centromeric region of DNA and provides an attachment point for the spindle microtubules.'},\n",
       "  'comment': 'Note that the kinetochore overlaps the centromeric DNA, but centromeric DNA is not part of the kinetochore.',\n",
       "  'children': [{'id': 'GO:0031617', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1990423', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000777', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000777': {'id': 'GO:0000777',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed chromosome kinetochore',\n",
       "  'definition': {'text': 'A multisubunit complex that is located at the centromeric region of a condensed chromosome and provides an attachment point for the spindle microtubules.'},\n",
       "  'children': [{'id': 'GO:0000444', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000778', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033551', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000939', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000940', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000778': {'id': 'GO:0000778',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed nuclear chromosome kinetochore',\n",
       "  'definition': {'text': 'A multisubunit complex that is located at the centromeric region of a condensed nuclear chromosome and provides an attachment point for the spindle microtubules.'},\n",
       "  'children': [{'id': 'GO:0000942', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000817', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000818', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000941', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000779': {'id': 'GO:0000779',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed chromosome, centromeric region',\n",
       "  'definition': {'text': 'The region of a condensed chromosome that includes the centromere and associated proteins, including the kinetochore. In monocentric chromosomes, this region corresponds to a single area of the chromosome, whereas in holocentric chromosomes, it is evenly distributed along the chromosome.'},\n",
       "  'comment': \"Note that this term can be used in place of the obsolete cellular component term 'centromere ; GO:0005698'. Use with caution because this term refers to a specific region of the chromosome and not a protein complex.\",\n",
       "  'synonyms': [{'name': 'condensed chromosome, centromere', 'type': 'related'},\n",
       "   {'name': 'condensed chromosome, pericentric region', 'type': 'related'},\n",
       "   {'name': 'condensed chromosome, centric region', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000777', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000780', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000780': {'id': 'GO:0000780',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed nuclear chromosome, centromeric region',\n",
       "  'definition': {'text': 'The region of a condensed nuclear chromosome that includes the centromere and associated proteins, including the kinetochore. In monocentric chromosomes, this region corresponds to a single area of the chromosome, whereas in holocentric chromosomes, it is evenly distributed along the chromosome.'},\n",
       "  'comment': \"Note that this term can be used in place of the obsolete cellular component term 'centromere ; GO:0005698'. Use with caution because this term refers to a specific region of the chromosome and not a protein complex.\",\n",
       "  'synonyms': [{'name': 'condensed nuclear chromosome, centromere',\n",
       "    'type': 'related'},\n",
       "   {'name': 'condensed nuclear chromosome, pericentric region',\n",
       "    'type': 'related'},\n",
       "   {'name': 'condensed nuclear chromosome, centric region', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000778', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000781': {'id': 'GO:0000781',\n",
       "  'isObsolete': False,\n",
       "  'name': 'chromosome, telomeric region',\n",
       "  'definition': {'text': 'The terminal region of a linear chromosome that includes the telomeric DNA repeats and associated proteins.'},\n",
       "  'comment': \"Note that this term can be used in place of the obsolete cellular component term 'telomere ; GO:0005696'. Use with caution because this term refers to a specific region of the chromosome and not a protein complex.\",\n",
       "  'synonyms': [{'name': 'telomere', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0031933', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0035390', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0000782', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903823', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0070565', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000784', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006348', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0035012', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000782': {'id': 'GO:0000782',\n",
       "  'isObsolete': False,\n",
       "  'name': 'telomere cap complex',\n",
       "  'definition': {'text': 'A complex of DNA and protein located at the end of a linear chromosome that protects and stabilizes a linear chromosome.'},\n",
       "  'comment': \"Note that this term can be used in place of the obsolete cellular component term 'telomere ; GO:0005696'. Use with caution because this term refers to a specific protein complex and not a region of the chromosome.\",\n",
       "  'children': [{'id': 'GO:0000783', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043769', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016233', 'relation': 'capable_of_part_of'}]},\n",
       " 'GO:0000783': {'id': 'GO:0000783',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear telomere cap complex',\n",
       "  'definition': {'text': 'A complex of DNA and protein located at the end of a linear chromosome in the nucleus that protects and stabilizes a linear chromosome.'},\n",
       "  'comment': \"Note that this term can be used in place of the obsolete cellular component term 'telomere ; GO:0005696'. Use with caution because this term refers to a specific protein complex and not a region of the chromosome.\",\n",
       "  'children': [{'id': 'GO:1990879', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070187', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000784': {'id': 'GO:0000784',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear chromosome, telomeric region',\n",
       "  'definition': {'text': 'The terminal region of a linear nuclear chromosome that includes the telomeric DNA repeats and associated proteins.'},\n",
       "  'comment': \"Note that this term can be used in place of the obsolete cellular component term 'telomere ; GO:0005696'. Use with caution because this term refers to a specific region of the chromosome and not a protein complex.\",\n",
       "  'synonyms': [{'name': 'nuclear chromosome, telomere', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000783', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005724', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000785': {'id': 'GO:0000785',\n",
       "  'isObsolete': False,\n",
       "  'name': 'chromatin',\n",
       "  'definition': {'text': 'The ordered and organized complex of DNA, protein, and sometimes RNA, that forms the chromosome.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20404130'}]},\n",
       "  'comment': 'Chromosomes include parts that are not part of the chromatin.  Examples include the kinetochore.',\n",
       "  'synonyms': [{'name': 'chromosome scaffold', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000792', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030527', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0000786', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000790', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000791', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000789', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035328', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035327', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061638', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000786': {'id': 'GO:0000786',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleosome',\n",
       "  'definition': {'text': 'A complex comprised of DNA wound around a multisubunit core and associated proteins, which forms the primary packing unit of DNA into higher order structures.'},\n",
       "  'children': [{'id': 'GO:0043505', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000787', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000788', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000787': {'id': 'GO:0000787',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytoplasmic nucleosome',\n",
       "  'definition': {'text': 'A complex comprised of DNA wound around a multisubunit core and associated proteins, which forms the primary packing unit of DNA in the cytoplasm into higher order structures.'},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000788': {'id': 'GO:0000788',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear nucleosome',\n",
       "  'definition': {'text': 'A complex comprised of DNA wound around a multisubunit core and associated proteins, which forms the primary packing unit of DNA in the nucleus into higher order structures.'},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000789': {'id': 'GO:0000789',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytoplasmic chromatin',\n",
       "  'definition': {'text': 'The ordered and organized complex of DNA, protein, and sometimes RNA, that forms the chromosome in the cytoplasm.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20404130'}]},\n",
       "  'children': [{'id': 'GO:0000787', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000790': {'id': 'GO:0000790',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear chromatin',\n",
       "  'definition': {'text': 'The ordered and organized complex of DNA, protein, and sometimes RNA, that forms the chromosome in the nucleus.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20404130'}]},\n",
       "  'children': [{'id': 'GO:0070603', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1904834', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001058', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0070823', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001055', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0070822', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001054', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0001057', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0032798', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001056', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0070210', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000788', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005720', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030874', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0034974', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0034967', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1990483', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0070211', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005726', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005719', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000981', 'relation': 'occurs_in'},\n",
       "   {'id': 'GO:0070209', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000791': {'id': 'GO:0000791',\n",
       "  'isObsolete': False,\n",
       "  'name': 'euchromatin',\n",
       "  'definition': {'text': 'A dispersed and relatively uncompacted form of chromatin.'},\n",
       "  'children': [{'id': 'GO:0005719', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000792': {'id': 'GO:0000792',\n",
       "  'isObsolete': False,\n",
       "  'name': 'heterochromatin',\n",
       "  'definition': {'text': 'A compact and highly condensed form of chromatin.'},\n",
       "  'children': [{'id': 'GO:0033553', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033503', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031934', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990421', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035985', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031933', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005720', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005721', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000793': {'id': 'GO:0000793',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed chromosome',\n",
       "  'definition': {'text': 'A highly compacted molecule of DNA and associated proteins resulting in a cytologically distinct structure.'},\n",
       "  'comment': \"Note that this term can be used to annotate gene products that localize to a mitotic chromosome in an organism that undergoes an 'open mitosis' in which the nuclear envelope breaks down during mitosis.\",\n",
       "  'synonyms': [{'name': 'metaphase chromosome', 'type': 'related'},\n",
       "   {'name': 'cytoplasmic mitotic chromosome', 'type': 'related'},\n",
       "   {'name': 'mitotic chromosome', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000779', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000794', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008278', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030915', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000796', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000777', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000794': {'id': 'GO:0000794',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed nuclear chromosome',\n",
       "  'definition': {'text': 'A highly compacted molecule of DNA and associated proteins resulting in a cytologically distinct nuclear chromosome.'},\n",
       "  'comment': \"Note that this term and its children can be used to annotate gene products that localize to a mitotic chromosome in an organism that undergoes a 'closed mitosis' in which the nuclear envelope does not break down during mitosis and for gene products that localize to a meiotic chromosome.\",\n",
       "  'synonyms': [{'name': 'meiotic chromosome', 'type': 'related'},\n",
       "   {'name': 'nuclear mitotic chromosome', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0035808', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0099086', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000780', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0106069', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000798', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030896', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005713', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000799', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005712', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000778', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0036224', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000795': {'id': 'GO:0000795',\n",
       "  'isObsolete': False,\n",
       "  'name': 'synaptonemal complex',\n",
       "  'definition': {'text': 'A proteinaceous scaffold found between homologous chromosomes during meiosis. It consists of 2 lateral elements and a central element, all running parallel to each other.  Transverse filaments connect the lateral elements to the central element.',\n",
       "   'xrefs': [{'dbCode': 'DOI', 'dbId': '10.5772/29752'}]},\n",
       "  'children': [{'id': 'GO:0000802', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000800', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000801', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000796': {'id': 'GO:0000796',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensin complex',\n",
       "  'definition': {'text': 'A multisubunit protein complex that plays a central role in chromosome condensation.'},\n",
       "  'synonyms': [{'name': 'SMC complex', 'type': 'related'},\n",
       "   {'name': '13S condensin complex', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0000797', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061814', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000799', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030261', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0000068', 'relation': 'capable_of_part_of'}]},\n",
       " 'GO:0000797': {'id': 'GO:0000797',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensin core heterodimer',\n",
       "  'definition': {'text': 'The core heterodimer of a condensin complex, a multisubunit protein complex that plays a central role in chromosome condensation.'},\n",
       "  'synonyms': [{'name': 'Smc2-Smc4 complex', 'type': 'narrow'},\n",
       "   {'name': '8S condensin complex', 'type': 'narrow'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000798': {'id': 'GO:0000798',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear cohesin complex',\n",
       "  'definition': {'text': 'A cohesin complex required for cohesion between sister chromatids that remain in the nucleus.'},\n",
       "  'comment': 'Note that this term and its children should be used to annotate gene products found in cohesin complexes in organisms that undergo closed mitosis (i.e. where the nuclear envelope does not break down, as in fungi). For organisms in which the nuclear envelope breaks down during mitosis, the parent should be used.',\n",
       "  'children': [{'id': 'GO:0034991', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034990', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000799': {'id': 'GO:0000799',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear condensin complex',\n",
       "  'definition': {'text': 'A multisubunit protein complex that plays a central role in the condensation of chromosomes that remain in the nucleus.'},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000800': {'id': 'GO:0000800',\n",
       "  'isObsolete': False,\n",
       "  'name': 'lateral element',\n",
       "  'definition': {'text': 'A proteinaceous core found between sister chromatids during meiotic prophase.'},\n",
       "  'comment': 'In species that have a synaptonemal complex, the lateral elements are part of this complex.  S.pombe is an example of a species that lacks a  (canonical) synaptonemal complex, but still has lateral elements.',\n",
       "  'synonyms': [{'name': 'axial element', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000801': {'id': 'GO:0000801',\n",
       "  'isObsolete': False,\n",
       "  'name': 'central element',\n",
       "  'definition': {'text': 'A structural unit of the synaptonemal complex found between the lateral elements.'},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000802': {'id': 'GO:0000802',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transverse filament',\n",
       "  'definition': {'text': 'A structural unit of the synaptonemal complex that spans the regions between the lateral elements and connects them.'},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000803': {'id': 'GO:0000803',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sex chromosome',\n",
       "  'definition': {'text': 'A chromosome involved in sex determination.'},\n",
       "  'children': [{'id': 'GO:0098579', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000807', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000806', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001741', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098577', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000805', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000804', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001739', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0046536', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007530', 'relation': 'capable_of_part_of'}]},\n",
       " 'GO:0000804': {'id': 'GO:0000804',\n",
       "  'isObsolete': False,\n",
       "  'name': 'W chromosome',\n",
       "  'definition': {'text': 'The sex chromosome present in females of species in which the female is the heterogametic sex; generally, the sex chromosome that pairs with the Z chromosome in the heterogametic sex. The W chromosome is absent from the cells of males and present in one copy in the somatic cells of females.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0321000382'},\n",
       "    {'dbCode': 'PMID', 'dbId': '20622855'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000805': {'id': 'GO:0000805',\n",
       "  'isObsolete': False,\n",
       "  'name': 'X chromosome',\n",
       "  'definition': {'text': 'The sex chromosome present in both sexes of species in which the male is the heterogametic sex. Two copies of the X chromosome are present in each somatic cell of females and one copy is present in males.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0582227089'},\n",
       "    {'dbCode': 'PMID', 'dbId': '20622855'},\n",
       "    {'dbCode': 'URL',\n",
       "     'dbId': 'https://en.wikipedia.org/wiki/XY_sex-determination_system'}]},\n",
       "  'children': [{'id': 'GO:0001740', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016456', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000806': {'id': 'GO:0000806',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Y chromosome',\n",
       "  'definition': {'text': 'The sex chromosome present in males of species in which the male is the heterogametic sex; generally, the sex chromosome that pairs with the X chromosome in the heterogametic sex. The Y chromosome is absent from the cells of females and present in one copy in the somatic cells of males.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0582227089'},\n",
       "    {'dbCode': 'PMID', 'dbId': '20622855'},\n",
       "    {'dbCode': 'URL',\n",
       "     'dbId': 'https://en.wikipedia.org/wiki/XY_sex-determination_system'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000807': {'id': 'GO:0000807',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Z chromosome',\n",
       "  'definition': {'text': 'The sex chromosome present in both sexes of species in which the female is the heterogametic sex. Two copies of the Z chromosome are present in each somatic cell of males and one copy is present in females.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0321000382'},\n",
       "    {'dbCode': 'PMID', 'dbId': '20622855'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000808': {'id': 'GO:0000808',\n",
       "  'isObsolete': False,\n",
       "  'name': 'origin recognition complex',\n",
       "  'definition': {'text': 'A multisubunit complex that is located at the replication origins of a chromosome.'},\n",
       "  'synonyms': [{'name': 'origin of replication recognition complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ORC', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000809', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005664', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000809': {'id': 'GO:0000809',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytoplasmic origin of replication recognition complex',\n",
       "  'definition': {'text': 'A multisubunit complex that is located at the replication origins of a chromosome in the cytoplasm.'},\n",
       "  'synonyms': [{'name': 'cytoplasmic ORC', 'type': 'exact'},\n",
       "   {'name': 'prokaryotic ORC', 'type': 'related'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000810': {'id': 'GO:0000810',\n",
       "  'isObsolete': False,\n",
       "  'name': 'diacylglycerol diphosphate phosphatase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: a 1,2-diacyl-sn-glycerol 3-diphosphate + H2O = a 1,2-diacyl-sn-glycerol 3-phosphate + phosphate.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.1.3.81'},\n",
       "    {'dbCode': 'MetaCyc', 'dbId': 'RXN-11277'},\n",
       "    {'dbCode': 'PMID', 'dbId': '8567632'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9452443'}]},\n",
       "  'comment': 'The enzyme entry for EC 3.1.3.81 states that this is a bifunctional enzyme (http://www.chem.qmul.ac.uk/iubmb/enzyme/EC3/1/3/81.html). However, reaction details are only provided for the cumulative enzymatic activity, therefore this term has not been split into two.',\n",
       "  'synonyms': [{'name': 'diacylglycerol pyrophosphate phosphatase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'DGPP phosphohydrolase activity', 'type': 'exact'},\n",
       "   {'name': 'DGPP phosphatase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016462', 'relation': 'is_a'}]},\n",
       " 'GO:0000811': {'id': 'GO:0000811',\n",
       "  'isObsolete': False,\n",
       "  'name': 'GINS complex',\n",
       "  'definition': {'text': 'A heterotetrameric protein complex that associates with replication origins, where it is required for the initiation of DNA replication, and with replication forks.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12730134'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16990792'},\n",
       "    {'dbCode': 'PMID', 'dbId': '17467990'}]},\n",
       "  'synonyms': [{'name': 'Go, Ichi, Ni and San complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000812': {'id': 'GO:0000812',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Swr1 complex',\n",
       "  'definition': {'text': 'A multisubunit protein complex that is involved in chromatin remodeling. It is required for the incorporation of the histone variant H2AZ into chromatin. In S. cerevisiae, the complex contains Swr1p, a Swi2/Snf2-related ATPase, and 12 additional subunits.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14645854'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14690608'},\n",
       "    {'dbCode': 'PMID', 'dbId': '19355820'}]},\n",
       "  'synonyms': [{'name': 'SWR-C', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000813': {'id': 'GO:0000813',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ESCRT I complex',\n",
       "  'definition': {'text': 'An endosomal sorting complex required for transport. It consists of the class E vacuolar protein sorting (Vps) proteins and interacts with ubiquitinated cargoes.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12892785'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12900393'}]},\n",
       "  'synonyms': [{'name': 'endosomal sorting complex required for transport',\n",
       "    'type': 'broad'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000814': {'id': 'GO:0000814',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ESCRT II complex',\n",
       "  'definition': {'text': 'An endosomal sorting complex required for transport and functions downstream of ESCRT I complex. It consists of the class E vacuolar protein sorting (Vps) proteins and is required for the membrane recruitment of ESCRT III complex and binds to ubiquitinated cargoes.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12892785'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12900393'}]},\n",
       "  'synonyms': [{'name': 'endosomal sorting complex required for transport',\n",
       "    'type': 'broad'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000815': {'id': 'GO:0000815',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ESCRT III complex',\n",
       "  'definition': {'text': 'An endosomal sorting complex required for transport. Consists of two soluble subcomplexes of highly charged coiled-coil proteins and is required for sorting and/or concentration of multivesicular body (MVB) cargoes.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12892785'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12900393'}]},\n",
       "  'synonyms': [{'name': 'endosomal sorting complex required for transport',\n",
       "    'type': 'broad'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000817': {'id': 'GO:0000817',\n",
       "  'isObsolete': False,\n",
       "  'name': 'COMA complex',\n",
       "  'definition': {'text': 'A multiprotein complex in yeast consisting of Ctf19p, Okp1p, Mcm21p, and Ame1p. This complex bridges the subunits that are in contact with centromeric DNA and the subunits bound to microtubules during kinetochore assembly.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14633972'}]},\n",
       "  'synonyms': [{'name': 'Ctf19p-Okp1p-Mcm1p-Ame1p complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000818': {'id': 'GO:0000818',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear MIS12/MIND complex',\n",
       "  'definition': {'text': 'A multiprotein kinetochore subcomplex that binds to centromeric chromatin and forms part of the inner kinetochore of a chromosome in the nucleus. It helps to recruit outer kinetochore subunits that will bind to microtubules. Nuclear localization arises in some organisms because the nuclear envelope is not broken down during mitosis. In S. cerevisiae, it consists of at least four proteins: Mtw1p, Nnf1p, Nsl1p, and Dsn1.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14633972'}]},\n",
       "  'synonyms': [{'name': 'Mtw1p Including Nnf1p-Nsl1p-Dsn1p complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'MIND complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000819': {'id': 'GO:0000819',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sister chromatid segregation',\n",
       "  'definition': {'text': 'The cell cycle process in which sister chromatids are organized and then physically separated and apportioned to two or more sets.'},\n",
       "  'children': [{'id': 'GO:0033045', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0007062', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045144', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033046', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0031134', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000070', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0051276', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098813', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007001', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051277', 'relation': 'is_a'}]},\n",
       " 'GO:0000820': {'id': 'GO:0000820',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of glutamine family amino acid metabolic process',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving amino acids of the glutamine family, comprising arginine, glutamate, glutamine and proline.'},\n",
       "  'synonyms': [{'name': 'regulation of glutamine family amino acid metabolism',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:2000211', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2000214', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000821', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0062132', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006521', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009064', 'relation': 'regulates'}]},\n",
       " 'GO:0000821': {'id': 'GO:0000821',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of arginine metabolic process',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving arginine, 2-amino-5-(carbamimidamido)pentanoic acid.',\n",
       "   'xrefs': [{'dbCode': 'CHEBI', 'dbId': '29016'}]},\n",
       "  'synonyms': [{'name': 'regulation of arginine metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1900081', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900079', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000820', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006525', 'relation': 'regulates'}]},\n",
       " 'GO:0000822': {'id': 'GO:0000822',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol hexakisphosphate binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with inositol hexakisphosphate.'},\n",
       "  'synonyms': [{'name': 'InsP6 binding', 'type': 'exact'},\n",
       "   {'name': 'IP6 binding', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008144', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043168', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043178', 'relation': 'is_a'}]},\n",
       " 'GO:0000823': {'id': 'GO:0000823',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol-1,4,5-trisphosphate 6-kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: D-myo-inositol 1,4,5-trisphosphate + ATP = D-myo-inositol 1,4,5,6-tetrakisphosphate + ADP + 2 H(+).',\n",
       "   'xrefs': [{'dbCode': 'MetaCyc', 'dbId': '2.7.1.151-RXN'}]},\n",
       "  'synonyms': [{'name': 'inositol polyphosphate multikinase activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'IpmK', 'type': 'broad'},\n",
       "   {'name': 'ATP:1D-myo-inositol-1,4,5-trisphosphate 6-phosphotransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'inositol trisphosphate 6-kinase activity', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0051766', 'relation': 'is_a'}]},\n",
       " 'GO:0000824': {'id': 'GO:0000824',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol tetrakisphosphate 3-kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + 1D-myo-inositol 1,4,5,6-tetrakisphosphate = ADP + 1D-myo-inositol 1,3,4,5,6-pentakisphosphate.'},\n",
       "  'synonyms': [{'name': '1D-myo-inositol-tetrakisphosphate 3-kinase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'inositol polyphosphate multikinase activity', 'type': 'broad'},\n",
       "   {'name': 'inositol 1,4,5,6-tetrakisphosphate 3-kinase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'IpmK', 'type': 'broad'},\n",
       "   {'name': 'EC:2.7.1.151', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0051765', 'relation': 'is_a'}]},\n",
       " 'GO:0000825': {'id': 'GO:0000825',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol tetrakisphosphate 6-kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 1D-myo-inositol 1,3,4,5-tetrakisphosphate + ATP = 1D-myo-inositol 1,3,4,5,6-pentakisphosphate + ADP.'},\n",
       "  'synonyms': [{'name': '1D-myo-inositol-tetrakisphosphate 6-kinase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'inositol 1,3,4,5-tetrakisphosphate 6-kinase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0051765', 'relation': 'is_a'}]},\n",
       " 'GO:0000827': {'id': 'GO:0000827',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol-1,3,4,5,6-pentakisphosphate kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + 1D-myo-inositol 1,3,4,5,6-pentakisphosphate = ADP + diphospho-1D-myo-inositol tetrakisphosphate. The isomeric configuration of diphospho-1D-myo-inositol tetrakisphosphate is unknown.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11311242'}]},\n",
       "  'synonyms': [{'name': 'EC:2.7.4.21', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016301', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016776', 'relation': 'is_a'}]},\n",
       " 'GO:0000828': {'id': 'GO:0000828',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol hexakisphosphate kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + 1D-myo-inositol 1,2,3,4,5,6-hexakisphosphate = ADP + diphospho-1D-myo-inositol-pentakisphosphate. The isomeric configuration of diphospho-1D-myo-inositol-pentakisphosphate (PP-IP5) is unknown.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16429326'}]},\n",
       "  'children': [{'id': 'GO:0052723', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000830', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000832', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000831', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052724', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016301', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016776', 'relation': 'is_a'}]},\n",
       " 'GO:0000829': {'id': 'GO:0000829',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol heptakisphosphate kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + diphospho-1D-myo-inositol-pentakisphosphate = ADP + bis(diphospho)-1D-myo-inositol-tetrakisphosphate. The isomeric configurations of the diphospho-1D-myo-inositol-pentakisphosphate (PP-IP5) and bis(diphospho)-1D-myo-inositol-tetrakisphosphate (bis-PP-IP4) are unknown.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16429326'}]},\n",
       "  'children': [{'id': 'GO:0000834', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033200', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000833', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016301', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016776', 'relation': 'is_a'}]},\n",
       " 'GO:0000830': {'id': 'GO:0000830',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol hexakisphosphate 4-kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + 1D-myo-inositol hexakisphosphate = ADP + 4-diphospho-1D-myo-inositol (1,2,3,5,6)pentakisphosphate.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16429326'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000828', 'relation': 'is_a'}]},\n",
       " 'GO:0000831': {'id': 'GO:0000831',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol hexakisphosphate 6-kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + 1D-myo-inositol hexakisphosphate = ADP + 6-diphospho-1D-myo-inositol (1,2,3,4,5)pentakisphosphate.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16429326'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000828', 'relation': 'is_a'}]},\n",
       " 'GO:0000832': {'id': 'GO:0000832',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol hexakisphosphate 5-kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + 1D-myo-inositol 1,2,3,4,5,6-hexakisphosphate = ADP + 5-diphospho-1D-myo-inositol (1,2,3,4,6)pentakisphosphate.',\n",
       "   'xrefs': [{'dbCode': 'MetaCyc', 'dbId': '2.7.1.152-RXN'}]},\n",
       "  'synonyms': [{'name': 'ATP:1D-myo-inositol-hexakisphosphate 5-phosphotransferase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000828', 'relation': 'is_a'}]},\n",
       " 'GO:0000833': {'id': 'GO:0000833',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol heptakisphosphate 4-kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + 5-diphospho-1D-myo-inositol (1,2,3,4,6)pentakisphosphate = 4,5-bisdiphosphoinositol-1D-myoinositol (1,2,3,6)tetrakisphosphate.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16429326'}]},\n",
       "  'synonyms': [{'name': 'diphosphoinositol-pentakisphosphate 4-kinase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'IP7 4-kinase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000829', 'relation': 'is_a'}]},\n",
       " 'GO:0000834': {'id': 'GO:0000834',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inositol heptakisphosphate 6-kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + 5-diphospho-1D-myo-inositol (1,2,3,4,6)pentakisphosphate = 5,6-bisdiphosphoinositol-1D-myoinositol (1,2,3,4)tetrakisphosphate.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16429326'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000829', 'relation': 'is_a'}]},\n",
       " 'GO:0000835': {'id': 'GO:0000835',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ER ubiquitin ligase complex',\n",
       "  'definition': {'text': 'A ubiquitin ligase complex found in the ER.'},\n",
       "  'children': [{'id': 'GO:1990389', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000836', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000837', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030433', 'relation': 'capable_of_part_of'}]},\n",
       " 'GO:0000836': {'id': 'GO:0000836',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Hrd1p ubiquitin ligase complex',\n",
       "  'definition': {'text': 'A multiprotein complex that recognizes and ubiquitinates proteins with misfolded luminal and membrane domains during ER-associated protein degradation (ERAD). In S. cerevisiae, this complex contains the ubiquitin ligase Hrd1p. In mammals, this complex contains the ubiquitin ligase HRD1 (Synoviolin) or AMFR (gp78).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16619026'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16873066'},\n",
       "    {'dbCode': 'PMID', 'dbId': '21454652'}]},\n",
       "  'synonyms': [{'name': 'HRD1 ubiquitin ligase complex', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000839', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000838', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000837': {'id': 'GO:0000837',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Doa10p ubiquitin ligase complex',\n",
       "  'definition': {'text': 'A multiprotein complex that recognizes and ubiquitinates membrane proteins with misfolded cytosolic domains during ER-associated protein degradation (ERAD). In S. cerevisiae, this complex contains the ubiquitin ligase Ssm4p/Doa10p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16873066'}]},\n",
       "  'synonyms': [{'name': 'Ssm4p ubiquitin ligase complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000838': {'id': 'GO:0000838',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Hrd1p ubiquitin ligase ERAD-M complex',\n",
       "  'definition': {'text': 'A multiprotein complex that recognizes and ubiquitinates proteins with misfolded membrane domains during ER-associated protein degradation (ERAD). In S. cerevisiae, this complex contains the ubiquitin ligase Hrd1p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16873066'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000839': {'id': 'GO:0000839',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Hrd1p ubiquitin ligase ERAD-L complex',\n",
       "  'definition': {'text': 'A multiprotein complex that recognizes and ubiquitinates proteins with misfolded luminal domains during ER-associated protein degradation (ERAD). In S. cerevisiae, this complex contains the ubiquitin ligase Hrd1p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16873065'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16873066'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000900': {'id': 'GO:0000900',\n",
       "  'isObsolete': False,\n",
       "  'name': 'translation repressor activity, mRNA regulatory element binding',\n",
       "  'definition': {'text': 'Antagonizes the ribosome-mediated translation of mRNA into a polypeptide via direct binding (through a selective and non-covalent interaction) to nucleic acid.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '29061112'},\n",
       "    {'dbCode': 'PMID', 'dbId': '7523370'}]},\n",
       "  'comment': 'Changed term label as requested in https://github.com/geneontology/go-ontology/issues/15049',\n",
       "  'synonyms': [{'name': 'translation repressor activity, nucleic acid binding',\n",
       "    'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030371', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090079', 'relation': 'is_a'}]},\n",
       " 'GO:0000901': {'id': 'GO:0000901',\n",
       "  'isObsolete': False,\n",
       "  'name': 'translation repressor activity, non-nucleic acid binding',\n",
       "  'definition': {'text': 'Antagonizes the ribosome-mediated translation of mRNA into a polypeptide but does not bind directly to nucleic acid.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030371', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045183', 'relation': 'is_a'}]},\n",
       " 'GO:0000902': {'id': 'GO:0000902',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell morphogenesis',\n",
       "  'definition': {'text': 'The developmental process in which the size or shape of a cell is generated and organized.'},\n",
       "  'synonyms': [{'name': 'cellular morphogenesis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0061159', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000767', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000904', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022604', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0009826', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048858', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0032989', 'relation': 'is_a'}]},\n",
       " 'GO:0000904': {'id': 'GO:0000904',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell morphogenesis involved in differentiation',\n",
       "  'definition': {'text': \"The change in form (cell shape and size) that occurs when relatively unspecialized cells, e.g. embryonic or regenerative cells, acquire specialized structural and/or functional features that characterize the cells, tissues, or organs of the mature organism or some other relatively stable phase of the organism's life history.\"},\n",
       "  'children': [{'id': 'GO:0030713', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001828', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034446', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061336', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098508', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010770', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0010090', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070309', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010091', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090171', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010771', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0048667', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048601', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036344', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090378', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010442', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0021943', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010769', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0060880', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003382', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001831', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048766', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048768', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048938', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009860', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000902', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048468', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007148', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045790', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045791', 'relation': 'is_a'}]},\n",
       " 'GO:0000905': {'id': 'GO:0000905',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sporocarp development involved in asexual reproduction',\n",
       "  'definition': {'text': 'The formation of a spore-bearing structure by fungus where spores will arise from asexual reproduction.'},\n",
       "  'synonyms': [{'name': 'fruiting body formation involved in asexual reproduction',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'imperfect stage fruiting body development', 'type': 'narrow'},\n",
       "   {'name': 'homokaryotic fruiting', 'type': 'related'},\n",
       "   {'name': 'haploid fruiting', 'type': 'related'},\n",
       "   {'name': 'monokaryotic fruiting', 'type': 'related'},\n",
       "   {'name': 'conidium development', 'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019954', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030584', 'relation': 'is_a'}]},\n",
       " 'GO:0000906': {'id': 'GO:0000906',\n",
       "  'isObsolete': False,\n",
       "  'name': '6,7-dimethyl-8-ribityllumazine synthase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 3,4-dihydroxy-2-butanone-4-phosphate + 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione = 6,7-dimethyl-8-ribityllumazine + phosphate.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '7559556'}]},\n",
       "  'synonyms': [{'name': 'lumazine synthase activity', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016765', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016766', 'relation': 'is_a'}]},\n",
       " 'GO:0000907': {'id': 'GO:0000907',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sulfonate dioxygenase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: sulfonate + 2-oxoglutarate + O2 = sulfite + aminoacetaldehyde + succinate + CO2.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.14.11.-'},\n",
       "    {'dbCode': 'PMID', 'dbId': '10482536'}]},\n",
       "  'synonyms': [{'name': 'sulphonate dioxygenase activity', 'type': 'exact'},\n",
       "   {'name': 'sulfonate/alpha-ketoglutarate dioxygenase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016706', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010302', 'relation': 'is_a'}]},\n",
       " 'GO:0000908': {'id': 'GO:0000908',\n",
       "  'isObsolete': False,\n",
       "  'name': 'taurine dioxygenase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 2-oxoglutarate + O(2) + taurine = aminoacetaldehyde + CO(2) + succinate + sulfite.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.14.11.17'},\n",
       "    {'dbCode': 'RHEA', 'dbId': '15909'}]},\n",
       "  'synonyms': [{'name': 'taurine, 2-oxoglutarate:O2 oxidoreductase (sulfite-forming)',\n",
       "    'type': 'exact'},\n",
       "   {'name': '2-aminoethanesulfonate dioxygenase activity', 'type': 'exact'},\n",
       "   {'name': 'alpha-ketoglutarate-dependent taurine dioxygenase activity',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1990205', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016706', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010302', 'relation': 'is_a'}]},\n",
       " 'GO:0000909': {'id': 'GO:0000909',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sporocarp development involved in sexual reproduction',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of a fruiting body organ over time, from its formation to the mature structure. The fruiting body is a spore bearing structure. In fungi, the sporocarp (also known as fruiting body) is a multicellular structure on which spore-producing structures, such as basidia or asci, are borne. The fruiting body is part of the sexual phase of a fungal life cycle, with the rest of the life cycle being characterized by vegetative mycelial growth. The sporocarp of a basidiomycete is known as a basidiocarp, while the fruiting body of an ascomycete is known as an ascocarp. A significant range of different shapes and morphologies is found in both basidiocarps and ascocarps; these features play an important role in the identification and taxonomy of fungi.'},\n",
       "  'synonyms': [{'name': 'fruiting body development involved in sexual reproduction',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'fruiting body formation involved in sexual reproduction',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'perfect stage fruiting body development', 'type': 'narrow'},\n",
       "   {'name': 'ascus development', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:1902059', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0070791', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902058', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1902060', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019953', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030584', 'relation': 'is_a'}]},\n",
       " 'GO:0000910': {'id': 'GO:0000910',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytokinesis',\n",
       "  'definition': {'text': 'The division of the cytoplasm and the plasma membrane of a cell and its partitioning into two daughter cells.'},\n",
       "  'comment': \"Note that this term should not be used for direct annotation. When annotating eukaryotic species, mitotic or meiotic cytokinesis should always be specified for manual annotation and for prokaryotic species use 'FtsZ-dependent cytokinesis ; GO:0043093' or Cdv-dependent cytokinesis ; GO:0061639.  Also, note that cytokinesis does not necessarily result in physical separation and detachment of the two daughter cells from each other.\",\n",
       "  'synonyms': [{'name': 'cytokinesis involved in cell cycle', 'type': 'exact'},\n",
       "   {'name': 'cell cycle cytokinesis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0032467', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0000917', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043093', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901901', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000911', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032466', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0032506', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032465', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0061640', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061639', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Electronic',\n",
       "  'parent': [{'id': 'GO:0022402', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051301', 'relation': 'part_of'}]},\n",
       " 'GO:0000911': {'id': 'GO:0000911',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cytokinesis by cell plate formation',\n",
       "  'definition': {'text': 'The process of dividing the cytoplasm of a parent cell where a structure forms in the cytoplasm and grows until reaching the plasma membrane, thereby completely separating the cytoplasms of adjacent progeny cells. An example of this is found in Arabidopsis thaliana.'},\n",
       "  'children': [{'id': 'GO:0010235', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000913', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000914', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000919', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010443', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000910', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033205', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016288', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007104', 'relation': 'is_a'}]},\n",
       " 'GO:0000912': {'id': 'GO:0000912',\n",
       "  'isObsolete': False,\n",
       "  'name': 'assembly of actomyosin apparatus involved in cytokinesis',\n",
       "  'definition': {'text': 'The assembly and arrangement of an apparatus composed of actin, myosin, and associated proteins that will function in cytokinesis.'},\n",
       "  'synonyms': [{'name': 'formation of actomyosin apparatus involved in cytokinesis',\n",
       "    'type': 'related'},\n",
       "   {'name': 'cytokinesis, formation of actomyosin apparatus', 'type': 'exact'},\n",
       "   {'name': 'actomyosin apparatus assembly involved in cytokinesis',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000915', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902407', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0022607', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032506', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061640', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071844', 'relation': 'is_a'}]},\n",
       " 'GO:0000913': {'id': 'GO:0000913',\n",
       "  'isObsolete': False,\n",
       "  'name': 'preprophase band assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of a set of components to form the preprophase band, a dense band of microtubules that marks the position in the cell where cytokinesis will occur in cells that perform cytokinesis by cell plate formation.'},\n",
       "  'synonyms': [{'name': 'preprophase band formation', 'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000911', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022402', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022607', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071844', 'relation': 'is_a'}]},\n",
       " 'GO:0000914': {'id': 'GO:0000914',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phragmoplast assembly',\n",
       "  'definition': {'text': 'The formation of a structure composed of actin, myosin, and associated proteins that will function in cytokinesis in cells that perform cytokinesis by cell plate formation. The structure usually contains antiparallel microtubules and membrane (often visible as vesicles).'},\n",
       "  'synonyms': [{'name': 'phragmoplast formation', 'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000911', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902407', 'relation': 'is_a'}]},\n",
       " 'GO:0000915': {'id': 'GO:0000915',\n",
       "  'isObsolete': False,\n",
       "  'name': 'actomyosin contractile ring assembly',\n",
       "  'definition': {'text': 'The process of assembly of a ring composed of actin, myosin, and associated proteins that will function in cytokinesis.'},\n",
       "  'synonyms': [{'name': 'cytokinesis, actomyosin ring biosynthesis',\n",
       "    'type': 'related'},\n",
       "   {'name': 'contractile ring assembly', 'type': 'exact'},\n",
       "   {'name': 'cytokinesis, actomyosin contractile ring formation',\n",
       "    'type': 'related'},\n",
       "   {'name': 'cytokinesis, contractile ring assembly', 'type': 'related'},\n",
       "   {'name': 'cytokinesis, actomyosin ring formation', 'type': 'related'},\n",
       "   {'name': 'cytokinesis, actomyosin contractile ring assembly',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'constriction ring assembly', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:2000433', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:2000708', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071520', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1903475', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2000431', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000689', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902575', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071519', 'relation': 'part_of'},\n",
       "   {'id': 'GO:2000432', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000912', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044837', 'relation': 'is_a'}]},\n",
       " 'GO:0000916': {'id': 'GO:0000916',\n",
       "  'isObsolete': False,\n",
       "  'name': 'actomyosin contractile ring contraction',\n",
       "  'definition': {'text': 'The process of an actomyosin ring getting smaller in diameter, in the context of cytokinesis that takes place as part of a cell cycle.'},\n",
       "  'synonyms': [{'name': 'contractile ring constriction', 'type': 'broad'},\n",
       "   {'name': 'cytokinesis, actomyosin ring contraction', 'type': 'exact'},\n",
       "   {'name': 'cytokinesis, contractile ring contraction', 'type': 'broad'},\n",
       "   {'name': 'contractile ring contraction involved in cell cycle cytokinesis',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0031991', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1902404', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0036213', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044837', 'relation': 'part_of'}]},\n",
       " 'GO:0000917': {'id': 'GO:0000917',\n",
       "  'isObsolete': False,\n",
       "  'name': 'division septum assembly',\n",
       "  'definition': {'text': 'The assembly and arrangement of a septum that spans the plasma membrane interface between progeny cells following cytokinesis. The progeny cells that form a division septum are not able to exchange intracellular material.'},\n",
       "  'synonyms': [{'name': 'formation of division septum involved in mitotic cell cycle',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'division septum formation involved in mitotic cell cycle',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'mitotic division septum assembly', 'type': 'narrow'},\n",
       "   {'name': 'division septum formation', 'type': 'exact'},\n",
       "   {'name': 'formation of division septum', 'type': 'exact'},\n",
       "   {'name': 'septin assembly and septum formation involved in mitotic cell cycle',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'septin assembly and septum biosynthesis involved in mitotic cell cycle',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'division septum assembly involved in cell cycle cytokinesis involved in mitotic cell cycle',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'division septum formation involved in cell cycle cytokinesis involved in mitotic cell cycle',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'division septum assembly involved in cell cycle cytokinesis',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'division septum formation involved in cell cycle cytokinesis',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'septin assembly and septum formation', 'type': 'related'},\n",
       "   {'name': 'septation', 'type': 'broad'},\n",
       "   {'name': 'septin assembly and septum biosynthesis', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0032955', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0010974', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0140278', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010973', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0000918', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000910', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090529', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033205', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0016288', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007104', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090530', 'relation': 'is_a'}]},\n",
       " 'GO:0000918': {'id': 'GO:0000918',\n",
       "  'isObsolete': False,\n",
       "  'name': 'division septum site selection',\n",
       "  'definition': {'text': 'The process of marking the site where a division septum will form.'},\n",
       "  'synonyms': [{'name': 'selection of site for barrier cell septum formation',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'selection of site for division septum formation',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'selection of site for barrier cell septum biosynthesis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'septum positioning', 'type': 'exact'},\n",
       "   {'name': 'septin assembly and septum biosynthesis', 'type': 'broad'},\n",
       "   {'name': 'septin assembly and septum formation', 'type': 'broad'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000917', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032506', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902411', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071937', 'relation': 'part_of'}]},\n",
       " 'GO:0000919': {'id': 'GO:0000919',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell plate assembly',\n",
       "  'definition': {'text': 'The process of assembly, maturation, and growth of the cell plate to the cell periphery in cells that divide by cell plate formation; often involves deposition of cell wall material in and around the phragmoplast.'},\n",
       "  'synonyms': [{'name': 'cell plate formation', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0009920', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000911', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022402', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022607', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071844', 'relation': 'is_a'}]},\n",
       " 'GO:0000920': {'id': 'GO:0000920',\n",
       "  'isObsolete': False,\n",
       "  'name': 'septum digestion after cytokinesis',\n",
       "  'definition': {'text': 'The process of physically separating the septal cell wall material by enzymatic digestion, that occurs after daughter cells are separated by cytokinesis.'},\n",
       "  'comment': \"This term should not be used to describe the last step of cytokinesis in organisms without a cell wall, ie, the cell resealing of the plasma membrane via abscission. Consider annotating to 'GO:0061952 midbody abscission' to capture this process.\",\n",
       "  'synonyms': [{'name': 'cytokinetic cell separation', 'type': 'related'},\n",
       "   {'name': 'mitotic cytokinetic cell separation', 'type': 'broad'},\n",
       "   {'name': 'cell separation after cytokinesis', 'type': 'related'},\n",
       "   {'name': 'cell separation following cytokinesis', 'type': 'exact'},\n",
       "   {'name': 'daughter cell separation', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:2001043', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1904652', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030995', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030994', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010590', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2001042', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1905391', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009987', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051301', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0050875', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044763', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008151', 'relation': 'is_a'}]},\n",
       " 'GO:0000921': {'id': 'GO:0000921',\n",
       "  'isObsolete': False,\n",
       "  'name': 'septin ring assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of septins and associated proteins to form an organized structure resembling a ring at the cell cortex.'},\n",
       "  'synonyms': [{'name': 'septin assembly and septum biosynthesis',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'septin assembly and septum formation', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:1990317', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0062163', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031106', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034622', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070925', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043623', 'relation': 'is_a'}]},\n",
       " 'GO:0000922': {'id': 'GO:0000922',\n",
       "  'isObsolete': False,\n",
       "  'name': 'spindle pole',\n",
       "  'definition': {'text': 'Either of the ends of a spindle, where spindle microtubules are organized; usually contains a microtubule organizing center and accessory molecules, spindle microtubules and astral microtubules.'},\n",
       "  'children': [{'id': 'GO:0031616', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0097431', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005827', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090619', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000923': {'id': 'GO:0000923',\n",
       "  'isObsolete': False,\n",
       "  'name': 'equatorial microtubule organizing center',\n",
       "  'definition': {'text': 'A microtubule organizing center formed by a band of gamma-tubulin that is recruited to a circumferential band of F-actin at the midpoint of a cell and which nucleates microtubules from the cell division site at the end of mitosis.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11792817'}]},\n",
       "  'synonyms': [{'name': 'EMTOC', 'type': 'exact'},\n",
       "   {'name': 'equatorial microtubule organising centre', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0055031', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000924': {'id': 'GO:0000924',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gamma-tubulin ring complex, centrosomal',\n",
       "  'definition': {'text': 'A centrosome-localized multiprotein complex composed of gamma-tubulin and other non-tubulin proteins assembled into a ring structure that is thought to be the unit of nucleation at the minus end of a microtubule. Gamma-tubulin small complexes are thought to be the core repeating units of the ring.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12134075'},\n",
       "    {'dbCode': 'PMID', 'dbId': '17021256'}]},\n",
       "  'synonyms': [{'name': 'gamma-tubulin large complex, centrosomal',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000927': {'id': 'GO:0000927',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gamma-tubulin small complex, centrosomal',\n",
       "  'definition': {'text': 'A centrosomal complex usually comprising two gamma-tubulin molecules, at least two conserved non-tubulin proteins that multimerize along with additional non-tubulin proteins in animal cells into larger functional complexes. Gamma-tubulin small complexes are thought to be the repeating unit making up the core of the gamma-tubulin ring complex. An example of this structure is found in Mus musculus.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12134075'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000928': {'id': 'GO:0000928',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gamma-tubulin small complex, spindle pole body',\n",
       "  'definition': {'text': 'A complex composed of two gamma-tubulin molecules and conserved non-tubulin proteins located in the spindle pole body and isolated by fractionation from cells. The complex, approximately 6S-9S, is analogous to the small complex in animal cells but contains fewer subunits, and is not thought to multimerize into larger functional units, like complexes in those organisms. An example of this structure is found in Saccharomyces cerevisiae.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12134075'}]},\n",
       "  'synonyms': [{'name': 'Tub4 complex', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0061495', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000930': {'id': 'GO:0000930',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gamma-tubulin complex',\n",
       "  'definition': {'text': 'A multiprotein complex composed of gamma-tubulin and other non-tubulin proteins. Gamma-tubulin complexes are localized to microtubule organizing centers, and play an important role in the nucleation of microtubules. The number and complexity of non-tubulin proteins associated with these complexes varies between species.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12134075'}]},\n",
       "  'children': [{'id': 'GO:0000931', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008275', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000931': {'id': 'GO:0000931',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gamma-tubulin large complex',\n",
       "  'definition': {'text': 'A complex of gamma tubulin and associated proteins thought to be formed by multimerization of gamma-tubulin small complexes. An example of this structure is found in Schizosaccharomyces pombe.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12134075'}]},\n",
       "  'children': [{'id': 'GO:0055033', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055032', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055031', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008274', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000932': {'id': 'GO:0000932',\n",
       "  'isObsolete': False,\n",
       "  'name': 'P-body',\n",
       "  'definition': {'text': 'A focus in the cytoplasm where mRNAs may become inactivated by decapping or some other mechanism. Protein and RNA localized to these foci are involved in mRNA degradation, nonsense-mediated mRNA decay (NMD), translational repression, and RNA-mediated gene silencing.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12730603'}]},\n",
       "  'synonyms': [{'name': 'P body', 'type': 'exact'},\n",
       "   {'name': 'cytoplasmic mRNA processing body', 'type': 'exact'},\n",
       "   {'name': 'cytoplasmic foci', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:1990500', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000933': {'id': 'GO:0000933',\n",
       "  'isObsolete': False,\n",
       "  'name': 'adventitious septum',\n",
       "  'definition': {'text': 'A cell septum whose formation is independent of nuclear division.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0471940526'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000934': {'id': 'GO:0000934',\n",
       "  'isObsolete': False,\n",
       "  'name': 'porous cell septum',\n",
       "  'definition': {'text': 'A septum or cross wall which does not entirely span the space between two portions of cell wall and may contain a specialized central pore structure. A porous septum allows the movement of organelles and/or cytoplasm between compartments.'},\n",
       "  'synonyms': [{'name': 'porous septum', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0000937', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000935': {'id': 'GO:0000935',\n",
       "  'isObsolete': False,\n",
       "  'name': 'division septum',\n",
       "  'definition': {'text': 'A cell septum which forms as part of the division site and functions in the compartmentalization of a cell into two daughter cells at division. A division septum spans a cell and does not allow exchange of organelles or cytoplasm between compartments.'},\n",
       "  'synonyms': [{'name': 'divison septum', 'type': 'exact'},\n",
       "   {'name': 'complete septum', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000936', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051077', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000936': {'id': 'GO:0000936',\n",
       "  'isObsolete': False,\n",
       "  'name': 'primary cell septum',\n",
       "  'definition': {'text': 'A cell septum that forms following nuclear division.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0471940526'}]},\n",
       "  'synonyms': [{'name': 'primary septum', 'type': 'broad'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000937': {'id': 'GO:0000937',\n",
       "  'isObsolete': False,\n",
       "  'name': 'dolipore septum',\n",
       "  'definition': {'text': 'A septum, or cross-wall, between two portions of a cell or hypha; contains a central pore around which the septum is swollen to form a barrel-shaped structure; pore is covered on each side of the septum by a septal pore cap (parenthosome).'},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000938': {'id': 'GO:0000938',\n",
       "  'isObsolete': False,\n",
       "  'name': 'GARP complex',\n",
       "  'definition': {'text': 'A quatrefoil tethering complex required for retrograde traffic from the early endosome back to the late Golgi and biogenesis of cytoplasmic vesicles.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10637310'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12077354'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12446664'}]},\n",
       "  'synonyms': [{'name': 'Vps fifty three tethering complex', 'type': 'exact'},\n",
       "   {'name': 'Golgi associated retrograde protein complex', 'type': 'exact'},\n",
       "   {'name': 'VFT tethering complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000939': {'id': 'GO:0000939',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed chromosome inner kinetochore',\n",
       "  'definition': {'text': 'The region of a condensed chromosome kinetochore closest to centromeric DNA; in mammals the CREST antigens (CENP proteins) are found in this layer; this layer may help define underlying centromeric chromatin structure and position of the kinetochore on the chromosome.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10619130'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11483983'}]},\n",
       "  'synonyms': [{'name': 'inner kinetochore plate', 'type': 'narrow'},\n",
       "   {'name': 'inner kinetochore of condensed chromosome', 'type': 'exact'},\n",
       "   {'name': 'inner centromere core complex', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0031511', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000941', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000940': {'id': 'GO:0000940',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed chromosome outer kinetochore',\n",
       "  'definition': {'text': 'The region of a condensed chromosome kinetochore most external to centromeric DNA; this outer region mediates kinetochore-microtubule interactions.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11483983'}]},\n",
       "  'synonyms': [{'name': 'outer kinetochore plate', 'type': 'narrow'},\n",
       "   {'name': 'outer kinetochore of condensed chromosome', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000942', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000941': {'id': 'GO:0000941',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed nuclear chromosome inner kinetochore',\n",
       "  'definition': {'text': 'The region of a condensed nuclear chromosome kinetochore closest to centromeric DNA; this layer may help define underlying centromeric chromatin structure and position of the kinetochore on the chromosome.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10619130'},\n",
       "    {'dbCode': 'PMID', 'dbId': '1148398'}]},\n",
       "  'synonyms': [{'name': 'inner kinetochore of condensed nuclear chromosome',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000942': {'id': 'GO:0000942',\n",
       "  'isObsolete': False,\n",
       "  'name': 'condensed nuclear chromosome outer kinetochore',\n",
       "  'definition': {'text': 'The region of a condensed nuclear chromosome kinetochore most external to centromeric DNA; this outer region mediates kinetochore-microtubule interactions.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '1148398'}]},\n",
       "  'synonyms': [{'name': 'outer kinetochore of condensed nuclear chromosome',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0042729', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000943': {'id': 'GO:0000943',\n",
       "  'isObsolete': False,\n",
       "  'name': 'retrotransposon nucleocapsid',\n",
       "  'definition': {'text': 'A complex of the retrotransposon RNA genome, reverse transcriptase, integrase, and associated molecules required for reproduction and integration of the retrotransposon into the host genome; the main structural molecule of the nucleocapsid is often a gag protein homolog.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10861903'}]},\n",
       "  'synonyms': [{'name': 'VLP', 'type': 'exact'},\n",
       "   {'name': 'Virus-like particle', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000947': {'id': 'GO:0000947',\n",
       "  'isObsolete': False,\n",
       "  'name': 'amino acid catabolic process to alcohol via Ehrlich pathway',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the catabolism of amino acids to produce alcohols with one carbon less than the starting amino acid. In S. cerevisiae, this is known to occur for leucine, isoleucine, valine, methionine, phenylalanine, tyrosine, or tryptophan. Often referred to as the Ehrlich pathway, these reactions generally occur during fermentation to produce a variety of alcohols, often collectively referred to as fusel alcohols. Depending on the redox state of the cells, carboxylic acid derivatives may be produced instead of alcohols.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18281432'}]},\n",
       "  'children': [{'id': 'GO:0000949', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000950', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000951', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000955', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046165', 'relation': 'is_a'}]},\n",
       " 'GO:0000948': {'id': 'GO:0000948',\n",
       "  'isObsolete': False,\n",
       "  'name': 'amino acid catabolic process to carboxylic acid via Ehrlich pathway',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the catabolism of amino acids to produce carboxylic acids with one carbon less than the starting amino acid. In S. cerevisiae, this is known to occur for leucine, isoleucine, valine, methionine, phenylalanine, tyrosine, or tryptophan. Often referred to as the Ehrlich pathway, these reactions generally occur during fermentation to produce a variety of carboxylic acids, sometimes collectively referred to as fusel acids. Depending on the redox state of the cells, alcohol derivatives may be produced instead of carboxylic acids.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18281432'}]},\n",
       "  'children': [{'id': 'GO:0000953', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000954', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000952', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000955', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046394', 'relation': 'is_a'}]},\n",
       " 'GO:0000949': {'id': 'GO:0000949',\n",
       "  'isObsolete': False,\n",
       "  'name': 'aromatic amino acid family catabolic process to alcohol via Ehrlich pathway',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the catabolism of aromatic amino acids to produce aromatic alcohols with one carbon less than the starting amino acid. In S. cerevisiae, this is known to occur for leucine, isoleucine, valine, methionine, phenylalanine, tyrosine, or tryptophan. When an aromatic family amino acid, phenylalanine, tyrosine, or tryptophan, is used as the substrate, 2-phenylethanol, 4-hydroxyphenylethanol, or tryptophol, respectively, is produced. Often referred to as the Ehrlich pathway, these reactions generally occur during fermentation to produce a variety of alcohols, often collectively referred to as fusel alcohols. Depending on the redox state of the cells, carboxylic acid derivatives may be produced instead of alcohols.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18281432'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000947', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009074', 'relation': 'is_a'}]},\n",
       " 'GO:0000950': {'id': 'GO:0000950',\n",
       "  'isObsolete': False,\n",
       "  'name': 'branched-chain amino acid catabolic process to alcohol via Ehrlich pathway',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the catabolism of branched chain amino acids to produce branched chain alcohols with one carbon less than the starting amino acid. In S. cerevisiae, this is known to occur for leucine, isoleucine, valine, methionine, phenylalanine, tyrosine, or tryptophan. When a branched chain family amino acid, leucine, isoleucine, or valine, is used as the substrate, 3-methylbutanol, 2-methylbutanol, or 2-methylpropanol, respectively, is produced. Often referred to as the Ehrlich pathway, these reactions generally occur during fermentation to produce a variety of alcohols, often collectively referred to as fusel alcohols. Depending on the redox state of the cells, carboxylic acid derivatives may be produced instead of alcohols.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18281432'}]},\n",
       "  'synonyms': [{'name': 'branched chain family amino acid catabolic process to alcohol via Ehrlich pathway',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000947', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009083', 'relation': 'is_a'}]},\n",
       " 'GO:0000951': {'id': 'GO:0000951',\n",
       "  'isObsolete': False,\n",
       "  'name': 'methionine catabolic process to 3-methylthiopropanol',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the catabolism of branched chain amino acids to produce branched chain alcohols with one carbon less than the starting amino acid. In S. cerevisiae, this is known to occur for leucine, isoleucine, valine, methionine, phenylalanine, tyrosine, or tryptophan. When methionine is used as the substrate, 3-methylthiopropanol is produced. Often referred to as the Ehrlich pathway, these reactions generally occur during fermentation to produce a variety of alcohols, often collectively referred to as fusel alcohols. Depending on the redox state of the cells, carboxylic acid derivatives may be produced instead of alcohols.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18281432'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000947', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009087', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044272', 'relation': 'is_a'}]},\n",
       " 'GO:0000952': {'id': 'GO:0000952',\n",
       "  'isObsolete': False,\n",
       "  'name': 'aromatic amino acid family catabolic process to carboxylic acid via Ehrlich pathway',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the catabolism of amino acids to produce carboxylic acids with one carbon less than the starting amino acid. In S. cerevisiae, this is known to occur for leucine, isoleucine, valine, methionine, phenylalanine, tyrosine, or tryptophan. When an aromatic family amino acid, phenylalanine, tyrosine, or tryptophan, is used as the substrate, 2-phenylethanoate, 4-hydroxyphenylethanoate, or 2-(Indol-3-yl)-ethanoate, respectively, is produced. Often referred to as the Ehrlich pathway, these reactions generally occur during fermentation to produce a variety of carboxylic acids, sometimes collectively referred to as fusel acids. Depending on the redox state of the cells, alcohol derivatives may be produced instead of carboxylic acids.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18281432'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000948', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009074', 'relation': 'is_a'}]},\n",
       " 'GO:0000953': {'id': 'GO:0000953',\n",
       "  'isObsolete': False,\n",
       "  'name': 'branched-chain amino acid catabolic process to carboxylic acid via Ehrlich pathway',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the catabolism of amino acids to produce carboxylic acids with one carbon less than the starting amino acid. In S. cerevisiae, this is known to occur for leucine, isoleucine, valine, methionine, phenylalanine, tyrosine, or tryptophan. When a branched chain family amino acid, leucine, isoleucine, or valine, is used as the substrate, 3-methylbutanoate, 2-methylbutanoate, or 2-methylpropanoate, respectively, is produced. Often referred to as the Ehrlich pathway, these reactions generally occur during fermentation to produce a variety of carboxylic acids, sometimes collectively referred to as fusel acids. Depending on the redox state of the cells, alcohol derivatives may be produced instead of carboxylic acids.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18281432'}]},\n",
       "  'synonyms': [{'name': 'branched chain family amino acid catabolic process to carboxylic acid via Ehrlich pathway',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000948', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009083', 'relation': 'is_a'}]},\n",
       " 'GO:0000954': {'id': 'GO:0000954',\n",
       "  'isObsolete': False,\n",
       "  'name': 'methionine catabolic process to 3-methylthiopropanoate',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the catabolism of amino acids to produce carboxylic acids with one carbon less than the starting amino acid. In S. cerevisiae, this is known to occur for leucine, isoleucine, valine, methionine, phenylalanine, tyrosine, or tryptophan. When methionine is used as the substrate, 3-methylthiopropanoate is produced. Often referred to as the Ehrlich pathway, these reactions generally occur during fermentation to produce a variety of carboxylic acids, sometimes collectively referred to as fusel acids. Depending on the redox state of the cells, alcohol derivatives may be produced instead of carboxylic acids.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18281432'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000948', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006633', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009087', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009403', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044272', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000037', 'relation': 'is_a'}]},\n",
       " 'GO:0000955': {'id': 'GO:0000955',\n",
       "  'isObsolete': False,\n",
       "  'name': 'amino acid catabolic process via Ehrlich pathway',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving the catabolism of amino acids to produce alcohols or carboxylic acids containing one carbon less than the starting amino acid. In S. cerevisiae, this is known to occur for leucine, isoleucine, valine, methionine, phenylalanine, tyrosine, or tryptophan. Often referred to as the Ehrlich pathway, these reactions generally occur during fermentation to produce a variety of alcohols, often collectively referred to as fusel alcohols. Depending on the redox state of the cells, carboxylic acid derivatives, sometimes referred to as fusel acids, may be produced instead of alcohols.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18281432'}]},\n",
       "  'children': [{'id': 'GO:0000947', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000948', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009063', 'relation': 'is_a'}]},\n",
       " 'GO:0000956': {'id': 'GO:0000956',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nuclear-transcribed mRNA catabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of nuclear-transcribed mRNAs in eukaryotic cells.'},\n",
       "  'synonyms': [{'name': 'nuclear mRNA catabolism', 'type': 'exact'},\n",
       "   {'name': 'nuclear mRNA degradation', 'type': 'exact'},\n",
       "   {'name': 'nuclear mRNA breakdown', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0031086', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000184', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071028', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000291', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000290', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0106095', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0000289', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070966', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000288', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071044', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000294', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033621', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070481', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006402', 'relation': 'is_a'}]},\n",
       " 'GO:0000957': {'id': 'GO:0000957',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial RNA catabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of RNA transcribed from the mitochondrial genome and occurring in the mitochondrion.'},\n",
       "  'children': [{'id': 'GO:2000827', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000958', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000960', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0000962', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0000961', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000959', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006401', 'relation': 'is_a'}]},\n",
       " 'GO:0000958': {'id': 'GO:0000958',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial mRNA catabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of mRNA transcribed from the mitochondrial genome and occurring in the mitochondrion.'},\n",
       "  'children': [{'id': 'GO:1905638', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1905637', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1905639', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000957', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006402', 'relation': 'is_a'}]},\n",
       " 'GO:0000959': {'id': 'GO:0000959',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial RNA metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving RNA transcribed from the mitochondrial genome and occurring in the mitochondrion.'},\n",
       "  'children': [{'id': 'GO:0000957', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006391', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900864', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006392', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006390', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006393', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070127', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000963', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016070', 'relation': 'is_a'}]},\n",
       " 'GO:0000960': {'id': 'GO:0000960',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of mitochondrial RNA catabolic process',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving catabolism in the mitochondrion of RNA transcribed from the mitochondrial genome.'},\n",
       "  'children': [{'id': 'GO:0000961', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000962', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905637', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000957', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0031329', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051252', 'relation': 'is_a'}]},\n",
       " 'GO:0000961': {'id': 'GO:0000961',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative regulation of mitochondrial RNA catabolic process',\n",
       "  'definition': {'text': 'Any process that stops, prevents, or reduces the frequency, rate or extent of the chemical reactions and pathways involving catabolism in the mitochondrion of RNA transcribed from the mitochondrial genome.'},\n",
       "  'children': [{'id': 'GO:1905638', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000957', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0000960', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902369', 'relation': 'is_a'}]},\n",
       " 'GO:0000962': {'id': 'GO:0000962',\n",
       "  'isObsolete': False,\n",
       "  'name': 'positive regulation of mitochondrial RNA catabolic process',\n",
       "  'definition': {'text': 'Any process that activates or increases the frequency, rate or extent of the chemical reactions and pathways involving catabolism in the mitochondrion of RNA transcribed from the mitochondrial genome.'},\n",
       "  'children': [{'id': 'GO:1905639', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000957', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0000960', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031331', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051254', 'relation': 'is_a'}]},\n",
       " 'GO:0000963': {'id': 'GO:0000963',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial RNA processing',\n",
       "  'definition': {'text': 'The conversion of a primary RNA molecule transcribed from a mitochondrial genome into one or more mature RNA molecules; occurs in the mitochondrion.'},\n",
       "  'children': [{'id': 'GO:0000964', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000965', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072684', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090646', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090615', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000959', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006396', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140053', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006394', 'relation': 'is_a'}]},\n",
       " 'GO:0000964': {'id': 'GO:0000964',\n",
       "  'isObsolete': False,\n",
       "  'name': \"mitochondrial RNA 5'-end processing\",\n",
       "  'definition': {'text': \"Any process involved in forming the mature 5' end of an RNA molecule transcribed from a mitochondrial genome; occurs in the mitochondrion.\"},\n",
       "  'synonyms': [{'name': \"mitochondrial RNA 5' end processing\",\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0097745', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000963', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000966', 'relation': 'is_a'}]},\n",
       " 'GO:0000965': {'id': 'GO:0000965',\n",
       "  'isObsolete': False,\n",
       "  'name': \"mitochondrial RNA 3'-end processing\",\n",
       "  'definition': {'text': \"Any process involved in forming the mature 3' end of an RNA molecule transcribed from a mitochondrial genome; occurs in the mitochondrion.\"},\n",
       "  'synonyms': [{'name': \"mitochondrial RNA 3' end processing\",\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:1990180', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090616', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000963', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031123', 'relation': 'is_a'}]},\n",
       " 'GO:0000966': {'id': 'GO:0000966',\n",
       "  'isObsolete': False,\n",
       "  'name': \"RNA 5'-end processing\",\n",
       "  'definition': {'text': \"Any process involved in forming the mature 5' end of an RNA molecule.\"},\n",
       "  'synonyms': [{'name': \"RNA 5' end processing\", 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000964', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034471', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006396', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006394', 'relation': 'is_a'}]},\n",
       " 'GO:0000967': {'id': 'GO:0000967',\n",
       "  'isObsolete': False,\n",
       "  'name': \"rRNA 5'-end processing\",\n",
       "  'definition': {'text': \"Any process involved in forming the mature 5' end of an rRNA molecule.\"},\n",
       "  'synonyms': [{'name': \"rRNA 5' end processing\", 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000472', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000465', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000477', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006364', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034471', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006365', 'relation': 'is_a'}]},\n",
       " 'GO:0000968': {'id': 'GO:0000968',\n",
       "  'isObsolete': False,\n",
       "  'name': 'tRNA exon ligation',\n",
       "  'definition': {'text': 'An RNA exon ligation process that rejoins two exons of a pre-tRNA which has had the intron removed.'},\n",
       "  'children': [{'id': 'GO:0000969', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072669', 'relation': 'capable_of_part_of'},\n",
       "   {'id': 'GO:0000970', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000971', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000378', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006388', 'relation': 'part_of'}]},\n",
       " 'GO:0000969': {'id': 'GO:0000969',\n",
       "  'isObsolete': False,\n",
       "  'name': 'tRNA exon ligation utilizing ATP as source of linkage phosphate',\n",
       "  'definition': {'text': \"A tRNA exon ligation process in which the splice junction phosphate is derived from exogenous ATP. This type of ligation to rejoin the 5' and 3' exons of a tRNA is observed in vertebrate species.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17786051'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000968', 'relation': 'is_a'}]},\n",
       " 'GO:0000970': {'id': 'GO:0000970',\n",
       "  'isObsolete': False,\n",
       "  'name': 'tRNA exon ligation utilizing GTP as source of linkage phosphate',\n",
       "  'definition': {'text': \"A tRNA exon ligation process in which the splice junction phosphate is derived from exogenous GTP. This type of ligation to rejoin the 5' and 3' exons of a tRNA is observed in the yeast Saccharomyces cerevisiae where the ligation reaction also produces a 2'-phosphate at the splice junction which is subsequently removed as part of the ligation process.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18217203'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9299409'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000968', 'relation': 'is_a'}]},\n",
       " 'GO:0000971': {'id': 'GO:0000971',\n",
       "  'isObsolete': False,\n",
       "  'name': \"tRNA exon ligation utilizing 2',3' cyclic phosphate of 5'-exon as source of linkage phosphate\",\n",
       "  'definition': {'text': \"A tRNA exon ligation process in which the splice junction phosphate is derived from the 2',3' cyclic phosphate at the 3'-end of the 5'-exon. This type of ligation to rejoin the 5' and 3' exons of a tRNA is observed in wheat, Chlamydomonas, and vertebrate species including humans.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17786051'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18217203'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9299409'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000968', 'relation': 'is_a'}]},\n",
       " 'GO:0000972': {'id': 'GO:0000972',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery',\n",
       "  'definition': {'text': \"The chromosome organization process in which the DNA sequence containing a gene transcribed by RNA polymerase II is maintained in a specific location at the nuclear periphery. In S. cerevisiae, this process involves cis-acting DNA sequences such as the TATA box and upstream activating sequence (UAS) elements, trans-acting transcriptional activators, and also the 3'-UTR of the transcript.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18614049'}]},\n",
       "  'children': [{'id': 'GO:0000973', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0051276', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007001', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051277', 'relation': 'is_a'}]},\n",
       " 'GO:0000973': {'id': 'GO:0000973',\n",
       "  'isObsolete': False,\n",
       "  'name': 'posttranscriptional tethering of RNA polymerase II gene DNA at nuclear periphery',\n",
       "  'definition': {'text': 'The chromosome organization process in which the DNA sequence containing a gene transcribed by RNA polymerase II is maintained in a specific location at the nuclear periphery even after transcription has been repressed.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17373856'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18614049'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000972', 'relation': 'is_a'}]},\n",
       " 'GO:0000974': {'id': 'GO:0000974',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Prp19 complex',\n",
       "  'definition': {'text': 'A protein complex consisting of Prp19 and associated proteins that is involved in the transition from the precatalytic spliceosome to the activated form that catalyzes step 1 of splicing, and which remains associated with the spliceosome through the second catalytic step. It is widely conserved, found in both yeast and mammals, though the exact composition varies. In S. cerevisiae, it contains Prp19p, Ntc20p, Snt309p, Isy1p, Syf2p, Cwc2p, Prp46p, Clf1p, Cef1p, and Syf1p.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16540691'},\n",
       "    {'dbCode': 'PMID', 'dbId': '19239890'}]},\n",
       "  'synonyms': [{'name': 'NTC', 'type': 'related'},\n",
       "   {'name': 'nineteen complex', 'type': 'narrow'},\n",
       "   {'name': 'Prp19/CDC5 complex', 'type': 'narrow'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0000976': {'id': 'GO:0000976',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific sequence of DNA that is part of a regulatory region that controls transcription of that section of the DNA. The transcribed region might be described as a gene, cistron, or operon.'},\n",
       "  'comment': 'The word \"promoter\" is used variously in the literature to describe the core promoter specifically or the entire proximal regulatory region (excluding any distal enhancers) including both the core promoter and the upstream region where activating transcription factors such as Gal4 in S. cerevisiae or catabolite activator protein (CAP) in E. coli bind. To minimize ambiguity in the use of the word \"promoter\" in GO, we have chosen the phrase \"transcription regulatory region\" in order to refer to all of the regulatory regions. Regulatory regions in the DNA which control initiation may include the \"core promoter\" where the basal transcription machinery binds, the \"core promoter proximal region\" where regulatory factors other than the basal machinery bind, and \"enhancer\" regions which are typically more distal from the core promoter. There are also additional regulatory regions, in both the DNA and the RNA transcript, which regulate elongation or termination of transcription. ANNOTATION NOTE: Regarding annotation to \"transcription regulatory region DNA binding\" (GO:0044212) and any of its is_a children, note that annotation to these terms specifies DNA binding only without any statement about transcription factor activity. To make an annotation about a function of transcription factor activity, consider \"sequence-specific DNA-binding transcription factor activity\" (GO:0003700) or its is_a children which have has_part relationships to the appropriate kind of \"transcription regulatory region DNA binding\".',\n",
       "  'children': [{'id': 'GO:0000984', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000987', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001147', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000977', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001046', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044212', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990837', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000975', 'relation': 'is_a'}]},\n",
       " 'GO:0000977': {'id': 'GO:0000977',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific sequence of DNA that is part of a regulatory region that controls the transcription of a gene or cistron by RNA polymerase II.'},\n",
       "  'comment': 'The word \"promoter\" is used variously in the literature to describe the core promoter specifically or the entire proximal regulatory region (excluding any distal enhancers) including both the core promoter and the upstream region where activating transcription factors such as Gal4 in S. cerevisiae or catabolite activator protein (CAP) in E. coli bind. To minimize ambiguity in the use of the word \"promoter\" in GO, we have chosen the phrase \"transcription regulatory region\" in order to refer to all of the regulatory regions. Regulatory regions in the DNA which control initiation may include the \"core promoter\" where the basal transcription machinery binds, the \"core promoter proximal region\" where regulatory factors other than the basal machinery bind, and \"enhancer\" regions which are typically more distal from the core promoter. There are also additional regulatory regions, in both the DNA and the RNA transcript, which regulate elongation or termination of transcription. ANNOTATION NOTE: Regarding annotation to \"transcription regulatory region DNA binding\" (GO:0044212) and any of its is_a children, note that annotation to these terms specifies DNA binding only without any statement about transcription factor activity. To make an annotation about a function of transcription factor activity, consider \"sequence-specific DNA-binding transcription factor activity\" (GO:0003700) or its is_a children which have has_part relationships to the appropriate kind of \"transcription regulatory region DNA binding\".',\n",
       "  'children': [{'id': 'GO:1903026', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1990647', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:1990590', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:1990907', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0000978', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903025', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1905636', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0000979', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990589', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0001162', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990617', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:1990378', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000976', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001012', 'relation': 'is_a'}]},\n",
       " 'GO:0000978': {'id': 'GO:0000978',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II cis-regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific upstream regulatory DNA sequence (transcription factor recognition sequence or binding site) located in cis relative to the transcription start site (i.e., on the same strand of DNA) of a gene transcribed by RNA polymerase II.'},\n",
       "  'comment': 'Note that the phrase \"upstream activating sequence\", or UAS is often used in S. cerevisiae literature to refer to cis-regulatory sequences. In bacteria such as E. coli, the phrase \"upstream activating sequence\", or UAS is usually a synonym for \"enhancer\".',\n",
       "  'synonyms': [{'name': 'RNA polymerase II proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA polymerase II promoter proximal region sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA polymerase II core promoter proximal region sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA polymerase II upstream activating sequence (UAS) sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA polymerase II distal enhancer sequence-specific DNA binding',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0010736', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070594', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070888', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044323', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035538', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035497', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034056', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044377', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032810', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071820', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070644', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000977', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000987', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001159', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001158', 'relation': 'is_a'}]},\n",
       " 'GO:0000979': {'id': 'GO:0000979',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II core promoter sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a DNA sequence that is part of the core promoter of a RNA polymerase II-transcribed gene.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381658'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000977', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001046', 'relation': 'is_a'}]},\n",
       " 'GO:0000981': {'id': 'GO:0000981',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA-binding transcription factor activity, RNA polymerase II-specific',\n",
       "  'definition': {'text': 'A protein or a member of a complex that interacts selectively and non-covalently with a specific DNA sequence (sometimes referred to as a motif) within the regulatory region of a RNA polymerase II-transcribed gene to modulate transcription. Regulatory regions include promoters (proximal and distal) and enhancers. Genes are transcriptional units.'},\n",
       "  'comment': 'Merged terms see https://github.com/geneontology/go-ontology/issues/15530 and https://github.com/geneontology/go-ontology/issues/16152; changed term label and definition https://github.com/geneontology/go-ontology/issues/16131',\n",
       "  'synonyms': [{'name': 'RNA polymerase II transcription factor activity, metal ion regulated core promoter proximal region sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II core promoter proximal region sequence-specific DNA binding transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase II core promoter proximal region sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor', 'type': 'broad'},\n",
       "   {'name': 'sequence-specific transcription regulatory region DNA binding RNA polymerase II transcription factor recruiting transcription factor activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'sequence-specific distal enhancer binding RNA polymerase II transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase II distal enhancer sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase II proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II distal enhancer sequence-specific DNA binding transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcription factor activity, zinc ion regulated proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'sequence-specific DNA binding RNA polymerase II transcription factor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'RNA polymerase II transcription factor activity, zinc ion regulated core promoter proximal region sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'copper ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcription factor activity, copper ion regulated proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcription factor activity, copper ion regulated core promoter proximal region sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcription factor activity, sequence-specific transcription regulatory region DNA binding',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'RNA polymerase II transcription factor activity, sequence-specific DNA binding',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'zinc ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcription factor activity, metal ion regulated proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcription factor activity, metal ion regulated sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'metal ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'metal ion regulated sequence-specific DNA binding RNA polymerase II transcription factor activity',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:1990378', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:1990647', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0004879', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990513', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0001227', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001228', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003700', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006357', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001151', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001071', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001131', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000130', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001199', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001130', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001204', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006358', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010551', 'relation': 'part_of'}]},\n",
       " 'GO:0000984': {'id': 'GO:0000984',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial-type RNA polymerase regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a sequence of DNA that is part of a regulatory region that controls the transcription of a gene or operon by a bacterial-type RNA polymerase.'},\n",
       "  'comment': 'The word \"promoter\" is used variously in the literature to describe the core promoter specifically or the entire proximal regulatory region (excluding any distal enhancers) including both the core promoter and the upstream region where activating transcription factors such as Gal4 in S. cerevisiae or catabolite activator protein (CAP) in E. coli bind. To minimize ambiguity in the use of the word \"promoter\" in GO, we have chosen the phrase \"transcription regulatory region\" in order to refer to all of the regulatory regions. Regulatory regions in the DNA which control initiation may include the \"core promoter\" where the basal transcription machinery binds and the \" promoter cis-regulatory region\" where regulatory factors other than the basal machinery bind. There are also additional regulatory regions, in both the DNA and the RNA transcript, which regulate elongation or termination of transcription. ANNOTATION NOTE: Regarding annotation to \"transcription regulatory region DNA binding\" (GO:0044212) and any of its is_a children, note that annotation to these terms specifies DNA binding only without any statement about transcription factor activity. To make an annotation about a function of transcription factor activity, consider \"sequence-specific DNA-binding transcription factor activity\" (GO:0003700) or its is_a children which have has_part relationships to the appropriate kind of \"transcription regulatory region DNA binding\".',\n",
       "  'synonyms': [{'name': 'eubacterial-type RNA polymerase regulatory region sequence-specific DNA binding',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001148', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000985', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000986', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000976', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001017', 'relation': 'is_a'}]},\n",
       " 'GO:0000985': {'id': 'GO:0000985',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial-type RNA polymerase core promoter sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a DNA sequence that is part of the core promoter of a bacterial gene.'},\n",
       "  'synonyms': [{'name': 'eubacterial-type RNA polymerase core promoter sequence-specific DNA binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000984', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001046', 'relation': 'is_a'}]},\n",
       " 'GO:0000986': {'id': 'GO:0000986',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial-type cis-regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific upstream cis-regulatory DNA sequence (transcription factor recognition sequence or binding site) located in the promoter of a bacterial gene (or cistron, or operon).'},\n",
       "  'synonyms': [{'name': 'bacterial-type RNA polymerase upstream activating sequence (UAS) sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'bacterial-type proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'bacterial-type RNA polymerase enhancer sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'eubacterial-type RNA polymerase regulatory transcription factor sequence-specific DNA binding',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'bacterial-type RNA polymerase core promoter proximal region sequence-specific DNA binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000984', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000987', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001159', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001158', 'relation': 'is_a'}]},\n",
       " 'GO:0000987': {'id': 'GO:0000987',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cis-regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific upstream regulatory DNA sequence (transcription factor recognition sequence or binding site) located in cis relative to the transcription start site (i.e., on the same strand of DNA) of a gene transcribed by some RNA polymerase. The proximal promoter is in cis with and relatively close to the core promoter.'},\n",
       "  'synonyms': [{'name': 'proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'core promoter proximal region DNA binding', 'type': 'narrow'},\n",
       "   {'name': 'promoter proximal region sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'core promoter proximal region sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'enhancer sequence-specific DNA binding', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0000992', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000986', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000978', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000976', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035326', 'relation': 'is_a'}]},\n",
       " 'GO:0000992': {'id': 'GO:0000992',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III cis-regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific upstream regulatory DNA sequence (transcription factor recognition sequence or binding site) located in cis relative to the transcription start site (i.e., on the same strand of DNA) of a gene transcribed by RNA polymerase III. The transcribed region might be contain a single gene or a cistron containing multiple genes.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'comment': 'To minimize ambiguity in the use of the word \"promoter\" in GO, we have chosen the phrase \"transcription regulatory region\" in order to refer to all of the regulatory regions. Regulatory regions in the DNA which control initiation may include the \"core promoter\" where the basal transcription machinery binds, the \"core promoter proximal region\" where regulatory factors other than the basal machinery bind, and \"enhancer\" regions which are typically more distal from the core promoter. There are also additional regulatory regions, in both the DNA and the RNA transcript, which regulate elongation or termination of transcription. ANNOTATION NOTE: Regarding annotation to \"transcription regulatory region DNA binding\" (GO:0044212) and any of its is_a children, note that annotation to these terms specifies DNA binding only without any statement about transcription factor activity. To make an annotation about a function of transcription factor activity, consider \"sequence-specific DNA-binding transcription factor activity\" (GO:0003700) or its is_a children which have has_part relationships to the appropriate kind of \"transcription regulatory region DNA binding\".',\n",
       "  'children': [{'id': 'GO:0001002', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001003', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001039', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001006', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000987', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001016', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001159', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001158', 'relation': 'is_a'}]},\n",
       " 'GO:0000993': {'id': 'GO:0000993',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II complex binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits.'},\n",
       "  'synonyms': [{'name': 'RNAP II core binding', 'type': 'exact'},\n",
       "   {'name': 'RNA polymerase II core binding', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0099122', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001139', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001099', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043175', 'relation': 'is_a'}]},\n",
       " 'GO:0000994': {'id': 'GO:0000994',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III core binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with RNA polymerase III core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of seventeen subunits.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0043175', 'relation': 'is_a'}]},\n",
       " 'GO:0000995': {'id': 'GO:0000995',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III general transcription initiation factor activity',\n",
       "  'definition': {'text': 'An activity that contributes to transcription start site selection and transcription initiation of genes transcribed by RNA polymerase III. Factors required for RNA polymerase III transcription initiation include TFIIIA, TFIIIB and TFIIIC.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'},\n",
       "    {'dbCode': 'Wikipedia', 'dbId': 'RNA_polymerase_III'}]},\n",
       "  'comment': 'Changed label, definition, see https://github.com/geneontology/go-ontology/issues/15789',\n",
       "  'synonyms': [{'name': 'core RNA polymerase III binding transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'sequence-specific DNA binding RNA polymerase III transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase III general initiation factor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcription factor activity, core RNA polymerase III binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase III transcription factor activity, sequence-specific DNA binding',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006383', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140223', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001035', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001009', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001041', 'relation': 'part_of'}]},\n",
       " 'GO:0001000': {'id': 'GO:0001000',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial-type RNA polymerase core enzyme binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with the bacterial-type RNA polymerase core enzyme, typically consisting of two alpha, one beta, one beta prime, and one omega subunit.'},\n",
       "  'comment': 'Should omega be included here?',\n",
       "  'synonyms': [{'name': 'eubacterial-type RNA polymerase core enzyme binding',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001052', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0043175', 'relation': 'is_a'}]},\n",
       " 'GO:0001001': {'id': 'GO:0001001',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial single-subunit type RNA polymerase binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a single subunit mitochondrial RNA polymerase enzyme, which is composed of a single catalytic subunit similar to the RNA polymerase enzymes from phages T3, T7, and SP6.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20701995'},\n",
       "    {'dbCode': 'PMID', 'dbId': '2088182'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001050', 'relation': 'is_a'}]},\n",
       " 'GO:0001002': {'id': 'GO:0001002',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III type 1 promoter sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a sequence of DNA that is a part of a type 1 promoter that controls transcription by RNA polymerase III. Type 1 promoters are found in 5S rRNA genes, downstream of the transcription start site within the sequence of the mature RNA, and require TFIIIA for recognition.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000992', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001030', 'relation': 'is_a'}]},\n",
       " 'GO:0001003': {'id': 'GO:0001003',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III type 2 promoter sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a sequence of DNA that is a part of a type 2 promoter that controls transcription by RNA polymerase III. Type 2 promoters consist of an A box and a B box downstream of the transcription start site within the sequence within the sequence of the mature RNA. Type 2 promoters are found in many tRNA genes as well as in other small RNAs.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000992', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001031', 'relation': 'is_a'}]},\n",
       " 'GO:0001004': {'id': 'GO:0001004',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III transcription regulator recruiting activity',\n",
       "  'definition': {'text': 'Initiating the assembly of the RNA polymerase III pre-initiation complex by binding to a control sequence in the intragenic region. This allows to recruit TFIIIB to the DNA at a site centered approximately 26 base pairs upstream of the start site of transcription. For tRNA genes, TFIIIC first associates with DNA, and then recruits TFIIIB. For 5S rRNA genes, TFIIIA binds to DNA first, followed by TFIIIC, which then recruits TFIIIB.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'},\n",
       "    {'dbCode': 'PMID', 'dbId': '23063749'},\n",
       "    {'dbCode': 'PMID', 'dbId': '27911719'},\n",
       "    {'dbCode': 'Wikipedia', 'dbId': 'RNA_polymerase_III'}]},\n",
       "  'comment': 'Merged terms as in https://github.com/geneontology/go-ontology/issues/14852',\n",
       "  'synonyms': [{'name': 'RNA polymerase III type 2 promoter sequence-specific DNA binding TFIIIB recruiting transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase III assembly factor activity, TFIIIB recruiting',\n",
       "    'type': 'related'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase III type 3 promoter TFIIIB recruiting',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase III type 2 promoter sequence-specific binding, TFIIIB recruiting',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase III type 1 promoter TFIIIB',\n",
       "    'type': 'related'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase III promoter sequence-specific binding, TFIIIB recruiting',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'SNAPc-type activity', 'type': 'related'},\n",
       "   {'name': 'RNA polymerase III type 1 promoter sequence-specific DNA binding TFIIIB recruiting transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase III type 1 promoter sequence-specific binding, TFIIIB recruiting',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase III type 3 promoter TFIIIB recruiting transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase III hybrid type promoter TFIIIB recruiting transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'TFIIIC-type activity', 'type': 'related'},\n",
       "   {'name': 'TFIIIA activity', 'type': 'related'},\n",
       "   {'name': 'type 3 RNA polymerase III promoter recognition',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'type 2 RNA polymerase III promoter recognition',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase III promoter sequence-specific DNA binding TFIIIB recruiting transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase III transcription factor recruiting activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'RNA polymerase III assembly factor activity', 'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001134', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070898', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001043', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001020', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001021', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000999', 'relation': 'part_of'}]},\n",
       " 'GO:0001006': {'id': 'GO:0001006',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III type 3 promoter sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a sequence of DNA that is a part of a type 3 promoter that controls transcription by RNA polymerase III (Pol III). A type 3 Pol III promoter is composed of elements upstream of the transcription start site, including a TATA box. The human U6 snRNA gene has a type 3 promoter. Type 3 Pol III promoters have not been observed in S. cerevisiae.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000992', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001032', 'relation': 'is_a'}]},\n",
       " 'GO:0001010': {'id': 'GO:0001010',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II sequence-specific DNA-binding transcription factor recruiting activity',\n",
       "  'definition': {'text': 'The function of binding to a specific DNA sequence and recruiting another transcription factor to the DNA in order to modulate transcription. The recruited factor may bind DNA directly, or may be colocalized via protein-protein interactions.'},\n",
       "  'synonyms': [{'name': 'transcription factor activity, sequence-specific DNA binding transcription factor recruiting',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sequence-specific DNA binding transcription factor recruiting transcription factor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcription factor activity, sequence-specific DNA-binding transcription factor recruiting',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0061629', 'relation': 'is_a'}]},\n",
       " 'GO:0001012': {'id': 'GO:0001012',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II regulatory region DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a DNA region that controls the transcription of a region of DNA by RNA polymerase II. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors.'},\n",
       "  'children': [{'id': 'GO:0000977', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044212', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000975', 'relation': 'is_a'}]},\n",
       " 'GO:0001013': {'id': 'GO:0001013',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I regulatory region DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a DNA region that controls the transcription of a region of DNA by RNA polymerase I. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors.'},\n",
       "  'children': [{'id': 'GO:0110016', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0001163', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044212', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000975', 'relation': 'is_a'}]},\n",
       " 'GO:0001014': {'id': 'GO:0001014',\n",
       "  'isObsolete': False,\n",
       "  'name': 'snoRNA transcription by RNA polymerase III',\n",
       "  'definition': {'text': 'The synthesis of small nucleolar RNA (snoRNA) from a DNA template by RNA polymerase III, originating at a type 2 RNA polymerase III promoter.'},\n",
       "  'synonyms': [{'name': 'snoRNA transcription from a type 2 RNA polymerase III promoter',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006383', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009302', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001035', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001009', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001041', 'relation': 'is_a'}]},\n",
       " 'GO:0001015': {'id': 'GO:0001015',\n",
       "  'isObsolete': False,\n",
       "  'name': 'snoRNA transcription by RNA polymerase II',\n",
       "  'definition': {'text': 'The synthesis of small nucleolar RNA (snoRNA) from a DNA template by RNA polymerase II, originating at an RNA polymerase II promoter.'},\n",
       "  'comment': 'Changed class label as requested in https://github.com/geneontology/go-ontology/issues/14854',\n",
       "  'synonyms': [{'name': 'snoRNA transcription from an RNA polymerase II promoter',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006366', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009302', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032568', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032569', 'relation': 'is_a'}]},\n",
       " 'GO:0001016': {'id': 'GO:0001016',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III regulatory region DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a DNA region that controls the transcription of a gene by RNA polymerase III. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'children': [{'id': 'GO:0001031', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001030', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000992', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001037', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001032', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044212', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000975', 'relation': 'is_a'}]},\n",
       " 'GO:0001017': {'id': 'GO:0001017',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial-type RNA polymerase regulatory region DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a DNA region that controls the transcription of a gene or operon by a bacterial-type RNA polymerase. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors.'},\n",
       "  'synonyms': [{'name': 'eubacterial-type RNA polymerase regulatory region DNA binding',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000984', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044212', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000975', 'relation': 'is_a'}]},\n",
       " 'GO:0001018': {'id': 'GO:0001018',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial promoter sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a DNA region that controls the transcription of the mitochondrial DNA.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20056105'}]},\n",
       "  'synonyms': [{'name': 'mitochondrial RNA polymerase core promoter sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'mitochondrial heavy strand promoter sense binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'LSPs binding', 'type': 'narrow'},\n",
       "   {'name': 'LSP non-coding strand binding', 'type': 'narrow'},\n",
       "   {'name': 'HSPas binding', 'type': 'narrow'},\n",
       "   {'name': 'mitochondrial light strand promoter anti-sense binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'HSP non-coding strand binding', 'type': 'narrow'},\n",
       "   {'name': 'HSP coding strand binding', 'type': 'narrow'},\n",
       "   {'name': 'mitochondrial RNA polymerase regulatory region sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'LSP coding strand binding', 'type': 'narrow'},\n",
       "   {'name': 'mitochondrial proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'LSPas binding', 'type': 'narrow'},\n",
       "   {'name': 'mitochondrial RNA polymerase core promoter proximal region sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'mitochondrial heavy strand promoter anti-sense binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'mitochondrial RNA polymerase regulatory region DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'mitochondrial light strand promoter sense binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'HSPs binding', 'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044212', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000975', 'relation': 'is_a'}]},\n",
       " 'GO:0001019': {'id': 'GO:0001019',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid RNA polymerase regulatory region DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a DNA region that controls the transcription of a region of DNA by a plastid RNA polymerase. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044212', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000975', 'relation': 'is_a'}]},\n",
       " 'GO:0001025': {'id': 'GO:0001025',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III general transcription initiation factor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase III transcription factor, any protein required to initiate or regulate transcription by RNA polymerase III.'},\n",
       "  'synonyms': [{'name': 'RNA polymerase III transcription factor binding',\n",
       "    'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0001155', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001156', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001154', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0140296', 'relation': 'is_a'}]},\n",
       " 'GO:0001030': {'id': 'GO:0001030',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III type 1 promoter DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a region of DNA that is a part of a type 1 promoter that controls transcription by RNA polymerase III. Type 1 promoters are found in 5S rRNA genes, downstream of the transcription start site within the sequence of the mature RNA, and require TFIIIA for recognition.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'children': [{'id': 'GO:0001002', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001016', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080084', 'relation': 'is_a'}]},\n",
       " 'GO:0001031': {'id': 'GO:0001031',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III type 2 promoter DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a region of DNA that is a part of a type 2 promoter that controls transcription by RNA polymerase III. Type 2 promoters consist of an A box and a B box downstream of the transcription start site within the sequence within the sequence of the mature RNA. Type 2 promoters are found in many tRNA genes as well as in other small RNAs.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'children': [{'id': 'GO:0001003', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001016', 'relation': 'is_a'}]},\n",
       " 'GO:0001032': {'id': 'GO:0001032',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III type 3 promoter DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a region of DNA that is a part of a type 3 promoter that controls transcription by RNA polymerase III. A type 3 Pol III promoter is composed of elements upstream of the transcription start site, including a TATA box. The human U6 snRNA gene has a type 3 promoter. Type 3 Pol III promoters have not been observed in S. cerevisiae.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'children': [{'id': 'GO:0001006', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001016', 'relation': 'is_a'}]},\n",
       " 'GO:0001037': {'id': 'GO:0001037',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III hybrid type promoter DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a region of DNA that is a part of a hybrid promoter that controls transcription by RNA polymerase III (Pol III). A hybrid Pol III promoter contains both regulatory elements both upstream and downstream of the transcription initiation site. An example gene with such a promoter is the S. cerevisiae U6 gene.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'children': [{'id': 'GO:0001039', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001016', 'relation': 'is_a'}]},\n",
       " 'GO:0001039': {'id': 'GO:0001039',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III hybrid type promoter sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a sequence of DNA that is a part of a hybrid type promoter that controls transcription by RNA polymerase III (Pol III). A hybrid Pol III promoter contains both regulatory elements both upstream and downstream of the transcription initiation site. An example gene with such a promoter is the S. cerevisiae U6 gene.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000992', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001037', 'relation': 'is_a'}]},\n",
       " 'GO:0001042': {'id': 'GO:0001042',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I core binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with RNA polymerase I core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of seventeen subunits.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0043175', 'relation': 'is_a'}]},\n",
       " 'GO:0001046': {'id': 'GO:0001046',\n",
       "  'isObsolete': False,\n",
       "  'name': 'core promoter sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a sequence of DNA that is part of a core promoter region. The core promoter is composed of the transcription start site and binding sites for the RNA polymerase and the basal transcription machinery. The transcribed region might be described as a gene, cistron, or operon.'},\n",
       "  'comment': 'To minimize ambiguity in the use of the word \"promoter\" in GO, we have chosen the phrase \"transcription regulatory region\" to refer to all of the regulatory regions. Regulatory regions in the DNA which control initiation  include the \"core promoter\" where the basal transcription machinery binds, and the \"cis-regulatory region\" where regulatory factors other than the basal machinery bind. There are also additional regulatory regions, in both the DNA and the RNA transcript, which regulate elongation or termination of transcription. ANNOTATION NOTE: Regarding annotation to \"transcription regulatory region DNA binding\" (GO:0044212) and any of its is_a children, note that annotation to these terms specifies DNA binding only without any statement about transcription factor activity. To make an annotation about a function of transcription factor activity, consider \"sequence-specific DNA-binding transcription factor activity\" (GO:0003700) or its is_a children which have has_part relationships to the appropriate kind of \"transcription regulatory region DNA binding\".',\n",
       "  'children': [{'id': 'GO:0000985', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000979', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001164', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000976', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001047', 'relation': 'is_a'}]},\n",
       " 'GO:0001047': {'id': 'GO:0001047',\n",
       "  'isObsolete': False,\n",
       "  'name': 'core promoter binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with the regulatory region composed of the transcription start site and binding sites for the basal transcription machinery. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors.'},\n",
       "  'children': [{'id': 'GO:1904796', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001046', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904798', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1904797', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044212', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000975', 'relation': 'is_a'}]},\n",
       " 'GO:0001048': {'id': 'GO:0001048',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase IV core binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with RNA polymerase IV core enzyme, a multisubunit eukaryotic nuclear RNA polymerase found in plants and involved in siRNA production.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '19110459'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0043175', 'relation': 'is_a'}]},\n",
       " 'GO:0001049': {'id': 'GO:0001049',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase V core binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with RNA polymerase V core enzyme, a multisubunit eukaryotic nuclear RNA polymerase found in plants and involved in production of noncoding transcripts at target loci for silencing.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '19110459'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0043175', 'relation': 'is_a'}]},\n",
       " 'GO:0001050': {'id': 'GO:0001050',\n",
       "  'isObsolete': False,\n",
       "  'name': 'single-subunit type RNA polymerase binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a single subunit RNA polymerase enzyme, which is composed of a single catalytic subunit similar to the RNA polymerase enzymes from phages T3, T7, and SP6.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20701995'}]},\n",
       "  'synonyms': [{'name': 'T3-type RNA polymerase binding', 'type': 'exact'},\n",
       "   {'name': 'SP6-type RNA polymerase binding', 'type': 'exact'},\n",
       "   {'name': 'T3/T7 type RNA polymerase binding', 'type': 'exact'},\n",
       "   {'name': 'T7-type RNA polymerase binding', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001001', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001051', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0070063', 'relation': 'is_a'}]},\n",
       " 'GO:0001051': {'id': 'GO:0001051',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid single-subunit type RNA polymerase binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a single subunit plastid RNA polymerase enzyme, which is composed of a single catalytic subunit similar to the RNA polymerase enzymes from phages T3, T7, and SP6.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20701995'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001050', 'relation': 'is_a'}]},\n",
       " 'GO:0001052': {'id': 'GO:0001052',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid PEP RNA polymerase core enzyme binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with the bacterial-type plastid PEP RNA polymerase core enzyme, typically consisting of two alpha, one beta, one beta prime, and one double prime subunit.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20701995'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001000', 'relation': 'is_a'}]},\n",
       " 'GO:0001054': {'id': 'GO:0001054',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase I specific promoter to direct initiation and catalyzes DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\"},\n",
       "  'synonyms': [{'name': 'DNA-directed RNA polymerase activity involved in transcription from RNA polymerase I promoter',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0005736', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003899', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006360', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000129', 'relation': 'is_a'}]},\n",
       " 'GO:0001055': {'id': 'GO:0001055',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase II specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\"},\n",
       "  'synonyms': [{'name': 'DNA-directed RNA polymerase activity involved in transcription from RNA polymerase II promoter',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003899', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006366', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000129', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032568', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032569', 'relation': 'part_of'}]},\n",
       " 'GO:0001056': {'id': 'GO:0001056',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase III activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase III specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\"},\n",
       "  'synonyms': [{'name': 'DNA-directed RNA polymerase activity involved in transcription from RNA polymerase III promoter',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1903623', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1903622', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0005666', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003899', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006383', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000129', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001035', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001009', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001041', 'relation': 'part_of'}]},\n",
       " 'GO:0001057': {'id': 'GO:0001057',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase IV activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase IV specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\"},\n",
       "  'synonyms': [{'name': 'DNA-directed RNA polymerase activity involved in transcription from RNA polymerase IV promoter',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000418', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001059', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0003899', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000129', 'relation': 'is_a'}]},\n",
       " 'GO:0001058': {'id': 'GO:0001058',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase V activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase V specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\"},\n",
       "  'synonyms': [{'name': 'DNA-directed RNA polymerase activity involved in transcription from RNA polymerase V promoter',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000419', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001060', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0003899', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000129', 'relation': 'is_a'}]},\n",
       " 'GO:0001059': {'id': 'GO:0001059',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription by RNA polymerase IV',\n",
       "  'definition': {'text': 'The synthesis of RNA from a DNA template by RNA polymerase IV, originating at a Pol IV-specific promoter.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '19110459'}]},\n",
       "  'comment': 'Changed class label as requested in https://github.com/geneontology/go-ontology/issues/14854',\n",
       "  'synonyms': [{'name': 'transcription from RNA polymerase IV promoter',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcription from RNA pol IV promoter', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001057', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006351', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061018', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006350', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061022', 'relation': 'is_a'}]},\n",
       " 'GO:0001060': {'id': 'GO:0001060',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription by RNA polymerase V',\n",
       "  'definition': {'text': 'The synthesis of RNA from a DNA template by RNA polymerase V, originating at a Pol V-specific promoter.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '19110459'}]},\n",
       "  'comment': 'Changed class label as requested in https://github.com/geneontology/go-ontology/issues/14854',\n",
       "  'synonyms': [{'name': 'transcription from RNA polymerase V promoter',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcription from RNA pol V promoter', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1904279', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1904281', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1904280', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0001058', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006351', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061018', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006350', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061022', 'relation': 'is_a'}]},\n",
       " 'GO:0001061': {'id': 'GO:0001061',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial-type RNA polymerase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a bacterial-type specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\"},\n",
       "  'children': [{'id': 'GO:0001062', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001063', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001121', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0003899', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000129', 'relation': 'is_a'}]},\n",
       " 'GO:0001062': {'id': 'GO:0001062',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid PEP-A RNA polymerase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a plastid PEP-A RNA polymerase II specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20701995'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001061', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042793', 'relation': 'part_of'}]},\n",
       " 'GO:0001063': {'id': 'GO:0001063',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid PEP-B RNA polymerase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a plastid PEP-B RNA polymerase II specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20701995'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001061', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042793', 'relation': 'part_of'}]},\n",
       " 'GO:0001064': {'id': 'GO:0001064',\n",
       "  'isObsolete': False,\n",
       "  'name': 'single subunit type RNA polymerase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a single-subunit-type RNA polymerase-specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\"},\n",
       "  'synonyms': [{'name': 'T3/T7 type RNA polymerase activity',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001065', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001066', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003899', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000129', 'relation': 'is_a'}]},\n",
       " 'GO:0001065': {'id': 'GO:0001065',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial single subunit type RNA polymerase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a single-subunit-type mitochondrial RNA polymerase-specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\"},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001064', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006390', 'relation': 'part_of'}]},\n",
       " 'GO:0001066': {'id': 'GO:0001066',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plastid single subunit type RNA polymerase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains a single subunit type plastid RNA polymerase specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20701995'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001064', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042793', 'relation': 'part_of'}]},\n",
       " 'GO:0001067': {'id': 'GO:0001067',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulatory region nucleic acid binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a nucleic acid region that regulates a nucleic acid-based process. Such processes include transcription, DNA replication, and DNA repair.'},\n",
       "  'children': [{'id': 'GO:0001069', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044212', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003676', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000496', 'relation': 'is_a'}]},\n",
       " 'GO:0001068': {'id': 'GO:0001068',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription regulatory region RNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a RNA region within the transcript that regulates the transcription of a region of DNA, which may be a gene, cistron, or operon.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001069', 'relation': 'is_a'}]},\n",
       " 'GO:0001069': {'id': 'GO:0001069',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulatory region RNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a RNA region that regulates a nucleic acid-based process. Such processes include transcription, DNA replication, and DNA repair.'},\n",
       "  'children': [{'id': 'GO:0001068', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001067', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003723', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000498', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044822', 'relation': 'is_a'}]},\n",
       " 'GO:0001070': {'id': 'GO:0001070',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA-binding transcription regulator activity',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA sequence in order to modulate transcription. This function is known to occur in phages and viruses, for example the lambda N and the HIV tat proteins are necessary to allow RNA polymerase to read through terminator sequences.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '1756726'}]},\n",
       "  'comment': 'Added taxon constraints, edited definition see https://github.com/geneontology/go-ontology/issues/15872',\n",
       "  'synonyms': [{'name': 'RNA binding transcription regulator activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'RNA binding transcription factor activity', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:1905257', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1905255', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1990969', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1905256', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0140110', 'relation': 'is_a'}]},\n",
       " 'GO:0001072': {'id': 'GO:0001072',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription antitermination factor activity, RNA binding',\n",
       "  'definition': {'text': 'Binds to RNA, typically within the nascent RNA transcript, to promote readthrough of a transcription termination site and thus extending the length of the RNA transcript produced. Examples of antitermination factors which bind the nascent RNA include the lambda N protein and the HIV-1 tat protein.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '8332211'}]},\n",
       "  'synonyms': [{'name': 'RNA binding transcription antitermination factor activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031564', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140110', 'relation': 'is_a'}]},\n",
       " 'GO:0001073': {'id': 'GO:0001073',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription antitermination factor activity, DNA binding',\n",
       "  'definition': {'text': 'Binds to DNA, typically within region of the promoter and transcribed region, to promote readthrough of a transcription termination site and thus extending the length of the RNA transcript produced. Examples of antitermination factors which bind DNA include the lambda Q protein.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '8332211'}]},\n",
       "  'synonyms': [{'name': 'DNA binding transcription antitermination factor activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031564', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140110', 'relation': 'is_a'}]},\n",
       " 'GO:0001079': {'id': 'GO:0001079',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nitrogen catabolite regulation of transcription from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'A transcription regulation process in which the presence of one nitrogen source leads to the modulation of the frequency, rate, or extent of transcription, from an RNA polymerase II promoter, of specific genes involved in the metabolism of other nitrogen sources.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '19104072'}]},\n",
       "  'synonyms': [{'name': 'regulation of transcription from RNA polymerase II promoter by nitrogen catabolites',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001080', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001081', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006357', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090293', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006358', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010551', 'relation': 'is_a'}]},\n",
       " 'GO:0001080': {'id': 'GO:0001080',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nitrogen catabolite activation of transcription from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'A transcription regulation process in which the presence of one nitrogen source leads to an increase in the frequency, rate, or extent of transcription, from an RNA polymerase II promoter, of specific genes involved in the metabolism of other nitrogen sources.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '19104072'}]},\n",
       "  'synonyms': [{'name': 'positive regulation of transcription from RNA polymerase II promoter by nitrogen catabolites',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001079', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090294', 'relation': 'is_a'}]},\n",
       " 'GO:0001081': {'id': 'GO:0001081',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nitrogen catabolite repression of transcription from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'A transcription regulation process in which the presence of one nitrogen source leads to a decrease in the frequency, rate, or extent of transcription, from an RNA polymerase II promoter, of specific genes involved in the metabolism of other nitrogen sources.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '19104072'}]},\n",
       "  'synonyms': [{'name': 'negative regulation of transcription from RNA polymerase II promoter by nitrogen catabolites',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001079', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090295', 'relation': 'is_a'}]},\n",
       " 'GO:0001085': {'id': 'GO:0001085',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II transcription factor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase II transcription factor, any protein required to initiate or regulate transcription by RNA polymerase II.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008134', 'relation': 'is_a'}]},\n",
       " 'GO:0001091': {'id': 'GO:0001091',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II general transcription initiation factor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a basal RNA polymerase II transcription factor, any of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'synonyms': [{'name': 'RNA polymerase II basal transcription factor binding',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001092', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001095', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001096', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001093', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001094', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001097', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001099', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140296', 'relation': 'is_a'}]},\n",
       " 'GO:0001092': {'id': 'GO:0001092',\n",
       "  'isObsolete': False,\n",
       "  'name': 'TFIIA-class transcription factor complex binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a general RNA polymerase II transcription factor belonging to the TFIIA complex, one of the complexes involved in formation of the preinitiation complex (PIC) by RNA polymerase II and defined as a basal or general transcription factor.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'synonyms': [{'name': 'TFIIA-class transcription factor binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001091', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044877', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032403', 'relation': 'is_a'}]},\n",
       " 'GO:0001093': {'id': 'GO:0001093',\n",
       "  'isObsolete': False,\n",
       "  'name': 'TFIIB-class transcription factor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a general RNA polymerase II transcription factor of the TFIIB class, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001091', 'relation': 'is_a'}]},\n",
       " 'GO:0001094': {'id': 'GO:0001094',\n",
       "  'isObsolete': False,\n",
       "  'name': 'TFIID-class transcription factor complex binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a general RNA polymerase II transcription factor belonging to the TFIID complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'synonyms': [{'name': 'TFIID-class transcription factor binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001091', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044877', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032403', 'relation': 'is_a'}]},\n",
       " 'GO:0001095': {'id': 'GO:0001095',\n",
       "  'isObsolete': False,\n",
       "  'name': 'TFIIE-class transcription factor complex binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a general RNA polymerase II transcription factor belonging to the TFIIE complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'synonyms': [{'name': 'TFIIE-class transcription factor binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001091', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044877', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032403', 'relation': 'is_a'}]},\n",
       " 'GO:0001096': {'id': 'GO:0001096',\n",
       "  'isObsolete': False,\n",
       "  'name': 'TFIIF-class transcription factor complex binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a general RNA polymerase II transcription factor belonging to the TFIIF complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'synonyms': [{'name': 'TFIIF-class transcription factor binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001091', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044877', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032403', 'relation': 'is_a'}]},\n",
       " 'GO:0001097': {'id': 'GO:0001097',\n",
       "  'isObsolete': False,\n",
       "  'name': 'TFIIH-class transcription factor complex binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a general RNA polymerase II transcription factor belonging to the TFIIH complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase II.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'synonyms': [{'name': 'TFIIH-class transcription factor binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001091', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044877', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032403', 'relation': 'is_a'}]},\n",
       " 'GO:0001098': {'id': 'GO:0001098',\n",
       "  'isObsolete': False,\n",
       "  'name': 'basal transcription machinery binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with the basal transcription machinery which is composed of the RNA polymerase core enzyme and the basal transcription factor(s), the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase.'},\n",
       "  'comment': 'Note that the definition of basal, or general, transcription factors has typically been done at a small number of well characterized activator-independent promoters. At an activator-dependent promoter, one or more additional factors are generally required in addition to the basal factors.',\n",
       "  'children': [{'id': 'GO:0001099', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001108', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005515', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045308', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001948', 'relation': 'is_a'}]},\n",
       " 'GO:0001099': {'id': 'GO:0001099',\n",
       "  'isObsolete': False,\n",
       "  'name': 'basal RNA polymerase II transcription machinery binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with the basal transcription machinery for RNA polymerase II which is composed of the RNA polymerase II core enzyme, a multisubunit eukaryotic nuclear RNA polymerase typically composed of twelve subunits, and the basal RNA polymerase II transcription factors, the minimal set of factors required for formation of the preinitiation complex (PIC) by the RNA polymerase.'},\n",
       "  'comment': 'Note that the definition of basal, or general, transcription factors has typically been done at a small number of well characterized activator-independent promoters. At an activator-dependent promoter, one or more additional factors are generally required in addition to the basal factors.',\n",
       "  'synonyms': [{'name': 'basal RNAP II transcription machinery binding',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0000993', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001091', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001098', 'relation': 'is_a'}]},\n",
       " 'GO:0001100': {'id': 'GO:0001100',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative regulation of exit from mitosis',\n",
       "  'definition': {'text': 'Any process involved in the inhibition of progression from anaphase/telophase (high mitotic CDK activity) to G1 (low mitotic CDK activity).'},\n",
       "  'synonyms': [{'name': 'inhibition of exit from mitosis', 'type': 'narrow'},\n",
       "   {'name': 'down regulation of exit from mitosis', 'type': 'exact'},\n",
       "   {'name': 'downregulation of exit from mitosis', 'type': 'exact'},\n",
       "   {'name': 'down-regulation of exit from mitosis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0031578', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007096', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010458', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0045839', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901991', 'relation': 'is_a'}]},\n",
       " 'GO:0001101': {'id': 'GO:0001101',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to acid chemical',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus by the chemical structure of the anion portion of a dissociated acid (rather than the acid acting as a proton donor). The acid chemical may be in gaseous, liquid or solid form.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://en.wikipedia.org/wiki/Acid'}]},\n",
       "  'comment': \"This term should be used to describe a response to a specific acid as a chemical. E.g., if an organism were responding to glutamate, then the response would be glutamate-specific; the organism is actually responding to the chemical structure of the anion portion of the dissociated acid. Note that this term is in the subset of terms that should not be used for direct gene product annotation. Instead, select a child term or, if no appropriate child term exists, please request a new term. Direct annotations to this term may be amended during annotation QC. If annotating experiments where an acid is playing a role as a proton donor, please annotate to GO:0010447 'response to acidic pH' instead.\",\n",
       "  'synonyms': [{'name': 'response to acid', 'type': 'broad'},\n",
       "   {'name': 'response to acid anion', 'type': 'related'},\n",
       "   {'name': 'response to oxoanion', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0097532', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080033', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070542', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901556', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009751', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010034', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903840', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010157', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009753', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901530', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009737', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009739', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051593', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901427', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010193', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901593', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901425', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903492', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033591', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901422', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905439', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904880', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009415', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031427', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904481', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903927', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080021', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902617', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033986', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071229', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043200', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070781', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010167', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080026', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903412', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032526', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905441', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071505', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Electronic',\n",
       "  'parent': [{'id': 'GO:0042221', 'relation': 'is_a'}]},\n",
       " 'GO:0001102': {'id': 'GO:0001102',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II activating transcription factor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase II transcription activating factor, a protein involved in positive regulation of transcription.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0033613', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061629', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001107', 'relation': 'is_a'}]},\n",
       " 'GO:0001103': {'id': 'GO:0001103',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II repressing transcription factor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase II transcription repressing factor, a protein involved in negative regulation of transcription.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0061629', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070491', 'relation': 'is_a'}]},\n",
       " 'GO:0001108': {'id': 'GO:0001108',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial-type RNA polymerase holo enzyme binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with the basal transcription machinery which is composed of a bacterial-type RNA polymerase core enzyme and a sigma factor, the minimal set of factors required for formation of the preinitiation complex (PIC) by a bacterial-type RNA polymerase.'},\n",
       "  'synonyms': [{'name': 'basal bacterial-type RNA polymerase transcription machinery binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001098', 'relation': 'is_a'}]},\n",
       " 'GO:0001109': {'id': 'GO:0001109',\n",
       "  'isObsolete': False,\n",
       "  'name': 'promoter clearance during DNA-templated transcription',\n",
       "  'definition': {'text': 'Any process involved in the transition from the initiation to the elongation phases of transcription by a DNA-dependent RNA polymerase, generally including a conformational change from the initiation conformation to the elongation conformation. Promoter clearance often involves breaking contact with transcription factors involved only in the initiation phase and making contacts with elongation specific factors.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15020047'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'synonyms': [{'name': 'promoter clearance during DNA-dependent transcription',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'promoter escape', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0001111', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001122', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001110', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001182', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006351', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032774', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061018', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006350', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061022', 'relation': 'part_of'}]},\n",
       " 'GO:0001110': {'id': 'GO:0001110',\n",
       "  'isObsolete': False,\n",
       "  'name': 'promoter clearance from RNA polymerase III promoter',\n",
       "  'definition': {'text': 'Any process involved in the transition from the initiation to the elongation phases of transcription by RNA polymerase III, generally including a conformational change from the initiation conformation to the elongation conformation. Promoter clearance often involves breaking contact with transcription factors involved only in the initiation phase.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001109', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006383', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001035', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001009', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001041', 'relation': 'part_of'}]},\n",
       " 'GO:0001111': {'id': 'GO:0001111',\n",
       "  'isObsolete': False,\n",
       "  'name': 'promoter clearance from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Any process involved in the transition from the initiation to the elongation phases of transcription by RNA polymerase II, generally including a conformational change from the initiation conformation to the elongation conformation. Promoter clearance often involves breaking contact with transcription factors involved only in the initiation phase and making contacts with elongation specific factors.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15020047'}]},\n",
       "  'synonyms': [{'name': 'promoter escape from RNA polymerase II promoter',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001109', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006366', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032568', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032569', 'relation': 'part_of'}]},\n",
       " 'GO:0001112': {'id': 'GO:0001112',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA-templated transcriptional open complex formation',\n",
       "  'definition': {'text': 'Any process involved in the melting of the DNA hybrid of the core promoter region within the transcriptional closed complex of an RNA polymerase preinitiation complex (PIC) to produce an open complex where the DNA duplex around the transcription initiation site is unwound to form the transcription bubble.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15020047'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'synonyms': [{'name': 'DNA-dependent transcriptional open complex formation',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'promoter melting', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0001127', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001113', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001120', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006352', 'relation': 'part_of'}]},\n",
       " 'GO:0001113': {'id': 'GO:0001113',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcriptional open complex formation at RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Any process involved in the melting of the DNA hybrid of the core promoter region within the transcriptional closed complex of an RNA polymerase II preinitiation complex (PIC) to produce an open complex where the DNA duplex around the transcription initiation site is unwound to form the transcription bubble.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15020047'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'synonyms': [{'name': 'RNA polymerase II promoter melting',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001177', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001112', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006367', 'relation': 'part_of'}]},\n",
       " 'GO:0001114': {'id': 'GO:0001114',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein-DNA-RNA complex',\n",
       "  'definition': {'text': 'A macromolecular complex containing protein, DNA, and RNA molecules.'},\n",
       "  'children': [{'id': 'GO:0097523', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001115': {'id': 'GO:0001115',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein-DNA-RNA complex subunit organization',\n",
       "  'definition': {'text': 'Any process in which macromolecules aggregate, disaggregate, or are modified, resulting in the formation, disassembly, or alteration of a protein-DNA-RNA complex.'},\n",
       "  'synonyms': [{'name': 'protein-DNA-RNA complex subunit organisation',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001117', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001119', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001116', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0043933', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034621', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034600', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071822', 'relation': 'is_a'}]},\n",
       " 'GO:0001116': {'id': 'GO:0001116',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein-DNA-RNA complex assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of proteins, DNA, and RNA molecules to form a protein-DNA-RNA complex.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001115', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034622', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043623', 'relation': 'is_a'}]},\n",
       " 'GO:0001117': {'id': 'GO:0001117',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein-DNA-RNA complex disassembly',\n",
       "  'definition': {'text': 'The disaggregation of a protein-DNA-RNA complex into its constituent components.'},\n",
       "  'children': [{'id': 'GO:0001118', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001115', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032984', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034623', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043241', 'relation': 'is_a'}]},\n",
       " 'GO:0001118': {'id': 'GO:0001118',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription ternary complex disassembly',\n",
       "  'definition': {'text': 'The disaggregation of a transcription ternary complex, composed of RNA polymerase, template DNA, and an RNA transcript, into its constituent components.'},\n",
       "  'synonyms': [{'name': 'transcription protein-DNA-RNA complex disassembly',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001117', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006353', 'relation': 'part_of'}]},\n",
       " 'GO:0001119': {'id': 'GO:0001119',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein-DNA-RNA complex remodeling',\n",
       "  'definition': {'text': 'The acquisition, loss, or modification of macromolecules within a protein-DNA-RNA complex, resulting in the alteration of an existing complex.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001115', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034367', 'relation': 'is_a'}]},\n",
       " 'GO:0001120': {'id': 'GO:0001120',\n",
       "  'isObsolete': False,\n",
       "  'name': 'protein-DNA complex remodeling',\n",
       "  'definition': {'text': 'The acquisition, loss, or modification of macromolecules within a protein-DNA complex, resulting in the alteration of an existing complex.'},\n",
       "  'children': [{'id': 'GO:0001112', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0034367', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071824', 'relation': 'is_a'}]},\n",
       " 'GO:0001121': {'id': 'GO:0001121',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial transcription',\n",
       "  'definition': {'text': 'The synthesis of RNA from a DNA template by a bacterial-type RNA polymerase, originating at a bacterial-type promoter.'},\n",
       "  'comment': 'Changed class label as requested in https://github.com/geneontology/go-ontology/issues/14854',\n",
       "  'synonyms': [{'name': 'transcription from bacterial-type RNA polymerase promoter',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001123', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001122', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001125', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001124', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001061', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006351', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061018', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006350', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061022', 'relation': 'is_a'}]},\n",
       " 'GO:0001122': {'id': 'GO:0001122',\n",
       "  'isObsolete': False,\n",
       "  'name': 'promoter clearance from bacterial-type RNA polymerase promoter',\n",
       "  'definition': {'text': 'Any process involved in the transition from the initiation to the elongation phase of transcription by a bacterial-type RNA polymerase, generally including a conformational change from the initiation conformation to the elongation conformation. Promoter clearance generally involves dissociation of the sigma initiation factor.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001109', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001121', 'relation': 'part_of'}]},\n",
       " 'GO:0001123': {'id': 'GO:0001123',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription initiation from bacterial-type RNA polymerase promoter',\n",
       "  'definition': {'text': 'Any process involved in the assembly of bacterial-type RNA polymerase preinitiation complex (PIC) at the core promoter region of a DNA template, resulting in the subsequent synthesis of RNA from that promoter. The initiation phase includes PIC assembly and the formation of the first few bonds in the RNA chain, including abortive initiation, which occurs when the first few nucleotides are repeatedly synthesized and then released. The initiation phase ends just before and does not include promoter clearance, or release, which is the transition between the initiation and elongation phases of transcription.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'children': [{'id': 'GO:0001127', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001126', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001176', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001121', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006352', 'relation': 'is_a'}]},\n",
       " 'GO:0001124': {'id': 'GO:0001124',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription elongation from bacterial-type RNA polymerase promoter',\n",
       "  'definition': {'text': 'The extension of an RNA molecule after transcription initiation and promoter clearance at a bacterial-type RNA polymerase promoter by the addition of ribonucleotides catalyzed by a bacterial-type RNA polymerase.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'synonyms': [{'name': 'RNA elongation from bacterial-type RNA polymerase promoter',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001194', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001121', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006354', 'relation': 'is_a'}]},\n",
       " 'GO:0001125': {'id': 'GO:0001125',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription termination from bacterial-type RNA polymerase promoter',\n",
       "  'definition': {'text': 'The process in which the synthesis of an RNA molecule by a bacterial-type RNA polymerase using a DNA template is completed.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001121', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006353', 'relation': 'is_a'}]},\n",
       " 'GO:0001126': {'id': 'GO:0001126',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial-type RNA polymerase preinitiation complex assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of proteins on a bacterial-type RNA polymerase promoter DNA to form the transcriptional preinitiation complex (PIC), required for transcription.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'synonyms': [{'name': 'bacterial-type RNA polymerase transcription PIC formation',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'bacterial-type RNA polymerase transcriptional preinitiation complex formation',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'bacterial-type RNA polymerase transcription PIC biosynthesis',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001123', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0070897', 'relation': 'is_a'}]},\n",
       " 'GO:0001127': {'id': 'GO:0001127',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcriptional open complex formation at bacterial-type RNA polymerase promoter',\n",
       "  'definition': {'text': 'Any process involved in the melting of the DNA hybrid of the core promoter region within a bacterial-type RNA polymerase promoter to produce an open complex where the DNA duplex around the transcription initiation site is unwound to form the transcription bubble.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001112', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001123', 'relation': 'part_of'}]},\n",
       " 'GO:0001134': {'id': 'GO:0001134',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription regulator recruiting activity',\n",
       "  'definition': {'text': 'The function of binding to an RNA polymerase (RNAP) transcription regulator and recruiting it to the general transcription machinery complex in order to modulate transcription initiation.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'comment': 'Changed subclasses https://github.com/geneontology/go-ontology/issues/15577',\n",
       "  'synonyms': [{'name': 'transcription factor activity, transcription factor recruiting',\n",
       "    'type': 'related'},\n",
       "   {'name': 'transcription factor recruiting transcription factor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcription factor recruiting activity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001186', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001004', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001135', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008134', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140110', 'relation': 'is_a'}]},\n",
       " 'GO:0001135': {'id': 'GO:0001135',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II transcription regulator recruiting activity',\n",
       "  'definition': {'text': 'The function of binding to an RNA polymerase II (RNAP II) transcription regulator and recruiting it to the general transcription machinery complex in order to modulate transcription initiation.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'comment': 'Merged terms see https://github.com/geneontology/go-ontology/issues/15795',\n",
       "  'synonyms': [{'name': 'TFIIH-class transcription factor recruiting transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor activity, TFIIF-class transcription factor recruiting',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor activity, TFIIE-class transcription factor recruiting',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase II transcription factor recruiting',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA polymerase II transcription factor recruiting transcription factor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcription factor activity, TFIIH-class transcription factor recruiting',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'TFIIE-class transcription factor recruiting transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'TFIIF-class transcription factor recruiting transcription factor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcription factor recruiting activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001134', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051123', 'relation': 'part_of'}]},\n",
       " 'GO:0001139': {'id': 'GO:0001139',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II complex recruiting activity',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase II (Pol II) complex, typically composed of twelve subunits, and with another protein, macromolecule, or complex, permitting those molecules to function in a coordinated way in order to facilitate the aggregation, arrangement and bonding together of proteins on an RNA polymerase II promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription by RNA polymerase.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16858867'}]},\n",
       "  'synonyms': [{'name': 'core RNA polymerase II recruiting transcription factor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcription factor activity, core RNA polymerase II recruiting',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000993', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051123', 'relation': 'part_of'}]},\n",
       " 'GO:0001145': {'id': 'GO:0001145',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mitochondrial RNA polymerase termination site sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a DNA sequence that controls mitochondrial transcription termination.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18391175'}]},\n",
       "  'synonyms': [{'name': 'transcription factor activity, mitochondrial RNA polymerase terminator site sequence-specific binding',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'mitochondrial RNA polymerase terminator site sequence-specific DNA binding transcription factor activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001147', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006393', 'relation': 'part_of'}]},\n",
       " 'GO:0001147': {'id': 'GO:0001147',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription termination site sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a sequence of DNA that promotes termination by RNA polymerase. The transcribed region might be described as a gene, cistron, or operon.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18280161'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18391175'}]},\n",
       "  'comment': 'Transcription termination sites can be recognized by the RNA polymerase (RNAP) itself or by another protein which interacts with the RNAP to promote transcription termination. Note that not all genes have a specific sequence that functions as a termination site; for most mRNAs transcribed by RNAP II termination is not mediated by a specific termination sequence, but is coupled to polyadenylation.',\n",
       "  'children': [{'id': 'GO:0001148', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001145', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000976', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001160', 'relation': 'is_a'}]},\n",
       " 'GO:0001148': {'id': 'GO:0001148',\n",
       "  'isObsolete': False,\n",
       "  'name': 'bacterial-type RNA polymerase termination site sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a sequence of DNA that is a terminator for bacterial-type RNA polymerase.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18280161'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000984', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001147', 'relation': 'is_a'}]},\n",
       " 'GO:0001154': {'id': 'GO:0001154',\n",
       "  'isObsolete': False,\n",
       "  'name': 'TFIIIB-class transcription factor complex binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a general RNA polymerase III transcription factor belonging to the TFIIB complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase III.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'synonyms': [{'name': 'TFIIIB-class transcription factor binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001025', 'relation': 'is_a'}]},\n",
       " 'GO:0001155': {'id': 'GO:0001155',\n",
       "  'isObsolete': False,\n",
       "  'name': 'TFIIIA-class transcription factor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase III transcription factor of the TFIIIA class, one of the factors involved in formation of the preinitiation complex (PIC) at RNA polymerase III promoters.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001025', 'relation': 'is_a'}]},\n",
       " 'GO:0001156': {'id': 'GO:0001156',\n",
       "  'isObsolete': False,\n",
       "  'name': 'TFIIIC-class transcription factor complex binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a general RNA polymerase III transcription factor belonging to the TFIIC complex, one of the factors involved in formation of the preinitiation complex (PIC) by RNA polymerase III.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'}]},\n",
       "  'synonyms': [{'name': 'TFIIIC-class transcription factor binding',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001025', 'relation': 'is_a'}]},\n",
       " 'GO:0001160': {'id': 'GO:0001160',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription termination site DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a region of DNA that promotes termination by RNA polymerase. The transcribed region might be described as a gene, cistron, or operon.'},\n",
       "  'children': [{'id': 'GO:0001147', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044212', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000975', 'relation': 'is_a'}]},\n",
       " 'GO:0001161': {'id': 'GO:0001161',\n",
       "  'isObsolete': False,\n",
       "  'name': 'intronic transcription regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an intronic DNA sequence that regulates the transcription of the transcript it is contained within.'},\n",
       "  'children': [{'id': 'GO:0001162', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0044213', 'relation': 'is_a'}]},\n",
       " 'GO:0001162': {'id': 'GO:0001162',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II intronic transcription regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase II intronic DNA sequence that regulates the transcription of the transcript it is contained within.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000977', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001161', 'relation': 'is_a'}]},\n",
       " 'GO:0001163': {'id': 'GO:0001163',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific sequence of DNA that is part of a regulatory region that controls the transcription of a gene or cistron by RNA polymerase I.'},\n",
       "  'children': [{'id': 'GO:0001165', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001164', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901839', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1901840', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001013', 'relation': 'is_a'}]},\n",
       " 'GO:0001164': {'id': 'GO:0001164',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I core promoter sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with the regulatory region composed of the transcription start site and binding sites for transcription factors of the RNA polymerase I transcription machinery. This site is often referred to as the CORE element. In mammalian cells, the CORE element functions in conjunction with the Upstream Control Element (UCE), while in fungi, protozoa, and plants, the CORE element functions without a UCE.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12865296'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14969726'},\n",
       "    {'dbCode': 'PMID', 'dbId': '8057832'}]},\n",
       "  'synonyms': [{'name': 'RNA polymerase I CORE element sequence-specific DNA binding',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'RNA polymerase I core promoter sequence-specific DNA binding',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'RNA polymerase I core element sequence-specific DNA binding',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase I CORE element binding transcription factor recruiting',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA polymerase I CORE element sequence-specific DNA binding transcription factor recruiting transcription factor activity',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001046', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001163', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001188', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001189', 'relation': 'part_of'}]},\n",
       " 'GO:0001165': {'id': 'GO:0001165',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I cis-regulatory region sequence-specific DNA binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific upstream regulatory DNA sequence (transcription factor recognition sequence or binding site) located in cis relative to the transcription start site (i.e., on the same strand of DNA) of a gene transcribed by RNA polymerase I. RNA polymerase I elements are referred to either enhancers or upstream control element (UCE, or alternately referred to as the upstream element).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12865296'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14969726'},\n",
       "    {'dbCode': 'PMID', 'dbId': '8057832'}]},\n",
       "  'synonyms': [{'name': 'RNA polymerase I enhancer sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA polymerase I upstream control element sequence-specific DNA binding',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA polymerase I upstream element sequence-specific DNA binding',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001163', 'relation': 'is_a'}]},\n",
       " 'GO:0001171': {'id': 'GO:0001171',\n",
       "  'isObsolete': False,\n",
       "  'name': 'reverse transcription',\n",
       "  'definition': {'text': 'A DNA synthesis process that uses RNA as the initial template for synthesis of DNA, but which also includes an RNase activity to remove the RNA strand of an RNA-DNA heteroduplex produced by the RNA-dependent synthesis step and use of the initial DNA strand as a template for DNA synthesis.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20358252'}]},\n",
       "  'children': [{'id': 'GO:1900270', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1900268', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0032199', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900269', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006278', 'relation': 'is_a'}]},\n",
       " 'GO:0001172': {'id': 'GO:0001172',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription, RNA-templated',\n",
       "  'definition': {'text': 'The cellular synthesis of RNA on a template of RNA.'},\n",
       "  'synonyms': [{'name': 'transcription, RNA-dependent', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0003968', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0097659', 'relation': 'is_a'}]},\n",
       " 'GO:0001173': {'id': 'GO:0001173',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA-templated transcriptional start site selection',\n",
       "  'definition': {'text': 'Any process involved in the selection of the specific location within the template strand of a DNA-dependent RNA polymerase promoter for hybridization of the cognate ribonucleotides and formation of first phosphodiester bond within the nascent transcript.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16826228'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18846104'}]},\n",
       "  'synonyms': [{'name': 'DNA-dependent transcriptional start site selection',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001174', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001175', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001176', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006352', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032774', 'relation': 'is_a'}]},\n",
       " 'GO:0001174': {'id': 'GO:0001174',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcriptional start site selection at RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Any process involved in the selection of the specific location within the template strand of an RNA polymerase II promoter for hybridization of the cognate ribonucleotides and formation of first phosphodiester bond within the nascent transcript.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16826228'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18846104'}]},\n",
       "  'children': [{'id': 'GO:0001178', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001173', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006367', 'relation': 'part_of'}]},\n",
       " 'GO:0001175': {'id': 'GO:0001175',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcriptional start site selection at RNA polymerase III promoter',\n",
       "  'definition': {'text': 'Any process involved in the selection of the specific location within the template strand of an RNA polymerase III promoter for hybridization of the cognate ribonucleotides and formation of first phosphodiester bond within the nascent transcript.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001173', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006384', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001024', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001022', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001023', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001036', 'relation': 'part_of'}]},\n",
       " 'GO:0001176': {'id': 'GO:0001176',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcriptional start site selection at bacterial-type RNA polymerase promoter',\n",
       "  'definition': {'text': 'Any process involved in the selection of the specific location within the template strand of a bacterial-type RNA polymerase promoter for hybridization of the cognate ribonucleotides and formation of first phosphodiester bond within the nascent transcript.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001123', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001173', 'relation': 'is_a'}]},\n",
       " 'GO:0001177': {'id': 'GO:0001177',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of transcriptional open complex formation at RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Any process that modulates the rate, frequency or extent of a process involved the melting of the DNA hybrid of the core promoter region within the transcriptional closed complex of an RNA polymerase II preinitiation complex (PIC) to produce an open complex where the DNA duplex around the transcription initiation site is unwound to form the transcription bubble.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001113', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0051128', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060260', 'relation': 'is_a'}]},\n",
       " 'GO:0001178': {'id': 'GO:0001178',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of transcriptional start site selection at RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Any process that modulates the rate, frequency or extent of a process involved in the selection of the specific location within the template strand of an RNA polymerase II promoter for hybridization of the cognate ribonucleotides and formation of first phosphodiester bond within the nascent transcript.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001174', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0060260', 'relation': 'is_a'}]},\n",
       " 'GO:0001179': {'id': 'GO:0001179',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I general transcription initiation factor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase I transcription factor, any protein required to initiate or regulate transcription by RNA polymerase I.'},\n",
       "  'synonyms': [{'name': 'RNA polymerase I transcription factor binding',\n",
       "    'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0001186', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0140296', 'relation': 'is_a'}]},\n",
       " 'GO:0001181': {'id': 'GO:0001181',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I general transcription initiation factor activity',\n",
       "  'definition': {'text': 'An activity that contributes to transcription start site selection and transcription initiation of genes transcribed by RNA polymerase I. Factors required for RNA polymerase I transcription initiation include upstream activation factor (UAF), core factor (CF), TATA binding protein (TBP) and RRN3.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11500378'},\n",
       "    {'dbCode': 'PMID', 'dbId': '28340337'},\n",
       "    {'dbCode': 'PMID', 'dbId': '28842442'}]},\n",
       "  'comment': 'Updated class label and definition, see https://github.com/geneontology/go-ontology/issues/14736',\n",
       "  'synonyms': [{'name': 'RNA polymerase I transcription general initiation factor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'general RNA polymerase I transcription factor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'core RNA polymerase I binding transcription factor activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'transcription factor activity, core RNA polymerase I binding',\n",
       "    'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006360', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140223', 'relation': 'is_a'}]},\n",
       " 'GO:0001182': {'id': 'GO:0001182',\n",
       "  'isObsolete': False,\n",
       "  'name': 'promoter clearance from RNA polymerase I promoter',\n",
       "  'definition': {'text': 'Any process involved in the transition from the initiation to the elongation phases of transcription by RNA polymerase I, generally including a conformational change from the initiation conformation to the elongation conformation. Promoter clearance often involves breaking contact with transcription factors involved only in the initiation phase and making contacts with elongation specific factors.'},\n",
       "  'children': [{'id': 'GO:0001184', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001109', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006360', 'relation': 'part_of'}]},\n",
       " 'GO:0001183': {'id': 'GO:0001183',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription elongation from RNA polymerase I promoter for nuclear large rRNA transcript',\n",
       "  'definition': {'text': 'The extension of an RNA molecule after transcription initiation and promoter clearance at an RNA polymerase I promoter for the nuclear large ribosomal RNA (rRNA) transcript by the addition of ribonucleotides catalyzed by RNA polymerase I.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006362', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042790', 'relation': 'part_of'}]},\n",
       " 'GO:0001184': {'id': 'GO:0001184',\n",
       "  'isObsolete': False,\n",
       "  'name': 'promoter clearance from RNA polymerase I promoter for nuclear large rRNA transcript',\n",
       "  'definition': {'text': 'Any process involved in the transition from the initiation to the elongation phases of transcription by RNA polymerase I at a promoter for the nuclear large ribosomal RNA (rRNA) transcript, generally including a conformational change from the initiation conformation to the elongation conformation. Promoter clearance often involves breaking contact with transcription factors involved only in the initiation phase and making contacts with elongation specific factors.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001182', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042790', 'relation': 'part_of'}]},\n",
       " 'GO:0001186': {'id': 'GO:0001186',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I transcription regulator recruiting activity',\n",
       "  'definition': {'text': 'The function of binding to an RNA polymerase I (RNAP I) transcription regulator and recruiting it to the general transcription machinery complex in order to modulate transcription initiation.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14969726'},\n",
       "    {'dbCode': 'PMID', 'dbId': '8057832'}]},\n",
       "  'synonyms': [{'name': 'RNA polymerase I transcription factor recruiting transcription factor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcription factor activity, RNA polymerase I transcription factor recruiting',\n",
       "    'type': 'related'},\n",
       "   {'name': 'RNA polymerase I transcription factor recruiting activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001134', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001179', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001188', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001189', 'relation': 'part_of'}]},\n",
       " 'GO:0001188': {'id': 'GO:0001188',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase I preinitiation complex assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of proteins on promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription from an RNA polymerase I promoter.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12381659'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14969726'},\n",
       "    {'dbCode': 'PMID', 'dbId': '8057832'}]},\n",
       "  'synonyms': [{'name': 'RNA polymerase I transcriptional preinitiation complex assembly',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'RNA polymerase I transcriptional preinitiation complex assembly at the promoter for the nuclear large rRNA transcript',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase I transcriptional preinitiation complex assembly at the promoter for the nucleolar primary rRNA transcript',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0001186', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001164', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006361', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0070897', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001180', 'relation': 'part_of'}]},\n",
       " 'GO:0001192': {'id': 'GO:0001192',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maintenance of transcriptional fidelity during DNA-templated transcription elongation',\n",
       "  'definition': {'text': 'Suppression of the occurrence of transcriptional errors, such as substitutions and/or insertions of nucleotides that do not correctly match the template base, during the process of transcription elongation on a DNA template.'},\n",
       "  'synonyms': [{'name': 'maintenance of transcriptional fidelity during DNA-dependent transcription elongation',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001193', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001195', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001194', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006354', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032774', 'relation': 'is_a'}]},\n",
       " 'GO:0001193': {'id': 'GO:0001193',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maintenance of transcriptional fidelity during DNA-templated transcription elongation from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Suppression of the occurrence of transcriptional errors, such as substitutions and/or insertions of nucleotides that do not correctly match the template base, during the process of transcription elongation from an RNA polymerase II promoter.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14531857'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16492753'},\n",
       "    {'dbCode': 'PMID', 'dbId': '17535246'}]},\n",
       "  'synonyms': [{'name': 'maintenance of transcriptional fidelity during DNA-dependent transcription elongation from RNA polymerase II promoter',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001192', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006368', 'relation': 'part_of'}]},\n",
       " 'GO:0001194': {'id': 'GO:0001194',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maintenance of transcriptional fidelity during DNA-templated transcription elongation from bacterial-type RNA polymerase promoter',\n",
       "  'definition': {'text': 'Suppression of the occurrence of transcriptional errors, such as substitutions and/or insertions of nucleotides that do not correctly match the template base, during the process of transcription elongation from a bacterial-type RNA polymerase promoter.'},\n",
       "  'synonyms': [{'name': 'maintenance of transcriptional fidelity during DNA-dependent transcription elongation from bacterial-type RNA polymerase promoter',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001124', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001192', 'relation': 'is_a'}]},\n",
       " 'GO:0001195': {'id': 'GO:0001195',\n",
       "  'isObsolete': False,\n",
       "  'name': 'maintenance of transcriptional fidelity during DNA-templated transcription elongation from RNA polymerase III promoter',\n",
       "  'definition': {'text': 'Suppression of the occurrence of transcriptional errors, such as substitutions and/or insertions of nucleotides that do not correctly match the template base, during the process of transcription elongation from a RNA polymerase III promoter.'},\n",
       "  'synonyms': [{'name': 'maintenance of transcriptional fidelity during DNA-dependent transcription elongation from RNA polymerase III promoter',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001192', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006385', 'relation': 'part_of'}]},\n",
       " 'GO:0001196': {'id': 'GO:0001196',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of mating-type specific transcription from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Any mating-type specific process that modulates the frequency, rate or extent of transcription from an RNA polymerase II promoter.'},\n",
       "  'children': [{'id': 'GO:0001197', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001198', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006357', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006366', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0007532', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006358', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010551', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032568', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0032569', 'relation': 'regulates'}]},\n",
       " 'GO:0001197': {'id': 'GO:0001197',\n",
       "  'isObsolete': False,\n",
       "  'name': 'positive regulation of mating-type specific transcription from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Any mating-type specific process that activates or increases the rate of transcription from an RNA polymerase II promoter.'},\n",
       "  'synonyms': [{'name': 'upregulation of transcription from an RNA polymerase II promoter, mating-type specific',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'stimulation of transcription from an RNA polymerase II promoter, mating-type specific',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'up regulation of transcription from an RNA polymerase II promoter, mating-type specific',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'activation of transcription from an RNA polymerase II promoter, mating-type specific',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'up-regulation of transcription from an RNA polymerase II promoter, mating-type specific',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001196', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006366', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0045895', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032568', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0032569', 'relation': 'regulates'}]},\n",
       " 'GO:0001198': {'id': 'GO:0001198',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative regulation of mating-type specific transcription from RNA polymerase II promoter',\n",
       "  'definition': {'text': 'Any mating-type specific process that stops, prevents or reduces the rate of transcription from an RNA polymerase II promoter.'},\n",
       "  'synonyms': [{'name': 'down regulation of RNA polymerase II transcription, mating-type specific',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'down-regulation of RNA polymerase II transcription, mating-type specific',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'downregulation of RNA polymerase II transcription, mating-type specific',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'inhibition of RNA polymerase II transcription, mating-type specific',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001196', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006366', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0045894', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032568', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0032569', 'relation': 'regulates'}]},\n",
       " 'GO:0001207': {'id': 'GO:0001207',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histone displacement',\n",
       "  'definition': {'text': 'The removal of histones, including histone dimers, from nucleosomes within chromatin.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15525516'},\n",
       "    {'dbCode': 'PMID', 'dbId': '17496903'},\n",
       "    {'dbCode': 'PMID', 'dbId': '21807128'}]},\n",
       "  'children': [{'id': 'GO:0001208', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0034728', 'relation': 'is_a'}]},\n",
       " 'GO:0001208': {'id': 'GO:0001208',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histone H2A-H2B dimer displacement',\n",
       "  'definition': {'text': 'The removal of a H2A-H2B histone dimer from a nucleosome within chromatin.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15525516'},\n",
       "    {'dbCode': 'PMID', 'dbId': '17496903'},\n",
       "    {'dbCode': 'PMID', 'dbId': '21807128'}]},\n",
       "  'children': [{'id': 'GO:1902649', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1902650', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1902651', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001207', 'relation': 'is_a'}]},\n",
       " 'GO:0001216': {'id': 'GO:0001216',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA-binding transcription activator activity',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific DNA sequence in order to activate or increase the frequency, rate or extent of transcription by a RNA polymerase.'},\n",
       "  'comment': 'Merged terms as requested in https://github.com/geneontology/go-ontology/issues/16534',\n",
       "  'synonyms': [{'name': 'bacterial-type RNA polymerase transcriptional activator activity, sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional activator activity, bacterial-type RNA polymerase core promoter proximal region sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'bacterial-type RNA polymerase core promoter proximal region sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'related'},\n",
       "   {'name': 'bacterial-type RNA polymerase transcriptional activator activity, metal ion regulated sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'metal ion regulated sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional activator activity, bacterial-type RNA polymerase proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001228', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003700', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045893', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001151', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001071', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001131', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000130', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001199', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001130', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001204', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061020', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045941', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043193', 'relation': 'part_of'}]},\n",
       " 'GO:0001217': {'id': 'GO:0001217',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA-binding transcription repressor activity',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a specific DNA sequence in order to stop, prevent, or reduce the frequency, rate or extent of transcription by a RNA polymerase.'},\n",
       "  'comment': 'Made term more general see https://github.com/geneontology/go-ontology/issues/16736',\n",
       "  'synonyms': [{'name': 'bacterial-type RNA polymerase transcriptional repressor activity, sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'bacterial-type RNA polymerase transcriptional repressor activity, cadmium ion regulated sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'bacterial-type RNA polymerase core promoter proximal region sequence-specific DNA binding transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'copper ion regulated sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'bacterial-type DNA binding transcription repressor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'cadmium ion regulated sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional repressor activity, bacterial-type RNA polymerase core promoter proximal region sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'bacterial-type RNA polymerase transcriptional repressor activity, copper ion regulated sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'metal ion regulated sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'sequence-specific DNA binding bacterial-type RNA polymerase transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'bacterial-type RNA polymerase transcriptional repressor activity, metal ion regulated sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional repressor activity, bacterial-type RNA polymerase proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0110001', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:0001227', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003700', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045892', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001151', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001071', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001131', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000130', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001199', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001130', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001204', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016481', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061021', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032582', 'relation': 'part_of'}]},\n",
       " 'GO:0001221': {'id': 'GO:0001221',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription cofactor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a transcription cofactor, any protein involved in regulation of transcription via protein-protein interactions with transcription factors and other transcription regulatory proteins. Cofactors do not bind DNA directly, but rather mediate protein-protein interactions between regulatory transcription factors and the basal transcription machinery.'},\n",
       "  'children': [{'id': 'GO:0001223', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001224', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001222', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008134', 'relation': 'is_a'}]},\n",
       " 'GO:0001222': {'id': 'GO:0001222',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription corepressor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a transcription corepressor, any protein involved in negative regulation of transcription via protein-protein interactions with transcription factors and other proteins that negatively regulate transcription. Transcription corepressors do not bind DNA directly, but rather mediate protein-protein interactions between repressing transcription factors and the basal transcription machinery.'},\n",
       "  'children': [{'id': 'GO:0001226', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001221', 'relation': 'is_a'}]},\n",
       " 'GO:0001223': {'id': 'GO:0001223',\n",
       "  'isObsolete': False,\n",
       "  'name': 'transcription coactivator binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a transcription coactivator, any protein involved in positive regulation of transcription via protein-protein interactions with transcription factors and other proteins that positively regulate transcription. Transcription coactivators do not bind DNA directly, but rather mediate protein-protein interactions between activating transcription factors and the basal transcription machinery.'},\n",
       "  'children': [{'id': 'GO:0001225', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001221', 'relation': 'is_a'}]},\n",
       " 'GO:0001224': {'id': 'GO:0001224',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II transcription cofactor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a transcription cofactor for RNA polymerase II, any protein involved in regulation of transcription via protein-protein interactions with RNA polymerase II transcription factors and other transcription regulatory proteins. Cofactors do not bind DNA directly, but rather mediate protein-protein interactions between regulatory transcription factors and the basal transcription machinery of RNA polymerase II.'},\n",
       "  'children': [{'id': 'GO:0001226', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001225', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001221', 'relation': 'is_a'}]},\n",
       " 'GO:0001225': {'id': 'GO:0001225',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II transcription coactivator binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase II transcription coactivator, any protein involved in positive regulation of transcription of RNA polymerase II via protein-protein interactions with transcription factors and other proteins that positively regulate transcription. Transcription coactivators do not bind DNA directly, but rather mediate protein-protein interactions between activating transcription factors and the basal transcription machinery of RNA polymerase II.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001223', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001224', 'relation': 'is_a'}]},\n",
       " 'GO:0001226': {'id': 'GO:0001226',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA polymerase II transcription corepressor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an RNA polymerase II transcription corepressor, any protein involved in negative regulation of transcription by RNA polymerase II via protein-protein interactions with transcription factors and other proteins that negatively regulate transcription. Transcription corepressors do not bind DNA directly, but rather mediate protein-protein interactions between repressing transcription factors and the basal transcription machinery of RNA polymerase II.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001222', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001224', 'relation': 'is_a'}]},\n",
       " 'GO:0001227': {'id': 'GO:0001227',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA-binding transcription repressor activity, RNA polymerase II-specific',\n",
       "  'definition': {'text': 'A protein or a member of a complex that interacts selectively and non-covalently with a specific DNA sequence (sometimes referred to as a motif) within the regulatory region of a RNA polymerase II-transcribed gene to repress or decrease transcription. Regulatory regions include promoters (proximal and distal) and enhancers. Genes are transcriptional units.'},\n",
       "  'comment': 'Changed term label and definition https://github.com/geneontology/go-ontology/issues/16131 and https://github.com/geneontology/go-ontology/issues/16152',\n",
       "  'synonyms': [{'name': 'RNA polymerase II transcriptional repressor activity, metal ion regulated core promoter proximal region sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'metal ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcriptional repressor activity, metal ion regulated proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional repressor activity, RNA polymerase II transcription regulatory region sequence-specific DNA binding',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcriptional repressor activity, metal ion regulated sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional repressor activity, RNA polymerase II proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'distal enhancer DNA-binding transcription repressor activity, RNA polymerase II-specific',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'sequence-specific distal enhancer binding RNA polymerase II transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II core promoter proximal region sequence-specific DNA binding transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional repressor activity, RNA polymerase II distal enhancer sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II distal enhancer sequence-specific DNA-binding transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'proximal promoter DNA-binding transcription repressor activity, RNA polymerase II-specific',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcription regulatory region sequence-specific DNA binding transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'metal ion regulated sequence-specific DNA binding transcription factor activity involved in negative regulation of transcription',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000122', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000981', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001217', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010553', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045816', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000982', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001133', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003705', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001200', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001202', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001203', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001201', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001219', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001220', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001141', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001218', 'relation': 'is_a'}]},\n",
       " 'GO:0001228': {'id': 'GO:0001228',\n",
       "  'isObsolete': False,\n",
       "  'name': 'DNA-binding transcription activator activity, RNA polymerase II-specific',\n",
       "  'definition': {'text': 'A protein or a member of a complex that interacts selectively and non-covalently with a specific DNA sequence (sometimes referred to as a motif) within the regulatory region of a RNA polymerase II-transcribed gene to activate or increase transcription. Regulatory regions include promoters (proximal and distal) and enhancers. Genes are transcriptional units.'},\n",
       "  'comment': 'Merged terms, see https://github.com/geneontology/go-ontology/issues/16152 and https://github.com/geneontology/go-ontology/issues/16534; changed term label and definition https://github.com/geneontology/go-ontology/issues/16131',\n",
       "  'synonyms': [{'name': 'RNA polymerase II transcriptional activator activity, zinc ion regulated proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcriptional activator activity, copper ion regulated proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcriptional activator activity, metal ion regulated proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional activator activity, metal ion regulated sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'distal enhancer DNA-binding transcription activator activity, RNA polymerase II-specific',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'proximal promoter DNA-binding transcription activator activity, RNA polymerase II-specific',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional activator activity, RNA polymerase II core promoter proximal region sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional activator activity, RNA polymerase II distal enhancer sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transcriptional activator activity, RNA polymerase II transcription regulatory region sequence-specific DNA binding',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'RNA polymerase II distal enhancer sequence-specific DNA-binding transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'copper ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'sequence-specific distal enhancer binding RNA polymerase II transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcriptional activator activity, metal ion regulated core promoter proximal region sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcription regulatory region sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'transcriptional activator activity, RNA polymerase II proximal promoter sequence-specific DNA binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'metal ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'metal ion regulated sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'zinc ion regulated core promoter proximal region sequence-specific DNA binding RNA polymerase II transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II transcriptional activator activity, zinc ion regulated core promoter proximal region sequence-specific binding',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'RNA polymerase II core promoter proximal region sequence-specific DNA binding transcription factor activity involved in positive regulation of transcription',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:1990589', 'relation': 'capable_of'},\n",
       "   {'id': 'GO:1990590', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000981', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001216', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045944', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0000982', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001133', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003705', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001200', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001202', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001203', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001201', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001215', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001140', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010552', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045817', 'relation': 'part_of'}]},\n",
       " 'GO:0001300': {'id': 'GO:0001300',\n",
       "  'isObsolete': False,\n",
       "  'name': 'chronological cell aging',\n",
       "  'definition': {'text': 'The process associated with progression of the cell from its inception to the end of its lifespan that occurs when the cell is in a non-dividing, or quiescent, state.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044934'}]},\n",
       "  'synonyms': [{'name': 'chronological cell ageing', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001320', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001305', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001323', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010978', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001324', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007569', 'relation': 'is_a'}]},\n",
       " 'GO:0001301': {'id': 'GO:0001301',\n",
       "  'isObsolete': False,\n",
       "  'name': 'progressive alteration of chromatin involved in cell aging',\n",
       "  'definition': {'text': 'Any chromatin organization process that occurs during the lifespan of the cell that results in changes in chromatin structure. Such changes may lead to gene dysregulation and ultimately to a loss in cell homeostasis, bringing about an aging phenotype.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044938'}]},\n",
       "  'synonyms': [{'name': 'progressive alteration of chromatin during cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'progressive alteration of chromatin during cell ageing',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'age-dependent accumulation of genetic damage',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001304', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0034652', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001305', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007580', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006325', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007569', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0016568', 'relation': 'is_a'}]},\n",
       " 'GO:0001302': {'id': 'GO:0001302',\n",
       "  'isObsolete': False,\n",
       "  'name': 'replicative cell aging',\n",
       "  'definition': {'text': 'The process associated with progression of the cell from its inception to the end of its lifespan that occurs as the cell continues cycles of growth and division.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044934'}]},\n",
       "  'synonyms': [{'name': 'replicative cell ageing', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001321', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1900062', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001308', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001314', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001304', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001303', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001313', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007569', 'relation': 'is_a'}]},\n",
       " 'GO:0001303': {'id': 'GO:0001303',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nucleolar fragmentation involved in replicative aging',\n",
       "  'definition': {'text': 'A nucleolar fragmentation process that gives rise to multiple rounded structures and that occurs in conjunction with increasing age in dividing cells.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9891807'}]},\n",
       "  'synonyms': [{'name': 'nucleolar fragmentation during replicative ageing',\n",
       "    'type': 'related'},\n",
       "   {'name': 'nucleolar fragmentation during replicative aging',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001302', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007576', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046616', 'relation': 'is_a'}]},\n",
       " 'GO:0001304': {'id': 'GO:0001304',\n",
       "  'isObsolete': False,\n",
       "  'name': 'progressive alteration of chromatin involved in replicative cell aging',\n",
       "  'definition': {'text': 'A process that results in changes in chromatin structure contributing to replicative cell aging.'},\n",
       "  'synonyms': [{'name': 'progressive alteration of chromatin during replicative cell ageing',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'progressive alteration of chromatin during replicative cell aging',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001307', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001308', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001301', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001302', 'relation': 'part_of'}]},\n",
       " 'GO:0001305': {'id': 'GO:0001305',\n",
       "  'isObsolete': False,\n",
       "  'name': 'progressive alteration of chromatin involved in chronological cell aging',\n",
       "  'definition': {'text': 'A process that results in changes in chromatin structure contributing to chronological cell aging, occurring in non-dividing cells.'},\n",
       "  'synonyms': [{'name': 'progressive alteration of chromatin during chronological cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'progressive alteration of chromatin during chronological cell ageing',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001300', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001301', 'relation': 'is_a'}]},\n",
       " 'GO:0001306': {'id': 'GO:0001306',\n",
       "  'isObsolete': False,\n",
       "  'name': 'age-dependent response to oxidative stress',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of oxidative stress, a state often resulting from exposure to high levels of reactive oxygen species, where the change varies according to the age of the cell or organism.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044938'}]},\n",
       "  'children': [{'id': 'GO:0001324', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001322', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001315', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006979', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007571', 'relation': 'part_of'}]},\n",
       " 'GO:0001307': {'id': 'GO:0001307',\n",
       "  'isObsolete': False,\n",
       "  'name': 'extrachromosomal circular DNA accumulation involved in replicative cell aging',\n",
       "  'definition': {'text': 'Increase in abundance of circular DNA molecules in dividing cells as they age. These molecules originate in the chromosome but are excised and circularized, often by intramolecular homologous recombination between direct tandem repeats, and then replicated independently of chromosomal replication.'},\n",
       "  'synonyms': [{'name': 'extrachromosomal circular DNA accumulation during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'extrachromosomal circular DNA accumulation during replicative cell ageing',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001313', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001314', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001310', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001304', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007580', 'relation': 'is_a'}]},\n",
       " 'GO:0001308': {'id': 'GO:0001308',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative regulation of chromatin silencing involved in replicative cell aging',\n",
       "  'definition': {'text': 'The process, which occurs as a dividing cell ages, leading to expression of genes that are typically not expressed due to silencing by regulatory proteins.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044934'}]},\n",
       "  'synonyms': [{'name': 'loss of chromatin silencing involved in replicative cell aging',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'loss of chromatin silencing during replicative cell ageing',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001302', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001304', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006342', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0031936', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016440', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0006345', 'relation': 'is_a'}]},\n",
       " 'GO:0001310': {'id': 'GO:0001310',\n",
       "  'isObsolete': False,\n",
       "  'name': 'extrachromosomal rDNA circle accumulation involved in replicative cell aging',\n",
       "  'definition': {'text': 'Increase in abundance of circular DNA molecules containing ribosomal DNA repeats in dividing cells as they age. These molecules originate in the chromosome but are excised and circularized, often by intramolecular homologous recombination between direct tandem repeats, and then replicated independently of chromosomal replication.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044934'}]},\n",
       "  'synonyms': [{'name': 'extrachromosomal ribosomal DNA circle accumulation during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'extrachromosomal rDNA circle accumulation during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'extrachromosomal rDNA circle accumulation during replicative cell ageing',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:1900007', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001311', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001312', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1900008', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1900009', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001307', 'relation': 'is_a'}]},\n",
       " 'GO:0001311': {'id': 'GO:0001311',\n",
       "  'isObsolete': False,\n",
       "  'name': 'formation of extrachromosomal circular rDNA by homologous recombination involved in replicative cell aging',\n",
       "  'definition': {'text': 'Excision from the chromosome and circularization of DNA molecules encoding ribosomal RNA in dividing cells as they age.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044934'}]},\n",
       "  'synonyms': [{'name': 'assembly of extrachromosomal circular rDNA by homologous recombination involved in replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'formation of extrachromosomal circular ribosomal DNA by homologous recombination during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'formation of extrachromosomal circular rDNA by homologous recombination during replicative cell ageing',\n",
       "    'type': 'related'},\n",
       "   {'name': 'formation of extrachromosomal circular rDNA by homologous recombination during replicative cell aging',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001310', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001313', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045458', 'relation': 'is_a'}]},\n",
       " 'GO:0001312': {'id': 'GO:0001312',\n",
       "  'isObsolete': False,\n",
       "  'name': 'replication of extrachromosomal rDNA circles involved in replicative cell aging',\n",
       "  'definition': {'text': 'Replication of rDNA following its excision from the chromosome of dividing cells as they age. Extrachromosomal rDNA forms a circle that contains at least one autonomously replicating sequence (ARS), which supports replication independent of chromosomal replication.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044934'}]},\n",
       "  'synonyms': [{'name': 'replication of extrachromosomal rDNA circles during replicative cell ageing',\n",
       "    'type': 'related'},\n",
       "   {'name': 'replication of extrachromosomal rDNA circles during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'replication of extrachromosomal ribosomal DNA circles during replicative cell aging',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001310', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001314', 'relation': 'is_a'}]},\n",
       " 'GO:0001313': {'id': 'GO:0001313',\n",
       "  'isObsolete': False,\n",
       "  'name': 'formation of extrachromosomal circular DNA involved in replicative cell aging',\n",
       "  'definition': {'text': 'Excision from the chromosome and circularization of a region of chromosomal DNA, generally, but not always, via homologous recombination between direct tandem repeats, in dividing cells as they age.'},\n",
       "  'synonyms': [{'name': 'assembly of extrachromosomal circular DNA involved in replicative cell aging',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'formation of extrachromosomal circular DNA during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'formation of extrachromosomal circular DNA during replicative cell ageing',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001311', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001302', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001307', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001325', 'relation': 'is_a'}]},\n",
       " 'GO:0001314': {'id': 'GO:0001314',\n",
       "  'isObsolete': False,\n",
       "  'name': 'replication of extrachromosomal circular DNA involved in replicative cell aging',\n",
       "  'definition': {'text': 'Replication of circular DNA following excision from the chromosome of dividing cells as they age; replication of extrachromosomal circular DNA generally occurs independently of chromosomal replication.'},\n",
       "  'synonyms': [{'name': 'replication of extrachromosomal circular DNA during replicative cell ageing',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'replication of extrachromosomal circular DNA during replicative cell aging',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001312', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001302', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001307', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001326', 'relation': 'is_a'}]},\n",
       " 'GO:0001315': {'id': 'GO:0001315',\n",
       "  'isObsolete': False,\n",
       "  'name': 'age-dependent response to reactive oxygen species',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of reactive oxygen species, where the change varies according to the age of the cell or organism.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044938'}]},\n",
       "  'synonyms': [{'name': 'age-dependent response to reactive oxygen intermediate',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'age-dependent response to reactive oxidative species',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'age-dependent response to ROI', 'type': 'exact'},\n",
       "   {'name': 'age-dependent response to ROS', 'type': 'exact'},\n",
       "   {'name': 'age-dependent response to active oxygen species',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'age-dependent response to AOS', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0072353', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000302', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001306', 'relation': 'is_a'}]},\n",
       " 'GO:0001316': {'id': 'GO:0001316',\n",
       "  'isObsolete': False,\n",
       "  'name': 'age-dependent response to reactive oxygen species involved in replicative cell aging',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) occurring during the process of replicative cell aging as a result of reactive oxygen species, where the change varies according to the age of the cell or organism.'},\n",
       "  'synonyms': [{'name': 'age-dependent response to ROI during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to active oxygen species during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to reactive oxygen species during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to reactive oxygen intermediate during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to reactive oxygen species during replicative cell ageing',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'age-dependent response to reactive oxidative species during replicative cell aging',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'age-dependent response to AOS during replicative cell aging',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'age-dependent response to ROS during replicative cell aging',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001317', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001322', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072353', 'relation': 'is_a'}]},\n",
       " 'GO:0001317': {'id': 'GO:0001317',\n",
       "  'isObsolete': False,\n",
       "  'name': 'accumulation of oxidatively modified proteins involved in replicative cell aging',\n",
       "  'definition': {'text': 'Accumulation of proteins that have undergone reactions with reactive oxygen species in aging dividing cells and exhibit modifications such as increased protein carbonyl content, oxidized methionine, protein hydrophobicity, and cross-linking.'},\n",
       "  'synonyms': [{'name': 'accumulation of oxidatively modified proteins during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'accumulation of oxidatively modified proteins during replicative cell ageing',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001318', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001319', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001316', 'relation': 'is_a'}]},\n",
       " 'GO:0001318': {'id': 'GO:0001318',\n",
       "  'isObsolete': False,\n",
       "  'name': 'formation of oxidatively modified proteins involved in replicative cell aging',\n",
       "  'definition': {'text': 'Chemical reaction, between proteins and reactive oxygen species, that occurs in dividing cells as they age and leads to a variety of changes in the affected proteins, including increases in protein carbonyl content, oxidized methionine, protein hydrophobicity, and cross-linking.'},\n",
       "  'synonyms': [{'name': 'formation of oxidatively modified proteins during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'formation of oxidatively modified proteins during replicative cell ageing',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001317', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006464', 'relation': 'is_a'}]},\n",
       " 'GO:0001319': {'id': 'GO:0001319',\n",
       "  'isObsolete': False,\n",
       "  'name': 'inheritance of oxidatively modified proteins involved in replicative cell aging',\n",
       "  'definition': {'text': 'A protein localization process in which progeny cells acquire, or are barred from acquiring, proteins that have been altered by reaction with reactive oxygen species in dividing aging cells.'},\n",
       "  'synonyms': [{'name': 'inheritance of oxidatively modified proteins during replicative cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'inheritance of oxidatively modified proteins during replicative cell ageing',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001317', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0008104', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008105', 'relation': 'is_a'}]},\n",
       " 'GO:0001320': {'id': 'GO:0001320',\n",
       "  'isObsolete': False,\n",
       "  'name': 'age-dependent response to reactive oxygen species involved in chronological cell aging',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) occurring in non-dividing cells as they age as a result of reactive oxygen species, where the change varies according to the age of the cell or organism.'},\n",
       "  'synonyms': [{'name': 'age-dependent response to active oxygen species during chronological cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to AOS during chronological cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to reactive oxygen species during chronological cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to ROI during chronological cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to reactive oxygen intermediate during chronological cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to ROS during chronological cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to reactive oxygen species during chronological cell ageing',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to reactive oxidative species during chronological cell aging',\n",
       "    'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001300', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001324', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072353', 'relation': 'is_a'}]},\n",
       " 'GO:0001321': {'id': 'GO:0001321',\n",
       "  'isObsolete': False,\n",
       "  'name': 'age-dependent general metabolic decline involved in replicative cell aging',\n",
       "  'definition': {'text': 'A process of general metabolic decline that arises in dividing cells as they age, and alters cellular metabolism to cause a decline in cell function.'},\n",
       "  'synonyms': [{'name': 'age-dependent general metabolic decline during replicative cell ageing',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent general metabolic decline involved in replicative cell aging',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001322', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001302', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007571', 'relation': 'is_a'}]},\n",
       " 'GO:0001322': {'id': 'GO:0001322',\n",
       "  'isObsolete': False,\n",
       "  'name': 'age-dependent response to oxidative stress involved in replicative cell aging',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) occurring in dividing cells as they age as a result of oxidative stress, where the change varies according to the age of the cell or organism.'},\n",
       "  'synonyms': [{'name': 'age-dependent response to oxidative stress during replicative cell ageing',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to oxidative stress during replicative cell aging',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001316', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001306', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001321', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0034599', 'relation': 'part_of'}]},\n",
       " 'GO:0001323': {'id': 'GO:0001323',\n",
       "  'isObsolete': False,\n",
       "  'name': 'age-dependent general metabolic decline involved in chronological cell aging',\n",
       "  'definition': {'text': 'A process of general metabolic decline that arises in non-dividing cells as they age, and alters cellular metabolism to cause a decline in cell function.'},\n",
       "  'synonyms': [{'name': 'age-dependent general metabolic decline during chronological cell ageing',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent general metabolic decline during chronological cell aging',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001324', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001300', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007571', 'relation': 'is_a'}]},\n",
       " 'GO:0001324': {'id': 'GO:0001324',\n",
       "  'isObsolete': False,\n",
       "  'name': 'age-dependent response to oxidative stress involved in chronological cell aging',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) occurring in non-dividing cells as they age as a result of oxidative stress, where the change varies according to the age of the cell or organism.'},\n",
       "  'synonyms': [{'name': 'age-dependent response to oxidative stress during chronological cell aging',\n",
       "    'type': 'related'},\n",
       "   {'name': 'age-dependent response to oxidative stress during chronological cell ageing',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001320', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001300', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001306', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001323', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0034599', 'relation': 'is_a'}]},\n",
       " 'GO:0001325': {'id': 'GO:0001325',\n",
       "  'isObsolete': False,\n",
       "  'name': 'formation of extrachromosomal circular DNA',\n",
       "  'definition': {'text': 'Excision from the chromosome and circularization of a region of chromosomal DNA, generally, but not always, via homologous recombination between direct tandem repeats.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12044938'}]},\n",
       "  'synonyms': [{'name': 'assembly of extrachromosomal circular DNA',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0090656', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001313', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006259', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022607', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055132', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071844', 'relation': 'is_a'}]},\n",
       " 'GO:0001326': {'id': 'GO:0001326',\n",
       "  'isObsolete': False,\n",
       "  'name': 'replication of extrachromosomal circular DNA',\n",
       "  'definition': {'text': 'Replication of circular DNA following excision from the chromosome; replication of extrachromosomal circular DNA generally occurs independently of chromosomal replication.'},\n",
       "  'children': [{'id': 'GO:0001314', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006260', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055133', 'relation': 'is_a'}]},\n",
       " 'GO:0001400': {'id': 'GO:0001400',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mating projection base',\n",
       "  'definition': {'text': 'The region where the mating projection meets the bulk of the cell, in unicellular fungi exposed to mating pheromone.'},\n",
       "  'synonyms': [{'name': 'base of shmoo tip', 'type': 'narrow'},\n",
       "   {'name': 'conjugation tube base', 'type': 'narrow'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001401': {'id': 'GO:0001401',\n",
       "  'isObsolete': False,\n",
       "  'name': 'SAM complex',\n",
       "  'definition': {'text': 'A large complex of the mitochondrial outer membrane that mediates sorting of some imported proteins to the outer membrane and their assembly in the membrane; functions after import of incoming proteins by the mitochondrial outer membrane translocase complex.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12891361'}]},\n",
       "  'comment': \"See also the cellular component term 'mitochondrial outer membrane translocase complex ; GO:0005742'.\",\n",
       "  'synonyms': [{'name': 'TOB complex', 'type': 'exact'},\n",
       "   {'name': 'mitochondrial sorting and assembly machinery complex',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001402': {'id': 'GO:0001402',\n",
       "  'isObsolete': False,\n",
       "  'name': 'signal transduction involved in filamentous growth',\n",
       "  'definition': {'text': 'Relaying of environmental signals promoting filamentous growth.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9728395'}]},\n",
       "  'synonyms': [{'name': 'signal transduction during filamentous growth',\n",
       "    'type': 'related'},\n",
       "   {'name': 'MAPKKK cascade (pseudohyphal growth)', 'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007165', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030447', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0023033', 'relation': 'is_a'}]},\n",
       " 'GO:0001403': {'id': 'GO:0001403',\n",
       "  'isObsolete': False,\n",
       "  'name': 'invasive growth in response to glucose limitation',\n",
       "  'definition': {'text': 'A growth pattern exhibited by budding haploid cells under certain growth conditions, in which cells retain the typical axial budding pattern of haploids, but become elongated and fail to separate after division; during growth on a solid substrate, this results in penetration of cells into the agar medium. An example of this process is found in Saccharomyces cerevisiae.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9728395'}]},\n",
       "  'comment': \"Note that this term should not be used to describe the invasion of host tissues by pathogenic organisms, which is described by the biological process term 'entry into host ; GO:0044409', nor should it be used to describe growth of diseased cells of an organism into the surrounding normal tissue, which is outside of the scope of GO.\",\n",
       "  'synonyms': [{'name': 'colony morphology', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:2000218', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:2000219', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:2000217', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0036267', 'relation': 'is_a'}]},\n",
       " 'GO:0001405': {'id': 'GO:0001405',\n",
       "  'isObsolete': False,\n",
       "  'name': 'PAM complex, Tim23 associated import motor',\n",
       "  'definition': {'text': 'Protein complex located on the matrix side of the mitochondrial inner membrane and associated with the TIM23 mitochondrial import inner membrane translocase complex (GO:0005744); ATPase motor activity to drive import of proteins into the mitochondrial matrix.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14517234'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14638855'}]},\n",
       "  'synonyms': [{'name': 'pre-sequence translocase-associated import motor',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'mitochondrial import motor', 'type': 'exact'},\n",
       "   {'name': 'PAM complex', 'type': 'exact'},\n",
       "   {'name': 'presequence translocase-associated import motor',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001406': {'id': 'GO:0001406',\n",
       "  'isObsolete': False,\n",
       "  'name': 'glycerophosphodiester transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of glycerophosphodiesters from one side of a membrane to the other. Glycerophosphodiesters are small molecules composed of glycerol-3-phosphate and an alcohol, for example, glycerophosphoinositol.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12912892'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001407', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0008514', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015605', 'relation': 'is_a'}]},\n",
       " 'GO:0001407': {'id': 'GO:0001407',\n",
       "  'isObsolete': False,\n",
       "  'name': 'glycerophosphodiester transmembrane transport',\n",
       "  'definition': {'text': 'The process in which a glycerophosphodiester is transported across a membrane. Glycerophosphodiesters are small molecules composed of glycerol-3-phosphate and an alcohol, for example, glycerophosphoinositol.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12912892'}]},\n",
       "  'children': [{'id': 'GO:0001406', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015711', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015748', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098656', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0099133', 'relation': 'is_a'}]},\n",
       " 'GO:0001408': {'id': 'GO:0001408',\n",
       "  'isObsolete': False,\n",
       "  'name': 'guanine nucleotide transport',\n",
       "  'definition': {'text': 'The directed movement of guanine nucleotides, GTP, GDP, and/or GMP, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore.'},\n",
       "  'children': [{'id': 'GO:1903790', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070731', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015865', 'relation': 'is_a'}]},\n",
       " 'GO:0001409': {'id': 'GO:0001409',\n",
       "  'isObsolete': False,\n",
       "  'name': 'guanine nucleotide transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of guanine nucleotides (GMP, GDP, and GTP) from one side of a membrane to the other.'},\n",
       "  'children': [{'id': 'GO:0010292', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140360', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905948', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015216', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903790', 'relation': 'part_of'}]},\n",
       " 'GO:0001410': {'id': 'GO:0001410',\n",
       "  'isObsolete': False,\n",
       "  'name': 'chlamydospore formation',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of the chlamydospore over time, from its formation to the mature structure. A chlamydospores is a mitotic (asexual) one-celled spore, produced primarily for survival, not dispersal, originating endogenously and singly within part of a pre-existing cell and possessing an inner secondary and often thickened cell wall. An example of this is found in Candida albicans.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '085199377X'},\n",
       "    {'dbCode': 'PMID', 'dbId': '14663094'}]},\n",
       "  'synonyms': [{'name': 'chlamydospore development', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0043936', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048468', 'relation': 'is_a'}]},\n",
       " 'GO:0001411': {'id': 'GO:0001411',\n",
       "  'isObsolete': False,\n",
       "  'name': 'hyphal tip',\n",
       "  'definition': {'text': 'The end, or tip, of a fungal hypha, where polarized growth occurs during hyphal elongation.'},\n",
       "  'children': [{'id': 'GO:0031521', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031562', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001501': {'id': 'GO:0001501',\n",
       "  'isObsolete': False,\n",
       "  'name': 'skeletal system development',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of the skeleton over time, from its formation to the mature structure. The skeleton is the bony framework of the body in vertebrates (endoskeleton) or the hard outer envelope of insects (exoskeleton or dermoskeleton).',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://www.stedmans.com/'}]},\n",
       "  'synonyms': [{'name': 'skeletal development', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0048706', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048705', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051216', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060348', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048731', 'relation': 'is_a'}]},\n",
       " 'GO:0001502': {'id': 'GO:0001502',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cartilage condensation',\n",
       "  'definition': {'text': 'The condensation of mesenchymal cells that have been committed to differentiate into chondrocytes.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0878932437'}]},\n",
       "  'children': [{'id': 'GO:1902026', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1904932', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1902027', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048705', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0051216', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098743', 'relation': 'is_a'}]},\n",
       " 'GO:0001503': {'id': 'GO:0001503',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ossification',\n",
       "  'definition': {'text': 'The formation of bone or of a bony substance, or the conversion of fibrous tissue or of cartilage into bone or a bony substance.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17572649'}]},\n",
       "  'comment': \"Note that this term does not have a 'developmental process' parent because ossification isn't necessarily developmental, can also occur as part of bone remodeling. Instead use 'ossification involved in bone maturation ; GO:0043931'.\",\n",
       "  'synonyms': [{'name': 'bone biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'osteogenesis', 'type': 'exact'},\n",
       "   {'name': 'bone formation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0030279', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0045778', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0036077', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043931', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036075', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036076', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030278', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0036072', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030282', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043932', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001649', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0032501', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044707', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050874', 'relation': 'is_a'}]},\n",
       " 'GO:0001504': {'id': 'GO:0001504',\n",
       "  'isObsolete': False,\n",
       "  'name': 'neurotransmitter uptake',\n",
       "  'definition': {'text': 'The directed movement of neurotransmitters into neurons or glial cells. This process leads to inactivation and recycling of neurotransmitters.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0123668387'}]},\n",
       "  'synonyms': [{'name': 'neurotransmitter import into neuron',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'neurotransmitter recycling', 'type': 'broad'},\n",
       "   {'name': 'neurotransmitter import into glial cell', 'type': 'narrow'},\n",
       "   {'name': 'neurotransmitter import', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0051580', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1903804', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051581', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0051615', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098712', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051582', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0098810', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0140009', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001505', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006836', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098657', 'relation': 'is_a'}]},\n",
       " 'GO:0001505': {'id': 'GO:0001505',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of neurotransmitter levels',\n",
       "  'definition': {'text': 'Any process that modulates levels of neurotransmitter.'},\n",
       "  'children': [{'id': 'GO:1901413', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051580', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080164', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046928', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901715', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045428', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001504', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905627', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060408', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007269', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042137', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904407', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042133', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900336', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904406', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0065008', 'relation': 'is_a'}]},\n",
       " 'GO:0001507': {'id': 'GO:0001507',\n",
       "  'isObsolete': False,\n",
       "  'name': 'acetylcholine catabolic process in synaptic cleft',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of acetylcholine that occurs in the synaptic cleft during synaptic transmission.'},\n",
       "  'synonyms': [{'name': 'acetylcholine breakdown in synaptic cleft',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'acetylcholine degradation in synaptic cleft', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006581', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007271', 'relation': 'part_of'}]},\n",
       " 'GO:0001508': {'id': 'GO:0001508',\n",
       "  'isObsolete': False,\n",
       "  'name': 'action potential',\n",
       "  'definition': {'text': 'A process in which membrane potential cycles through a depolarizing spike, triggered in response to depolarization above some threshold, followed by repolarization. This cycle is driven by the flow of ions through various voltage gated channels with different thresholds and ion specificities.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '978-0-07-139011-8'}]},\n",
       "  'comment': 'Action potentials typically propagate across excitable membranes. This class covers both action potentials that propagate and those that fail to do so.',\n",
       "  'children': [{'id': 'GO:0019228', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098870', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097364', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0100001', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0086001', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0086011', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0086010', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045759', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0045760', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0098900', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0099610', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0042391', 'relation': 'is_a'}]},\n",
       " 'GO:0001510': {'id': 'GO:0001510',\n",
       "  'isObsolete': False,\n",
       "  'name': 'RNA methylation',\n",
       "  'definition': {'text': \"Posttranscriptional addition of a methyl group to either a nucleotide or 2'-O ribose in a polyribonucleotide. Usually uses S-adenosylmethionine as a cofactor.\"},\n",
       "  'children': [{'id': 'GO:0030488', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990437', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031167', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008173', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0036261', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0080009', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061715', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036265', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990744', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009451', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043414', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016547', 'relation': 'is_a'}]},\n",
       " 'GO:0001512': {'id': 'GO:0001512',\n",
       "  'isObsolete': False,\n",
       "  'name': 'dihydronicotinamide riboside quinone reductase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 1-(beta-D-ribofuranosyl)-1,4-dihydronicotinamide + a quinone = 1-(beta-D-ribofuranosyl)nicotinamide + a hydroquinone.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.10.99.2'}]},\n",
       "  'synonyms': [{'name': 'N-ribosyldihydronicotinamide dehydrogenase (quinone) activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'NRH:quinone oxidoreductase 2 activity', 'type': 'narrow'},\n",
       "   {'name': 'QR2 activity', 'type': 'related'},\n",
       "   {'name': 'quinone reductase 2 activity', 'type': 'related'},\n",
       "   {'name': 'ribosyldihydronicotinamide dehydrogenase (quinone) activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'NAD(P)H:quinone oxidoreductase 2 activity', 'type': 'exact'},\n",
       "   {'name': 'NQO(2) activity', 'type': 'exact'},\n",
       "   {'name': 'NQO2', 'type': 'related'},\n",
       "   {'name': 'NAD(P)H:quinone oxidoreductase2', 'type': 'related'},\n",
       "   {'name': 'NAD(P)H:quinone oxidoreductase-2', 'type': 'related'},\n",
       "   {'name': '1-(beta-D-ribofuranosyl)-1,4-dihydronicotinamide:quinone oxidoreductase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016491', 'relation': 'is_a'}]},\n",
       " 'GO:0001514': {'id': 'GO:0001514',\n",
       "  'isObsolete': False,\n",
       "  'name': 'selenocysteine incorporation',\n",
       "  'definition': {'text': 'The incorporation of selenocysteine into a peptide; uses a special tRNA that recognizes the UGA codon as selenocysteine, rather than as a termination codon. Selenocysteine is synthesized from serine before its incorporation; it is not a posttranslational modification of peptidyl-cysteine.',\n",
       "   'xrefs': [{'dbCode': 'RESID', 'dbId': 'AA0022'}]},\n",
       "  'children': [{'id': 'GO:1904569', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1904570', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1904571', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006451', 'relation': 'is_a'}]},\n",
       " 'GO:0001515': {'id': 'GO:0001515',\n",
       "  'isObsolete': False,\n",
       "  'name': 'opioid peptide activity',\n",
       "  'definition': {'text': 'Naturally occurring peptide that is an opioid (any non-alkaloid having an opiate-like effect that can be reversed by naloxone or other recognized morphine antagonist). These include Leu- and Met-enkephalin, dynorphin and neoendorphin, alpha, beta, gamma and delta endorphins formed from beta-lipotropin, various pronase-resistant peptides such as beta casamorphin, and other peptides whose opiate-like action seems to be indirect.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048018', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071884', 'relation': 'is_a'}]},\n",
       " 'GO:0001516': {'id': 'GO:0001516',\n",
       "  'isObsolete': False,\n",
       "  'name': 'prostaglandin biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of prostaglandins, any of a group of biologically active metabolites which contain a cyclopentane ring.'},\n",
       "  'synonyms': [{'name': 'prostaglandin synthesis', 'type': 'exact'},\n",
       "   {'name': 'prostaglandin formation', 'type': 'exact'},\n",
       "   {'name': 'prostaglandin biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'prostaglandin anabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0031393', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0019371', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0100008', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031392', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0031394', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006693', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046457', 'relation': 'is_a'}]},\n",
       " 'GO:0001517': {'id': 'GO:0001517',\n",
       "  'isObsolete': False,\n",
       "  'name': 'N-acetylglucosamine 6-O-sulfotransferase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + N-acetyl-D-glucosamine = adenosine 3',5'-bisphosphate + N-acetyl-D-glucosamine 6-sulfate.\"},\n",
       "  'synonyms': [{'name': 'N-acetylglucosamine 6-O-sulphotransferase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008146', 'relation': 'is_a'}]},\n",
       " 'GO:0001518': {'id': 'GO:0001518',\n",
       "  'isObsolete': False,\n",
       "  'name': 'voltage-gated sodium channel complex',\n",
       "  'definition': {'text': 'A sodium channel in a cell membrane whose opening is governed by the membrane potential.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'voltage gated sodium channel complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'voltage-dependent sodium channel complex', 'type': 'exact'},\n",
       "   {'name': 'voltage-sensitive sodium channel complex', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005248', 'relation': 'capable_of'}]},\n",
       " 'GO:0001519': {'id': 'GO:0001519',\n",
       "  'isObsolete': False,\n",
       "  'name': 'peptide amidation',\n",
       "  'definition': {'text': 'The posttranslational conversion of C-terminal glycine-extended peptides to C-terminal alpha-amidated peptides. Occurs to over half of all peptide hormones to give bioactive peptides. This is a two step process catalyzed by a peptidyl-glycine alpha-hydroxylating monooxygenase and a peptidyl-alpha-hydroxyglycine alpha-amidating lyase. In some organisms, this process is catalyzed by two separate enzymes, whereas in higher organisms, one polypeptide catalyzes both reactions.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11028916'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031179', 'relation': 'is_a'}]},\n",
       " 'GO:0001520': {'id': 'GO:0001520',\n",
       "  'isObsolete': False,\n",
       "  'name': 'outer dense fiber',\n",
       "  'definition': {'text': 'A supramolecular fiber found in the flagella of mammalian sperm that surrounds the nine microtubule doublets. These dense fibers are stiff and noncontractile. In human, they consist of about 10 major and at least 15 minor proteins, where all major proteins are ODF1, ODF2 or ODF2-related proteins.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0824072820'},\n",
       "    {'dbCode': 'PMID', 'dbId': '10381817'},\n",
       "    {'dbCode': 'PMID', 'dbId': '21586547'},\n",
       "    {'dbCode': 'PMID', 'dbId': '25361759'}]},\n",
       "  'synonyms': [{'name': 'outer dense fibre', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001522': {'id': 'GO:0001522',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pseudouridine synthesis',\n",
       "  'definition': {'text': 'The intramolecular conversion of uridine to pseudouridine within an RNA molecule. This posttranscriptional base modification occurs in tRNA, rRNA, and snRNAs.'},\n",
       "  'synonyms': [{'name': 'pseudouridylation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0031120', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004730', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0031119', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031118', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990481', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009451', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016547', 'relation': 'is_a'}]},\n",
       " 'GO:0001523': {'id': 'GO:0001523',\n",
       "  'isObsolete': False,\n",
       "  'name': 'retinoid metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving retinoids, any member of a class of isoprenoids that contain or are derived from four prenyl groups linked head-to-tail. Retinoids include retinol and retinal and structurally similar natural derivatives or synthetic compounds, but need not have vitamin A activity.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'retinoid metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0042572', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042573', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042574', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006776', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016101', 'relation': 'is_a'}]},\n",
       " 'GO:0001525': {'id': 'GO:0001525',\n",
       "  'isObsolete': False,\n",
       "  'name': 'angiogenesis',\n",
       "  'definition': {'text': 'Blood vessel formation when new vessels emerge from the proliferation of pre-existing blood vessels.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0878932453'}]},\n",
       "  'synonyms': [{'name': 'blood vessel formation from pre-existing blood vessels',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0045766', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0045765', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0016525', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0060055', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072104', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001569', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060978', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002041', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002040', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048514', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048646', 'relation': 'is_a'}]},\n",
       " 'GO:0001527': {'id': 'GO:0001527',\n",
       "  'isObsolete': False,\n",
       "  'name': 'microfibril',\n",
       "  'definition': {'text': 'Extracellular matrix components occurring independently or along with elastin. Thought to have force-bearing functions in tendon. In addition to fibrillins, microfibrils may contain other associated proteins.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '27026396'}]},\n",
       "  'synonyms': [{'name': 'extended fibrils', 'type': 'exact'},\n",
       "   {'name': 'fibrillin', 'type': 'related'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001530': {'id': 'GO:0001530',\n",
       "  'isObsolete': False,\n",
       "  'name': 'lipopolysaccharide binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with lipopolysaccharide.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11079463'}]},\n",
       "  'synonyms': [{'name': 'endotoxin binding', 'type': 'broad'},\n",
       "   {'name': 'LPS binding', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008289', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097367', 'relation': 'is_a'}]},\n",
       " 'GO:0001531': {'id': 'GO:0001531',\n",
       "  'isObsolete': False,\n",
       "  'name': 'interleukin-21 receptor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with the interleukin-21 receptor.'},\n",
       "  'synonyms': [{'name': 'IL-21', 'type': 'narrow'},\n",
       "   {'name': 'interleukin-21 receptor ligand', 'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005126', 'relation': 'is_a'}]},\n",
       " 'GO:0001532': {'id': 'GO:0001532',\n",
       "  'isObsolete': False,\n",
       "  'name': 'interleukin-21 receptor activity',\n",
       "  'definition': {'text': 'Combining with interleukin-21 and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity.'},\n",
       "  'synonyms': [{'name': 'IL-21R', 'type': 'exact'},\n",
       "   {'name': 'IL-21 receptor activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004896', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0038114', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0004907', 'relation': 'is_a'}]},\n",
       " 'GO:0001533': {'id': 'GO:0001533',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cornified envelope',\n",
       "  'definition': {'text': 'A type of plasma membrane that has been modified through addition of distinct intracellular and extracellular components, including ceramide, found in cornifying epithelial cells (corneocytes).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11112355'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11590230'},\n",
       "    {'dbCode': 'PMID', 'dbId': '15803139'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001534': {'id': 'GO:0001534',\n",
       "  'isObsolete': False,\n",
       "  'name': 'radial spoke',\n",
       "  'definition': {'text': 'Protein complex that links the outer microtubule doublet of the ciliary or flagellum axoneme with the sheath that surrounds the central pair of microtubules. Composed of a stalk that attaches to each doublet microtubule and a globular structure (spoke head) that projects toward the central pair of microtubules.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0124325653'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9450971'}]},\n",
       "  'children': [{'id': 'GO:0001535', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001536', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001535': {'id': 'GO:0001535',\n",
       "  'isObsolete': False,\n",
       "  'name': 'radial spoke head',\n",
       "  'definition': {'text': 'Protein complex forming part of eukaryotic flagellar apparatus.'},\n",
       "  'synonyms': [{'name': 'radial spokehead', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001536': {'id': 'GO:0001536',\n",
       "  'isObsolete': False,\n",
       "  'name': 'radial spoke stalk',\n",
       "  'definition': {'text': 'Globular portion of the radial spoke that projects towards the central pair of microtubules.'},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001537': {'id': 'GO:0001537',\n",
       "  'isObsolete': False,\n",
       "  'name': 'N-acetylgalactosamine 4-O-sulfotransferase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + N-acetyl-D-galactosamine = adenosine 3',5'-bisphosphate + N-acetyl-D-galactosamine 4-sulfate.\",\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.8.2.-'}]},\n",
       "  'synonyms': [{'name': 'N-acetylgalactosamine 4-O-sulphotransferase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008146', 'relation': 'is_a'}]},\n",
       " 'GO:0001539': {'id': 'GO:0001539',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cilium or flagellum-dependent cell motility',\n",
       "  'definition': {'text': 'Cell motility due to movement of eukaryotic cilia or bacterial-type flagella or archaeal-type flagella.'},\n",
       "  'comment': \"Note that we deem eukaryotic cilia and microtubule-based flagella to be equivalent, while the bacterial- and archaeal-type flagella have a different structure. The former are microtubule-based structures that lash back and forth and are present only in eukaryotes, while the latter achieve motility by rotation. Bacterial- and archaeal-type flagella are superficially similar but have a different molecular composition and fine structure. These three structures never co-exist in the same organism. Therefore, GO:0001539 'cilium or flagellum-dependent cell motility' is in the subset of terms that should not be used for direct gene product annotation. Instead, select a child term. Direct annotations to GO:0001539 'cilium or flagellum-dependent cell motility' may be amended during annotation QC.\",\n",
       "  'synonyms': [{'name': 'ciliary/flagellar motility', 'type': 'exact'},\n",
       "   {'name': 'ciliary or bacterial-type flagellar motility',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0060285', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097588', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Electronic',\n",
       "  'parent': [{'id': 'GO:0048870', 'relation': 'is_a'}]},\n",
       " 'GO:0001540': {'id': 'GO:0001540',\n",
       "  'isObsolete': False,\n",
       "  'name': 'amyloid-beta binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with amyloid-beta peptide/protein and/or its precursor.'},\n",
       "  'synonyms': [{'name': 'beta-amyloid binding', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0042277', 'relation': 'is_a'}]},\n",
       " 'GO:0001541': {'id': 'GO:0001541',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ovarian follicle development',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of the ovarian follicle over time, from its formation to the mature structure.'},\n",
       "  'synonyms': [{'name': 'follicular phase', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:2000355', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0042699', 'relation': 'part_of'},\n",
       "   {'id': 'GO:2000386', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0001544', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001545', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001546', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0035471', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001547', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001551', 'relation': 'part_of'},\n",
       "   {'id': 'GO:2000354', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008585', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048608', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061039', 'relation': 'part_of'}]},\n",
       " 'GO:0001542': {'id': 'GO:0001542',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ovulation from ovarian follicle',\n",
       "  'definition': {'text': 'The process leading to the rupture of the follicle, releasing the centrally located oocyte into the oviduct. An example of this is found in Mus musculus.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'children': [{'id': 'GO:0001543', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008585', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030728', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061039', 'relation': 'part_of'}]},\n",
       " 'GO:0001543': {'id': 'GO:0001543',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ovarian follicle rupture',\n",
       "  'definition': {'text': 'Disruption of theca cell layer releasing follicular fluid and/or the oocyte.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001542', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'}]},\n",
       " 'GO:0001544': {'id': 'GO:0001544',\n",
       "  'isObsolete': False,\n",
       "  'name': 'initiation of primordial ovarian follicle growth',\n",
       "  'definition': {'text': 'Increase in size of primordial follicles including proliferation and shape changes of granulosa and/or theca cells until oocyte is surrounded by one layer of cuboidal shaped granulosa cells (primary follicle).',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001541', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'}]},\n",
       " 'GO:0001545': {'id': 'GO:0001545',\n",
       "  'isObsolete': False,\n",
       "  'name': 'primary ovarian follicle growth',\n",
       "  'definition': {'text': 'Increase in size of primary follicles including oocyte growth and granulosa and/or theca cell proliferation until more than one layer of granulosa cells is present (preantral follicle).',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'children': [{'id': 'GO:0043929', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043930', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001541', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0040007', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048590', 'relation': 'is_a'}]},\n",
       " 'GO:0001546': {'id': 'GO:0001546',\n",
       "  'isObsolete': False,\n",
       "  'name': 'preantral ovarian follicle growth',\n",
       "  'definition': {'text': 'Increase in size of follicles surrounded by two or more layers of granulosa cells up to the onset of antrum formation.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001541', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0040007', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048162', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048590', 'relation': 'is_a'}]},\n",
       " 'GO:0001547': {'id': 'GO:0001547',\n",
       "  'isObsolete': False,\n",
       "  'name': 'antral ovarian follicle growth',\n",
       "  'definition': {'text': 'Increase in size of antral follicles due to cell proliferation and/or growth of the antral cavity.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'children': [{'id': 'GO:0001550', 'relation': 'part_of'},\n",
       "   {'id': 'GO:2000388', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:2000387', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001548', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001549', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001541', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0003006', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048589', 'relation': 'is_a'}]},\n",
       " 'GO:0001548': {'id': 'GO:0001548',\n",
       "  'isObsolete': False,\n",
       "  'name': 'follicular fluid formation in ovarian follicle antrum',\n",
       "  'definition': {'text': 'The menstrual cycle process that results in the formation of one central cavity separating the oocyte/cumulus complex from mural granulosa and theca cells during the various stages of oogenesis.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'synonyms': [{'name': 'ovarian follicle antrum/follicular fluid biosynthesis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ovarian follicle antrum/follicular fluid formation',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0003003', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003004', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003005', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001547', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'}]},\n",
       " 'GO:0001549': {'id': 'GO:0001549',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cumulus cell differentiation',\n",
       "  'definition': {'text': 'The process in which a subpopulation of granulosa cells surrounding the oocyte acquires the specialized features of an ovarian cumulus cell.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'synonyms': [{'name': 'ovarian cumulus cell differentiation',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0045594', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0045592', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0045593', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001547', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0003006', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022412', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030855', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048165', 'relation': 'part_of'}]},\n",
       " 'GO:0001550': {'id': 'GO:0001550',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ovarian cumulus expansion',\n",
       "  'definition': {'text': 'Increase in size of the cumulus surrounding the oocyte including change in morphology due to proliferation and dispersion of cumulus cells.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'synonyms': [{'name': 'ovarian cumulus growth', 'type': 'related'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001547', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0040007', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048165', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048590', 'relation': 'is_a'}]},\n",
       " 'GO:0001551': {'id': 'GO:0001551',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ovarian follicle endowment',\n",
       "  'definition': {'text': 'Association of oocytes with supporting epithelial granulosa cells to form primordial follicles.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001541', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'}]},\n",
       " 'GO:0001552': {'id': 'GO:0001552',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ovarian follicle atresia',\n",
       "  'definition': {'text': 'A periodic process in which immature ovarian follicles degenerate and are subsequently re-absorbed.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18638134'},\n",
       "    {'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003006', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008585', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061039', 'relation': 'part_of'}]},\n",
       " 'GO:0001553': {'id': 'GO:0001553',\n",
       "  'isObsolete': False,\n",
       "  'name': 'luteinization',\n",
       "  'definition': {'text': 'The set of processes resulting in differentiation of theca and granulosa cells into luteal cells and in the formation of a corpus luteum after ovulation.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'synonyms': [{'name': 'luteal phase', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0042701', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042702', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008585', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061039', 'relation': 'part_of'}]},\n",
       " 'GO:0001554': {'id': 'GO:0001554',\n",
       "  'isObsolete': False,\n",
       "  'name': 'luteolysis',\n",
       "  'definition': {'text': 'The lysis or structural demise of the corpus luteum. During normal luteolysis, two closely related events occur. First, there is loss of the capacity to synthesize and secrete progesterone (functional luteolysis) followed by loss of the cells that comprise the corpus luteum (structural luteolysis). Preventing luteolysis is crucial to maintain pregnancy.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10617764'},\n",
       "    {'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'children': [{'id': 'GO:0061364', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061363', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008585', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022602', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061039', 'relation': 'part_of'}]},\n",
       " 'GO:0001555': {'id': 'GO:0001555',\n",
       "  'isObsolete': False,\n",
       "  'name': 'oocyte growth',\n",
       "  'definition': {'text': 'The developmental growth process in which an oocyte irreversibly increases in size over time by accretion and biosynthetic production of matter similar to that already present.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'children': [{'id': 'GO:0030715', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003006', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022412', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048588', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048599', 'relation': 'part_of'}]},\n",
       " 'GO:0001556': {'id': 'GO:0001556',\n",
       "  'isObsolete': False,\n",
       "  'name': 'oocyte maturation',\n",
       "  'definition': {'text': 'A developmental process, independent of morphogenetic (shape) change, that is required for an oocyte to attain its fully functional state. Oocyte maturation commences after reinitiation of meiosis commonly starting with germinal vesicle breakdown, and continues up to the second meiotic arrest prior to fertilization.',\n",
       "   'xrefs': [{'dbCode': 'URL', 'dbId': 'http://ovary.stanford.edu'}]},\n",
       "  'children': [{'id': 'GO:1900194', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1900193', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1900195', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1903537', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003006', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022412', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048469', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048599', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0071695', 'relation': 'is_a'}]},\n",
       " 'GO:0001558': {'id': 'GO:0001558',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of cell growth',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate, extent or direction of cell growth.'},\n",
       "  'children': [{'id': 'GO:2000220', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051510', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061387', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061388', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001560', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030307', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030308', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061050', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903041', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903859', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061389', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1902890', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016049', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0040008', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051128', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048591', 'relation': 'regulates'}]},\n",
       " 'GO:0001559': {'id': 'GO:0001559',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of cell growth by detection of nuclear:cytoplasmic ratio',\n",
       "  'definition': {'text': 'Any process in which the size of the nucleus with respect to the cytoplasm modulates the frequency, rate or extent of cell growth, the irreversible increase in size of a cell over time.'},\n",
       "  'synonyms': [{'name': 'regulation of cell growth by nuclear:cytoplasmic ratio',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'interpretation of nuclear:cytoplasmic ratio to regulate cell growth',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'regulation of cell growth by sensing of nuclear:cytoplasmic ratio',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'detection of nuclear:cytoplasmic ratio to regulate cell growth',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016049', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0016475', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061387', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048591', 'relation': 'regulates'}]},\n",
       " 'GO:0001560': {'id': 'GO:0001560',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of cell growth by extracellular stimulus',\n",
       "  'definition': {'text': 'Any process in which external signals modulate the frequency, rate or extent of cell growth, the irreversible increase in size of a cell over time.'},\n",
       "  'synonyms': [{'name': 'regulation of cell growth by detection of exogenous stimulus',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'regulation of growth by external signal', 'type': 'exact'},\n",
       "   {'name': 'regulation of growth by exogenous stimulus', 'type': 'exact'},\n",
       "   {'name': 'regulation of growth by external stimulus', 'type': 'exact'},\n",
       "   {'name': 'interpretation of external signals that regulate cell growth',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'regulation of growth by external stimuli', 'type': 'exact'},\n",
       "   {'name': 'regulation of cell growth by sensing of exogenous stimulus',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'regulation of growth by exogenous signal', 'type': 'exact'},\n",
       "   {'name': 'regulation of growth by exogenous stimuli', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001558', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031668', 'relation': 'part_of'}]},\n",
       " 'GO:0001561': {'id': 'GO:0001561',\n",
       "  'isObsolete': False,\n",
       "  'name': 'fatty acid alpha-oxidation',\n",
       "  'definition': {'text': 'A metabolic pathway by which 3-methyl branched fatty acids are degraded. These compounds are not degraded by the normal peroxisomal beta-oxidation pathway, because the 3-methyl blocks the dehydrogenation of the hydroxyl group by hydroxyacyl-CoA dehydrogenase. The 3-methyl branched fatty acid is converted in several steps to pristenic acid, which can then feed into the beta-oxidative pathway.',\n",
       "   'xrefs': [{'dbCode': 'URL',\n",
       "     'dbId': 'http://www.peroxisome.org/Scientist/Biochemistry/alpha-oxidation.html'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009062', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019395', 'relation': 'is_a'}]},\n",
       " 'GO:0001562': {'id': 'GO:0001562',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to protozoan',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus from a protozoan.'},\n",
       "  'synonyms': [{'name': 'resistance to pathogenic protozoa',\n",
       "    'type': 'related'},\n",
       "   {'name': 'response to protozoon', 'type': 'exact'},\n",
       "   {'name': 'response to protozoa', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001563', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0099141', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042832', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0051707', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042828', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009613', 'relation': 'is_a'}]},\n",
       " 'GO:0001563': {'id': 'GO:0001563',\n",
       "  'isObsolete': False,\n",
       "  'name': 'detection of protozoan',\n",
       "  'definition': {'text': 'The series of events in which a stimulus from a protozoan is received and converted into a molecular signal.'},\n",
       "  'synonyms': [{'name': 'detection of protozoon', 'type': 'exact'},\n",
       "   {'name': 'perception of protozoa', 'type': 'related'},\n",
       "   {'name': 'detection of protozoa', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001562', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098543', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042833', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051824', 'relation': 'is_a'}]},\n",
       " 'GO:0001565': {'id': 'GO:0001565',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phorbol ester receptor activity',\n",
       "  'definition': {'text': 'Combining with a phorbol ester and transmitting the signal to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10506570'}]},\n",
       "  'children': [{'id': 'GO:0001566', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0038023', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004872', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019041', 'relation': 'is_a'}]},\n",
       " 'GO:0001566': {'id': 'GO:0001566',\n",
       "  'isObsolete': False,\n",
       "  'name': 'non-kinase phorbol ester receptor activity',\n",
       "  'definition': {'text': 'Combining with a phorbol ester and transmitting the signal by a mechanism independent of kinase activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10506570'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001565', 'relation': 'is_a'}]},\n",
       " 'GO:0001567': {'id': 'GO:0001567',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cholesterol 25-hydroxylase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: AH(2) + cholesterol + O(2) = 25-hydroxycholesterol + A + H(2)O.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.14.99.38'},\n",
       "    {'dbCode': 'RHEA', 'dbId': '21104'}]},\n",
       "  'synonyms': [{'name': 'cholesterol 25-monooxygenase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cholesterol,hydrogen-donor:oxygen oxidoreductase (25-hydroxylating) activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008395', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016705', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008394', 'relation': 'is_a'}]},\n",
       " 'GO:0001568': {'id': 'GO:0001568',\n",
       "  'isObsolete': False,\n",
       "  'name': 'blood vessel development',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of a blood vessel over time, from its formation to the mature structure. The blood vessel is the vasculature carrying blood.',\n",
       "   'xrefs': [{'dbCode': 'UBERON', 'dbId': '0001981'}]},\n",
       "  'children': [{'id': 'GO:0060674', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035779', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001955', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072012', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060841', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048514', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060840', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060837', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060976', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001944', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048856', 'relation': 'is_a'}]},\n",
       " 'GO:0001569': {'id': 'GO:0001569',\n",
       "  'isObsolete': False,\n",
       "  'name': 'branching involved in blood vessel morphogenesis',\n",
       "  'definition': {'text': 'The process of coordinated growth and sprouting of blood vessels giving rise to the organized vascular system.'},\n",
       "  'synonyms': [{'name': 'patterning of blood vessels', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:1905553', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1905555', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1905554', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001525', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048514', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048754', 'relation': 'is_a'}]},\n",
       " 'GO:0001570': {'id': 'GO:0001570',\n",
       "  'isObsolete': False,\n",
       "  'name': 'vasculogenesis',\n",
       "  'definition': {'text': 'The differentiation of endothelial cells from progenitor cells during blood vessel development, and the de novo formation of blood vessels and tubes.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '8999798'}]},\n",
       "  'synonyms': [{'name': 'vascular morphogenesis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0022009', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2001213', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:2001212', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0035441', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060979', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2001214', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030154', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048514', 'relation': 'part_of'}]},\n",
       " 'GO:0001571': {'id': 'GO:0001571',\n",
       "  'isObsolete': False,\n",
       "  'name': 'non-tyrosine kinase fibroblast growth factor receptor activity',\n",
       "  'definition': {'text': 'Combining with fibroblast growth factor (FGF) and transmitting the signal from one side of the membrane to the other by a mechanism independent of tyrosine kinase activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11418238'}]},\n",
       "  'synonyms': [{'name': 'non-tyrosine kinase FGFR activity', 'type': 'exact'},\n",
       "   {'name': 'non-tyrosine kinase FGF receptor activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004888', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004926', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0099600', 'relation': 'is_a'}]},\n",
       " 'GO:0001572': {'id': 'GO:0001572',\n",
       "  'isObsolete': False,\n",
       "  'name': 'lactosylceramide biosynthetic process',\n",
       "  'definition': {'text': \"The chemical reactions and pathways resulting in the formation of lactosylceramides, Gal-beta-(1->4)-Glc-beta(1->1') ceramides, any compound formed by the replacement of the glycosidic C1 hydroxyl group of lactose by a ceramide group. They are the precursors of both gangliosides and globosides.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'},\n",
       "    {'dbCode': 'ISBN', 'dbId': '0471586501'}]},\n",
       "  'synonyms': [{'name': 'lactosylceramide biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'lactosylceramide formation', 'type': 'exact'},\n",
       "   {'name': 'lactosylceramide anabolism', 'type': 'exact'},\n",
       "   {'name': 'lactosylceramide synthesis', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006688', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046478', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046513', 'relation': 'is_a'}]},\n",
       " 'GO:0001573': {'id': 'GO:0001573',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ganglioside metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving ceramide oligosaccharides carrying in addition to other sugar residues, one or more sialic acid residues.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'ganglioside metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0006689', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001574', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006672', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006687', 'relation': 'is_a'}]},\n",
       " 'GO:0001574': {'id': 'GO:0001574',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ganglioside biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of ceramide oligosaccharides carrying in addition to other sugar residues, one or more sialic acid residues.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'ganglioside formation', 'type': 'exact'},\n",
       "   {'name': 'ganglioside anabolism', 'type': 'exact'},\n",
       "   {'name': 'ganglioside biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'ganglioside synthesis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0010706', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001573', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006688', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046513', 'relation': 'is_a'}]},\n",
       " 'GO:0001575': {'id': 'GO:0001575',\n",
       "  'isObsolete': False,\n",
       "  'name': 'globoside metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving globosides, globotetraosylceramides, ceramides containing a core structure of GalNAc-beta-(1->3)-Gal-alpha-(1->4)-Glc(I). Globosides are the major neutral glycosphingolipid in normal kidneys and erythrocytes.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'globoside metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001576', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006687', 'relation': 'is_a'}]},\n",
       " 'GO:0001576': {'id': 'GO:0001576',\n",
       "  'isObsolete': False,\n",
       "  'name': 'globoside biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of a ceramide with a core structure of GalNAc-beta-(1->3)-Gal-alpha-(1->4)-Glc(I).',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'globoside formation', 'type': 'exact'},\n",
       "   {'name': 'globoside synthesis', 'type': 'exact'},\n",
       "   {'name': 'globoside anabolism', 'type': 'exact'},\n",
       "   {'name': 'globoside biosynthesis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0010707', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001575', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006688', 'relation': 'is_a'}]},\n",
       " 'GO:0001578': {'id': 'GO:0001578',\n",
       "  'isObsolete': False,\n",
       "  'name': 'microtubule bundle formation',\n",
       "  'definition': {'text': 'A process that results in a parallel arrangement of microtubules.'},\n",
       "  'synonyms': [{'name': 'microtubule bundling', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0035082', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903562', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903563', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0000226', 'relation': 'is_a'}]},\n",
       " 'GO:0001579': {'id': 'GO:0001579',\n",
       "  'isObsolete': False,\n",
       "  'name': 'medium-chain fatty acid transport',\n",
       "  'definition': {'text': 'The directed movement of medium-chain fatty acids into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore. A medium-chain fatty acid is a fatty acid with a chain length of between C6 and C12.',\n",
       "   'xrefs': [{'dbCode': 'CHEBI', 'dbId': '59554'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0015908', 'relation': 'is_a'}]},\n",
       " 'GO:0001580': {'id': 'GO:0001580',\n",
       "  'isObsolete': False,\n",
       "  'name': 'detection of chemical stimulus involved in sensory perception of bitter taste',\n",
       "  'definition': {'text': 'The series of events required for a bitter taste stimulus to be received and converted to a molecular signal.'},\n",
       "  'synonyms': [{'name': 'sensory detection of bitter taste', 'type': 'exact'},\n",
       "   {'name': 'sensory detection of chemical stimulus during perception of bitter taste',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sensory transduction of bitter taste', 'type': 'exact'},\n",
       "   {'name': 'sensory transduction of chemical stimulus during perception of bitter taste',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'bitter taste detection', 'type': 'exact'},\n",
       "   {'name': 'perception of bitter taste, sensory transduction of chemical stimulus',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'perception of bitter taste, detection of chemical stimulus',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0033038', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0050912', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050913', 'relation': 'part_of'}]},\n",
       " 'GO:0001581': {'id': 'GO:0001581',\n",
       "  'isObsolete': False,\n",
       "  'name': 'detection of chemical stimulus involved in sensory perception of sour taste',\n",
       "  'definition': {'text': 'The series of events required for a sour taste stimulus to be received and converted to a molecular signal.'},\n",
       "  'synonyms': [{'name': 'sensory transduction of sour taste', 'type': 'exact'},\n",
       "   {'name': 'sour taste detection', 'type': 'exact'},\n",
       "   {'name': 'perception of sour taste, detection of chemical stimulus',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sensory transduction of chemical stimulus during perception of sour taste',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sensory detection of sour taste', 'type': 'exact'},\n",
       "   {'name': 'sensory detection of chemical stimulus during perception of sour taste',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'perception of sour taste, sensory transduction of chemical stimulus',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0033040', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0050912', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050915', 'relation': 'part_of'}]},\n",
       " 'GO:0001582': {'id': 'GO:0001582',\n",
       "  'isObsolete': False,\n",
       "  'name': 'detection of chemical stimulus involved in sensory perception of sweet taste',\n",
       "  'definition': {'text': 'The series of events required for a sweet taste stimulus to be received and converted to a molecular signal.'},\n",
       "  'synonyms': [{'name': 'sensory detection of sweet taste', 'type': 'exact'},\n",
       "   {'name': 'sensory detection of chemical stimulus during perception of sweet taste',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'perception of sweet taste, sensory transduction of chemical stimulus',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'perception of sweet taste, detection of chemical stimulus',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sensory transduction of chemical stimulus during perception of sweet taste',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sweet taste detection', 'type': 'exact'},\n",
       "   {'name': 'sensory transduction of sweet taste', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0033041', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0050912', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050916', 'relation': 'part_of'}]},\n",
       " 'GO:0001583': {'id': 'GO:0001583',\n",
       "  'isObsolete': False,\n",
       "  'name': 'detection of chemical stimulus involved in sensory perception of salty taste',\n",
       "  'definition': {'text': 'The series of events required for a salty taste stimulus to be received and converted to a molecular signal.'},\n",
       "  'synonyms': [{'name': 'sensory transduction of salty taste',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sensory transduction of chemical stimulus during perception of salty taste',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sensory detection of chemical stimulus during perception of salty taste',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'sensory detection of salty taste', 'type': 'exact'},\n",
       "   {'name': 'salty taste detection', 'type': 'exact'},\n",
       "   {'name': 'perception of salty taste, sensory transduction of chemical stimulus',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'perception of salty taste, detection of chemical stimulus',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0033039', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0050912', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050914', 'relation': 'part_of'}]},\n",
       " 'GO:0001586': {'id': 'GO:0001586',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Gi/o-coupled serotonin receptor activity',\n",
       "  'definition': {'text': 'Combining with serotonin and transmitting the signal across the membrane by activation of the Gi/o subunit of an associated cytoplasmic heterotrimeric G protein complex. The Gi/o subunit subsequently inhibits adenylate cyclase and results in a decrease in cyclic AMP (cAMP) levels.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18571247'}]},\n",
       "  'synonyms': [{'name': 'serotonin receptor activity, coupled via Gi/o',\n",
       "    'type': 'exact'},\n",
       "   {'name': '5-HT1 receptor activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004993', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001585', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016609', 'relation': 'is_a'}]},\n",
       " 'GO:0001587': {'id': 'GO:0001587',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Gq/11-coupled serotonin receptor activity',\n",
       "  'definition': {'text': 'Combining with serotonin and transmitting the signal across the membrane by activation of the Gq/11 subunit of an associated cytoplasmic heterotrimeric G protein complex. The Gq/11 subunit subsequently activates phospholipase C and results in an increase in inositol triphosphate (IP3) levels.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18571247'},\n",
       "    {'dbCode': 'PMID', 'dbId': '18703043'}]},\n",
       "  'synonyms': [{'name': 'serotonin receptor activity, coupled via Gq/11',\n",
       "    'type': 'exact'},\n",
       "   {'name': '5-HT2 receptor activity', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004993', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001585', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016609', 'relation': 'is_a'}]},\n",
       " 'GO:0001588': {'id': 'GO:0001588',\n",
       "  'isObsolete': False,\n",
       "  'name': 'dopamine neurotransmitter receptor activity, coupled via Gs',\n",
       "  'definition': {'text': 'Combining with the neurotransmitter dopamine and activating adenylate cyclase via coupling to Gs to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0953351033'},\n",
       "    {'dbCode': 'IUPHAR_RECEPTOR', 'dbId': '2252'},\n",
       "    {'dbCode': 'IUPHAR_RECEPTOR', 'dbId': '2260'}]},\n",
       "  'synonyms': [{'name': 'dopamine D1 receptor activity', 'type': 'narrow'},\n",
       "   {'name': 'dopamine D5 receptor activity', 'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004952', 'relation': 'is_a'}]},\n",
       " 'GO:0001591': {'id': 'GO:0001591',\n",
       "  'isObsolete': False,\n",
       "  'name': 'dopamine neurotransmitter receptor activity, coupled via Gi/Go',\n",
       "  'definition': {'text': 'Combining with the neurotransmitter dopamine and activating adenylate cyclase via coupling to Gi/Go to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0953351033'},\n",
       "    {'dbCode': 'IUPHAR_RECEPTOR', 'dbId': '2254'},\n",
       "    {'dbCode': 'IUPHAR_RECEPTOR', 'dbId': '2256'},\n",
       "    {'dbCode': 'IUPHAR_RECEPTOR', 'dbId': '2258'}]},\n",
       "  'synonyms': [{'name': 'dopamine D2 receptor activity', 'type': 'narrow'},\n",
       "   {'name': 'dopamine D3 receptor activity', 'type': 'narrow'},\n",
       "   {'name': 'dopamine D4 receptor activity', 'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004952', 'relation': 'is_a'}]},\n",
       " 'GO:0001594': {'id': 'GO:0001594',\n",
       "  'isObsolete': False,\n",
       "  'name': 'trace-amine receptor activity',\n",
       "  'definition': {'text': 'Combining with a trace amine to initiate a change in cell activity. Trace amines are biogenic amines that are synthesized from aromatic amino acids and are substrates for monoamine oxidase, and are therefore detectable only at trace levels in mammals.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '19325074'}]},\n",
       "  'children': [{'id': 'GO:1990081', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008227', 'relation': 'is_a'}]},\n",
       " 'GO:0001595': {'id': 'GO:0001595',\n",
       "  'isObsolete': False,\n",
       "  'name': 'angiotensin receptor activity',\n",
       "  'definition': {'text': 'Combining with angiotensin to initiate a change in cell activity.'},\n",
       "  'children': [{'id': 'GO:0004945', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001596', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008528', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0038166', 'relation': 'part_of'}]},\n",
       " 'GO:0001596': {'id': 'GO:0001596',\n",
       "  'isObsolete': False,\n",
       "  'name': 'angiotensin type I receptor activity',\n",
       "  'definition': {'text': 'An angiotensin receptor activity that acts via Gq-mediated activation of phospholipase C followed by phosphoinositide hydrolysis and Ca2+ signaling, and may act via additional signaling mechanisms.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10977869'}]},\n",
       "  'synonyms': [{'name': 'PLC-activating angiotensin receptor activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001595', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0086097', 'relation': 'part_of'}]},\n",
       " 'GO:0001601': {'id': 'GO:0001601',\n",
       "  'isObsolete': False,\n",
       "  'name': 'peptide YY receptor activity',\n",
       "  'definition': {'text': 'Combining with gut peptide YY to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9315606'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004983', 'relation': 'is_a'}]},\n",
       " 'GO:0001602': {'id': 'GO:0001602',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pancreatic polypeptide receptor activity',\n",
       "  'definition': {'text': 'Combining with pancreatic polypeptide PP to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9315606'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004983', 'relation': 'is_a'}]},\n",
       " 'GO:0001604': {'id': 'GO:0001604',\n",
       "  'isObsolete': False,\n",
       "  'name': 'urotensin II receptor activity',\n",
       "  'definition': {'text': 'Combining with urotensin II to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '15102493'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008528', 'relation': 'is_a'}]},\n",
       " 'GO:0001605': {'id': 'GO:0001605',\n",
       "  'isObsolete': False,\n",
       "  'name': 'adrenomedullin receptor activity',\n",
       "  'definition': {'text': 'Combining with adrenomedullin to initiate a change in cell activity.'},\n",
       "  'synonyms': [{'name': 'G10D receptor', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:1903143', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0097642', 'relation': 'is_a'}]},\n",
       " 'GO:0001607': {'id': 'GO:0001607',\n",
       "  'isObsolete': False,\n",
       "  'name': 'neuromedin U receptor activity',\n",
       "  'definition': {'text': 'Combining with neuromedin U to initiate a change in cell activity.'},\n",
       "  'synonyms': [{'name': 'NMUR activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008188', 'relation': 'is_a'}]},\n",
       " 'GO:0001608': {'id': 'GO:0001608',\n",
       "  'isObsolete': False,\n",
       "  'name': 'G protein-coupled nucleotide receptor activity',\n",
       "  'definition': {'text': 'Combining with a nucleotide and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex.',\n",
       "   'xrefs': [{'dbCode': 'IUPHAR_GPCR', 'dbId': '1294'}]},\n",
       "  'synonyms': [{'name': 'G-protein coupled nucleotide receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'nucleotide receptor activity, G protein coupled',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'G protein coupled nucleotide receptor activity', 'type': 'exact'},\n",
       "   {'name': 'nucleotide receptor activity, G-protein coupled',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0045028', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071553', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004930', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016526', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001623', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001622', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001625', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001624', 'relation': 'is_a'}]},\n",
       " 'GO:0001609': {'id': 'GO:0001609',\n",
       "  'isObsolete': False,\n",
       "  'name': 'G protein-coupled adenosine receptor activity',\n",
       "  'definition': {'text': 'Combining with adenosine and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9755289'}]},\n",
       "  'synonyms': [{'name': 'G-protein coupled adenosine receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'G protein coupled adenosine receptor activity', 'type': 'exact'},\n",
       "   {'name': 'A1 adenosine receptor activity, G-protein coupled',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'adenosine receptor activity, G protein coupled', 'type': 'exact'},\n",
       "   {'name': 'adenosine nucleotide receptor', 'type': 'broad'},\n",
       "   {'name': 'G-protein-coupled A3 adenosine receptor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'G-protein-coupled A2B adenosine receptor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'G-protein-coupled A2A adenosine receptor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'G-protein-coupled A1 adenosine receptor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'G protein coupled A3 adenosine receptor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'G protein coupled A2B adenosine receptor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'G protein coupled A2A adenosine receptor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'G protein coupled A1 adenosine receptor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'A3 adenosine receptor activity, G-protein coupled',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'A3 adenosine receptor activity, G protein coupled',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'A2B adenosine receptor activity, G-protein coupled',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'A2B adenosine receptor activity, G protein coupled',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'A2A adenosine receptor activity, G-protein coupled',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'A2A adenosine receptor activity, G protein coupled',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'A1 adenosine receptor activity, G protein coupled',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'P1 receptor', 'type': 'exact'},\n",
       "   {'name': 'adenosine receptor activity, G-protein coupled',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0045032', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045031', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001973', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0004930', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035586', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016526', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001623', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001622', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001625', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001624', 'relation': 'is_a'}]},\n",
       " 'GO:0001614': {'id': 'GO:0001614',\n",
       "  'isObsolete': False,\n",
       "  'name': 'purinergic nucleotide receptor activity',\n",
       "  'definition': {'text': 'Combining with a purine nucleotide and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity.'},\n",
       "  'synonyms': [{'name': 'P2 receptor', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001646', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045028', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016502', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035586', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035590', 'relation': 'part_of'}]},\n",
       " 'GO:0001616': {'id': 'GO:0001616',\n",
       "  'isObsolete': False,\n",
       "  'name': 'growth hormone secretagogue receptor activity',\n",
       "  'definition': {'text': 'Combining with ghrelin to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '17983853'}]},\n",
       "  'synonyms': [{'name': 'ghrelin receptor activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004930', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016526', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001623', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001622', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001625', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001624', 'relation': 'is_a'}]},\n",
       " 'GO:0001618': {'id': 'GO:0001618',\n",
       "  'isObsolete': False,\n",
       "  'name': 'virus receptor activity',\n",
       "  'definition': {'text': 'Combining with a virus component and mediating entry of the virus into the cell.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '7621403'},\n",
       "    {'dbCode': 'UniProtKB-KW', 'dbId': 'KW-1183'}]},\n",
       "  'synonyms': [{'name': 'viral receptor activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0046718', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0140272', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019063', 'relation': 'part_of'}]},\n",
       " 'GO:0001621': {'id': 'GO:0001621',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ADP receptor activity',\n",
       "  'definition': {'text': 'Combining with ADP and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11196645'}]},\n",
       "  'synonyms': [{'name': 'K101 receptor', 'type': 'narrow'},\n",
       "   {'name': 'platelet ADP receptor activity', 'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0045028', 'relation': 'is_a'}]},\n",
       " 'GO:0001626': {'id': 'GO:0001626',\n",
       "  'isObsolete': False,\n",
       "  'name': 'nociceptin receptor activity',\n",
       "  'definition': {'text': 'Combining with the peptide nociceptin, and transmitting the signal across the membrane by activating an associated G-protein.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '18670432'}]},\n",
       "  'synonyms': [{'name': 'nociceptin/orphanin-FQ receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'OFQ receptor activity', 'type': 'exact'},\n",
       "   {'name': 'orphanin-FQ receptor activity', 'type': 'exact'},\n",
       "   {'name': 'ORPH receptor', 'type': 'exact'},\n",
       "   {'name': 'X-opioid receptor activity', 'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004985', 'relation': 'is_a'}]},\n",
       " 'GO:0001631': {'id': 'GO:0001631',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cysteinyl leukotriene receptor activity',\n",
       "  'definition': {'text': 'Combining with a cysteinyl leukotriene to initiate a change in cell activity. Cysteinyl leukotrienes are leukotrienes that contain a peptide group based on cysteine.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'CysLT receptor', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004974', 'relation': 'is_a'}]},\n",
       " 'GO:0001632': {'id': 'GO:0001632',\n",
       "  'isObsolete': False,\n",
       "  'name': 'leukotriene B4 receptor activity',\n",
       "  'definition': {'text': 'Combining with leukotriene B4, LTB4, to initiate a change in cell activity. Leukotriene B4 is also known as (6Z, 8E, 10E, 14Z)-(5S, 12R)-5,12-dihydroxyicosa-6,8,10,14-tetraen-1-oate.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'BLT receptor', 'type': 'narrow'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004974', 'relation': 'is_a'}]},\n",
       " 'GO:0001634': {'id': 'GO:0001634',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pituitary adenylate cyclase-activating polypeptide receptor activity',\n",
       "  'definition': {'text': 'A G protein-coupled receptor that interacts with pituitary adenylate cyclase-activating polypeptide.'},\n",
       "  'synonyms': [{'name': 'pituitary adenylate cyclase activating polypeptide receptor',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'pituitary adenylyl cyclase activating protein receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'pituitary adenylate cyclase activating protein receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'pituitary adenylate cyclase-activating peptide receptor activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'pituitary adenylate cyclase-activating polypeptide receptor activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'PACAP receptor', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004930', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016526', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001623', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001622', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001625', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001624', 'relation': 'is_a'}]},\n",
       " 'GO:0001635': {'id': 'GO:0001635',\n",
       "  'isObsolete': False,\n",
       "  'name': 'calcitonin gene-related peptide receptor activity',\n",
       "  'definition': {'text': 'Combining with a calcitonin gene-related polypeptide (CGRP) to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12037140'}]},\n",
       "  'synonyms': [{'name': 'CGRP receptor', 'type': 'exact'},\n",
       "   {'name': 'calcitonin gene-related polypeptide receptor activity',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1990406', 'relation': 'capable_of'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0097642', 'relation': 'is_a'}]},\n",
       " 'GO:0001637': {'id': 'GO:0001637',\n",
       "  'isObsolete': False,\n",
       "  'name': 'G protein-coupled chemoattractant receptor activity',\n",
       "  'definition': {'text': 'Combining with a chemoattractant and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex.'},\n",
       "  'synonyms': [{'name': 'G-protein chemoattractant receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'G-protein coupled chemoattractant receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'G protein chemoattractant receptor activity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0004950', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008528', 'relation': 'is_a'}]},\n",
       " 'GO:0001639': {'id': 'GO:0001639',\n",
       "  'isObsolete': False,\n",
       "  'name': 'PLC activating G protein-coupled glutamate receptor activity',\n",
       "  'definition': {'text': 'A G protein-coupled receptor that binds glutamate and is linked to the inositol 1,4,5-trisphosphate/calcium signaling system.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9016303'}]},\n",
       "  'synonyms': [{'name': 'group I metabotropic glutamate receptor activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'PLC activating metabotropic glutamate receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'Group I metabotropic glutamate receptor', 'type': 'related'},\n",
       "   {'name': 'phospholipase C activating metabotropic glutamate receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'PLC activating G-protein coupled glutamate receptor activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007206', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098988', 'relation': 'is_a'}]},\n",
       " 'GO:0001640': {'id': 'GO:0001640',\n",
       "  'isObsolete': False,\n",
       "  'name': 'adenylate cyclase inhibiting G protein-coupled glutamate receptor activity',\n",
       "  'definition': {'text': 'Combining with glutamate and transmitting the signal across the membrane by activating the alpha-subunit of an associated heterotrimeric G-protein complex to inhibit downstream adenylate cyclase activity.'},\n",
       "  'synonyms': [{'name': 'adenylyl cyclase inhibiting metabotropic glutamate receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'adenylate cyclase inhibiting metabotropic glutamate receptor activity',\n",
       "    'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0001641', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001642', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007196', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0098988', 'relation': 'is_a'}]},\n",
       " 'GO:0001641': {'id': 'GO:0001641',\n",
       "  'isObsolete': False,\n",
       "  'name': 'group II metabotropic glutamate receptor activity',\n",
       "  'definition': {'text': 'A G protein-coupled receptor that is activated by trans-1-aminocyclopentane-1,3-dicarboxylic acid (t-ACPD) and inhibits adenylate cyclase activity.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001640', 'relation': 'is_a'}]},\n",
       " 'GO:0001642': {'id': 'GO:0001642',\n",
       "  'isObsolete': False,\n",
       "  'name': 'group III metabotropic glutamate receptor activity',\n",
       "  'definition': {'text': 'A G protein-coupled receptor that is activated by L-AP-4 and inhibits adenylate cyclase activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9016303'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001640', 'relation': 'is_a'}]},\n",
       " 'GO:0001646': {'id': 'GO:0001646',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cAMP receptor activity',\n",
       "  'definition': {'text': \"Combining with cAMP (cyclic AMP, adenosine 3',5'-cyclophosphate) and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity.\"},\n",
       "  'synonyms': [{'name': \"3',5' cAMP receptor activity\", 'type': 'exact'},\n",
       "   {'name': \"3',5'-cAMP receptor activity\", 'type': 'exact'},\n",
       "   {'name': \"adenosine 3',5'-cyclophosphate receptor activity\",\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cyclic AMP receptor activity', 'type': 'exact'},\n",
       "   {'name': 'class E G-protein coupled receptor', 'type': 'broad'},\n",
       "   {'name': 'class E GPCR', 'type': 'broad'},\n",
       "   {'name': 'class E G protein coupled receptor', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001614', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004930', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016526', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001623', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001622', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001625', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001624', 'relation': 'is_a'}]},\n",
       " 'GO:0001647': {'id': 'GO:0001647',\n",
       "  'isObsolete': False,\n",
       "  'name': 'G protein-coupled cytokinin receptor activity',\n",
       "  'definition': {'text': 'Combining with cytokinin and transmitting the signal across the membrane by activating an associated G-protein; promotes the exchange of GDP for GTP on the alpha subunit of a heterotrimeric G-protein complex.'},\n",
       "  'synonyms': [{'name': 'G-protein coupled cytokinin receptor activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'cytokinin receptor activity, G-protein coupled', 'type': 'exact'},\n",
       "   {'name': 'G protein coupled cytokinin receptor activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004930', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009884', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016526', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001623', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001622', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001625', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001624', 'relation': 'is_a'}]},\n",
       " 'GO:0001648': {'id': 'GO:0001648',\n",
       "  'isObsolete': False,\n",
       "  'name': 'proteinase activated receptor activity',\n",
       "  'definition': {'text': 'A G protein-coupled peptide receptor activity that is initiated by cleavage of the N terminus of the receptor by a serine protease, resulting in the generation of a new tethered ligand that interacts with the receptor.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11356985'}]},\n",
       "  'children': [{'id': 'GO:1900276', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1900278', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1900277', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008528', 'relation': 'is_a'}]},\n",
       " 'GO:0001649': {'id': 'GO:0001649',\n",
       "  'isObsolete': False,\n",
       "  'name': 'osteoblast differentiation',\n",
       "  'definition': {'text': 'The process whereby a relatively unspecialized cell acquires the specialized features of an osteoblast, a mesodermal or neural crest cell that gives rise to bone.',\n",
       "   'xrefs': [{'dbCode': 'CL', 'dbId': '0000062'},\n",
       "    {'dbCode': 'GO_REF', 'dbId': '0000034'}]},\n",
       "  'synonyms': [{'name': 'osteoblast cell differentiation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0045669', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0045668', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0045667', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0044339', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0002076', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0002051', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001503', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030154', 'relation': 'is_a'}]},\n",
       " 'GO:0001650': {'id': 'GO:0001650',\n",
       "  'isObsolete': False,\n",
       "  'name': 'fibrillar center',\n",
       "  'definition': {'text': 'A structure found most metazoan nucleoli, but not usually found in lower eukaryotes; surrounded by the dense fibrillar component; the zone of transcription from multiple copies of the pre-rRNA genes is in the border region between these two structures.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10754561'}]},\n",
       "  'synonyms': [{'name': 'fibrillar centre', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001651': {'id': 'GO:0001651',\n",
       "  'isObsolete': False,\n",
       "  'name': 'dense fibrillar component',\n",
       "  'definition': {'text': 'A structure found in the nucleolus, which contains newly synthesized preribosomal RNA (pre-rRNA) and a collection of proteins.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10754561'}]},\n",
       "  'synonyms': [{'name': 'pars fibrosa', 'type': 'broad'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001652': {'id': 'GO:0001652',\n",
       "  'isObsolete': False,\n",
       "  'name': 'granular component',\n",
       "  'definition': {'text': 'A structure found in the nucleolus, which contains nearly completed preribosomal particles destined for the cytoplasm.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10754561'}]},\n",
       "  'synonyms': [{'name': 'pars granulosa', 'type': 'broad'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001653': {'id': 'GO:0001653',\n",
       "  'isObsolete': False,\n",
       "  'name': 'peptide receptor activity',\n",
       "  'definition': {'text': 'Combining with an extracellular or intracellular peptide to initiate a change in cell activity.'},\n",
       "  'synonyms': [{'name': 'endogenous peptide receptor activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'exogenous peptide receptor activity', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0043404', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016941', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008528', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0038023', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004872', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019041', 'relation': 'is_a'}]},\n",
       " 'GO:0001654': {'id': 'GO:0001654',\n",
       "  'isObsolete': False,\n",
       "  'name': 'eye development',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of the eye over time, from its formation to the mature structure. The eye is the organ of sight.'},\n",
       "  'children': [{'id': 'GO:0055034', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048749', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043010', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048592', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007423', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0150063', 'relation': 'part_of'}]},\n",
       " 'GO:0001655': {'id': 'GO:0001655',\n",
       "  'isObsolete': False,\n",
       "  'name': 'urogenital system development',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of the urogenital system over time, from its formation to the mature structure.'},\n",
       "  'children': [{'id': 'GO:0030850', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072001', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0035844', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048731', 'relation': 'is_a'}]},\n",
       " 'GO:0001656': {'id': 'GO:0001656',\n",
       "  'isObsolete': False,\n",
       "  'name': 'metanephros development',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of the metanephros over time, from its formation to the mature structure. In mammals, the metanephros is the excretory organ of the fetus, which develops into the mature kidney and is formed from the rear portion of the nephrogenic cord. The metanephros is an endocrine and metabolic organ that filters the blood and excretes the end products of body metabolism in the form of urine.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0192800752'}]},\n",
       "  'children': [{'id': 'GO:0035788', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061290', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072075', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072207', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072215', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0072206', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072208', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1900200', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072268', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072211', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072212', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072210', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072203', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072202', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072205', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072213', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072204', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072214', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072298', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072217', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0003338', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072216', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001822', 'relation': 'is_a'}]},\n",
       " 'GO:0001657': {'id': 'GO:0001657',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ureteric bud development',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of the ureteric bud over time, from its formation to the mature structure.'},\n",
       "  'children': [{'id': 'GO:0072138', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0035503', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0072099', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060675', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0035502', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0072164', 'relation': 'is_a'}]},\n",
       " 'GO:0001658': {'id': 'GO:0001658',\n",
       "  'isObsolete': False,\n",
       "  'name': 'branching involved in ureteric bud morphogenesis',\n",
       "  'definition': {'text': 'The process in which the branching structure of the ureteric bud is generated and organized. The ureteric bud is an epithelial tube that grows out from the metanephric duct. The bud elongates and branches to give rise to the ureter and kidney collecting tubules.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16916378'}]},\n",
       "  'synonyms': [{'name': 'ureteric bud branching', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0060680', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061291', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090189', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0090191', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0060681', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090190', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0060679', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060678', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048754', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060675', 'relation': 'part_of'}]},\n",
       " 'GO:0001659': {'id': 'GO:0001659',\n",
       "  'isObsolete': False,\n",
       "  'name': 'temperature homeostasis',\n",
       "  'definition': {'text': 'A homeostatic process in which an organism modulates its internal body temperature.'},\n",
       "  'synonyms': [{'name': 'thermoregulation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0031649', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042309', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060086', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0106106', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031653', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048871', 'relation': 'is_a'}]},\n",
       " 'GO:0001660': {'id': 'GO:0001660',\n",
       "  'isObsolete': False,\n",
       "  'name': 'fever generation',\n",
       "  'definition': {'text': 'The heat generation process that results in a rise in body temperature above the normal, often as a response to infection.'},\n",
       "  'synonyms': [{'name': 'pyrexia', 'type': 'broad'}],\n",
       "  'children': [{'id': 'GO:0031620', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0031621', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0031622', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0100011', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0100010', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0071811', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0100009', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0100008', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006953', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031649', 'relation': 'is_a'}]},\n",
       " 'GO:0001661': {'id': 'GO:0001661',\n",
       "  'isObsolete': False,\n",
       "  'name': 'conditioned taste aversion',\n",
       "  'definition': {'text': 'A conditioned aversion to a specific chemical compound as a result of that compound being coupled with a noxious stimulus.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9920659'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007631', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008306', 'relation': 'is_a'}]},\n",
       " 'GO:0001662': {'id': 'GO:0001662',\n",
       "  'isObsolete': False,\n",
       "  'name': 'behavioral fear response',\n",
       "  'definition': {'text': 'An acute behavioral change resulting from a perceived external threat.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '9920659'}]},\n",
       "  'synonyms': [{'name': 'behavioural fear response', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:2000986', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:2000822', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000987', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002209', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042596', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001663', 'relation': 'is_a'}]},\n",
       " 'GO:0001664': {'id': 'GO:0001664',\n",
       "  'isObsolete': False,\n",
       "  'name': 'G protein-coupled receptor binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a G protein-coupled receptor.'},\n",
       "  'synonyms': [{'name': 'G protein coupled receptor ligand', 'type': 'narrow'},\n",
       "   {'name': 'G-protein-coupled receptor ligand', 'type': 'narrow'},\n",
       "   {'name': 'G protein coupled receptor binding', 'type': 'exact'},\n",
       "   {'name': 'G-protein coupled receptor binding', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0031774', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031753', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031795', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031775', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042379', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031859', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031856', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031811', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031855', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031876', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031690', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031871', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031883', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035256', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031717', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005119', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031889', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031716', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031714', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031711', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031762', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031784', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031531', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031767', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031849', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031745', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031769', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031788', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050780', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031768', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031789', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031806', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031821', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0005109', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071855', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031862', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031893', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061761', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031685', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031628', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031700', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031701', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0031761', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005102', 'relation': 'is_a'}]},\n",
       " 'GO:0001665': {'id': 'GO:0001665',\n",
       "  'isObsolete': False,\n",
       "  'name': 'alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: CMP-N-acetylneuraminate + glycano-(1->3)-(N-acetyl-alpha-D-galactosaminyl)-glycoprotein = CMP + glycano-[(2->6)-alpha-N-acetylneuraminyl]-(N-acetyl-D-galactosaminyl)-glycoprotein.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.4.99.3'}]},\n",
       "  'synonyms': [{'name': 'GalNAc alpha-2,6-sialyltransferase I activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'CMP-N-acetylneuraminate:glycano-1,3-(N-acetyl-alpha-D-galactosaminyl)-glycoprotein alpha-2,6-N-acetylneuraminyltransferase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008373', 'relation': 'is_a'}]},\n",
       " 'GO:0001666': {'id': 'GO:0001666',\n",
       "  'isObsolete': False,\n",
       "  'name': 'response to hypoxia',\n",
       "  'definition': {'text': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating lowered oxygen tension. Hypoxia, defined as a decline in O2 levels below normoxic levels of 20.8 - 20.95%, results in metabolic adaptation at both the cellular and organismal level.'},\n",
       "  'comment': \"Note that this term should not be confused with 'response to anoxia ; GO:0034059'. Note that in laboratory studies, hypoxia is typically studied at O2 concentrations ranging from 0.1 - 5%.\",\n",
       "  'synonyms': [{'name': 'response to intermittent hypoxia', 'type': 'narrow'},\n",
       "   {'name': 'response to lowered oxygen tension', 'type': 'exact'},\n",
       "   {'name': 'response to sustained hypoxia', 'type': 'narrow'},\n",
       "   {'name': 'response to hypoxic stress', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0070483', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990910', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071456', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006950', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036293', 'relation': 'is_a'}]},\n",
       " 'GO:0001667': {'id': 'GO:0001667',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ameboidal-type cell migration',\n",
       "  'definition': {'text': 'Cell migration that is accomplished by extension and retraction of a pseudopodium.'},\n",
       "  'comment': 'Note that this term refers to a mode of migration rather than to any particular cell type.',\n",
       "  'synonyms': [{'name': 'amoeboid cell migration', 'type': 'exact'},\n",
       "   {'name': 'ameboid cell migration', 'type': 'exact'},\n",
       "   {'name': 'amoeboidal cell migration', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0006932', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010761', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0021806', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0003371', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090497', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042074', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010631', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008078', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001755', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003366', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0003365', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0003363', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061450', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016477', 'relation': 'is_a'}]},\n",
       " 'GO:0001669': {'id': 'GO:0001669',\n",
       "  'isObsolete': False,\n",
       "  'name': 'acrosomal vesicle',\n",
       "  'definition': {'text': 'A structure in the head of a spermatozoon that contains acid hydrolases, and is concerned with the breakdown of the outer membrane of the ovum during fertilization. It lies just beneath the plasma membrane and is derived from the lysosome.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0124325653'},\n",
       "    {'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': 'acrosomal granule', 'type': 'exact'},\n",
       "   {'name': 'acrosome', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0002080', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043159', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043160', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001671': {'id': 'GO:0001671',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ATPase activator activity',\n",
       "  'definition': {'text': 'Binds to and increases the ATP hydrolysis activity of an ATPase.'},\n",
       "  'synonyms': [{'name': 'ATPase stimulator activity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0072587', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008047', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016887', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0032781', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060590', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010577', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004002', 'relation': 'positively_regulates'}]},\n",
       " 'GO:0001672': {'id': 'GO:0001672',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of chromatin assembly or disassembly',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate or extent of chromatin assembly or disassembly.'},\n",
       "  'synonyms': [{'name': 'regulation of chromatin assembly/disassembly',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0010848', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010847', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045799', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045798', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006333', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1902275', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903308', 'relation': 'is_a'}]},\n",
       " 'GO:0001673': {'id': 'GO:0001673',\n",
       "  'isObsolete': False,\n",
       "  'name': 'male germ cell nucleus',\n",
       "  'definition': {'text': 'The nucleus of a male germ cell, a reproductive cell in males.',\n",
       "   'xrefs': [{'dbCode': 'CL', 'dbId': '0000015'}]},\n",
       "  'synonyms': [{'name': 'male germ-cell nucleus', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0035843', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001674': {'id': 'GO:0001674',\n",
       "  'isObsolete': False,\n",
       "  'name': 'female germ cell nucleus',\n",
       "  'definition': {'text': 'The nucleus of the female germ cell, a reproductive cell in females.',\n",
       "   'xrefs': [{'dbCode': 'CL', 'dbId': '0000021'}]},\n",
       "  'synonyms': [{'name': 'female germ-cell nucleus', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0042585', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001675': {'id': 'GO:0001675',\n",
       "  'isObsolete': False,\n",
       "  'name': 'acrosome assembly',\n",
       "  'definition': {'text': 'The formation of the acrosome from the spermatid Golgi.'},\n",
       "  'synonyms': [{'name': 'acrosome formation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0120211', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003006', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007286', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010927', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0022412', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033363', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0070925', 'relation': 'is_a'}]},\n",
       " 'GO:0001676': {'id': 'GO:0001676',\n",
       "  'isObsolete': False,\n",
       "  'name': 'long-chain fatty acid metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving long-chain fatty acids, A long-chain fatty acid is a fatty acid with a chain length between C13 and C22.',\n",
       "   'xrefs': [{'dbCode': 'CHEBI', 'dbId': '15904'}]},\n",
       "  'synonyms': [{'name': 'long-chain fatty acid metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0036102', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0051121', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901751', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0004467', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0036109', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019369', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042759', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1900533', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2001302', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042758', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1903512', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2001304', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043651', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006631', 'relation': 'is_a'}]},\n",
       " 'GO:0001677': {'id': 'GO:0001677',\n",
       "  'isObsolete': False,\n",
       "  'name': 'formation of translation initiation ternary complex',\n",
       "  'definition': {'text': 'Formation of a complex between aminoacylated initiator methionine tRNA, GTP, and initiation factor 2 (either eIF2 in eukaryotes, or IF2 in prokaryotes). In prokaryotes, fMet-tRNA (initiator) is used rather than Met-tRNA (initiator).'},\n",
       "  'synonyms': [{'name': 'translation initiation ternary complex assembly',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1901192', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1901190', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1901191', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006413', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0006454', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006440', 'relation': 'part_of'}]},\n",
       " 'GO:0001678': {'id': 'GO:0001678',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cellular glucose homeostasis',\n",
       "  'definition': {'text': 'A cellular homeostatic process involved in the maintenance of an internal steady state of glucose within a cell or between a cell and its external environment.'},\n",
       "  'synonyms': [{'name': 'cell glucose homeostasis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0071333', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0042593', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055082', 'relation': 'is_a'}]},\n",
       " 'GO:0001680': {'id': 'GO:0001680',\n",
       "  'isObsolete': False,\n",
       "  'name': \"tRNA 3'-terminal CCA addition\",\n",
       "  'definition': {'text': \"Post-transcriptional addition of the terminal 3' CCA sequence to a tRNA which does not encode this sequence within the primary transcript. CCA addition proceeds by the sequential addition of CTP, CTP, and then ATP to the 3' end of the tRNA, yielding a diphosphate with each nucleotide addition.\",\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.7.7.72'}]},\n",
       "  'synonyms': [{'name': 'transfer-RNA nucleotidyltransferase activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'CTP:tRNA cytidylyltransferase activity', 'type': 'narrow'},\n",
       "   {'name': 'transfer ribonucleic acid nucleotidyl transferase activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'tRNA-nucleotidyltransferase activity', 'type': 'broad'},\n",
       "   {'name': 'ATP(CTP)-tRNA nucleotidyltransferase activity', 'type': 'broad'},\n",
       "   {'name': 'ATP(CTP):tRNA nucleotidyltransferase activity', 'type': 'broad'},\n",
       "   {'name': 'CCA-adding enzyme activity', 'type': 'exact'},\n",
       "   {'name': 'transfer RNA adenylyltransferase activity', 'type': 'narrow'},\n",
       "   {'name': 'transfer ribonucleic adenylyl (cytidylyl) transferase activity',\n",
       "    'type': 'related'},\n",
       "   {'name': 'tRNA adenylyl(cytidylyl)transferase activity', 'type': 'broad'},\n",
       "   {'name': 'tRNA CCA-diphosphorylase activity', 'type': 'related'},\n",
       "   {'name': 'ribonucleic cytidylyltransferase activity', 'type': 'narrow'},\n",
       "   {'name': 'ribonucleic cytidylic cytidylic adenylic pyrophosphorylase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'CTP(ATP):tRNA nucleotidyltransferase activity', 'type': 'broad'},\n",
       "   {'name': 'CCA tRNA nucleotidyltransferase activity', 'type': 'broad'},\n",
       "   {'name': '-C-C-A pyrophosphorylase activity', 'type': 'related'},\n",
       "   {'name': 'transfer ribonucleate adenyltransferase activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'CCA-adding enzyme, tRNA adenylyltransferase, tRNA cytidylyltransferase, tRNA CCA-pyrophosphorylase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'tRNA cytidylyltransferase activity', 'type': 'narrow'},\n",
       "   {'name': 'transfer ribonucleate cytidylyltransferase activity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'transfer ribonucleate nucleotidyltransferase activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'transfer ribonucleic-terminal trinucleotide nucleotidyltransferase activity',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'transfer ribonucleate adenylyltransferase activity',\n",
       "    'type': 'narrow'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0042780', 'relation': 'is_a'}]},\n",
       " 'GO:0001681': {'id': 'GO:0001681',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sialate O-acetylesterase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: N-acetyl-O-acetylneuraminate (free or glycosidically bound) + H2O = N-acetylneuraminate + acetate.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '3.1.1.53'},\n",
       "    {'dbCode': 'PMID', 'dbId': '1991039'}]},\n",
       "  'synonyms': [{'name': 'sialate 9(4)-O-acetylesterase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'N-acetylneuraminate acetyltransferase activity', 'type': 'exact'},\n",
       "   {'name': 'N-acyl-O-acetylneuraminate O-acetylhydrolase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008126', 'relation': 'is_a'}]},\n",
       " 'GO:0001682': {'id': 'GO:0001682',\n",
       "  'isObsolete': False,\n",
       "  'name': \"tRNA 5'-leader removal\",\n",
       "  'definition': {'text': \"Generation of the mature 5'-end of the tRNA, usually via an endonucleolytic cleavage by RNase P.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11592395'}]},\n",
       "  'synonyms': [{'name': \"tRNA 5' leader removal\", 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0099116', 'relation': 'is_a'}]},\n",
       " 'GO:0001691': {'id': 'GO:0001691',\n",
       "  'isObsolete': False,\n",
       "  'name': 'pseudophosphatase activity',\n",
       "  'definition': {'text': 'Maintains the phosphorylation state of certain molecules by associating with them and preventing them from associating with active phosphatases, and thus inhibiting the enzyme activity without interacting with the enzyme. Often pertains to proteins belonging to dual-specificity phosphatase family but lacking critical active site residues.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019212', 'relation': 'is_a'}]},\n",
       " 'GO:0001692': {'id': 'GO:0001692',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histamine metabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways involving histamine, a physiologically active amine, found in plant and animal tissue and released from mast cells as part of an allergic reaction in humans.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0395825172'}]},\n",
       "  'synonyms': [{'name': 'histamine metabolism', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001695', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001694', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006576', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042133', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052803', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097164', 'relation': 'is_a'}]},\n",
       " 'GO:0001694': {'id': 'GO:0001694',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histamine biosynthetic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the formation of histamine, a physiologically active amine, found in plant and animal tissue and released from mast cells as part of an allergic reaction in humans.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0395825172'}]},\n",
       "  'synonyms': [{'name': 'histamine biosynthesis', 'type': 'exact'},\n",
       "   {'name': 'histamine anabolism', 'type': 'exact'},\n",
       "   {'name': 'histamine formation', 'type': 'exact'},\n",
       "   {'name': 'histamine synthesis', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001692', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0018130', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0019438', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042136', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042401', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901362', 'relation': 'is_a'}]},\n",
       " 'GO:0001695': {'id': 'GO:0001695',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histamine catabolic process',\n",
       "  'definition': {'text': 'The chemical reactions and pathways resulting in the breakdown of histamine, a physiologically active amine, found in plant and animal tissue and released from mast cells as part of an allergic reaction in humans.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0395825172'}]},\n",
       "  'synonyms': [{'name': 'histamine breakdown', 'type': 'exact'},\n",
       "   {'name': 'histamine catabolism', 'type': 'exact'},\n",
       "   {'name': 'histamine degradation', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001692', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042135', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042402', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0052805', 'relation': 'is_a'}]},\n",
       " 'GO:0001696': {'id': 'GO:0001696',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gastric acid secretion',\n",
       "  'definition': {'text': 'The regulated release of gastric acid (hydrochloric acid) by parietal or oxyntic cells during digestion.'},\n",
       "  'synonyms': [{'name': 'hydrochloric acid secretion', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0001699', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001698', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060454', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0060455', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0060453', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001697', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0022600', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046717', 'relation': 'is_a'}]},\n",
       " 'GO:0001697': {'id': 'GO:0001697',\n",
       "  'isObsolete': False,\n",
       "  'name': 'histamine-induced gastric acid secretion',\n",
       "  'definition': {'text': 'The regulated release of gastric acid induced by the interaction of histamine with H2 type receptor receptors with subsequent activation of adenylate cyclase and elevation of intracellular cyclic AMP.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001696', 'relation': 'is_a'}]},\n",
       " 'GO:0001698': {'id': 'GO:0001698',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gastrin-induced gastric acid secretion',\n",
       "  'definition': {'text': 'The regulated release of gastric acid induced by the interaction of gastrin with its receptor.'},\n",
       "  'children': [{'id': 'GO:1903639', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1903640', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1903641', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001696', 'relation': 'is_a'}]},\n",
       " 'GO:0001699': {'id': 'GO:0001699',\n",
       "  'isObsolete': False,\n",
       "  'name': 'acetylcholine-induced gastric acid secretion',\n",
       "  'definition': {'text': 'The regulated release of gastric acid by parietal cells in response to acetylcholine.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001696', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905145', 'relation': 'part_of'}]},\n",
       " 'GO:0001700': {'id': 'GO:0001700',\n",
       "  'isObsolete': False,\n",
       "  'name': 'embryonic development via the syncytial blastoderm',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of the embryo over time, from zygote formation through syncytial blastoderm to the hatching of the first instar larva. An example of this process is found in Drosophila melanogaster.'},\n",
       "  'children': [{'id': 'GO:0046665', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0061321', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030589', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0035190', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0008258', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007390', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007391', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009792', 'relation': 'is_a'}]},\n",
       " 'GO:0001701': {'id': 'GO:0001701',\n",
       "  'isObsolete': False,\n",
       "  'name': 'in utero embryonic development',\n",
       "  'definition': {'text': 'The process whose specific outcome is the progression of the embryo in the uterus over time, from formation of the zygote in the oviduct, to birth. An example of this process is found in Mus musculus.'},\n",
       "  'children': [{'id': 'GO:0001892', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001824', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0043009', 'relation': 'is_a'}]},\n",
       " 'GO:0001702': {'id': 'GO:0001702',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gastrulation with mouth forming second',\n",
       "  'definition': {'text': 'A gastrulation process in which the initial invagination becomes the anus and the mouth forms second.'},\n",
       "  'synonyms': [{'name': 'deuterostomic gastrulation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0055112', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0055111', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0055110', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0055109', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090009', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0055113', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007369', 'relation': 'is_a'}]},\n",
       " 'GO:0001703': {'id': 'GO:0001703',\n",
       "  'isObsolete': False,\n",
       "  'name': 'gastrulation with mouth forming first',\n",
       "  'definition': {'text': 'A gastrulation process in which the initial invagination becomes the mouth and the anus forms second.'},\n",
       "  'synonyms': [{'name': 'protostomic gastrulation', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0010004', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007369', 'relation': 'is_a'}]},\n",
       " 'GO:0001704': {'id': 'GO:0001704',\n",
       "  'isObsolete': False,\n",
       "  'name': 'formation of primary germ layer',\n",
       "  'definition': {'text': 'The formation of the ectoderm, mesoderm and endoderm during gastrulation.'},\n",
       "  'children': [{'id': 'GO:0001705', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001706', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060795', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001707', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007369', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048646', 'relation': 'is_a'}]},\n",
       " 'GO:0001705': {'id': 'GO:0001705',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ectoderm formation',\n",
       "  'definition': {'text': 'The formation of ectoderm during gastrulation.'},\n",
       "  'children': [{'id': 'GO:0001712', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001704', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007398', 'relation': 'part_of'}]},\n",
       " 'GO:0001706': {'id': 'GO:0001706',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endoderm formation',\n",
       "  'definition': {'text': 'The formation of the endoderm during gastrulation.'},\n",
       "  'synonyms': [{'name': 'endoblast formation', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0035987', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001704', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007492', 'relation': 'part_of'}]},\n",
       " 'GO:0001707': {'id': 'GO:0001707',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mesoderm formation',\n",
       "  'definition': {'text': 'The process that gives rise to the mesoderm. This process pertains to the initial formation of the structure from unspecified parts.'},\n",
       "  'children': [{'id': 'GO:0048320', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007509', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1905903', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0048333', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048370', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048391', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905904', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0048341', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905902', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001704', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048332', 'relation': 'part_of'}]},\n",
       " 'GO:0001708': {'id': 'GO:0001708',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell fate specification',\n",
       "  'definition': {'text': 'The process involved in the specification of cell identity. Once specification has taken place, a cell will be committed to differentiate down a specific pathway if left in its normal environment.'},\n",
       "  'children': [{'id': 'GO:0009957', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007501', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0014817', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090628', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072327', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010234', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0021780', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060847', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009996', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0042659', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0042660', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0035153', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048866', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060573', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042679', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001714', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001715', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060912', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048665', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035857', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042694', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010158', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048626', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090610', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0045165', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048869', 'relation': 'is_a'}]},\n",
       " 'GO:0001709': {'id': 'GO:0001709',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell fate determination',\n",
       "  'definition': {'text': 'A process involved in cell fate commitment. Once determination has taken place, a cell becomes committed to differentiate down a particular pathway regardless of its environment.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0878932437'}]},\n",
       "  'children': [{'id': 'GO:0042680', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007521', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007500', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045167', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0035028', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048664', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007493', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046845', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007278', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060848', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072326', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007425', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060582', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007403', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905935', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0048867', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030716', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905934', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0007518', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060913', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035858', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1905933', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001713', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0045165', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048869', 'relation': 'is_a'}]},\n",
       " 'GO:0001710': {'id': 'GO:0001710',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mesodermal cell fate commitment',\n",
       "  'definition': {'text': 'The cell differentiation process that results in commitment of a cell to become part of the mesoderm.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0878932437'}]},\n",
       "  'synonyms': [{'name': 'mesoderm cell fate commitment', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0048393', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048372', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007500', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007501', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048322', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048343', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048333', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060795', 'relation': 'is_a'}]},\n",
       " 'GO:0001711': {'id': 'GO:0001711',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endodermal cell fate commitment',\n",
       "  'definition': {'text': 'The cell differentiation process that results in commitment of a cell to become part of the endoderm.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0878932437'}]},\n",
       "  'synonyms': [{'name': 'endoderm cell fate commitment', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001714', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007493', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0035987', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060795', 'relation': 'is_a'}]},\n",
       " 'GO:0001712': {'id': 'GO:0001712',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ectodermal cell fate commitment',\n",
       "  'definition': {'text': 'The cell differentiation process that results in commitment of a cell to become part of the ectoderm.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0878932437'}]},\n",
       "  'synonyms': [{'name': 'ectoderm cell fate commitment', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001713', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001715', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001705', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0010668', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060795', 'relation': 'is_a'}]},\n",
       " 'GO:0001713': {'id': 'GO:0001713',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ectodermal cell fate determination',\n",
       "  'definition': {'text': 'The cell fate determination process that results in a cell becoming capable of differentiating autonomously into an ectoderm cell regardless of its environment; upon determination, the cell fate cannot be reversed.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0878932437'}]},\n",
       "  'synonyms': [{'name': 'ectoderm cell fate determination', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001709', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001712', 'relation': 'part_of'}]},\n",
       " 'GO:0001714': {'id': 'GO:0001714',\n",
       "  'isObsolete': False,\n",
       "  'name': 'endodermal cell fate specification',\n",
       "  'definition': {'text': 'The cell fate determination process that results in a cell becoming capable of differentiating autonomously into an endoderm cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed.'},\n",
       "  'synonyms': [{'name': 'endoderm cell fate specification', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0042664', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0060799', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060807', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042663', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001708', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001711', 'relation': 'part_of'}]},\n",
       " 'GO:0001715': {'id': 'GO:0001715',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ectodermal cell fate specification',\n",
       "  'definition': {'text': 'The cell fate determination process that results in a cell becoming becomes capable of differentiating autonomously into an ectoderm cell in an environment that is neutral with respect to the developmental pathway; upon specification, the cell fate can be reversed.'},\n",
       "  'synonyms': [{'name': 'ectoderm cell fate specification', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0042666', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0042665', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001708', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001712', 'relation': 'part_of'}]},\n",
       " 'GO:0001716': {'id': 'GO:0001716',\n",
       "  'isObsolete': False,\n",
       "  'name': 'L-amino-acid oxidase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: a L-amino acid + H2O + O2 = a 2-oxo acid + NH3 + hydrogen peroxide.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.4.3.2'}]},\n",
       "  'synonyms': [{'name': 'ophio-amino-acid oxidase activity', 'type': 'exact'},\n",
       "   {'name': 'L-amino-acid:oxygen oxidoreductase (deaminating)',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0033736', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050029', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008734', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050025', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016641', 'relation': 'is_a'}]},\n",
       " 'GO:0001717': {'id': 'GO:0001717',\n",
       "  'isObsolete': False,\n",
       "  'name': 'conversion of seryl-tRNAsec to selenocys-tRNAsec',\n",
       "  'definition': {'text': 'The modification process that results in the conversion of serine, carried by a specialized tRNA(ser) (which can read a UGA anticodon), to selenocysteine.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '155581073X'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019988', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030630', 'relation': 'is_a'}]},\n",
       " 'GO:0001720': {'id': 'GO:0001720',\n",
       "  'isObsolete': False,\n",
       "  'name': 'conversion of lysyl-tRNA to pyrrolysyl-tRNA',\n",
       "  'definition': {'text': 'The modification process that results in the conversion of lysine, carried by a specialized lysine-accepting tRNA (possessing a CUA anticodon), to pyrrolysine (a lysine with an amide linkage to a (4R,5R)-4-substituted pyrroline-5-carboxylate).',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12029131'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12029132'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12121639'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019988', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030630', 'relation': 'is_a'}]},\n",
       " 'GO:0001725': {'id': 'GO:0001725',\n",
       "  'isObsolete': False,\n",
       "  'name': 'stress fiber',\n",
       "  'definition': {'text': 'A contractile actin filament bundle that consists of short actin filaments with alternating polarity, cross-linked by alpha-actinin and possibly other actin bundling proteins, and with myosin present in a periodic distribution along the fiber.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '16651381'}]},\n",
       "  'synonyms': [{'name': 'actin cable', 'type': 'related'},\n",
       "   {'name': 'stress fibre', 'type': 'exact'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001726': {'id': 'GO:0001726',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ruffle',\n",
       "  'definition': {'text': 'Projection at the leading edge of a crawling cell; the protrusions are supported by a microfilament meshwork.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0124325653'}]},\n",
       "  'synonyms': [{'name': 'membrane ruffle', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0032587', 'relation': 'part_of'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001727': {'id': 'GO:0001727',\n",
       "  'isObsolete': False,\n",
       "  'name': 'lipid kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the phosphorylation of a simple or complex lipid.'},\n",
       "  'children': [{'id': 'GO:0001729', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090219', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0043550', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0008481', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090218', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016301', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046834', 'relation': 'part_of'}]},\n",
       " 'GO:0001729': {'id': 'GO:0001729',\n",
       "  'isObsolete': False,\n",
       "  'name': 'ceramide kinase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: ATP + ceramide = ADP + ceramide-1-phosphate.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.7.1.138'}]},\n",
       "  'synonyms': [{'name': 'ATP:ceramide 1-phosphotransferase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'acylsphingosine kinase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001727', 'relation': 'is_a'}]},\n",
       " 'GO:0001730': {'id': 'GO:0001730',\n",
       "  'isObsolete': False,\n",
       "  'name': \"2'-5'-oligoadenylate synthetase activity\",\n",
       "  'definition': {'text': \"Catalysis of the reaction: ATP = pppA(2'p5'A)n oligomers. This reaction requires the binding of double-stranded RNA.\",\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'synonyms': [{'name': '2-5A synthetase activity', 'type': 'exact'},\n",
       "   {'name': \"oligo-2',5'-adenylate synthetase activity\", 'type': 'exact'},\n",
       "   {'name': \"(2-5')oligo(A) synthetase activity\", 'type': 'exact'},\n",
       "   {'name': \"2'-5' oligoadenylate synthetase activity\", 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0070566', 'relation': 'is_a'}]},\n",
       " 'GO:0001731': {'id': 'GO:0001731',\n",
       "  'isObsolete': False,\n",
       "  'name': 'formation of translation preinitiation complex',\n",
       "  'definition': {'text': 'The joining of the small ribosomal subunit, ternary complex, and mRNA.'},\n",
       "  'comment': \"See also the cellular component terms 'eukaryotic 43S preinitiation complex ; GO:0016282', 'eukaryotic 48S initiation complex ; GO:0033290', and their children.\",\n",
       "  'synonyms': [{'name': 'formation of translation pre-initiation complex',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'translation preinitiation complex assembly', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1901195', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1901194', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1901193', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002183', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'}]},\n",
       " 'GO:0001732': {'id': 'GO:0001732',\n",
       "  'isObsolete': False,\n",
       "  'name': 'formation of cytoplasmic translation initiation complex',\n",
       "  'definition': {'text': 'Joining of the large subunit, with release of IF2/eIF2 and IF3/eIF3. This leaves the functional ribosome at the AUG, with the methionyl/formyl-methionyl-tRNA positioned at the P site.'},\n",
       "  'synonyms': [{'name': 'translation initiation complex assembly',\n",
       "    'type': 'broad'},\n",
       "   {'name': 'cytoplasmic translation initiation complex assembly',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'formation of translation initiation complex', 'type': 'broad'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002183', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0022618', 'relation': 'is_a'}]},\n",
       " 'GO:0001733': {'id': 'GO:0001733',\n",
       "  'isObsolete': False,\n",
       "  'name': 'galactosylceramide sulfotransferase activity',\n",
       "  'definition': {'text': \"Catalysis of the reaction: 3'-phosphoadenosine 5'-phosphosulfate + a galactosylceramide = adenosine 3',5'-bisphosphate + a galactosylceramidesulfate.\",\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '2.8.2.11'},\n",
       "    {'dbCode': 'PMID', 'dbId': '10727929'}]},\n",
       "  'synonyms': [{'name': \"3'-phosphoadenylyl-sulfate:galactosylceramide 3'-sulfotransferase activity\",\n",
       "    'type': 'exact'},\n",
       "   {'name': 'GSase', 'type': 'exact'},\n",
       "   {'name': \"3'-phosphoadenosine-5'-phosphosulfate-cerebroside sulfotransferase activity\",\n",
       "    'type': 'exact'},\n",
       "   {'name': 'galactosylceramide sulphotransferase activity', 'type': 'exact'},\n",
       "   {'name': 'galactocerebroside sulfotransferase activity', 'type': 'exact'},\n",
       "   {'name': 'glycosphingolipid sulfotransferase activity', 'type': 'exact'},\n",
       "   {'name': 'glycolipid sulfotransferase activity', 'type': 'exact'},\n",
       "   {'name': 'galactolipid sulfotransferase activity', 'type': 'exact'},\n",
       "   {'name': 'cerebroside sulfotransferase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0050694', 'relation': 'is_a'}]},\n",
       " 'GO:0001734': {'id': 'GO:0001734',\n",
       "  'isObsolete': False,\n",
       "  'name': 'mRNA (N6-adenosine)-methyltransferase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: S-adenosyl-L-methionine + RRACH = S-adenosyl-L-homocysteine + RRm6ACH; R is a purine, and H is C, A, or U.'},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0008174', 'relation': 'is_a'}]},\n",
       " 'GO:0001735': {'id': 'GO:0001735',\n",
       "  'isObsolete': False,\n",
       "  'name': 'prenylcysteine oxidase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: S-prenyl-L-cysteine + O2 + H2O = a prenal + L-cysteine + H2O2.'},\n",
       "  'synonyms': [{'name': 'prenylcysteine lyase activity', 'type': 'exact'},\n",
       "   {'name': 'S-prenyl-L-cysteine:oxygen oxidoreductase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016670', 'relation': 'is_a'}]},\n",
       " 'GO:0001736': {'id': 'GO:0001736',\n",
       "  'isObsolete': False,\n",
       "  'name': 'establishment of planar polarity',\n",
       "  'definition': {'text': 'Coordinated organization of groups of cells in the plane of an epithelium, such that they all orient to similar coordinates.'},\n",
       "  'synonyms': [{'name': 'establishment of planar cell polarity',\n",
       "    'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0042067', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001737', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042252', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048104', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003424', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072046', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090251', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042247', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090250', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090176', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042249', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0090175', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001738', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007164', 'relation': 'is_a'}]},\n",
       " 'GO:0001737': {'id': 'GO:0001737',\n",
       "  'isObsolete': False,\n",
       "  'name': 'establishment of imaginal disc-derived wing hair orientation',\n",
       "  'definition': {'text': 'Orientation of hairs in the imaginal disc-derived wing along a proximal-distal axis, such that each cell of the wing produces one wing hair which points in a distal direction.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11239465'}]},\n",
       "  'synonyms': [{'name': 'establishment of wing hair orientation',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0035320', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001736', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009886', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035317', 'relation': 'part_of'}]},\n",
       " 'GO:0001738': {'id': 'GO:0001738',\n",
       "  'isObsolete': False,\n",
       "  'name': 'morphogenesis of a polarized epithelium',\n",
       "  'definition': {'text': 'The morphogenetic process in which the anatomical structures of a polarized epithelium are generated and organized. A polarized epithelium is an epithelium where the epithelial sheet is oriented with respect to the planar axis.'},\n",
       "  'synonyms': [{'name': 'epithelial polarization', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0030859', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001736', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002009', 'relation': 'is_a'}]},\n",
       " 'GO:0001739': {'id': 'GO:0001739',\n",
       "  'isObsolete': False,\n",
       "  'name': 'sex chromatin',\n",
       "  'definition': {'text': 'Chromatin that is part of a sex chromosome.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'}]},\n",
       "  'children': [{'id': 'GO:0098578', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0098580', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001740': {'id': 'GO:0001740',\n",
       "  'isObsolete': False,\n",
       "  'name': 'Barr body',\n",
       "  'definition': {'text': 'A structure found in a female mammalian cell containing an unpaired X chromosome that has become densely heterochromatic, silenced and localized at the nuclear periphery.',\n",
       "   'xrefs': [{'dbCode': 'NIF_Subcellular', 'dbId': 'sao1571698684'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001741': {'id': 'GO:0001741',\n",
       "  'isObsolete': False,\n",
       "  'name': 'XY body',\n",
       "  'definition': {'text': 'A structure found in a male mammalian spermatocyte containing an unpaired X chromosome that has become densely heterochromatic, silenced and localized at the nuclear periphery.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '20622855'},\n",
       "    {'dbCode': 'URL',\n",
       "     'dbId': 'https://en.wikipedia.org/wiki/XY_sex-determination_system'}]},\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001742': {'id': 'GO:0001742',\n",
       "  'isObsolete': False,\n",
       "  'name': 'oenocyte differentiation',\n",
       "  'definition': {'text': 'The process in which a relatively unspecialized cell acquires the specialized features of an oenocyte. Oenocytes are large secretory cells found in clusters underlying the epidermis of larval abdominal segments.'},\n",
       "  'synonyms': [{'name': 'oenocyte cell differentiation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0007438', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030154', 'relation': 'is_a'}]},\n",
       " 'GO:0001743': {'id': 'GO:0001743',\n",
       "  'isObsolete': False,\n",
       "  'name': 'optic placode formation',\n",
       "  'definition': {'text': 'The initial developmental process that will lead to the formation of an eye.'},\n",
       "  'children': [{'id': 'GO:0046619', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048598', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060788', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048828', 'relation': 'part_of'}]},\n",
       " 'GO:0001744': {'id': 'GO:0001744',\n",
       "  'isObsolete': False,\n",
       "  'name': 'optic lobe placode formation',\n",
       "  'definition': {'text': 'Establishment of the optic placode, a thickened area of densely packed ectoderm cells directly overlying the optic vesicle in the early embryo. In Drosophila, for example, the placode appears in the dorsolateral region of the head in late stage 11 embryos and is the precursor to the larval visual system.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '8402833'}]},\n",
       "  'synonyms': [{'name': \"optic lobe and Bolwig's organ precursor formation\",\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0060788', 'relation': 'is_a'}]},\n",
       " 'GO:0001745': {'id': 'GO:0001745',\n",
       "  'isObsolete': False,\n",
       "  'name': 'compound eye morphogenesis',\n",
       "  'definition': {'text': 'The morphogenetic process in which the anatomical structures of the compound eye are generated and organized. The adult compound eye is a precise assembly of 700-800 ommatidia. Each ommatidium is composed of 20 cells, identified by cell type and position. An example of compound eye morphogenesis is found in Drosophila melanogaster.'},\n",
       "  'synonyms': [{'name': 'insect-type retina morphogenesis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0046667', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0008583', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001751', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007458', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0016330', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042067', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042675', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048750', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0062056', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0035736', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048592', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048749', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048748', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007456', 'relation': 'part_of'}]},\n",
       " 'GO:0001746': {'id': 'GO:0001746',\n",
       "  'isObsolete': False,\n",
       "  'name': \"Bolwig's organ morphogenesis\",\n",
       "  'definition': {'text': \"The morphogenetic process in which the anatomical structures of the larval eye in Drosophila are generated and organized. The larval eye in Drosophila is a relatively simple sensory system composed of Bolwig's organs: two clusters, each composed of 12 photoreceptor cells from which axons extend in a single fascicle to the brain.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '6185380'},\n",
       "    {'dbCode': 'URL',\n",
       "     'dbId': 'http://sdb.bio.purdue.edu/fly/torstoll/tailess.htm#bolwigs'}]},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048592', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0055034', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048748', 'relation': 'is_a'}]},\n",
       " 'GO:0001748': {'id': 'GO:0001748',\n",
       "  'isObsolete': False,\n",
       "  'name': 'optic lobe placode development',\n",
       "  'definition': {'text': \"The process whose specific outcome is the progression of the optic placode over time, from its formation to the mature structure. During embryonic stage 12 the placode starts to invaginate, forming a pouch. Cells that will form Bolwig's organ segregate from the ventral lip of this pouch, remaining in the head epidermis. The remainder of the invagination loses contact with the outer surface and becomes the optic lobe. An example of this process is found in Drosophila melanogaster.\",\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '8402833'}]},\n",
       "  'synonyms': [{'name': 'optic placode development', 'type': 'broad'},\n",
       "   {'name': \"optic lobe and Bolwig's organ precursor development\",\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0048856', 'relation': 'is_a'}]},\n",
       " 'GO:0001750': {'id': 'GO:0001750',\n",
       "  'isObsolete': False,\n",
       "  'name': 'photoreceptor outer segment',\n",
       "  'definition': {'text': 'The outer segment of a vertebrate photoreceptor that contains a stack of membrane discs embedded with photoreceptor proteins.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0824072820'},\n",
       "    {'dbCode': 'PMID', 'dbId': '19501669'},\n",
       "    {'dbCode': 'PMID', 'dbId': '26574505'},\n",
       "    {'dbCode': 'PMID', 'dbId': '6771304'}]},\n",
       "  'children': [{'id': 'GO:0042622', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0097381', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0120199', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0120200', 'relation': 'is_a'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001751': {'id': 'GO:0001751',\n",
       "  'isObsolete': False,\n",
       "  'name': 'compound eye photoreceptor cell differentiation',\n",
       "  'definition': {'text': 'The process in which a relatively unspecialized cell acquires the specialized features of an eye photoreceptor cell.'},\n",
       "  'children': [{'id': 'GO:0048054', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0110116', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0110118', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0048052', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042051', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001752', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0045465', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045466', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0110117', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0048056', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001745', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001754', 'relation': 'is_a'}]},\n",
       " 'GO:0001752': {'id': 'GO:0001752',\n",
       "  'isObsolete': False,\n",
       "  'name': 'compound eye photoreceptor fate commitment',\n",
       "  'definition': {'text': 'The process in which the developmental fate of a cell becomes restricted such that it will develop into a compound eye photoreceptor cell. A photoreceptor cell is a cell that responds to incident electromagnetic radiation. Different classes of photoreceptor have different spectral sensitivities and express different photosensitive pigments.'},\n",
       "  'children': [{'id': 'GO:0007460', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007462', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007463', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007464', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007465', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001751', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042706', 'relation': 'is_a'}]},\n",
       " 'GO:0001754': {'id': 'GO:0001754',\n",
       "  'isObsolete': False,\n",
       "  'name': 'eye photoreceptor cell differentiation',\n",
       "  'definition': {'text': 'The process in which a relatively unspecialized cell acquires the specialized features of a photoreceptor cell, as found in the eye, the primary visual organ of most organisms.'},\n",
       "  'children': [{'id': 'GO:0042462', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001751', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042706', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0060219', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0046530', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048592', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007467', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048748', 'relation': 'part_of'}]},\n",
       " 'GO:0001755': {'id': 'GO:0001755',\n",
       "  'isObsolete': False,\n",
       "  'name': 'neural crest cell migration',\n",
       "  'definition': {'text': 'The characteristic movement of cells from the dorsal ridge of the neural tube to a variety of locations in a vertebrate embryo.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0878932437'}]},\n",
       "  'children': [{'id': 'GO:0003253', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036484', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1901166', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003147', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001667', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0014032', 'relation': 'part_of'}]},\n",
       " 'GO:0001756': {'id': 'GO:0001756',\n",
       "  'isObsolete': False,\n",
       "  'name': 'somitogenesis',\n",
       "  'definition': {'text': 'The formation of mesodermal clusters that are arranged segmentally along the anterior posterior axis of an embryo.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0721662544'}]},\n",
       "  'synonyms': [{'name': 'formation of mesodermal clusters', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0090245', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090246', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090243', 'relation': 'part_of'},\n",
       "   {'id': 'GO:1902359', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090244', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0090242', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032525', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0014807', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001757', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009952', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035282', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0043009', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0048646', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061053', 'relation': 'part_of'}]},\n",
       " 'GO:0001757': {'id': 'GO:0001757',\n",
       "  'isObsolete': False,\n",
       "  'name': 'somite specification',\n",
       "  'definition': {'text': 'The process in which individual somites establish identity during embryogenesis.'},\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001756', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0007379', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0009880', 'relation': 'is_a'}]},\n",
       " 'GO:0001758': {'id': 'GO:0001758',\n",
       "  'isObsolete': False,\n",
       "  'name': 'retinal dehydrogenase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: retinal + NAD+ + H2O = retinoate + NADH. Acts on both 11-trans and 13-cis forms of retinal.',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '1.2.1.36'}]},\n",
       "  'synonyms': [{'name': 'cytosolic retinal dehydrogenase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'retinal:NAD+ oxidoreductase activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016620', 'relation': 'is_a'}]},\n",
       " 'GO:0001759': {'id': 'GO:0001759',\n",
       "  'isObsolete': False,\n",
       "  'name': 'organ induction',\n",
       "  'definition': {'text': 'The interaction of two or more cells or tissues that causes them to change their fates and specify the development of an organ.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0878932437'}]},\n",
       "  'synonyms': [{'name': 'induction of an organ', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0060492', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060514', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061132', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003129', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0003156', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010092', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0031128', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0110110', 'relation': 'is_a'}]},\n",
       " 'GO:0001760': {'id': 'GO:0001760',\n",
       "  'isObsolete': False,\n",
       "  'name': 'aminocarboxymuconate-semialdehyde decarboxylase activity',\n",
       "  'definition': {'text': 'Catalysis of the reaction: 2-amino-3-carboxymuconate 6-semialdehyde + H(+) = 2-aminomuconate 6-semialdehyde + CO(2).',\n",
       "   'xrefs': [{'dbCode': 'EC', 'dbId': '4.1.1.45'},\n",
       "    {'dbCode': 'RHEA', 'dbId': '16557'}]},\n",
       "  'synonyms': [{'name': '2-amino-3-(3-oxoprop-1-en-1-yl)but-2-enedioate carboxy-lyase (2-aminomuconate-semialdehyde-forming)',\n",
       "    'type': 'exact'},\n",
       "   {'name': '2-amino-3-(3-oxoprop-1-en-1-yl)but-2-enedioate carboxy-lyase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'ACMSD activity', 'type': 'exact'},\n",
       "   {'name': 'picolinic acid decarboxylase activity', 'type': 'exact'},\n",
       "   {'name': 'picolinic acid carboxylase activity', 'type': 'exact'},\n",
       "   {'name': 'alpha-amino-beta-carboxymuconate-epsilon-semialdehyde beta-decarboxylase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'alpha-amino-beta-carboxymuconate-epsilon-semialdehade decarboxylase activity',\n",
       "    'type': 'exact'},\n",
       "   {'name': '2-amino-3-(3-oxoprop-2-enyl)but-2-enedioate carboxy-lyase activity',\n",
       "    'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016831', 'relation': 'is_a'}]},\n",
       " 'GO:0001761': {'id': 'GO:0001761',\n",
       "  'isObsolete': False,\n",
       "  'name': 'beta-alanine transmembrane transporter activity',\n",
       "  'definition': {'text': 'Enables the transfer of beta-alanine from one side of a membrane to the other. Beta-alanine is 3-aminopropanoic acid.'},\n",
       "  'synonyms': [{'name': 'beta-alanine transporter activity', 'type': 'broad'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001762', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0005326', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015175', 'relation': 'is_a'}]},\n",
       " 'GO:0001762': {'id': 'GO:0001762',\n",
       "  'isObsolete': False,\n",
       "  'name': 'beta-alanine transport',\n",
       "  'definition': {'text': 'The directed movement of beta-alanine, 3-aminopropanoic acid, into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore.'},\n",
       "  'children': [{'id': 'GO:0001761', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0006836', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0015804', 'relation': 'is_a'}]},\n",
       " 'GO:0001763': {'id': 'GO:0001763',\n",
       "  'isObsolete': False,\n",
       "  'name': 'morphogenesis of a branching structure',\n",
       "  'definition': {'text': 'The process in which the anatomical structures of branches are generated and organized. A branch is a division or offshoot from a main stem. Examples in animals would include blood vessels, nerves, lymphatics and other endothelial or epithelial tubes.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0721662544'}]},\n",
       "  'synonyms': [{'name': 'branching morphogenesis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0061137', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0080181', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048755', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0010223', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061138', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0060688', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009653', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032501', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044707', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050874', 'relation': 'is_a'}]},\n",
       " 'GO:0001764': {'id': 'GO:0001764',\n",
       "  'isObsolete': False,\n",
       "  'name': 'neuron migration',\n",
       "  'definition': {'text': 'The characteristic movement of an immature neuron from germinal zones to specific positions where they will reside as they mature.',\n",
       "   'xrefs': [{'dbCode': 'CL', 'dbId': '0000540'}]},\n",
       "  'synonyms': [{'name': 'neuron guidance', 'type': 'related'},\n",
       "   {'name': 'neuronal migration', 'type': 'exact'},\n",
       "   {'name': 'neuron chemotaxis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0021947', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0097475', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0003393', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2001222', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0021852', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2001223', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1904937', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1904936', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0021948', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0021828', 'relation': 'is_a'},\n",
       "   {'id': 'GO:2001224', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0016477', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048699', 'relation': 'part_of'}]},\n",
       " 'GO:0001765': {'id': 'GO:0001765',\n",
       "  'isObsolete': False,\n",
       "  'name': 'membrane raft assembly',\n",
       "  'definition': {'text': 'The aggregation, arrangement and bonding together of a set of components to form a membrane raft, a small (10-200 nm), heterogeneous, highly dynamic, sterol- and sphingolipid-enriched membrane domains that compartmentalizes cellular processes.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12648772'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12803918'},\n",
       "    {'dbCode': 'PMID', 'dbId': '16645198'}]},\n",
       "  'synonyms': [{'name': 'membrane raft formation', 'type': 'related'},\n",
       "   {'name': 'lipid raft assembly', 'type': 'exact'},\n",
       "   {'name': 'lipid raft formation', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0044854', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031579', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071709', 'relation': 'is_a'}]},\n",
       " 'GO:0001766': {'id': 'GO:0001766',\n",
       "  'isObsolete': False,\n",
       "  'name': 'membrane raft polarization',\n",
       "  'definition': {'text': 'The clustering and aggregation of membrane rafts at a single cellular pole during activation of particular cell types, such as lymphocytes.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12615889'}]},\n",
       "  'synonyms': [{'name': 'lipid raft polarization', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0044858', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0031580', 'relation': 'is_a'}]},\n",
       " 'GO:0001767': {'id': 'GO:0001767',\n",
       "  'isObsolete': False,\n",
       "  'name': 'establishment of lymphocyte polarity',\n",
       "  'definition': {'text': 'The directed orientation of lymphocyte signaling molecules and associated membrane rafts towards a chemokine gradient or a contact point with an appropriate activating cell.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11244041'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12615889'}]},\n",
       "  'synonyms': [{'name': 'lymphocyte polarization', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001768', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001770', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001769', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030010', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046649', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0030468', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0000283', 'relation': 'is_a'}]},\n",
       " 'GO:0001768': {'id': 'GO:0001768',\n",
       "  'isObsolete': False,\n",
       "  'name': 'establishment of T cell polarity',\n",
       "  'definition': {'text': 'The directed orientation of T cell signaling molecules and associated membrane rafts towards a chemokine gradient or a contact point with antigen presenting cell.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11244041'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12615889'}]},\n",
       "  'synonyms': [{'name': 'T cell polarization', 'type': 'exact'},\n",
       "   {'name': 'establishment of T-lymphocyte polarity', 'type': 'exact'},\n",
       "   {'name': 'establishment of T-cell polarity', 'type': 'exact'},\n",
       "   {'name': 'establishment of T lymphocyte polarity', 'type': 'exact'},\n",
       "   {'name': 'T-cell polarization', 'type': 'exact'},\n",
       "   {'name': 'T lymphocyte polarization', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1903903', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1903904', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1903905', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001767', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042110', 'relation': 'part_of'}]},\n",
       " 'GO:0001769': {'id': 'GO:0001769',\n",
       "  'isObsolete': False,\n",
       "  'name': 'establishment of B cell polarity',\n",
       "  'definition': {'text': 'The directed orientation of B cell signaling molecules and associated membrane rafts towards a chemokine gradient of a contact point with an antigen displaying cell.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12615889'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9692889'}]},\n",
       "  'comment': \"Note that 'antigen displaying cell' in this term encompasses cell types such as follicular dendritic cells which display the unprocessed antigens of a B cell, leading to activation of the B cells and antigen uptake and processing by the B cells.\",\n",
       "  'synonyms': [{'name': 'establishment of B-cell polarity', 'type': 'exact'},\n",
       "   {'name': 'B cell polarization', 'type': 'exact'},\n",
       "   {'name': 'B lymphocyte polarization', 'type': 'exact'},\n",
       "   {'name': 'B-cell polarization', 'type': 'exact'},\n",
       "   {'name': 'establishment of B lymphocyte polarity', 'type': 'exact'},\n",
       "   {'name': 'establishment of B-lymphocyte polarity', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001767', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042113', 'relation': 'part_of'}]},\n",
       " 'GO:0001770': {'id': 'GO:0001770',\n",
       "  'isObsolete': False,\n",
       "  'name': 'establishment of natural killer cell polarity',\n",
       "  'definition': {'text': 'The directed orientation of natural killer cell signaling molecules and associated membrane rafts towards a chemokine gradient or a contact point with a cell displaying natural killer cell activating ligands.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12615886'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9759849'}]},\n",
       "  'synonyms': [{'name': 'natural killer cell polarization', 'type': 'exact'},\n",
       "   {'name': 'establishment of NK cell polarity', 'type': 'exact'},\n",
       "   {'name': 'NK cell polarization', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001767', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030101', 'relation': 'part_of'}]},\n",
       " 'GO:0001771': {'id': 'GO:0001771',\n",
       "  'isObsolete': False,\n",
       "  'name': 'immunological synapse formation',\n",
       "  'definition': {'text': 'The formation of an area of close contact between a lymphocyte (T-, B-, or natural killer cell) and a target cell through the clustering of particular signaling and adhesion molecules and their associated membrane rafts on both the lymphocyte and target cell, which facilitates activation of the lymphocyte, transfer of membrane from the target cell to the lymphocyte, and in some situations killing of the target cell through release of secretory granules and/or death-pathway ligand-receptor interaction.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11244041'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11376330'}]},\n",
       "  'synonyms': [{'name': 'formation of immunological synapse',\n",
       "    'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:2000522', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:2000521', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:2000520', 'relation': 'regulates'},\n",
       "   {'id': 'GO:2000526', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009988', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046649', 'relation': 'part_of'}]},\n",
       " 'GO:0001772': {'id': 'GO:0001772',\n",
       "  'isObsolete': False,\n",
       "  'name': 'immunological synapse',\n",
       "  'definition': {'text': 'An area of close contact between a lymphocyte (T-, B-, or natural killer cell) and a target cell formed through the clustering of particular signaling and adhesion molecules and their associated membrane rafts on both the lymphocyte and the target cell and facilitating activation of the lymphocyte, transfer of membrane from the target cell to the lymphocyte, and in some situations killing of the target cell through release of secretory granules and/or death-pathway ligand-receptor interaction.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '11244041'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11376300'}]},\n",
       "  'synonyms': [{'name': 'supramolecular activation cluster', 'type': 'exact'},\n",
       "   {'name': 'c-SMAC', 'type': 'narrow'}],\n",
       "  'aspect': 'cellular_component',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': []},\n",
       " 'GO:0001773': {'id': 'GO:0001773',\n",
       "  'isObsolete': False,\n",
       "  'name': 'myeloid dendritic cell activation',\n",
       "  'definition': {'text': 'The change in morphology and behavior of a dendritic cell resulting from exposure to a cytokine, chemokine, cellular ligand, or soluble factor.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'children': [{'id': 'GO:0043011', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002277', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030885', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0030887', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0030886', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002274', 'relation': 'is_a'}]},\n",
       " 'GO:0001774': {'id': 'GO:0001774',\n",
       "  'isObsolete': False,\n",
       "  'name': 'microglial cell activation',\n",
       "  'definition': {'text': 'The change in morphology and behavior of a microglial cell resulting from exposure to a cytokine, chemokine, cellular ligand, or soluble factor.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '10626665'},\n",
       "    {'dbCode': 'PMID', 'dbId': '10695728'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12580336'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9893949'}]},\n",
       "  'children': [{'id': 'GO:1903979', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:1903978', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1903980', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0002282', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002269', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042116', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061900', 'relation': 'is_a'}]},\n",
       " 'GO:0001775': {'id': 'GO:0001775',\n",
       "  'isObsolete': False,\n",
       "  'name': 'cell activation',\n",
       "  'definition': {'text': 'A change in the morphology or behavior of a cell resulting from exposure to an activating factor such as a cellular or soluble ligand.'},\n",
       "  'children': [{'id': 'GO:0007407', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042118', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061900', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0007343', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002266', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002263', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071892', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050865', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0032980', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044566', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0014719', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050866', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0030168', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072537', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0045321', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050867', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0009987', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0050875', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0044763', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0008151', 'relation': 'is_a'}]},\n",
       " 'GO:0001776': {'id': 'GO:0001776',\n",
       "  'isObsolete': False,\n",
       "  'name': 'leukocyte homeostasis',\n",
       "  'definition': {'text': 'The process of regulating the proliferation and elimination of cells of the immune system such that the total number of cells of a particular cell type within a whole or part of an organism is stable over time in the absence of an outside stimulus.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'comment': 'Note that this term represents the return of immune system cell levels to stable numbers following an immune response as well as the proliferation and elimination of cells of the immune system required to maintain stable numbers in the absence of an outside stimulus.',\n",
       "  'synonyms': [{'name': 'leucocyte homeostasis', 'type': 'exact'},\n",
       "   {'name': 'immune cell homeostasis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:1990960', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0035702', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001780', 'relation': 'is_a'},\n",
       "   {'id': 'GO:1990959', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0036145', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033023', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002260', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0061519', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002376', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0048872', 'relation': 'is_a'}]},\n",
       " 'GO:0001777': {'id': 'GO:0001777',\n",
       "  'isObsolete': False,\n",
       "  'name': 'T cell homeostatic proliferation',\n",
       "  'definition': {'text': 'The non-specific expansion of T cell populations within a whole or part of an organism to reach to a total number of T cells which will then remain stable over time in the absence of an external stimulus.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'synonyms': [{'name': 'T-lymphocyte homeostatic proliferation',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'T-cell homeostatic proliferation', 'type': 'exact'},\n",
       "   {'name': 'resting T cell proliferation', 'type': 'exact'},\n",
       "   {'name': 'resting T-cell proliferation', 'type': 'exact'},\n",
       "   {'name': 'T lymphocyte homeostatic proliferation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0046013', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0046014', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0042103', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0042098', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042592', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0043029', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0042111', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032845', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032844', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0032846', 'relation': 'part_of'}]},\n",
       " 'GO:0001778': {'id': 'GO:0001778',\n",
       "  'isObsolete': False,\n",
       "  'name': 'plasma membrane repair',\n",
       "  'definition': {'text': 'The resealing of a cell plasma membrane after cellular wounding due to, for instance, mechanical stress.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12925704'}]},\n",
       "  'children': [{'id': 'GO:1905684', 'relation': 'regulates'},\n",
       "   {'id': 'GO:1905686', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1905685', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0007009', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0042060', 'relation': 'part_of'}]},\n",
       " 'GO:0001779': {'id': 'GO:0001779',\n",
       "  'isObsolete': False,\n",
       "  'name': 'natural killer cell differentiation',\n",
       "  'definition': {'text': 'The process in which a relatively unspecialized cell acquires the specialized features of a natural killer cell.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'comment': \"Note that immunologists typically use the word 'development' to refer to cells of B or T cell lineages undergoing the process that GO describes as 'cell differentiation'.\",\n",
       "  'synonyms': [{'name': 'NK cell differentiation', 'type': 'exact'},\n",
       "   {'name': 'natural killer cell development', 'type': 'related'}],\n",
       "  'children': [{'id': 'GO:0002325', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032825', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0032823', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0032824', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030098', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0030101', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046650', 'relation': 'is_a'}]},\n",
       " 'GO:0001780': {'id': 'GO:0001780',\n",
       "  'isObsolete': False,\n",
       "  'name': 'neutrophil homeostasis',\n",
       "  'definition': {'text': 'The process of regulating the proliferation and elimination of neutrophils such that the total number of neutrophils within a whole or part of an organism is stable over time in the absence of an outside stimulus.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12752675'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12960266'}]},\n",
       "  'comment': 'Note that this term represents the return of neutrophil levels to stable numbers following an immune response as well as the proliferation and elimination of neutrophils required to maintain stable numbers in the absence of an outside stimulus.',\n",
       "  'children': [{'id': 'GO:0097350', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001781', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001776', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002262', 'relation': 'is_a'}]},\n",
       " 'GO:0001781': {'id': 'GO:0001781',\n",
       "  'isObsolete': False,\n",
       "  'name': 'neutrophil apoptotic process',\n",
       "  'definition': {'text': 'Any apoptotic process in a neutrophil, any of the immature or mature forms of a granular leukocyte that in its mature form has a nucleus with three to five lobes connected by slender threads of chromatin, and cytoplasm containing fine inconspicuous granules and stainable by neutral dyes.',\n",
       "   'xrefs': [{'dbCode': 'CL', 'dbId': '0000775'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12752675'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12960266'}]},\n",
       "  'synonyms': [{'name': 'programmed cell death, neutrophils', 'type': 'exact'},\n",
       "   {'name': 'programmed cell death of neutrophils by apoptosis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'neutrophil programmed cell death by apoptosis', 'type': 'exact'},\n",
       "   {'name': 'apoptosis of neutrophils', 'type': 'exact'},\n",
       "   {'name': 'neutrophil apoptosis', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0033031', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0033029', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0033030', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001780', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0006925', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0033028', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0071887', 'relation': 'is_a'}]},\n",
       " 'GO:0001782': {'id': 'GO:0001782',\n",
       "  'isObsolete': False,\n",
       "  'name': 'B cell homeostasis',\n",
       "  'definition': {'text': 'The process of regulating the proliferation and elimination of B cells such that the total number of B cells within a whole or part of an organism is stable over time in the absence of an outside stimulus.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12956429'}]},\n",
       "  'comment': 'Note that this term represents the return of B cell levels to stable numbers following an immune response as well as the proliferation and elimination of B cells required to maintain stable numbers in the absence of an outside stimulus.',\n",
       "  'synonyms': [{'name': 'B-lymphocyte homeostasis', 'type': 'exact'},\n",
       "   {'name': 'B lymphocyte homeostasis', 'type': 'exact'},\n",
       "   {'name': 'B-cell homeostasis', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0002901', 'relation': 'part_of'},\n",
       "   {'id': 'GO:0001922', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002260', 'relation': 'is_a'}]},\n",
       " 'GO:0001783': {'id': 'GO:0001783',\n",
       "  'isObsolete': False,\n",
       "  'name': 'B cell apoptotic process',\n",
       "  'definition': {'text': 'Any apoptotic process in a B cell, a lymphocyte of B lineage with the phenotype CD19-positive and capable of B cell mediated immunity.',\n",
       "   'xrefs': [{'dbCode': 'CL', 'dbId': '0000236'},\n",
       "    {'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'synonyms': [{'name': 'B cell apoptosis', 'type': 'narrow'},\n",
       "   {'name': 'B-lymphocyte programmed cell death by apoptosis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'B-lymphocyte apoptosis', 'type': 'exact'},\n",
       "   {'name': 'B-cell programmed cell death by apoptosis', 'type': 'exact'},\n",
       "   {'name': 'B-cell apoptosis', 'type': 'exact'},\n",
       "   {'name': 'B lymphocyte programmed cell death by apoptosis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'B lymphocyte apoptosis', 'type': 'exact'},\n",
       "   {'name': 'B cell programmed cell death by apoptosis', 'type': 'exact'},\n",
       "   {'name': 'programmed cell death, B-lymphocytes', 'type': 'exact'},\n",
       "   {'name': 'programmed cell death, B-cells', 'type': 'exact'},\n",
       "   {'name': 'programmed cell death, B lymphocytes', 'type': 'exact'},\n",
       "   {'name': 'programmed cell death, B cells', 'type': 'exact'},\n",
       "   {'name': 'programmed cell death of B-lymphocytes by apoptosis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'programmed cell death of B-cells by apoptosis', 'type': 'exact'},\n",
       "   {'name': 'programmed cell death of B lymphocytes by apoptosis',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'programmed cell death of B cells by apoptosis', 'type': 'exact'},\n",
       "   {'name': 'apoptosis of B-lymphocytes', 'type': 'exact'},\n",
       "   {'name': 'apoptosis of B cells', 'type': 'exact'},\n",
       "   {'name': 'apoptosis of B lymphocytes', 'type': 'exact'},\n",
       "   {'name': 'apoptosis of B-cells', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0002516', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002903', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0002902', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0071948', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002901', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002904', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:1990117', 'relation': 'part_of'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0070227', 'relation': 'is_a'}]},\n",
       " 'GO:0001784': {'id': 'GO:0001784',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phosphotyrosine residue binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a phosphorylated tyrosine residue within a protein.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '14636584'}]},\n",
       "  'synonyms': [{'name': 'phosphotyrosine binding', 'type': 'related'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0045309', 'relation': 'is_a'}]},\n",
       " 'GO:0001785': {'id': 'GO:0001785',\n",
       "  'isObsolete': False,\n",
       "  'name': 'prostaglandin J receptor activity',\n",
       "  'definition': {'text': 'Combining with prostaglandin J (PGJ(2)), a metabolite of prostaglandin D (PGD(2)) to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'PMID', 'dbId': '12878180'}]},\n",
       "  'synonyms': [{'name': 'PGJ receptor activity', 'type': 'related'},\n",
       "   {'name': 'PGJ(2) receptor activity', 'type': 'exact'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0004955', 'relation': 'is_a'}]},\n",
       " 'GO:0001786': {'id': 'GO:0001786',\n",
       "  'isObsolete': False,\n",
       "  'name': 'phosphatidylserine binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with phosphatidylserine, a class of glycophospholipids in which a phosphatidyl group is esterified to the hydroxyl group of L-serine.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0198506732'},\n",
       "    {'dbCode': 'PMID', 'dbId': '12000961'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0005543', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0072341', 'relation': 'is_a'}]},\n",
       " 'GO:0001787': {'id': 'GO:0001787',\n",
       "  'isObsolete': False,\n",
       "  'name': 'natural killer cell proliferation',\n",
       "  'definition': {'text': 'The expansion of a natural killer cell population by cell division.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'synonyms': [{'name': 'NK cell proliferation', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0032817', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0032818', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0002324', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0032819', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0030101', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0046651', 'relation': 'is_a'}]},\n",
       " 'GO:0001788': {'id': 'GO:0001788',\n",
       "  'isObsolete': False,\n",
       "  'name': 'antibody-dependent cellular cytotoxicity',\n",
       "  'definition': {'text': 'Cytolysis of target cells by natural killer cells, eosinophils, neutrophils, monocytes, or macrophages following engagement of antibodies bound to the target cells by Fc receptors on the effector cells.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'},\n",
       "    {'dbCode': 'PMID', 'dbId': '11677095'},\n",
       "    {'dbCode': 'PMID', 'dbId': '9581795'}]},\n",
       "  'synonyms': [{'name': 'type VI hypersensitivity', 'type': 'exact'},\n",
       "   {'name': 'antibody dependent cell death', 'type': 'exact'},\n",
       "   {'name': 'antibody dependent cell killing', 'type': 'exact'},\n",
       "   {'name': 'antibody-dependent cell death', 'type': 'exact'},\n",
       "   {'name': 'ADCC', 'type': 'exact'},\n",
       "   {'name': 'antibody-dependent cell killing', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001813', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001814', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0001815', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001794', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001909', 'relation': 'is_a'}]},\n",
       " 'GO:0001790': {'id': 'GO:0001790',\n",
       "  'isObsolete': False,\n",
       "  'name': 'polymeric immunoglobulin binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with a J-chain-containing polymeric immunoglobulin of the IgA or IgM isotypes.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019865', 'relation': 'is_a'}]},\n",
       " 'GO:0001791': {'id': 'GO:0001791',\n",
       "  'isObsolete': False,\n",
       "  'name': 'IgM binding',\n",
       "  'definition': {'text': 'Interacting selectively and non-covalently with an immunoglobulin of the IgM isotype.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019865', 'relation': 'is_a'}]},\n",
       " 'GO:0001792': {'id': 'GO:0001792',\n",
       "  'isObsolete': False,\n",
       "  'name': 'polymeric immunoglobulin receptor activity',\n",
       "  'definition': {'text': 'Combining with a J-chain-containing polymeric immunoglobulin of the IgA or IgM isotypes via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019763', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016489', 'relation': 'is_a'}]},\n",
       " 'GO:0001793': {'id': 'GO:0001793',\n",
       "  'isObsolete': False,\n",
       "  'name': 'IgM receptor activity',\n",
       "  'definition': {'text': 'Combining with an immunoglobulin of the IgM isotype via the Fc region, and transmitting the signal from one side of the membrane to the other to initiate a change in cell activity.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'children': [{'id': 'GO:0002173', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002172', 'relation': 'is_a'}],\n",
       "  'aspect': 'molecular_function',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0019763', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016489', 'relation': 'is_a'}]},\n",
       " 'GO:0001794': {'id': 'GO:0001794',\n",
       "  'isObsolete': False,\n",
       "  'name': 'type IIa hypersensitivity',\n",
       "  'definition': {'text': 'An inflammatory response resulting in cell death mediated by activation of the classical complement pathway or induction of effector cell phagocytosis or cytolysis mechanisms via complement or Fc receptors following the binding of antibodies to cell surface antigens on a target cell.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'comment': 'Note that some type IIb hypersensitivity responses (GO:0001795) are referred to simply as type II hypersensitivity in the earlier literature, but are mechanistically distinct from type IIa hypersensitivity.',\n",
       "  'children': [{'id': 'GO:0001798', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0001796', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001788', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001797', 'relation': 'negatively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002445', 'relation': 'is_a'}]},\n",
       " 'GO:0001795': {'id': 'GO:0001795',\n",
       "  'isObsolete': False,\n",
       "  'name': 'type IIb hypersensitivity',\n",
       "  'definition': {'text': 'An inflammatory response resulting in cell death or dysfunction mediated by the direct binding of antibody to cellular receptors.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'comment': 'Note that some type IIa hypersensitivity response (GO:0001794) are referred to simply as type II hypersensitivity in the earlier literature, but are mechanistically distinct from type IIb hypersensitivity.',\n",
       "  'synonyms': [{'name': 'type V hypersensitivity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001800', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0001799', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0001801', 'relation': 'positively_regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002445', 'relation': 'is_a'}]},\n",
       " 'GO:0001796': {'id': 'GO:0001796',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of type IIa hypersensitivity',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate, or extent of type IIa hypersensitivity, a type of inflammatory response.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'children': [{'id': 'GO:0001798', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001813', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001797', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001794', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0002892', 'relation': 'is_a'}]},\n",
       " 'GO:0001797': {'id': 'GO:0001797',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative regulation of type IIa hypersensitivity',\n",
       "  'definition': {'text': 'Any process that stops, prevents, or reduces the rate of type IIa hypersensitivity, a type of inflammatory response.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'synonyms': [{'name': 'inhibition of type IIa hypersensitivity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'downregulation of type IIa hypersensitivity', 'type': 'exact'},\n",
       "   {'name': 'down regulation of type IIa hypersensitivity', 'type': 'exact'},\n",
       "   {'name': 'down-regulation of type IIa hypersensitivity', 'type': 'exact'}],\n",
       "  'children': [{'id': 'GO:0001814', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001794', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0001796', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002893', 'relation': 'is_a'}]},\n",
       " 'GO:0001798': {'id': 'GO:0001798',\n",
       "  'isObsolete': False,\n",
       "  'name': 'positive regulation of type IIa hypersensitivity',\n",
       "  'definition': {'text': 'Any process that activates or increases the frequency, rate or extent of type IIa hypersensitivity, a type of inflammatory response.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'synonyms': [{'name': 'up regulation of type IIa hypersensitivity',\n",
       "    'type': 'exact'},\n",
       "   {'name': 'up-regulation of type IIa hypersensitivity', 'type': 'exact'},\n",
       "   {'name': 'upregulation of type IIa hypersensitivity', 'type': 'exact'},\n",
       "   {'name': 'activation of type IIa hypersensitivity', 'type': 'narrow'},\n",
       "   {'name': 'stimulation of type IIa hypersensitivity', 'type': 'narrow'}],\n",
       "  'children': [{'id': 'GO:0001815', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001794', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0001796', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002894', 'relation': 'is_a'}]},\n",
       " 'GO:0001799': {'id': 'GO:0001799',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of type IIb hypersensitivity',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate, or extent of type IIb hypersensitivity, a type of inflammatory response.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'children': [{'id': 'GO:0001800', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001801', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001795', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0002892', 'relation': 'is_a'}]},\n",
       " 'GO:0001800': {'id': 'GO:0001800',\n",
       "  'isObsolete': False,\n",
       "  'name': 'negative regulation of type IIb hypersensitivity',\n",
       "  'definition': {'text': 'Any process that stops, prevents, or reduces the rate of type IIb hypersensitivity, a type of inflammatory response.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'synonyms': [{'name': 'inhibition of type IIb hypersensitivity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'down-regulation of type IIb hypersensitivity', 'type': 'exact'},\n",
       "   {'name': 'downregulation of type IIb hypersensitivity', 'type': 'exact'},\n",
       "   {'name': 'down regulation of type IIb hypersensitivity', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001795', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0001799', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002893', 'relation': 'is_a'}]},\n",
       " 'GO:0001801': {'id': 'GO:0001801',\n",
       "  'isObsolete': False,\n",
       "  'name': 'positive regulation of type IIb hypersensitivity',\n",
       "  'definition': {'text': 'Any process that activates or increases the frequency, rate or extent of type IIb hypersensitivity, a type of inflammatory response.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'synonyms': [{'name': 'stimulation of type IIb hypersensitivity',\n",
       "    'type': 'narrow'},\n",
       "   {'name': 'activation of type IIb hypersensitivity', 'type': 'narrow'},\n",
       "   {'name': 'up regulation of type IIb hypersensitivity', 'type': 'exact'},\n",
       "   {'name': 'up-regulation of type IIb hypersensitivity', 'type': 'exact'},\n",
       "   {'name': 'upregulation of type IIb hypersensitivity', 'type': 'exact'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001795', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0001799', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002894', 'relation': 'is_a'}]},\n",
       " 'GO:0001802': {'id': 'GO:0001802',\n",
       "  'isObsolete': False,\n",
       "  'name': 'type III hypersensitivity',\n",
       "  'definition': {'text': 'An inflammatory response resulting from recognition of immune complexes via complement or Fc receptors on effector cells leading to activation of neutrophils and other leukocytes and damage to bystander tissue.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'comment': 'Note that the Arthus reaction is an example of type III hypersensitivity.',\n",
       "  'children': [{'id': 'GO:0001804', 'relation': 'negatively_regulates'},\n",
       "   {'id': 'GO:0001805', 'relation': 'positively_regulates'},\n",
       "   {'id': 'GO:0001803', 'relation': 'regulates'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0002444', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002524', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0016064', 'relation': 'is_a'}]},\n",
       " 'GO:0001803': {'id': 'GO:0001803',\n",
       "  'isObsolete': False,\n",
       "  'name': 'regulation of type III hypersensitivity',\n",
       "  'definition': {'text': 'Any process that modulates the frequency, rate, or extent of type III hypersensitivity, a type of inflammatory response.',\n",
       "   'xrefs': [{'dbCode': 'ISBN', 'dbId': '0781735149'}]},\n",
       "  'children': [{'id': 'GO:0001804', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0001805', 'relation': 'is_a'}],\n",
       "  'aspect': 'biological_process',\n",
       "  'usage': 'Unrestricted',\n",
       "  'parent': [{'id': 'GO:0001802', 'relation': 'regulates'},\n",
       "   {'id': 'GO:0002883', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002886', 'relation': 'is_a'},\n",
       "   {'id': 'GO:0002889', 'relation': 'is_a'}]},\n",
       " ...}"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "nary"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "* Owlready2 * Warning: optimized Cython parser module 'owlready2_optimized' is not available, defaulting to slower Python implementation\n"
     ]
    }
   ],
   "source": [
    "#import owlready2\n",
    "\n",
    "import types\n",
    "from owlready2 import *\n",
    "onto = get_ontology(\"http://test.org/onto.owl\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "#classmaker function to make FTC classes\n",
    "\n",
    "def classmaker(term, parentclass):\n",
    "                classnameA = 'GOA' + str(term)\n",
    "                classnameP = 'GOP' + str(term)\n",
    "                with onto:\n",
    "                    classname = types.new_class(classnameA, (parentclass,))\n",
    "                    classname = types.new_class(classnameP, (parentclass,))\n",
    "                return 0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "#define core classes\n",
    "\n",
    "with onto:\n",
    "    class therapeutic_agent(Thing):\n",
    "        namespace = onto\n",
    "    class biological_process(Thing):\n",
    "        namespace = onto\n",
    "    class molecular_function(Thing):\n",
    "        namespace = onto\n",
    "    class drug(Thing):\n",
    "        namespace = onto"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "#recursive classmakers\n",
    "\n",
    "\n",
    "def recursive_classmaker_down(term, parent):\n",
    "    parent = getattr(onto, parent)\n",
    "    with onto:\n",
    "        nc = types.new_class(term, (parent,))\n",
    "    parent = getattr(onto, term)\n",
    "    for item in nary[term]['children']:\n",
    "        recursive_classmaker_down(item['id'][3:], parent)\n",
    "    return 0\n",
    "\n",
    "\n",
    "\n",
    "def recursive_classmaker_up(term, parent):\n",
    "    if parent not in [str(s) for s in list(onto.classes())]:\n",
    "        for item in nary[('GO:' + term)]['parent']:\n",
    "            recursive_classmaker_up(parent, item['id'][3:])\n",
    "            \n",
    "    if nary[('GO:' + term)]['parent'] == []:\n",
    "        with onto:\n",
    "            nc = types.new_class(parent, (biological_process,))\n",
    "        #recursive_classmaker_down(term, parent)\n",
    "    return 0\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Using classmaker to make FTC classes\n",
    "\n",
    "for key, value in kilnary.items():\n",
    "    if value['aspect'] == 'biological_process':\n",
    "        for item in value['children']:\n",
    "            if item['relation'] == 'positively_regulates' or 'negatively_regulates' or 'regulates':\n",
    "                recursive_classmaker_up(str(item['id'][3:]), value['id'][3:])\n",
    "            if item['relation'] == 'capable of' or 'part_of' or 'capable_of_part_of' or 'has_part':\n",
    "                recursive_classmaker_up(str(item['id'][3:]), value['id'][3:])\n",
    "            \n",
    "            \n",
    "            \n",
    "#             if item['relation'] == 'negatively_regulates':\n",
    "#                 recursive_classmaker_up(str(item['id'][3:]), value['id'][3:])\n",
    "#             if item['relation'] == 'regulates':\n",
    "#                 recursive_classmaker_up(str(item['id'][3:]), value['id'][3:])\n",
    "#     if value['aspect'] == 'molecular_function':\n",
    "#         for item in value['children']:\n",
    "#             if item['relation'] == 'positively_regulates':\n",
    "#                 classmaker(str(item['id'][3:]), therapeutic_agent)\n",
    "#             if item['relation'] == 'negatively_regulates':\n",
    "#                 classmaker(str(item['id'][3:]), therapeutic_agent)\n",
    "#             if item['relation'] == 'regulates':\n",
    "#                 classmaker(str(item['id'][3:]), therapeutic_agent)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "onto.save(file = \"testowl2\", format = \"rdfxml\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[onto.therapeutic_agent,\n",
       " onto.biological_process,\n",
       " onto.molecular_function,\n",
       " onto.drug,\n",
       " onto.0008150,\n",
       " onto.0007582,\n",
       " onto.0000004,\n",
       " onto.0044699,\n",
       " onto.0008152,\n",
       " onto.0044236,\n",
       " onto.0044710,\n",
       " onto.0071704,\n",
       " onto.0009987,\n",
       " onto.0050875,\n",
       " onto.0044763,\n",
       " onto.0008151,\n",
       " onto.0043170,\n",
       " onto.0044237,\n",
       " onto.0043283,\n",
       " onto.0044259,\n",
       " onto.0006807,\n",
       " onto.0006725,\n",
       " onto.0034641,\n",
       " onto.0044238,\n",
       " onto.0046483,\n",
       " onto.1901360,\n",
       " onto.0006139,\n",
       " onto.0055134,\n",
       " onto.0044260,\n",
       " onto.0090304,\n",
       " onto.0034960,\n",
       " onto.0050896,\n",
       " onto.0051869,\n",
       " onto.0006950,\n",
       " onto.0051716,\n",
       " onto.0033554,\n",
       " onto.0006259,\n",
       " onto.0006974,\n",
       " onto.0055132,\n",
       " onto.0034984,\n",
       " onto.0065007,\n",
       " onto.0050789,\n",
       " onto.0050791,\n",
       " onto.0019222,\n",
       " onto.0050794,\n",
       " onto.0044246,\n",
       " onto.0051244,\n",
       " onto.0031323,\n",
       " onto.0051171,\n",
       " onto.0080090,\n",
       " onto.0019219,\n",
       " onto.0060255,\n",
       " onto.0048583,\n",
       " onto.0080134,\n",
       " onto.0080135,\n",
       " onto.0006281,\n",
       " onto.0051052,\n",
       " onto.2001020,\n",
       " onto.0048519,\n",
       " onto.0043118,\n",
       " onto.0009892,\n",
       " onto.0044252,\n",
       " onto.0048523,\n",
       " onto.0051243,\n",
       " onto.0031324,\n",
       " onto.0051172,\n",
       " onto.0010605,\n",
       " onto.0045934,\n",
       " onto.0048585,\n",
       " onto.0048518,\n",
       " onto.0043119,\n",
       " onto.0009893,\n",
       " onto.0044253,\n",
       " onto.0048522,\n",
       " onto.0051242,\n",
       " onto.0031325,\n",
       " onto.0051173,\n",
       " onto.0010604,\n",
       " onto.0045935,\n",
       " onto.0048584,\n",
       " onto.0071840,\n",
       " onto.0071841,\n",
       " onto.0016043,\n",
       " onto.0044235,\n",
       " onto.0071842,\n",
       " onto.0006996,\n",
       " onto.1902589,\n",
       " onto.0007049,\n",
       " onto.0007059,\n",
       " onto.0022402,\n",
       " onto.0051276,\n",
       " onto.0098813,\n",
       " onto.0007001,\n",
       " onto.0051277,\n",
       " onto.0048285,\n",
       " onto.0000278,\n",
       " onto.0007067,\n",
       " onto.0000280,\n",
       " onto.1903047,\n",
       " onto.0007010,\n",
       " onto.0007017,\n",
       " onto.0000226,\n",
       " onto.0000819,\n",
       " onto.0140014,\n",
       " onto.0007051,\n",
       " onto.1902850,\n",
       " onto.0043146,\n",
       " onto.0000070,\n",
       " onto.0007052,\n",
       " onto.0051231,\n",
       " onto.0016359,\n",
       " onto.0030472,\n",
       " onto.0000071,\n",
       " onto.0043148,\n",
       " onto.0051726,\n",
       " onto.0000074,\n",
       " onto.0051128,\n",
       " onto.0033043,\n",
       " onto.0032886,\n",
       " onto.0051493,\n",
       " onto.0010564,\n",
       " onto.0070507,\n",
       " onto.0007346,\n",
       " onto.0051783,\n",
       " onto.0033044,\n",
       " onto.0051983,\n",
       " onto.0090224,\n",
       " onto.0007088,\n",
       " onto.0033045,\n",
       " onto.0000022,\n",
       " onto.0032887,\n",
       " onto.0033047,\n",
       " onto.0060236,\n",
       " onto.0045786,\n",
       " onto.0051129,\n",
       " onto.0010639,\n",
       " onto.0010948,\n",
       " onto.0051985,\n",
       " onto.2001251,\n",
       " onto.0045930,\n",
       " onto.0051784,\n",
       " onto.0033046,\n",
       " onto.0045839,\n",
       " onto.0045787,\n",
       " onto.0051130,\n",
       " onto.0010638,\n",
       " onto.0045931,\n",
       " onto.0051785,\n",
       " onto.0090068,\n",
       " onto.0045840,\n",
       " onto.0051984,\n",
       " onto.2001252,\n",
       " onto.0051495,\n",
       " onto.0044085,\n",
       " onto.0071843,\n",
       " onto.0022607,\n",
       " onto.0071844,\n",
       " onto.0009056,\n",
       " onto.0044712,\n",
       " onto.0044243,\n",
       " onto.0044248,\n",
       " onto.0061919,\n",
       " onto.0006914,\n",
       " onto.0016238,\n",
       " onto.0007033,\n",
       " onto.0016236,\n",
       " onto.0044086,\n",
       " onto.0034262,\n",
       " onto.0009894,\n",
       " onto.0031329,\n",
       " onto.0009896,\n",
       " onto.0010506,\n",
       " onto.0031331,\n",
       " onto.0044087,\n",
       " onto.0070925,\n",
       " onto.1905037,\n",
       " onto.0009895,\n",
       " onto.0031330,\n",
       " onto.0051179,\n",
       " onto.1902578,\n",
       " onto.0033036,\n",
       " onto.0051641,\n",
       " onto.1902580,\n",
       " onto.0008104,\n",
       " onto.0051640,\n",
       " onto.0008105,\n",
       " onto.0022613,\n",
       " onto.0051234,\n",
       " onto.0050657,\n",
       " onto.0051236,\n",
       " onto.0006913,\n",
       " onto.0000063,\n",
       " onto.0050658,\n",
       " onto.0051168,\n",
       " onto.0015031,\n",
       " onto.0034613,\n",
       " onto.0046907,\n",
       " onto.0015831,\n",
       " onto.0016249,\n",
       " onto.1902582,\n",
       " onto.0006886,\n",
       " onto.0032779,\n",
       " onto.0006405,\n",
       " onto.0006611,\n",
       " onto.0031503,\n",
       " onto.0071166,\n",
       " onto.0097349,\n",
       " onto.0071426,\n",
       " onto.0015931,\n",
       " onto.0006403,\n",
       " onto.0051169,\n",
       " onto.0015833,\n",
       " onto.0045184,\n",
       " onto.0070727,\n",
       " onto.0006810,\n",
       " onto.0015457,\n",
       " onto.0044765,\n",
       " onto.0015460,\n",
       " onto.0071702,\n",
       " onto.0071705,\n",
       " onto.0042886,\n",
       " onto.0033750,\n",
       " onto.0042254,\n",
       " onto.0051656,\n",
       " onto.0071428,\n",
       " onto.0033753,\n",
       " onto.0007046,\n",
       " onto.0032879,\n",
       " onto.0051049,\n",
       " onto.0060341,\n",
       " onto.0032386,\n",
       " onto.0051051,\n",
       " onto.0032387,\n",
       " onto.0046822,\n",
       " onto.0032880,\n",
       " onto.0070201,\n",
       " onto.0090087,\n",
       " onto.0051223,\n",
       " onto.1903827,\n",
       " onto.0033157,\n",
       " onto.0051235,\n",
       " onto.0045185,\n",
       " onto.0051651,\n",
       " onto.0033365,\n",
       " onto.0032507,\n",
       " onto.0034504,\n",
       " onto.0072595,\n",
       " onto.0044744,\n",
       " onto.1904950,\n",
       " onto.0051224,\n",
       " onto.1903828,\n",
       " onto.0065008,\n",
       " onto.0046823,\n",
       " onto.0046825,\n",
       " onto.0051457,\n",
       " onto.0090317,\n",
       " onto.0032239,\n",
       " onto.0032240,\n",
       " onto.0046831,\n",
       " onto.0090069,\n",
       " onto.2000197,\n",
       " onto.0000054,\n",
       " onto.0051050,\n",
       " onto.0032388,\n",
       " onto.1904951,\n",
       " onto.0051222,\n",
       " onto.1903829,\n",
       " onto.0046824,\n",
       " onto.0090316,\n",
       " onto.0032241,\n",
       " onto.0044089,\n",
       " onto.0051225,\n",
       " onto.0051227,\n",
       " onto.0051226,\n",
       " onto.0090307,\n",
       " onto.0031023,\n",
       " onto.0051300,\n",
       " onto.0051426,\n",
       " onto.1902115,\n",
       " onto.0090169,\n",
       " onto.0061804,\n",
       " onto.0110100,\n",
       " onto.1905047,\n",
       " onto.0032888,\n",
       " onto.1901673]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(onto.classes())"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
